# Finding proteins with overrepresentation of dipeptide motifs.

The aim of the study is to develop a method, based on an analysis of amino acid composition of the proteome of Arabidopsis thaliana, enabling identification of proteins, which exhibit significant overrepresentation of dipeptide motifs.

The identification of protein domains is one of the key elements of protein annotation. Amino acid sequences of some protein domains consist of multiple repeats of two amino acids. An example of that can be recently discovered WG/GW domains, crucial to binding Argonaute proteins in the process of RNA interference. Given their lack of a tertiary structure and a low degree of sequence similarity (<30%), WG/GW domains are very difficult to identify, because standard methods of protein domain annotation are based on quantitative protein similarity. WG/GW repeats are probably one of many repeats of pairs of amino acids that build functional protein domains. Undoubtedly there exists an array of these types of repeats containing other amino acids, that function in many cellular processes.


I identified **967** Arabidopsis thaliana proteins which exhibit significant (**p < 10^-4**) overrepresentation of dipeptide motifs. Some proteins (33%) exhibit overrepresentation of more than one motif - a total of **2337** occurences of overrepresentation of dipeptides were found in those proteins. Out of 210 possible combinations of two aminoacids, 205 exhibit at least one overrepresentation. The developed method also identified all four WG/GW proteins with experimentally confirmed function of binding Argonaute proteins. For 524 overrepresentations in A.thaliana at least one orthologous protein exhibits overrepresentation of the same dipeptide motif, which can indicate the evolutionary and functional conservation of analysed motifs.

## Results are available in json format in the full_fixed_results directory.
