
<!doctype html>
<html>
    <head>
        <title>GI-IG</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT5G11250.2</h1>
        <h3>AT5G11250.2 pep chromosome:TAIR10:5:3587978:3592471:-1 gene:AT5G11250 transcript:AT5G11250.2 gene_biotype:protein_coding transcript_biotype:protein_coding description:Disease resistance protein (TIR-NBS-LRR class) [Source:UniProtKB/TrEMBL;Acc:Q9LFN1]</h3>
        <pre>
length  1189
count   20
ratio   0.00
zscore  5.62






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT5G11250.2     MASS-------------------------------------------------------F
Bra006094.1-P   MDSS-------------------------------------------------------F
Bra027510.1-P   MKVRRRRKTAAADQGLWKYNVSETVHLFHGVLQYASLSCLLQYGSEQPRNVFCDVMSLIL
Bra006146.1-P   ------------------------------------------------------------
Bra006556.1-P   MDSSFFLTTVAAA<span class="highlight">I</span><span class="highlight">G</span>-------------------------------------------LF
AES87066        ------------------------------------------------------------
VIT_18s0041g024 ------------------------------------------------------------
                                                                            

AT5G11250.2     FL------------------------------------TTVAAA<span class="highlight">I</span><span class="highlight">G</span>FFTLLRKLRFHH-D
Bra006094.1-P   LL------------------------------------TTLAAA<span class="highlight">I</span><span class="highlight">G</span>LLTVLRKLRFHQED
Bra027510.1-P   IL<span class="highlight">G</span><span class="highlight">I</span>WRTPLHLKTLESSSAVDVLEVEPEETVKEVKKKQCMLWAMLGLLTIQRKLRSHLDN
Bra006146.1-P   --------------------------------------------------MTKRKFHEVD
Bra006556.1-P   IL------------------------------------TTVAAA<span class="highlight">I</span><span class="highlight">G</span>LFTLLRKLRLHQDD
AES87066        ------------------------------------------------------------
VIT_18s0041g024 ------------------------------------------------------------
                                                                            

AT5G11250.2     NKEKKS-TSLSPSSSPSSLSPSSVPPPSSSCIWTHQVFPSFSGEDVRRDFLSHIQMEFQR
Bra006094.1-P   NREKDP-SSSSTS---SSLSPSTVHLPSSSHIWTHDVFPSFRGEDVRRDFFSHIQREFER
Bra027510.1-P   NDEKD--SSSSPS---SSLSPSSTVPPPSSRIWMHDVFPSFRGEDVRKDFLSHIHMEFQR
Bra006146.1-P   NGENNYFSSCSPS---SSLSPSIVPPP-STRIWTHHVFPSFRGEDVRRKFFSHIQMELER
Bra006556.1-P   NREKDS-SSSSPS---SSLSPSPVPPPSSSRIWTHHVFPSFRGEDVRKDFLSHIQMEFQR
AES87066        ------------------------------------------------------------
VIT_18s0041g024 ------------------------------------------------------------
                                                                            

AT5G11250.2     M<span class="highlight">G</span><span class="highlight">I</span>TPFVDNEIKRGES<span class="highlight">I</span><span class="highlight">G</span>PELLRAIRGSKIAIILLSRNYASSKWCLDELVEIMKCREEYG
Bra006094.1-P   K<span class="highlight">G</span><span class="highlight">I</span>TPFIDNEIKRGES<span class="highlight">I</span><span class="highlight">G</span>PELIRAIRGSKIAIILLSRNYASSKWCLDELVEIMKCREEFG
Bra027510.1-P   K<span class="highlight">G</span><span class="highlight">I</span>TPFVDNEIKRGES<span class="highlight">I</span><span class="highlight">G</span>PELVRAIRGSKIAIILISRNYASSKWCLDELVEIMKCREELG
Bra006146.1-P   K<span class="highlight">G</span><span class="highlight">I</span>TPFIDNEIRRGES<span class="highlight">I</span><span class="highlight">G</span>PELIRAIRGSKIAIILISRNYASSKWCLDELVEIMKCREELG
Bra006556.1-P   K<span class="highlight">G</span><span class="highlight">I</span>TPFIDNEIKRRDD<span class="highlight">I</span><span class="highlight">G</span>PELIRAIRGSKIAIILLSRNYASSKWCLDELVEIMKCREELG
AES87066        ---------------------------------------------MEEV-----------
VIT_18s0041g024 ----------------------------------------------------MQCK----
                                                                            

AT5G11250.2     QTVMAIFYKVDPSDVKNLTGDFGKVFRKTCAGKPKKD<span class="highlight">I</span><span class="highlight">G</span>RWRQAWEKVATVAGYHSINW-
Bra006094.1-P   QTVMAIFHKVDPSDVKKLTGDFGKFFKKTCAGKAKDCIERWRQALAKVATIAGYHSSNW-
Bra027510.1-P   QTVMAIFYRVDPSDVKKLAGDFGRVFRKTCAGKTKDN<span class="highlight">I</span><span class="highlight">G</span>RWRQALAKVATVAGYHSSNW-
Bra006146.1-P   QTVMPVFYEVDPSNVKKLTGDFGKVFRKTCAGKTKECIKRWRQAFAKVATIAGYHSSNWL
Bra006556.1-P   QTVMAIFYRVDPSDVKKLAGDFGRVFKKTCAGRTKENIERWRQALAKVATIAGYHSSNW-
AES87066        ----------------------------------------------------------W-
VIT_18s0041g024 ------------------------------------------------------------
                                                                            

AT5G11250.2     -------------------------------------------DNEAAMIKKIATDISNI
Bra006094.1-P   -------------------------------------------DNEADMIKKIATDTSNM
Bra027510.1-P   -------------------------------------------DNEAAMIKKIATDISNM
Bra006146.1-P   VLFLYSCFLYTFLINLTHPTNFIFLYGCEYDYLMDALNSPIIRDNEADMIKKITTDISNM
Bra006556.1-P   -------------------------------------------DNEAAMIKKIATDISDM
AES87066        ------------------------------------------------------------
VIT_18s0041g024 ------------------------------------------------------------
                                                                            

AT5G11250.2     LINSTPSRDFDGLVGMRAHLEKMKPLLCLDTDEVRI<span class="highlight">I</span><span class="highlight">G</span><span class="highlight">I</span>WGPP<span class="highlight">G</span><span class="highlight">I</span><span class="highlight">G</span>KTTIARVVYNQLSH
Bra006094.1-P   LNNFTPSNDFDGLVGMGAHWENLKSILCLGSDEVRM<span class="highlight">I</span><span class="highlight">G</span><span class="highlight">I</span>WGPP<span class="highlight">G</span><span class="highlight">I</span><span class="highlight">G</span>KTTIARVAFNQLSN
Bra027510.1-P   LNSFTPLSDFDGLVGMGAHLEKMEPLLGLGSDEVRM<span class="highlight">I</span><span class="highlight">G</span><span class="highlight">I</span>WGPP<span class="highlight">G</span><span class="highlight">I</span><span class="highlight">G</span>KTTIARVAYNQLSE
Bra006146.1-P   LNNFTPSNDLDEL<span class="highlight">I</span><span class="highlight">G</span>MEAQLEKMKPLLCLGSDEVRM<span class="highlight">I</span><span class="highlight">G</span><span class="highlight">I</span>WGPP<span class="highlight">G</span><span class="highlight">I</span><span class="highlight">G</span>KTTIARFAFNQLSN
Bra006556.1-P   LNNFTPSNDFDGLVGMGAHLEKMEPLLCLGSDEVRM<span class="highlight">I</span><span class="highlight">G</span><span class="highlight">I</span>WGPP<span class="highlight">G</span><span class="highlight">I</span><span class="highlight">G</span>KTTIARVAYSKLSN
AES87066        ------------------------------------------------------------
VIT_18s0041g024 ------------------------------------------------------------
                                                                            

AT5G11250.2     SFQLSVFMENIKANYTRPTGSDDYSAKLQLQQMFMSQITKQKDIEIPHLG----VAQDRL
Bra006094.1-P   SFQLSVFMDDLKANSSR-LCSDDYSVKLQLQQQFMSQITDHKDMVVSHFG----VVSNRL
Bra027510.1-P   GFQLSIFMDDIKVNYSR-LCSDDYSAKLQLQQQFISQITNHKDMVVSHLG----VASNRL
Bra006146.1-P   RFQLSVFMDDLKPNPTR-LCSDDYSLKLQLHQRFMSQITNHKDMVVSHLG----VASNRL
Bra006556.1-P   NFQLSVFMDDLKANYTR-LCSDDYSLKLQLQQQFMSQITNQKDMVVSHLG----VASNRL
AES87066        -----------KRN----------NGQISLQEQPLCCIEGATEIKIPTVKSGSVILKDRL
VIT_18s0041g024 ------------------------------------------------------------
                                                                            

AT5G11250.2     KDKKVLVVLDGVNQSVQLDAMAKEAWWFGPGSRIIITTQDQKLFRAH<span class="highlight">G</span><span class="highlight">I</span>NHIYKVDFPPT
Bra006094.1-P   RDKKVLVVLDGVNRSVQLDAMAKETWWFGPGSRIIITTQDQKLFRAH<span class="highlight">G</span><span class="highlight">I</span>NHIYEVNLPTN
Bra027510.1-P   KDKKVLVVLDGVDRSVQLDAIAKEALWFCPGSRIIITTQDHKLFRAH<span class="highlight">G</span><span class="highlight">I</span>NHIYKVDFPTN
Bra006146.1-P   KDKKVLVVLDGVDRSVQLDAVAKEAWWFGPGSRIIITTQDRRIFRAHGVNHIYKVDFPTS
Bra006556.1-P   KDKKVLVVLDGVDQSVQLEAMAKETWWFGPGSRIIITAQDQKLFRAHGVNLIYKVNFPTD
AES87066        QHKRVLLLLDNVYRLEQLQALCGSHEWFGEGSKIFIMTTDRHLLKEHGVDHVYRVKELEE
VIT_18s0041g024 ------------------------------------------------------------
                                                                            

AT5G11250.2     EEALQIFCMYAFGQNSPKDGFQNLAWKVINLAGNLPLGLRIMGSYFRGMSREEWKKSLPR
Bra006094.1-P   DEALQIFCTYCFGQNFPKYGFEELAREVTSLSGELPLGLRVMGSYLRGMSKEDWTNSLPR
Bra027510.1-P   DEALQIFCTYSFGQKFPKDGFEELAREVTKLAGELPLGLRVMGSYFRGMSKQEWINSLPR
Bra006146.1-P   DAALQIFCTYSFGQKSPKDGFEELARELTQLSGELPLGLRVMGSYFKGMSKQEWINAIPR
Bra006556.1-P   DEALQIFCTYSFGQKSPKDGFEELAREVTRLAGELPLGLRVMGSYFRGMSKQEWTNSLPR
AES87066        SESYELFNWAAFNQACLPEGFAELSRQVVAYCGGLPLALELLGLYLHGSEVLKWKGVLRS
VIT_18s0041g024 ------------------------------------------------------------
                                                                            

AT5G11250.2     LE--SSLDADIQSILKFSYDALDDEDKNLFLHIACFFNGKEIKILEEHLAKKFVEVRQRL
Bra006094.1-P   LR--DSLDTDIQSILKFSYDALDDEDKDLFLHIACFFSSEQIHKMEEHLAKRFLYVRQRL
Bra027510.1-P   LK--TSLDSDIQSILKFSYDALDDEDKYLFLHIACFFNSQKIHKVEEHLAKKFLEVRQRL
Bra006146.1-P   LR--TSLDSD<span class="highlight">I</span><span class="highlight">G</span>SILKFSYDALDDEDKYLFLYIACCYKSEWINEVEEYLAKKFVEVRQRL
Bra006556.1-P   LK--TSLDSDIRSILKFSYDALDDEDKDLFLHIACFFNSQEIHKVEEHLAKKFLEVRQRL
AES87066        LERFSIPSGQLVEALEKSFRYLSDEEKQIFLDIACFL<span class="highlight">I</span><span class="highlight">G</span>MNQNDVLQTLNMSMQCTTLQI
VIT_18s0041g024 ------------------------------------------------------------
                                                                            

AT5G11250.2     NVLAEKSLISFSNWGTIEMHKLLAKLGGEIVRNQSIHEPGQRQFLFDGEEICDVLNGDAA
Bra006094.1-P   KVLAEKSLISID-SGRIRMHSLLEKLGREIVCKQSIHEPGQRQFLYDKRDICEVLTGGAT
Bra027510.1-P   NVLAEKSLISIE-SGYIKMHSLLEKLGRVIVCKQSIHEPGQRQFLFDDIEICEVLTGDAM
Bra006146.1-P   NVLVDKSLISISLLGVIEMHSLLKKLGREIVCKQS-QEPGQRQFLYDEREVCEVLTGDAT
Bra006556.1-P   NVLAEKSLISID-SGVITMHSLLEKLGREIVCKQSIHEPRLRQFLWEETEIFEVLTGDTT
AES87066        SLLEDKSFVTIDENNKLQMHVMQQAMARDIVTRQSSKKTDQ-------------------
VIT_18s0041g024 ------------------------------------------------------------
                                                                            

AT5G11250.2     GSKSV<span class="highlight">I</span><span class="highlight">G</span><span class="highlight">I</span>DFHYIIE-EEFDMNERVFEGMSNLQFLRFDCDHDTLQLSRGLSYLSRKLQLL
Bra006094.1-P   GSKSV<span class="highlight">I</span><span class="highlight">G</span><span class="highlight">I</span>KFEYYRIREEIDISEKAFEGMSNLQFLKVCGFTDALQITGGLNYLSHKLRLL
Bra027510.1-P   GSKNV<span class="highlight">I</span><span class="highlight">G</span>MKLDYYKIKEELDISDKAFEGMSNLQFLKVYGYSDALQLTRGLNYLPRKLRLL
Bra006146.1-P   GSKSV<span class="highlight">I</span><span class="highlight">G</span><span class="highlight">I</span>NLDYSREGKEIDISEKAFEGMSNLQFLKVSCSHFTMKSTRGLSYLPHKLRLL
Bra006556.1-P   GSKSV<span class="highlight">I</span><span class="highlight">G</span><span class="highlight">I</span>KLKYNTEGEKIEISEKAFEGMSNLQFLKVSGYSHPLQLTRGLNYISHKLRFL
AES87066        ----------------------------MS--------------------GYLPSNMGLF
VIT_18s0041g024 -------------------------------------------LHVSDDFKFHYDELRYL
                                                                   :   ::  :

AT5G11250.2     DWIYFPMTCLPSTVNVEFLIELNLTHSK-LDMLWEGVKPLHNLRQMDLSYSVNLKELPDL
Bra006094.1-P   EWRHFPMTCLPCTVNLEFLVELVMPYSK-LEKLWEGCKPLRCLKWMDLGYSVNLKELPDL
Bra027510.1-P   EWSRFPMRYFPSTLNLEFLVELTMRHSK-LEKLWE<span class="highlight">G</span><span class="highlight">I</span>KPLRSLKWMNLSYSVNLKELPDL
Bra006146.1-P   KWSHCPMTCFPCNVNFEFLVELSMSNSK-LEKLWEVTKCSK--KKKKKKKKKEKKELPDL
Bra006556.1-P   QWTHFPMTCLPSILNLEFLVELIMHTSK-LEKLWEGTKPLRCLKWMDLSYSENLKELPDL
AES87066        D------------------------------------RP------------------PHA
VIT_18s0041g024 HWDEYPCESLPFDFESENLVHFCMPRSRHLTQLWKGQKVFGNLEFVDVSYSQYLKETPDF
                .                                    :                   *. 

AT5G11250.2     STAINLRK------------------------LILSNCSSLIKLPSC<span class="highlight">I</span><span class="highlight">G</span>NAINLEDLDLN
Bra006094.1-P   STATNLEK------------------------LYLYDCSSLVKLPSMSGN--SLEKLN<span class="highlight">I</span><span class="highlight">G</span>
Bra027510.1-P   STATRLKKLNLNACSSLMKLPPA<span class="highlight">I</span><span class="highlight">G</span>YANNLEVLNLRNCSSLIKLPCLARNATSLEKLY<span class="highlight">I</span><span class="highlight">G</span>
Bra006146.1-P   STATNLKR------------------------LNLSNCSSLIKLPSLPGN--SMKELYIK
Bra006556.1-P   STATNLE-------------------------LDLSNCSSLIKLPYLNGN--SLEKLY<span class="highlight">I</span><span class="highlight">G</span>
AES87066        STTR------------------------------------LVR-----------------
VIT_18s0041g024 SRATNLEV------------------------LVLKGCTNLRKVH---------------
                * :                                     * :                 

AT5G11250.2     GCSSLVELPSF-GDAINLQKLLLRYCSNLVELPSS<span class="highlight">I</span><span class="highlight">G</span>NAINLRELDLYYCSSLIRLPSSI
Bra006094.1-P   GCSSLVEFPSF<span class="highlight">I</span><span class="highlight">G</span>NAVNLQELDLSSFPNLLELPSYVGNATNLEYLDLRNCLNMVELPLSL
Bra027510.1-P   GCSSLVEFPSF<span class="highlight">I</span><span class="highlight">G</span>NAVNLQELDLSSFPNLLELPSYVGNATNLEKLCLNNCLYVVELPLSL
Bra006146.1-P   GCSSLVEFPSF<span class="highlight">I</span><span class="highlight">G</span>NAVNLETLDLSSLPNLLELPSFVENATNLKKL---------------
Bra006556.1-P   GCSSLVEFPSFIENAVSLRKLDLTSYPNLLELPSYVGNATNLDELYLSNCLDLVELPLSL
AES87066        --------PS----------------PN--------------------------------
VIT_18s0041g024 --------PS-------------------------LGYLSKLILLNLENCTNLEHLP-SI
                        **                                                  

AT5G11250.2     GNAINLLILDLNGCSNLLELPSS<span class="highlight">I</span><span class="highlight">G</span>NAINLQKLDLRRCAKL-LELPSS<span class="highlight">I</span><span class="highlight">G</span>NAINLQNLLL
Bra006094.1-P   RNLKKLKRLRLKGCSKLEVLPTNI-NLEYLNELDIAGCSSLDLGDFST<span class="highlight">I</span><span class="highlight">G</span>NAVNLRELNI
Bra027510.1-P   GNLQKLQKLILKGCNKLEFLPSNI-NLESLEKLDLSGCSSLDLAGLSIIENAVNLQTLDI
Bra006146.1-P   ------------------------------------------------------------
Bra006556.1-P   GNLQKLKKLVLKGCSKLEVFPTNF-NVESLEILCLAGCSSLDLGGCST<span class="highlight">I</span><span class="highlight">G</span>NVPSLRMLNL
AES87066        ------------------------------------------------------------
VIT_18s0041g024 RWLVSLETLILSGCSKLEKLPEVPQHMPYLSKLCLDGTAITDFSGWSELGN---------
                                                                            

AT5G11250.2     DDCSSLLELPSS<span class="highlight">I</span><span class="highlight">G</span>NATNLVYMNLSNCSNLVELPLS<span class="highlight">I</span><span class="highlight">G</span>NLQKLQELILKGCSKLEDLPIN
Bra006094.1-P   SSLPQLLEVPSF<span class="highlight">I</span><span class="highlight">G</span>NATNLENLVLSSCSKLVELPLF<span class="highlight">I</span><span class="highlight">G</span>NLQKLRWLRLEGCIRLEVLPTN
Bra027510.1-P   SRLPQLLELPSFIRNATNLEDFNLSGCSNLVELPLFIVNLQKLKMLRLEGCGKLEVLPTN
Bra006146.1-P   ----------------------DLRFCSNLVELPFS<span class="highlight">I</span><span class="highlight">G</span>NLQKLWWLELQGCSKLEVLPTN
Bra006556.1-P   RSLPQLLDLPSF<span class="highlight">I</span><span class="highlight">G</span>NAINLYYLDLSGCSNLVELPVF<span class="highlight">I</span><span class="highlight">G</span>NLQKLYMLGLEGCSKLEFLPTN
AES87066        ------------------------------------------------------------
VIT_18s0041g024 -----------FQENSG-------------------------------------------
                                                                            

AT5G11250.2     INLESLDILVLNDCSMLKRFPEISTNVRALYLCGTAIEEVPLSIRSWPRLDELLMSYFDN
Bra006094.1-P   INLESLLELNLSDCSMLKSFPQISTSLEKLNLRGTAIEQVPPSIRSWPHLKELHMSYFEN
Bra027510.1-P   INLESLDVLNLSNCSMLKSFPQISTNIRALDLRGTAVEEVPPSIVSWPRLDQLQMSYFEN
Bra006146.1-P   INLKSLYFLNLSDCSMLKSFPQISTNLEKLDLRGTAIEQVPPSIRSRPCSDILKMSYFEN
Bra006556.1-P   INLESLSWLNLRDCSMLKCFPQISTNIRDLDLTGTAIEQVPPSIRSWPRLEDLTMSYFEN
AES87066        ------------QCDINGGRPD----------S<span class="highlight">G</span><span class="highlight">I</span>LIKGVP-------------MDFGDT
VIT_18s0041g024 -NLDCLNELN-SDDSTIRQLPSSSVVLRNHNASPSSAPRRSHSIR---------------
                            : .     *.                  .                   

AT5G11250.2     LVEFPH-VLDIITNLDLSGKEIQEVPPLIKRISRLQTLILKGYRKVVSLPQIPDSLKWID
Bra006094.1-P   LKEFPH-ALERITSLSLTDTEIQEVPPLVKQISRLNRFFLSGCRKLVRLPPISESTHSIY
Bra027510.1-P   IKEFPH-ALERITWLCLTDREIREIPPWVKNISRLNEFVLKGCRKLVSVPPISESIYFVD
Bra006146.1-P   LKESPH-ALERITELWLTDTEIQELPPWVKKISRLSQLVVKGCRKLVSVPPLSDSIRYID
Bra006556.1-P   LKEFPH-ALERITELCLTDTDIQELPPWVKQISCLNSFVLKGCRKLVSIPPISDSIRFLD
AES87066        L-----------------------------------------------------------
VIT_18s0041g024 ----PHCTLTSLTYLNLSGTSIIRLPWNLERLFMLQRLELTNCRRLQALPVLPSSIERMN
                                                                            

AT5G11250.2     AEDCESLERLDC-SFHNPEITLFFGKCFKLN--QEARDLIIQTPTKQ-------------
Bra006094.1-P   ANDCDSLEILEC-SFSDQIRRLTFANCFKLN--QEARDLIIQASSEH-------------
Bra027510.1-P   ARDCESLEMLEC-SFHNPHVWLYFDNCFKLN--QEARDLIIQN-SEN-------------
Bra006146.1-P   ASDCESLEMIEC-SFPNQFVWLKVANCFKLN--QEARNLIIQK-SEF-------------
Bra006556.1-P   ASDCESLEILEC-SFHNQISRLNFANCFKLN--QEARDLIIQN-SRE-------------
AES87066        ------------------------------------------------------------
VIT_18s0041g024 ASNCTSLELVSPQSVFKRFGGFLFGNCFKLRNCHSKMEHDVQSVASHVVPGAWRSTYASW
                                                                            

AT5G11250.2     ---------AVLPGREVPAYFTHRASGGS------LTIKLNERPLPTSMRYKACILLVRI
Bra006094.1-P   ---------AVLPGGQVPPYFTHRATGGG-----PLTIKLNQNPLPESMTFKACILLLNK
Bra027510.1-P   ---------AVLPGGQVPAYFTHRATGGG-----PLTIKLNEKSLPKSMSFKFCILLVNK
Bra006146.1-P   ---------AVLPGGQVPAYFTHRA<span class="highlight">I</span><span class="highlight">G</span>GG-----PLTIKLNDNPLPKSMRFKACILLLNK
Bra006556.1-P   ---------AVLPGGQVPAYFTHRATGGG-----PLSIKLNEKPLPKSLRFKACILLVDK
AES87066        ----------------IPSLFRK---------------SLN-------------------
VIT_18s0041g024 HPNV<span class="highlight">G</span><span class="highlight">I</span>PFSTVFPGSEIPDWFRHHSQGHEINIEVPPDWYINSNFLGFALSAVMAP-----
                                :*  * :                :*                   

AT5G11250.2     G-DYGA--HDRDKWLHVAFGVRSTTLGHIIYSGLTEH--------------LYTFEIEEK
Bra006094.1-P   G-DHEACSKEKSRRVDVVNNNIIFGL----YPALAEH--------------LYTFRINLE
Bra027510.1-P   GVVDDACFKEKSTEVEVLYRNRSHTL---TYQPLEEH--------------LYIFQFREE
Bra006146.1-P   G-DHDTCYNEELTQVEVKFKYGSKTL----YLPLAGH--------------LYTFRFGAN
Bra006556.1-P   G-DHDACSKEKSTEVFAMYKNRHHRL----YPALAQH--------------LYTFEVEKE
AES87066        ------------------------------------------------------------
VIT_18s0041g024 --QHDS---------------RAWYM----YCDLDTHDLNSNSHRICSFFGSWTYQLQHT
                                                                            

AT5G11250.2     VTSSELVF--------EFKVEN-GNHWK-------------<span class="highlight">I</span><span class="highlight">G</span>ECGLRQLS---------
Bra006094.1-P   VTSRKLLF--------EFKLMSDDDDWK-------------<span class="highlight">I</span><span class="highlight">G</span>EC<span class="highlight">G</span><span class="highlight">I</span>VQQ----------
Bra027510.1-P   VTSSELLF--------EFKLKSDDEVWQ-------------<span class="highlight">I</span><span class="highlight">G</span>ECGLVQHMEVPSC----
Bra006146.1-P   VSSNELLF--------EFKLKN-DDVWK-------------<span class="highlight">I</span><span class="highlight">G</span>ECGLVQDLEVPFMLM--
Bra006556.1-P   VTSSELLF--------EFKLES-DDVWK-------------<span class="highlight">I</span><span class="highlight">G</span>EC<span class="highlight">G</span><span class="highlight">I</span>VQHLECISMMQMQ
AES87066        ------------------------------------------------------------
VIT_18s0041g024 PIESDHVWLAYVPSFLSFSCEK----WSHIKFSFSSSGGCVVKSCGF-----CPVYIK--
                                                                            

AT5G11250.2     --------------------------------------
Bra006094.1-P   --------------------------------------
Bra027510.1-P   --------------------------------------
Bra006146.1-P   ---------------------------NTMR------A
Bra006556.1-P   RSSNTQNGDGHVATLGSGLGLGGKPTRSVVR------P
AES87066        --------------------------------------
VIT_18s0041g024 --GTSDEGD---YSSGSAL---DEPRRHAAKPSRISYQ
                                                      


<a href='GI_IG-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            