
<!doctype html>
<html>
    <head>
        <title>HQ-QH</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT1G27430.1</h1>
        <h3>AT1G27430.1 pep chromosome:TAIR10:1:9520825:9527280:-1 gene:AT1G27430 transcript:AT1G27430.1 gene_biotype:protein_coding transcript_biotype:protein_coding description:GYF domain-containing protein [Source:UniProtKB/TrEMBL;Acc:F4HSW8]</h3>
        <pre>
length  1492
count   14
ratio   0.00
zscore  10.65






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT1G27430.1     MAEGKFDLPDDLIFSKSSDQL-------------------------KELASDNSIPLSPQ
Bra010950.1-P   MFESR-------------------------------------------------------
Bra010951.1-P   ------------------------------------------------------------
VIT_01s0146g001 MAESKLDLPDDLISTKPSDQFWTATVVASGGNDDEKALMGLADESKDQLASESSIPLSPQ
                                                                            

AT1G27430.1     WLYTKSSEYKMDVRSPTPVPMGNPSDPNPKDAWRLDAPEDKKDWKKIVHENETSRRWREE
Bra010950.1-P   ------------------------------------------------------------
Bra010951.1-P   ------------------------------------------------------------
VIT_01s0146g001 WLYSKPNETKMETRAPNSAALGNSTDPNQKEGWRLDASEDKKDWRKIATDTESNRRWREE
                                                                            

AT1G27430.1     ERETGLLGARKVDRRKTERRIDSVSSRETGDIKNAAASDRWNDVNSRAAV--HEPRRDNK
Bra010950.1-P   -----------------------------------------EDVRSPAPVPIGSRRRDNK
Bra010951.1-P   ------------------------------------------------------------
VIT_01s0146g001 ERETGLLGGRR-NLRKVDRRVDTVSIRESIDSRALPTSERWHDGSNRNSV--HETRRDSK
                                                                            

AT1G27430.1     WSSRWGPDDKEKEARCEKVDINKDKEEPQSESQSVV-SNVRATSERDSDTRDKWRPRHRM
Bra010950.1-P   WSSRWGPDDKEKDAKV-------DKEEP--ETQSVLGSSLRAT---DSDARDKWRPRHRM
Bra010951.1-P   -----------------------------------------------------------M
VIT_01s0146g001 WSSRWGPEEREKESRTEKRP-DVDKEDAHSDNQSFV-GSNRPAPERDSDSRDKWRPRHRM
                                                                           *

AT1G27430.1     E-SQSGGPSSYRAAPGFGLDRGRAEGPNLGFTVGRGRASTIG-----RGSSTSLIGAGSA
Bra010950.1-P   EQLQSGGPNSYRAAPGFGLDKGRSEGPNLGFAVGRGRA---------RGTWSTFFGAGGF
Bra010951.1-P   EQVQSGGPASYRAAPGFGLDKGRSEGPNIGFTVGRGRA---------RGTWSAFFGAGGF
VIT_01s0146g001 E-LHSGGPTSYRAAPGFGIERARLEGSHVGFAIGRGRSTALGSTPVLRSSSAGPIGGAQF
                *  :**** *********:::.* **.::**::****:         *.: :  :*..  

AT1G27430.1     L------------SPVFRYPRGKLLDMYRKQKPDSSLGRILTEMDEVASITQVALIEPLA
Bra010950.1-P   LRNERVPGKP---APMCRYVRGKLLDLYRNQKPD----RMPIDMEDVDSVTQVDLIEPLA
Bra010951.1-P   LRNESVPGKP---APTCRYVRGKLLDLYRNQKPD----RMPIDMEEVDSVTQVALIEPLA
VIT_01s0146g001 ERNGNVTGKLNLLDDTLCYPRGKLLDIYRRKKLDPSFATMPENMEETPHITLGDFIEPLA
                                  * ******:**.:* *     :  :*::.  :*   :*****

AT1G27430.1     FIAPDAEEEANLNGIWKGRIISSEVYTSSGEESLGGNSLLKCRIPESGETKVDGALLGFM
Bra010950.1-P   FIAPDAEEEESLKGIWKGRITGSEAHISPGEESLAEKSL--------GETKVDGGLLGVV
Bra010951.1-P   FIAPGVEEEESLKGIWKGRITSSEAHTSPGEESLAEESM--------GETKVDGALLGVM
VIT_01s0146g001 FVAPDAEEEVILRDIWKGKITSSGVVYNSFRKGRTTENVTGIEDLESPKEK-QGILPSIT
                *:**..***  *..****:* .* .  .. .:.   :.:         : * :* * .. 

AT1G27430.1     NGD----------NGSMKNNDSGLLGSH------------NGGLGAASSVPRLNSVASES
Bra010950.1-P   SGD----------SVSMHNSNSGLLGSH------------GGGLWGASESD---------
Bra010951.1-P   SGD----------SVSMHNSNSGVLGSH------------NGGLWGASESD---------
VIT_01s0146g001 TKEIADTFPEGVNDGAYQDDDSGISFNYNMTKNMIDEMDANQGEGKYSVAGMDDMISTVS
                . :          . : ::.:**:  .:            . *    *            

AT1G27430.1     YGSGGAGYQLSHGSPEA------------VRSVFTKSSVLDGSESVVG------------
Bra010950.1-P   --------QVSHGSPEA------------VRSGFTKSSVLNAGEPLV-------------
Bra010951.1-P   --------QVSRGSPEA------------VRSAFDKSSVLDADEPIV-------------
VIT_01s0146g001 KGSSLCGVSEMSGANRTASQLKAVENEHLANSDFTKHDKLDNITSAASFDIGCGLPDISN
                        .   *: .:            ..* * * . *:   . .             

AT1G27430.1     ------SFEQAYTGKLQQPDTEVDHSEGA------MPPEEFLFLYIDPQGVIQGPFIGSD
Bra010950.1-P   --------EQDYTGKLQQPEIEVNHSEGT------LPPEEFMFTYIDPQGVIQGPFIGSD
Bra010951.1-P   ------------TGKLQQPDIEVNHSEGT------LPPEEFMFSYIDPQGVIQGPFIGSD
VIT_01s0146g001 SIFALPSPKHSLSSNMQ----HLNSTGGTNLLGRGIPPEDFSLHYLDPQGEIQGPFLGVD
                            :.::*    .:: : *:      :***:* : *:**** *****:* *

AT1G27430.1     IISWFEQGFFGTDLQVRLASAPEGTPFQDLGRVMSYIKAESVHAHISDQKSELEETSLKA
Bra010950.1-P   IISWFEQGFFGTDLQVRLATAPEGTPFQDLGSVMSYLKTESMQAHINDQIIELEQTSRKA
Bra010951.1-P   IISWFEQGFFGTDLQVRLATAPEGTPFQDLGSVMSYLKTESMQAHINDQIIELEQTSRKA
VIT_01s0146g001 IISWFKQGFFGIDLPVRLSDAPEGIPFQDLGEIMPHLKTKD-GANSTDASSELEHAGILG
                *****:***** ** ***: **** ****** :*.::*::.  *: .*   ***.:.  .

AT1G27430.1     NSEAGGSVAH----VAESNDSSSLTGISRSFSVYNNPSGQDNFQRKSESEVYGRPPHAED
Bra010950.1-P   NSEIGLSFAP----VEDSNGSA---------------LGHDNVQMKSKSEAYVKPPHLDD
Bra010951.1-P   NSEIGLSFAP----VQESNGSA---------------LGHDNVQIQSKSEAYVKPPHVDD
VIT_01s0146g001 ANLEASSPAPGPVPVPDIADTTALNDHHWSLSEFDGLSSQNFQQRKSEREGPLQLSYSDG
                 .  . * *     * :  .::                .::  * :*: *   : .: :.

AT1G27430.1     QSFLDFSAQDEEIVFPGR--ARVSGYASSVKSSTSMHDALMEFSGHSDIPVEVTTAATRN
Bra010950.1-P   RSFLDYSAQDEEIVFPGR--AGVSGYGSA-KSSTSMHDALMGVSGHSAVPVESAKVATQN
Bra010951.1-P   RSFLDFSAQDEEIVFPGR--ARVSGYGSA-KSSTSMHDALMGVSGHPAIPVESAKVATQN
VIT_01s0146g001 QSFHDFSPQDEEIVFPGRPGSGGGGYPIG-KPSRSTQDPLANPITYSSLPNELTEPVMAN
                :** *:*.**********  :  .**  . *.* * :*.*     :. :* * :  .  *

AT1G27430.1     QNENKLHPFGVLWSELEGGSTPVNPLPNRSS---------GAMGEPSCSIENRPINSRRN
Bra010950.1-P   QNENKLHPFGVLWSELESSSTPVDLLPNRSY-----------------------------
Bra010951.1-P   QNENKLHPFGVLWSELESSSTPVDLLPNRSY-----------------------------
VIT_01s0146g001 QNDNKL<span class="highlight">H</span><span class="highlight">Q</span>FGLLWSELE-GAHPTHAQPSNLSSSIGRLGPLGAMAGSTPDAEAFSDVYRRN
                **:**** **:****** .: *..  *..                               

AT1G27430.1     SQIDPNISLDALSGNRMSQFEHESNFFNHGDQLPSN<span class="highlight">Q</span><span class="highlight">H</span><span class="highlight">H</span><span class="highlight">Q</span>Q----HFQNRDML-SHLHIG
Bra010950.1-P   -----------------------------------DTGHDQ----KYLDQ----------
Bra010951.1-P   -----------------------------------D---DQ----KYLDQ----------
VIT_01s0146g001 ILSNPNSYQDATATRHLSHIEQDSNRFDLAEQLMRQQFQQQLQQRQLQQQNLLSSHAHLN
                                                   :   :*    :  ::          

AT1G27430.1     DQDLEHL-ITLQLQQQQKIQM--QQQQKIQLQQQQKIQLQ<span class="highlight">Q</span><span class="highlight">H</span><span class="highlight">Q</span>LEQE<span class="highlight">H</span><span class="highlight">Q</span>L<span class="highlight">H</span><span class="highlight">Q</span>KLLQEQQQ
Bra010950.1-P   ARDLDHL-LTLKSQQQ-------QQQQKIQLQQQQKIQLQQRQLEQEYQLQQKLLREQQQ
Bra010951.1-P   AQDLDHLTLQLQQQQQQKIQL<span class="highlight">Q</span><span class="highlight">H</span><span class="highlight">Q</span>QQQKIQLQQQQKIQLQQRQLEQEYQLQQKLLQEQQQ
VIT_01s0146g001 ESLLEQV-ASRNHMH<span class="highlight">H</span><span class="highlight">Q</span>RLANQPF<span class="highlight">H</span><span class="highlight">Q</span>KQMLLQEQKQAQARQALLEQYNSV----------
                   *:::    :  ::        :*::: **:*:: * :*  ***  .:          

AT1G27430.1     SHARQLHFQQILQGQTPDTRFGQSHDFPRSNSVDQMLLEQQMLNELQKSSGHPSQNFAPY
Bra010950.1-P   SHARNLHFQQILQGQTPDSRLGQSHDFHRLNNVDQMILEQQLMDELQ-SSGHRSQNFTPY
Bra010951.1-P   SHARNLHFQQILQGQTPDSRLGQSHDFHRLNNVDQMLLEQQLMDELQ-SSGHRSQNFTPY
VIT_01s0146g001 --------PTILRGM---------------------------------------------
                          **:*                                              

AT1G27430.1     IE<span class="highlight">Q</span><span class="highlight">H</span>AAGNFGRFTHEG<span class="highlight">H</span><span class="highlight">Q</span>RELLEQLFSTQMQSQYGQKQSQYGQMQS--------------
Bra010950.1-P   MEQLAAGNFGQLPHEG<span class="highlight">H</span><span class="highlight">Q</span>KELLEQLLSEQMQSQYRQMHS<span class="highlight">Q</span><span class="highlight">H</span>GHMQSEPIRSLEYQLLQQD
Bra010951.1-P   MEQLAAGNFGQLHHEG<span class="highlight">H</span><span class="highlight">Q</span>KELLEQLLSEQMQSQYRQMHS<span class="highlight">Q</span><span class="highlight">H</span>GHMQSEPIRSLEYQLLQQE
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G27430.1     -----------------------<span class="highlight">Q</span><span class="highlight">H</span>GQLQSEPIRSLEYQLLQQEQLMQLANGVRHNTLLE
Bra010950.1-P   QLMRLANGGRQNTLLEELRHIDP<span class="highlight">Q</span><span class="highlight">H</span>GQLQSEPIRSLEYQWMQQEQLMQLANRARNNTLLE
Bra010951.1-P   QLMRLANGGRQNTLLEELRHIDP<span class="highlight">Q</span><span class="highlight">H</span>GQMQSEPIRSLEYQIMQQEQLMQLADRARHNTLLE
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G27430.1     EQRHIDPLWPSDHSDQLLRTHPGIHRSHSSAGFRPLDF<span class="highlight">H</span><span class="highlight">Q</span>QQQRPHFEDQFSQLERNRSY
Bra010950.1-P   EHRHIDPLWPSNRNDQLLGAHPGIHRSHSSAGFRPVDF<span class="highlight">H</span><span class="highlight">Q</span>QQQGPPFEDHFSHLERNLSY
Bra010951.1-P   EHRHIDPLWPSNRNDQLLGAHPGIHRSHSSAGFRPVDF<span class="highlight">H</span><span class="highlight">Q</span>QQQRPPFEDHFSHLERNLSY
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G27430.1     QQQLRLELLEHGLPFERS------ASGLNLDAVNGLGLSQGLELRDATAHMQSSGRLGNS
Bra010950.1-P   QQQLNQELFEQGLPFERS------TSGLNLDAVKGLNFS---ELRD--AQMQSSGRLGNS
Bra010951.1-P   QQQLNQELFEQGLPFERSTSLPRNTSGLNLDAVKGLNLS---ELRD--AQMQSSGRLGNS
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G27430.1     TPGFS<span class="highlight">H</span><span class="highlight">Q</span>NPRIPLGESHFSHLEPTEGRWSGADTQLAGDWAESQFRRSNMDTEHDKMRSEI
Bra010950.1-P   TPGFS<span class="highlight">H</span><span class="highlight">Q</span>NPHIQLGEPHFSEMEPRKERWHGADTQLAGGWAETQFHRPNTEADHHKMRSEM
Bra010951.1-P   TPGFS<span class="highlight">H</span><span class="highlight">Q</span>NPHIQLGEPHFSEMEPRKERWHGADTQLAGGWPETQFHRSNTEADHHKMRSEM
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G27430.1     RRLGEDPNSWMVGGSTDDKSKQLFMELL<span class="highlight">H</span><span class="highlight">Q</span>RPG<span class="highlight">H</span><span class="highlight">Q</span>SAESPNMNRGYPYDRMVPSGLTPGI
Bra010950.1-P   RRAGEDSNSWMVDGHTDEKSKQLFMELL<span class="highlight">H</span><span class="highlight">Q</span>RPG<span class="highlight">H</span><span class="highlight">Q</span>PLESPSMNRGEPYDRMAASGF----
Bra010951.1-P   SRVGEDSNSWMVDGHTDEKSKQLFMELL<span class="highlight">H</span><span class="highlight">Q</span>RPG<span class="highlight">H</span><span class="highlight">Q</span>PLESPSMNRGEPYDRMAASGF----
VIT_01s0146g001 --------------------------LIHL------------------------------
                                          *:*                               

AT1G27430.1     QTLGGLSDHGGNQNVSSAFGDRSFSDEQVNRVPGYGNNMGSLHHNSSLLSGIIDAGRSTQ
Bra010950.1-P   ----GFADHGGRQNASSSFGSHTSSDEHVNGLPGDRNYMGSLQRDNSLLSGSTDGGR--K
Bra010951.1-P   ----GFADHGGRQNASSSFGSHASSDEHVNGLPGDGNYMGSLQRDNSLLSGSTDGGR--K
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G27430.1     NETQAFSNMFGMNKDANDINTWNNVPPKNEGMGRMMSYDAQDRMGKQAVLDSLIQEELPV
Bra010950.1-P   NETKDFSNMLGMSKDVNDIRTWNNAPPKKEGAG-LMSFEAQDRMGKQAVMDSLVQGEVPV
Bra010951.1-P   NETKDFSNMLGMSKDVNDIRTWNNAPPKKEGAG-LMSFEAQDRMGKQAVMDSLVQGEVPV
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G27430.1     GTPGQQSSFNISDRYSDNLVGEDRRKDRLVVPSHGQNSVLLKRPPSSHSSSSHEGLLERM
Bra010950.1-P   ATLGRQSSSSISESYSDNLVGE-VRKDRLVVPSHGQVSVLLKRPP-----SSHEGLLEQM
Bra010951.1-P   ATLGRQSSSSISGMYLPEF--------RFRFP----------------------------
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G27430.1     SDTASRAAASSYSGIEGGVRRESGAAGNKGSTSEAASFSEMLKKS---NSMKKVAAE-ST
Bra010950.1-P   SDATNRT-----------------AVGNKGSK---ASFSEMLKNSSSNSSMKKVAAEPSS
Bra010951.1-P   ------------------------------------------------------------
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G27430.1     DATEGSKGGGGKKKGKKGRQIDPALLGFKVTSN--------RILMGEIHR----------
Bra010950.1-P   DPNEGNKGGGGKKKGKKGRQLDPALLGFKKTYKADSQTGKWRSLTTFIKRGFVPETTTSL
Bra010951.1-P   ------------------------------------------------------------
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G27430.1     ---------ADDF
Bra010950.1-P   FQIMRTAASGDTA
Bra010951.1-P   -------------
VIT_01s0146g001 -------------
                             


<a href='HQ_QH-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            