
<!doctype html>
<html>
    <head>
        <title>HQ-QH</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT4G02180.1</h1>
        <h3>AT4G02180.1 pep chromosome:TAIR10:4:959964:963678:1 gene:AT4G02180 transcript:AT4G02180.1 gene_biotype:protein_coding transcript_biotype:protein_coding description:DC1 domain-containing protein [Source:UniProtKB/TrEMBL;Acc:Q9ZU00]</h3>
        <pre>
length  989
count   11
ratio   0.00
zscore  8.24






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT4G02180.1     MGLINQWFPLDTLATPERIRMACFDEFTSPFYMTMDSEGVLMPLVHEHLMMPWNDLRKGD
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   ----------------------------------MESEGVSFPLIHEHPMKPWNDLRKGD
                                                                            

AT4G02180.1     CCGLSKAVSVGYYCKSCDFFAHKKCGES-SEFI<span class="highlight">Q</span><span class="highlight">H</span>PSHPNHTLQLRSSEGCNSCNLCGRT
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   CCGRFVSVSDGYYCINCDFFVHKKCGDNLSEFIVHPSHPNHTLRIRSKKGFCICRLCGRE
                                                                            

AT4G02180.1     ISDLFYRCDLCDFDVDLYCAKYPPPEVIDIPETHHHKLNLHKELIEFNCDAKCGKITGAE
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   TWNTFYRCKVCDFKVDLDCAKHPPPLVIENFETHSHKLTFLKELTEFYCCAKCGK-AGDG
                                                                            

AT4G02180.1     FPYVCYECQLPFHVDCVWRPS-ELKHPSEVNHSYHSLHPLKLLSGQLPDNCDGKCRLCAR
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   FTYKCEECDVAFHVDCVWHPTGELNYSPEVSHSYHSLHPLKLLRGPPPDYSDGKCRLCEN
                                                                            

AT4G02180.1     KIDDRLFYHCSPCNFTLDLRCVLNPPQQSLLNLKAHD<span class="highlight">H</span><span class="highlight">Q</span>LTLLPRLLSFTCNACGLNGDR
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   EVDEELFYHCSSCNFTLDMRCVLNSPPQSLVDLKAHG<span class="highlight">H</span><span class="highlight">Q</span>LTLLPRLFSFSCNACGLKGDR
                                                                            

AT4G02180.1     SPYTCVQCDFMI<span class="highlight">H</span><span class="highlight">Q</span>DCLDLPRVININ<span class="highlight">Q</span><span class="highlight">H</span>DHRVSRTSVPGVGN------------------
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   SPYMCVQCDFMI<span class="highlight">H</span><span class="highlight">Q</span>DCLGLPRVININRHDHRVSRTLILGVVNSVCGVCRKKVDWTCGGFS
                                                                            

AT4G02180.1     ------------------------------------------------------------
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   CQRCPGYVVHSKCATREDVWNGKELEGIPEEEEDVEPYVVVDENTI<span class="highlight">Q</span><span class="highlight">H</span>FSHKEHNIRLHG
                                                                            

AT4G02180.1     ------------------------------------------------------------
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   NGVLKEHNLRLHGNGVLYEDNKWCSACTHPIGLQSFYDCIDCDFSLHHKCAECPKKKWHV
                                                                            

AT4G02180.1     ------------------------------------------------------------
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   LHNERLTLVTNKELRVFSCDACNRTSNGFIYKDGDKMLDVLCGSITEPFVHPSHPEHPLY
                                                                            

AT4G02180.1     ------------------------------------------------------------
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   YIPTGELCDGCSNIERRMLRCVEGDCEYVLGFKCATLPQVVKHRIDDHPLSLCYGEKKAS
                                                                            

AT4G02180.1     ------------------------------------------------------------
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   GKYWCDICEERTYPNNWFYTCKDQRASLHTECVLGDFEGLMPRSVAKLWGESYEVELNNS
                                                                            

AT4G02180.1     ------------------------------------------------------------
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   IEGAFVQGLGWAALEEVKCGDASHKRIKPGNLLSCGPGYYKIPSINDIPFNFNVSLLKAG
                                                                            

AT4G02180.1     ------------------------------------------------------------
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   ALVHVYTVLVTHGGVEMGQGLHTKVAQVSASAFNIPLSSVFVSETSTGKVIDFLRTERTI
                                                                            

AT4G02180.1     ------------------------------------------------------------
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   TVKQCITSVITHISPSFVLFAIKDAIKAARAEVGLGRKWFPLVTPATPERIRIACFDEFT
                                                                            

AT4G02180.1     ------------------------------------------------------------
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   SPFNGPLDNKRHRGSTHRVLPPTVVGGTAPATSFRRLSPQCGTQQAPRCRRAEGGAARET
                                                                            

AT4G02180.1     ------------------------------------------------------------
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   EEHPTRNEALNDQQQTGPREALSEAIEEVREAMLQYTKCADPTESAARRERMRKAEEEGE
                                                                            

AT4G02180.1     ------------------------------------------------------------
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   LEETALLMIQANQDKAAMEEELPQDMKTSAERIPAILRLGPSPIAVSEATAVPEKKKLGR
                                                                            

AT4G02180.1     ---------------------------------------------------------SKG
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   PPGRRRV<span class="highlight">H</span><span class="highlight">Q</span>SPRLIKGSNSKKRKTLQTKPPTVRRKLNTDDAVDTRQRDLCIRERKMESEG
                                                                            

AT4G02180.1     VLLPWFHRKHIMMPWNDMRKGDCCGSLESVTDGFYCKSCDIFIHKKCGE-SSGIIDHSSH
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   VLLPLIH-EHPMMPWNDLRRGDCCGRYVSQSDGYYCKLCDFFVHKECGELTEFIIDHPAH
                                                                            

AT4G02180.1     PDHTLELNRYPNKSCNLCGRSKGVNVCYR-CDHCYYQLDLYCAKYPPPEFIDIPETHHHK
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   PDHTLRLQGIRGHVCDICER-KIKNLCYRCCDGCDFDMDLHCAKYPPPDVIDNFETHSHK
                                                                            

AT4G02180.1     LTLLKERIEFDCDAKCGETGDGFAYKCPECDLFFHVDCVWHP--------PE--VNHPLE
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   LTLVKKNTVFDCSAKCGK-AYGLSYKCEECDVAFPVECVWHPEASELNYSPERLVNYFII
                                                                            

AT4G02180.1     VNHSYHPWHPLKLHTGQPPHYSDGTCRLCARNIDDRLFYHCSPCNFTLDLRCVLNPPPQS
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   VHLVTSLWVSIVF-----------------------LIHH---KNLLLIRKCMITNSP--
                                                                            

AT4G02180.1     LLNLKAHD<span class="highlight">H</span><span class="highlight">Q</span>LILLPRLRSFTCNACGLSGDRSPYICIQCDFMI<span class="highlight">H</span><span class="highlight">Q</span>DCLDLPRLINVNRHD
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   ----FFHDSILLLV------MLAVCGLKGDRSPYACFDCGFMI<span class="highlight">H</span><span class="highlight">Q</span>DCLGLPRVININRHD
                                                                            

AT4G02180.1     HRVSRTSVLG--VVNSVCGVC<span class="highlight">H</span><span class="highlight">Q</span>KVDWTCGGFSCQRCSSYVVHSKCATRRDVWNGKELEG
Bra036235.1-P   ------------------------------------------------------------
Bra036288.1-P   HRVSRTSVLGDAAMNSVCGVCRKKVDWTCGGFSCKRCPGYVIHSKCATRDDVWNGEELEG
                                                                            

AT4G02180.1     V-PEEIEDIEPYVVIDDNTI<span class="highlight">Q</span><span class="highlight">H</span>FSHKEHYL-RLHVNDVLCNDNKRCKACTHPIFLQSFYS
Bra036235.1-P   --------------------------------------MKEQNKRCSACTHPVGLLSFYG
Bra036288.1-P   VHEEEEEDTEPYVVIDDNTIRHFSHEEHNLRRIHGNGIMYEEQKRCKACTHPIGLKSFYD
                                                      : :::***.*****: * ***.

AT4G02180.1     CMDCDFLL<span class="highlight">H</span><span class="highlight">Q</span>NCAGFPRMKRHVLHNERLTLVTN-ETKLFQCAPCDRWSNGFRY<span class="highlight">Q</span><span class="highlight">H</span>GYKSL
Bra036235.1-P   CTECDFSL<span class="highlight">H</span><span class="highlight">Q</span>KCAECPTRKWHVLHNERLTLVTNKELEVFDCYACKRNSNGFMYKHGTKKL
Bra036288.1-P   CMDCDFSL<span class="highlight">H</span><span class="highlight">Q</span>KCAECPKTKRHVLHNDRLTLVTNEELADFLCKGCDLNYNGFMYKHGDIEL
                * :*** ***:**  *  * *****:******* *   * *  *.   *** *:**  .*

AT4G02180.1     DLQCGSISEPFVHPSHPDHPLYHTLLDGRNEICDGCKKSWYYVLSCIEDDCRFVLCFKCL
Bra036235.1-P   DVLCGSISEPFTHPSHPHHPLYYTLIE-KEELCNGCNGREYFILKCIE-------GFTCA
Bra036288.1-P   DVLCGSISEPFVHPSHPHHPLYYMPTE-DFEICNGCNRMELHVLRCIEIDCVFVLGFKCA
                *: ********.*****.****:   :   *:*:**:    .:* ***        *.* 

AT4G02180.1     TLPQVVKHRVDDDPLLLCYG--EKACGKYWCDICEKETNPETWFYTCKD<span class="highlight">H</span><span class="highlight">Q</span>SSMHTMCVL
Bra036235.1-P   TLPQVVNHRVDDHPLSLCYGEEEEASGKYWPDICERETNPNNWFYACKNHLACLHIKCVL
Bra036288.1-P   TLPQVVKHRVDDHPLSLCYGEEEEASGKYWCDICERETNPKEWFYTCKDQRASLHTKCVL
                ******:*****.** ****  *:*.**** ****:****: ***:**:: :.:*  ***

AT4G02180.1     GDSVGIMPKSTIMGWYKYYEVVLNNSISRPVCEMCKSRCIFPTIMKILEASNEYVCSLAC
Bra036235.1-P   GDSSGFMPSSVATFWTRSFEVVLNDSVTLPFCSRCKSRCMYPINLKLLGRSSTYICSINC
Bra036288.1-P   GDLGRFMPRSVVKFQNRSFEVVLNKSVTRPFCSKCKSRCIYPIILKLLGISETSFCSFEC
                **   :** *.     : :*****.*:: *.*. *****::*  :*:*  *.  .**: *

AT4G02180.1     VL---ERG---
Bra036235.1-P   AS--HWRGTTI
Bra036288.1-P   AVLFYWFGEK-
                .      *   


<a href='HQ_QH-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            