
<!doctype html>
<html>
    <head>
        <title>HQ-QH</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT4G36870.2</h1>
        <h3>AT4G36870.2 pep chromosome:TAIR10:4:17368915:17374011:1 gene:AT4G36870 transcript:AT4G36870.2 gene_biotype:protein_coding transcript_biotype:protein_coding gene_symbol:BLH2 description:BEL1-like homeodomain protein 2 [Source:UniProtKB/Swiss-Prot;Acc:Q9SW80]</h3>
        <pre>
length  739
count   15
ratio   0.00
zscore  11.45






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT4G36870.2     MGIT--------------KTSP----------------------------------NTTI
Bra011713.1-P   ------------------------------------------------------------
Bra010577.1-P   ------------------------------------------------------------
AET04003        MGIA--------------TTPS--------------SIPSTFSNSSSIQNYHSSFNSSIL
AES94832        MGVA--------------SPV---------------PFPSSSSYSSIIQ-LKT<span class="highlight">H</span><span class="highlight">Q</span>NQRTL
KRH76894        ------------------------------------------------------------
KRH08328        MGIA--------------TPSS--------------SLPSILSHSKT<span class="highlight">H</span><span class="highlight">Q</span>-----------
KRH28575        ------------------------------------------------------------
KRH69971        MGIA--------------TPSS--------------SLPSILSHSKT-Q-----------
Zm00001d033898_ MGVA--------------APPY--------------ETTTYVSQRTKIN-IAAISAAAAT
BGIOSGA013510-P MGIA--------------APPCQ-----------PGQTTTFVISQPTSS-KSSPAAIVVR
Os03t0732100-01 MGIA--------------APPCQ-----------PGQTTTFVISQPTSS-KSSPAAIVVR
VIT_18s0001g118 ------------------------------------------------------------
VIT_04s0023g010 MSKAENEKKGGQKGKIEKAPTKTHLMPSLFLFTTP-PLPPVFSHSKVHN-------PIPI
                                                                            

AT4G36870.2     LLKTF--HNN-SMSQDYHHHHHHN<span class="highlight">Q</span><span class="highlight">H</span><span class="highlight">Q</span>GG-IFNF-SNGFDRS---DSPNLTTQQ---KQE
Bra011713.1-P   ------------------------------------------------------------
Bra010577.1-P   ------------------------------------------------------------
AET04003        EKSQN--STN-SMSQDYHHHH<span class="highlight">H</span><span class="highlight">Q</span>NH<span class="highlight">H</span><span class="highlight">Q</span>QG-IFAF-SNGFERS---NTPENNNPH---QQQ
AES94832        DCS----TSN-SMSQDY---------QQA-IFGFSSNGFERS---SSSQQQQQQ------
KRH76894        ------------MSQDF---------QQG-IFSF-PNGLERG---RVMSPQ---------
KRH08328        -QQIH--HSN-SMSQDYHH-----HH<span class="highlight">H</span><span class="highlight">Q</span>G-IFSF-PNGFERS-AATTMT<span class="highlight">H</span><span class="highlight">Q</span>DPHHHHHHH
KRH28575        ------------MSQDF---------QQG-IFSF-PNGLERG---RLVSPQ---------
KRH69971        -QQIH--HSNSSMSQDY-H-----HH<span class="highlight">H</span><span class="highlight">Q</span>G-IFSF-PNGFERSAAATTMT<span class="highlight">H</span><span class="highlight">Q</span>DPH------
Zm00001d033898_ RAATA--ASM-SHSQGFH------QGSAG-VYGYSSDGFDRP-DDSG--<span class="highlight">H</span><span class="highlight">Q</span>DHH--QQQQ
BGIOSGA013510-P PPATASSSSM-SHSQGFH------QPSSGAVFGFSSDGFDGR-PGSGQD<span class="highlight">H</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">H</span>E-QQQQQ
Os03t0732100-01 PPATASSSSM-SHSQGFH------QPSSGAVFGFSSDGFDGR-PGSGQD<span class="highlight">H</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">H</span>E-QQQQQ
VIT_18s0001g118 ------------------------------------------------------------
VIT_04s0023g010 AHKSN--FAN-SMSQDF---------<span class="highlight">H</span><span class="highlight">Q</span>G-IFSF-SNGFERS---AVT<span class="highlight">H</span><span class="highlight">Q</span>EQQQ---QQQ
                                                                            

AT4G36870.2     H--QRVEMDEESSVAG----------------------GRIP--VYESA--------GML
Bra011713.1-P   ------------------------------------------------------------
Bra010577.1-P   ------------------------------------------------------------
AET04003        ---QQIQRDKVR-IQDFDSQQPPQPPPPPLIGNIEEEQEALP--VYETT--------GML
AES94832        --QQQIQRDKVRLLQGLN-----------LSEGEGDERGG----VYETA--------GML
KRH76894        ---QQIRRDKVR-FE-----------AAPLVGIEEEE----P--VYETA--------GML
KRH08328        <span class="highlight">H</span><span class="highlight">Q</span>QQQIRRDKVR-VQGFEPP----P-QQTLLPIEEDESGSLP--VYETA--------GML
KRH28575        ---QQIRRDKVR-FEA----------AAPLVGIEEEE----P--VYETA--------GML
KRH69971        --QQQIRRDKVR-VQGFEPP----PS<span class="highlight">H</span><span class="highlight">Q</span>TLVPIEEDESGSLP--VYETA--------GML
Zm00001d033898_ HVAQQSRRDKLR-VQGFDP-AA--GGGHGLLPIEGDEHGG-PGSMYEHA-AAAAGASNML
BGIOSGA013510-P HVAQQSRRDKLR-VQGFDPAAA--AAGHGLLPIEGDEHGAEPGAMYDHAEAAAAGASNML
Os03t0732100-01 HVAQQSRRDKLR-VQGFDPAAA--AAGHGLLPIEGDEHGAEPGAMYDHAEAAAAGASNML
VIT_18s0001g118 ------------------------------------------------------------
VIT_04s0023g010 HITQQIRRDKLR-VQGFEP-------PPPLVAIEEEESGGLP--VYETA--------GML
                                                                            

AT4G36870.2     SEMFNF---PGSS-GGGRDLD------------LGQSFR---------------------
Bra011713.1-P   ------------------------------------------------------------
Bra010577.1-P   ------------------------------------------------------------
AET04003        SEMFNF---PHGA-SAAELLEQQQQQQQQQ-MASTRTPQ---------------APHGIG
AES94832        SEMFNF---ADP--STAELLE-------------TATFR--SSSSSSSRQ----LPQTTA
KRH76894        SEMFNF---PP----ATELLEQ<span class="highlight">Q</span><span class="highlight">H</span>---------ATATFR--AARQA--------------
KRH08328        SEMFNF---PPG---AAELLEQQQQQQQQQPMAMTTTFR--ASPSA---------RAVGS
KRH28575        SEMFNF---PP----ATELLEQ<span class="highlight">Q</span><span class="highlight">H</span>---------ATATFR--AARQA--------------
KRH69971        SEMFNF---TPG---ATELLE--QQQQQQQPMATTT---------A---------RAVGS
Zm00001d033898_ SEMFNFPA-PPSGPSATELLASQ----------MGANYRFGGLRQQQQQGPGGVVAGLS-
BGIOSGA013510-P SEMFNFPSQPPTGPSATELLASQ----------MNANYRF-GFRQA---------AGLAG
Os03t0732100-01 SEMFNFPSQPPTGPSATELLASQ----------MNANYRF-GFRQA---------AGLAG
VIT_18s0001g118 ------------------------------------------------------------
VIT_04s0023g010 SEMFNF---GPGATTAAELLE<span class="highlight">H</span><span class="highlight">Q</span>----------LPSNYR--NPRPA--------TAVTGV
                                                                            

AT4G36870.2     ------SN-----R-QLLE---------------------------------------EQ
Bra011713.1-P   ------------------------------------------------------------
Bra010577.1-P   ------------------------------------------------------------
AET04003        NSSDWYGN-----R-QTMLT------DSKNHH------NNRDSSSSSSI------FH<span class="highlight">H</span><span class="highlight">Q</span>N
AES94832        SENDWYGNNN---R-QGIQ---------------------------------------<span class="highlight">Q</span><span class="highlight">H</span>
KRH76894        --GEWYGN-----R-Q------------------------------------------QQ
KRH08328        GGSEWYGN-----R-QGMLSGLGPLGDSKNHHHHHGSVNSRDSSSSSIVQN<span class="highlight">Q</span><span class="highlight">H</span>HHHHN<span class="highlight">H</span><span class="highlight">Q</span>
KRH28575        --GEWYGN-----R--------------------------------------------QQ
KRH69971        GGSEWYGN-----R-QGMLSNLGPLGDSKN-HHHHGSVNSRDSSSSSIVQN<span class="highlight">Q</span><span class="highlight">H</span>HHHHNHH
Zm00001d033898_ GDGGWFVGAGAAGRAAGLVL-----GGGAN----MGSLGETSSPK-------------QQ
BGIOSGA013510-P GEGGWFGGGGAAGR-TGLVL------GGAS----LGSLGETSSPK-------------QQ
Os03t0732100-01 GEGGWFGGGGAAGR-TGLVL------GGAS----LGSLGETSSPK-------------QQ
VIT_18s0001g118 ------------------------------------------------------------
VIT_04s0023g010 SNSEWYGS-----R-QGMVGGLGPLGDSKNQ-----NVNNRDSLA-------------<span class="highlight">Q</span><span class="highlight">H</span>
                                                                            

AT4G36870.2     <span class="highlight">H</span><span class="highlight">Q</span>NIPAMNATDSATATAAAMQLFLMNPPPPQQPPSPSSTTSPRSHH------NSSTLHML
Bra011713.1-P   ------------------------------------------------------------
Bra010577.1-P   ------------------------------------------------------------
AET04003        H<span class="highlight">H</span><span class="highlight">Q</span>MSNINA-----ESAAAMQLFLMNPQ-----TTRSPSPPPQPHPSST---NSSTLHML
AES94832        QQQISSINVAD---DSAAAMQLFLMNPSQS<span class="highlight">H</span><span class="highlight">Q</span>TTTSSSS-PPPTH<span class="highlight">H</span><span class="highlight">Q</span>N-----SSTLHML
KRH76894        QQQISGINA-----DSAAAMQLFLMNPQ------TRSPS-PPQSHTTP-----SSTLHML
KRH08328        H<span class="highlight">H</span><span class="highlight">Q</span>MSSINA-----DSAAAMQLFLMNPQ-----TTRSPSPPPPPPPPP-----SSTLHML
KRH28575        QQQISGINA-----DSAAAMQLFLMNPQ------TRSPS-PPQTHATP-----SSTLHML
KRH69971        H<span class="highlight">H</span><span class="highlight">Q</span>MSSINA-----DSAAAMQLFLMNPQ-----TTRSPS-PPPPPPPS-----SSTLHML
Zm00001d033898_ GS-MAGLAT-----DPAAAMQLFLMNPQQQQQ-----SRSSPPSPSDV-----QV<span class="highlight">Q</span><span class="highlight">H</span>HEA
BGIOSGA013510-P ASGMAGLAA-----DPAAAMHLFLMNPQ-QQQQQSRSSTSPPPSDAQS-----AI<span class="highlight">H</span><span class="highlight">Q</span><span class="highlight">H</span>HE
Os03t0732100-01 ASGMAGLAA-----DPAAAMHLFLMNPQQQQQQQSRSSTSPPPSDAQS-----AI<span class="highlight">H</span><span class="highlight">Q</span><span class="highlight">H</span>HE
VIT_18s0001g118 ------------------------------------------------------------
VIT_04s0023g010 H<span class="highlight">H</span><span class="highlight">Q</span>ISSINA-----DSAAAMQLFLMNPQ------PRSPSPPPQPHPHPHPPATSSTLHML
                                                                            

AT4G36870.2     LPSPSTNTTH<span class="highlight">H</span><span class="highlight">Q</span>NYTNHMSM<span class="highlight">H</span><span class="highlight">Q</span>LP<span class="highlight">H</span><span class="highlight">Q</span><span class="highlight">H</span>---<span class="highlight">H</span><span class="highlight">Q</span>QIS---------TWQSSPD-------HH
Bra011713.1-P   ------------------------------------------------------------
Bra010577.1-P   ------------------------------------------------------------
AET04003        LPNPSTT-----------NLQGYNSGA---GGSFG-------QFTW--AQEGGGGGATSS
AES94832        LPNPPNN-----------SLQGFPN-----SGNFG-------QFTW--NST-------TT
KRH76894        LPNPSSN-----------SLQGFTGSAA--GGSFG-------QFTW--VPE-------GA
KRH08328        LPT-------------------FPPGS---GGSFS-------QFTW--LPD-------TT
KRH28575        LPNPSSNS----------SLQGFTGSAA--GGSFG-------QFTW--VPE-------SA
KRH69971        LPNT------------------FPPGS---GGSFG-------QFTW--LPD-------TT
Zm00001d033898_ ------------------AFQAFGNAG----APFG-------------------------
BGIOSGA013510-P ------------------AFQAFGGAG---AAAFG-------------------------
Os03t0732100-01 ------------------AFQAFGGAG---AAAFG-------------------------
VIT_18s0001g118 ------------------------------------------------------------
VIT_04s0023g010 LPNQST------------SLQGFATASAPGGGAFGASVIPPSQFTW--VPD-------SG
                                                                            

AT4G36870.2     HHH----HNSQTEIGTVHVENSGGHGGQGLSLSLSSS---LEAAAKA-EEYRN-------
Bra011713.1-P   ------------------------------------------------------------
Bra010577.1-P   ------------------------------------------------------------
AET04003        H<span class="highlight">H</span><span class="highlight">Q</span>---LNNNQPEMASV-VE------GQGLSLSLSSS---LQ-HHHS-EDLRMGDAA--A
AES94832        T<span class="highlight">H</span><span class="highlight">Q</span>-----------EGA-VEGQ---SVQGLSLSLSSS---LE-AAKAEEELRMGGGGGDG
KRH76894        <span class="highlight">H</span><span class="highlight">Q</span>Q-----------GGV-VE------GQGLSLSLSSS---LE-AAKA-EELRMGDS----
KRH08328        QEGGGP--------ST---EGP--GHGQGLSLSLSSS---LE-AAKA-EELRMGNS----
KRH28575        <span class="highlight">H</span><span class="highlight">Q</span>Q-----------GGV-VE------GQGLSLSLSSS---LE-AAKA-EELRMGDS----
KRH69971        QE-GGP--------STV-VEGPGHGHGQGLSLSLSSS---IE-AAKA-EELRMGDS----
Zm00001d033898_ ---GGA-G------TGV-VE------GQGLSLSLSPSLQQLEMAKQA-EELRVRD-----
BGIOSGA013510-P ---GGA-A------AGV-VE------GQGLSLSLSPSLQQLEMAKQA-EELRVRD-----
Os03t0732100-01 ---GGA-A------AGV-VE------GQGLSLSLSPSLQQLEMAKQA-EELRVRD-----
VIT_18s0001g118 ------------------------------------------------------------
VIT_04s0023g010 HES----GNNPSEIGGI-VE------GQGLSLSLSSSL<span class="highlight">Q</span><span class="highlight">H</span>LE-AAKA-EELRMGDS----
                                                                            

AT4G36870.2     ---IYYGA------NSS----NASP---H<span class="highlight">H</span><span class="highlight">Q</span>YNQFKTLLANSS<span class="highlight">Q</span><span class="highlight">H</span>H<span class="highlight">H</span><span class="highlight">Q</span>VL----------
Bra011713.1-P   ------------------------------------------------------------
Bra010577.1-P   ------------------------------------------------------------
AET04003        GGFLYYNQ------GGP----GTS-------YK-------NLGSVHHHHLQG----GGLG
AES94832        GSFMYNNYNIQGGSGGP----SSST----YPYK----------LN<span class="highlight">H</span><span class="highlight">Q</span>QALNLQLQGGGTI
KRH76894        -GFLYYN<span class="highlight">H</span><span class="highlight">Q</span>QGGGGGGGGGGPSSSSSAVQFQYK-------NNNSHH<span class="highlight">H</span><span class="highlight">Q</span>ALHLQ---GAMG
KRH08328        -GFLYYNQA----SGGP----SSYKST--------------LGGHH<span class="highlight">H</span><span class="highlight">Q</span>ALLA--------
KRH28575        -GFLYYN<span class="highlight">H</span><span class="highlight">Q</span>QGGGGGGG----SSSSSAVQFQYK-------NNNNNH<span class="highlight">H</span><span class="highlight">Q</span>ALHLQ---GAMG
KRH69971        -GFLYYNQA----SGGP----SSYKST--------------LGGHH<span class="highlight">H</span><span class="highlight">Q</span>ALLG--------
Zm00001d033898_ -GVLYFNRQQQQQ---------------------------------QQQLPMALH-GQVG
BGIOSGA013510-P -GVLYFNRQQQQQQAAA----AAAS--------------------VQQQLPMALALVQWS
Os03t0732100-01 -GVLYFNRQQQQQQAAA----AAAS--------------------VQQQLPMALH-GQVG
VIT_18s0001g118 ------------------------------------------------------------
VIT_04s0023g010 -GMLFYGQ------GGG----SSSA---QYPYK-------SLGG-<span class="highlight">H</span><span class="highlight">Q</span>QPLHLQ---GGVG
                                                                            

AT4G36870.2     ------NQFRSSPA-----------ASSSSMAAV-NILRNSRYT-TAAQEL-LEEFCSVG
Bra011713.1-P   ------------------------------------------------------------
Bra010577.1-P   ------------------------------------------------------------
AET04003        Q---INHI<span class="highlight">H</span><span class="highlight">Q</span>GHVG------FGSS-SSSSSLGVV-NVLRNSKYV-KAAQEL-LEEFCSVG
AES94832        NGYQLLQSHNQGQG------YGSV--------VV-NVLRNSKYM-KPTQEL-LQEFCSVG
KRH76894        HD----NN<span class="highlight">H</span><span class="highlight">Q</span>GHVG------FG-----SSSLGVV-NVLRNSKYV-KAAQEL-LEEFCSVG
KRH08328        ------QT<span class="highlight">H</span><span class="highlight">Q</span>GHVG------FGAASSSTSSLGVV-NALRNSKYA-KAAQEL-LEEFCSVG
KRH28575        HD----NN<span class="highlight">H</span><span class="highlight">Q</span>GHVG------FG-----SSSLGVV-NVLRNSKYA-KAAQEL-LEEFCSVG
KRH69971        ------QA<span class="highlight">H</span><span class="highlight">Q</span>GNVG------FGAASSSTSSLGVV-NALRNSKYA-KAAQEL-LEEFCSVG
Zm00001d033898_ A-----LGQQLHVG------YGPA-------GVA-GVLRNSKYT-RAAQEL-LEEFCSVG
BGIOSGA013510-P A-----LLVGTHAARAGVVRLAPS-------GLSLGVLLQLQCTPRAAQRAPRRSSAAFG
Os03t0732100-01 V-----LGQQLHGGGYG----GPA-------GVA-GVLRNSKYT-RAAQEL-LEEFCSVG
VIT_18s0001g118 ------------------------------------------------------------
VIT_04s0023g010 ------HN<span class="highlight">H</span><span class="highlight">Q</span>VHVG------FG------SSLGAV-NVMRNSKYV-KAAQEL-LEEFCSVG
                                                                            

AT4G36870.2     RGFL-KK-NKLGNS--SNPNTCGGDGGG------SSPSSAGA-NKEH----PPLSASDRI
Bra011713.1-P   ------------------------------------------------------------
Bra010577.1-P   ------------------------------------------------------------
AET04003        RGQF-KK-NKFSRQL-SNPNSNQGGGGGSV--G-GGGASSSS-SKDV----SPLSPADRI
AES94832        RGQFIKK-TKFNRQNSSNPNNCSSNVGDSI--------PSSS-SKDT--PPLPLSAADRI
KRH76894        RGQF-KK-SKFNRQN-SNPNSNP-----------GGGGSSPS-SKDAPPPPPPLSAADRI
KRH08328        RGQF-KK-NKFNRQL-SNPSSNLRGS--------GGGASSSS-SKDV----PPLSAADRI
KRH28575        RGQF-KK-SKFNRQN-SNPNSN------------AGGGASPS-SKDA-PPPPPLSAADRI
KRH69971        RGQF-KK-NKFNRQL-SNPSSNLGGSGG----G-GGGASSSS-SKDI----PPLSAADRI
Zm00001d033898_ RGQI-KG-GGRGASA-SNPNNNPGNKGGASSSGAAAQSPSSAPNKEP----PQLSPADRF
BGIOSGA013510-P RGQI-KGSGGRGSAP-NNPNSSKAAAGAASSSG-AAQSPSLA-SKEP----PQLSPADRF
Os03t0732100-01 RGQI-KGGGGRGSAP-NNPNSSKA---AASSSG-AAQSPSSA-SKEP----PQLSPADRF
VIT_18s0001g118 ------------------------------------------------------------
VIT_04s0023g010 RGQF-KK-NKFGRHN-TNPNSNPGGGSA------GGGGSSSS-SKDL----PPLSAADRI
                                                                            

AT4G36870.2     E<span class="highlight">H</span><span class="highlight">Q</span>RRKVKLLTMLEE--------------------VDRRYNHYCEQMQMVVNSFDIVMGH
Bra011713.1-P   -----------MLEE--------------------VDRRYNHYCEQMQMVVNSFDIVMGH
Bra010577.1-P   -----------MLEE--------------------VDRRYNHYCEQMQMVVNSFDIVMGH
AET04003        E<span class="highlight">H</span><span class="highlight">Q</span>RRKVKLLTMLDE--------------------VDRRYSHYCEQMQMVVNSFDLVMGF
AES94832        E<span class="highlight">H</span><span class="highlight">Q</span>RRKVKLLSMLDE--------------------VDRRYNHYCEQMQMVVNSFDVMMGF
KRH76894        E<span class="highlight">H</span><span class="highlight">Q</span>RRKVKLLSMLDE--------------------VDRRYNHYCEQMQMVVNSFDLMMGF
KRH08328        E<span class="highlight">H</span><span class="highlight">Q</span>RRKVKLLTMLDE--------------------VDRRYSHYCEQMHMVVNAFDMVMGF
KRH28575        E<span class="highlight">H</span><span class="highlight">Q</span>RRKVKLLSMLDE--------------------VDRRYNHYCEQMQMVVNSFDLMMGF
KRH69971        E<span class="highlight">H</span><span class="highlight">Q</span>RRKVKLLTMLDE--------------------VDRRYSHYCEQMHMVVNSFDMVMGF
Zm00001d033898_ EQQRKKAKLISMLDE--------------------VDRRYNHYCDQMQMVVNFFDSVMGF
BGIOSGA013510-P E<span class="highlight">H</span><span class="highlight">Q</span>RKKAKLISMLDEARVIRVASLPGEKV--FYMGVDRRYNHYCDQMQMVVNFFDSVMGF
Os03t0732100-01 E<span class="highlight">H</span><span class="highlight">Q</span>RKKAKLISMLDE--------------------VDRRYNHYCDQMQMVVNFFDSVMGF
VIT_18s0001g118 -----------MLDEARPTG-WKLLGNPM--ISIIVDSRYNHYCEQIQVVVNSFDSKMGF
VIT_04s0023g010 E<span class="highlight">H</span><span class="highlight">Q</span>RRKVKLLSMLDEAWKTN-RKFRQVSMNILYTRVDRRYNHYCEQMQMVVNSFDLVMGF
                           **:*                    ** **.***:*:::*** **  **.

AT4G36870.2     GAALPYTALAQKAMSRHFRCLKDAVAAQLKQSCELLGDKDAAGISSSGLTKGETPRLRLL
Bra011713.1-P   GAAIPYTALAQKAMSRHFRCLKDAVAAQLRQSCELLGEKDAAGLSSSGLTKGDTPRLRLL
Bra010577.1-P   GAALPYTALAQKAMSRHFRCLKDAVAAQLRQSCELLGDKDAAGISSSGLTKGETPRLRLL
AET04003        GAAVPYTALAQKAMSRHFRCLKDAITAQLKHSCELLGEKDGVG--TSGLTKGETPRLKLL
AES94832        GAAVPYTALAQKAMSRHFRCLKDAITTQVKQSCELLGEKEGAG---GGLTKGETPRLKVL
KRH76894        GAAVPYTALAQKAMSRHFRCLKEAITAQLKQSCEVLGDKDGAG-SSGGLTKGETPRLKML
KRH08328        GAAVPYTALAQKAMSRHFRCLKDAITAQLKHSCEVLGEKDGAG--NSGLTKGETPRLKML
KRH28575        GAAVPYTALAQKAMSRHFRCLKEAITAQLKQSCEVLGEKDGAG-NSGGLTKGETPRLKML
KRH69971        GAAVPYTALAQKAMSRHFRCLKDAITAQLKHSCEVLGEKDGAG--NSGLTKGETPRLKML
Zm00001d033898_ GAATPYTALAQKAMSRHFRCLKDAIAAQLRATCELLGEKDAGT--SSGLTKGETPRLRAI
BGIOSGA013510-P GAATPYTALAQKAMSRHFRCLKDAIAAQLRGTCEALGEKDAGT--GSGLTKGETPRLRAI
Os03t0732100-01 GAATPYTALAQKAMSRHFRCLKDAIAAQLRGTCEALGEKDAGT--GSGLTKGETPRLRAI
VIT_18s0001g118 GAANPYTTLARKAMSRHFRCMKDAILAQLKISCELLGEKDVMA--ASGLSKGETPRLRLL
VIT_04s0023g010 GAAVPYTALAQKAMSRHFRCLKDAIAVQLKHSCELLGEKDPSG--TSGVTKGETPRLRLL
                *** ***:**:*********:*:*: .*:: :** **:*:      .*::**:****: :

AT4G36870.2     EQSLRQNRAF<span class="highlight">H</span><span class="highlight">Q</span>MGMMEQEAWRPQRGLPERSVNILRAWLFEHFLHPYPSDADKHLLARQT
Bra011713.1-P   EQSLRQQRAF<span class="highlight">H</span><span class="highlight">Q</span>MGMMEQEAWRPQRGLPERSVNILRAWLFEHFLHPYPSDADKHLLARQT
Bra010577.1-P   EQSLRQQRAF<span class="highlight">H</span><span class="highlight">Q</span>MGMMEQEAWRPQRGLPERSVNILRAWLFEHFLHPYPSDADKHLLARQT
AET04003        EQSLRQQRAF<span class="highlight">H</span><span class="highlight">Q</span>MGMMEQEAWRPQRGLPERSVNILRAWLFEHFLHPYPSDADKHLLARQT
AES94832        EQSLRQQRAF<span class="highlight">H</span><span class="highlight">Q</span>MGMMDQEAWRPQRGLPDRSVNVLRAWLFEHFLHPYPSDADKHLLARQT
KRH76894        EQSLRQQRAF<span class="highlight">H</span><span class="highlight">Q</span>MGMMEQEAWRPQRGLPERSVNILRAWLFEHFLHPYPSDADKHLLARQT
KRH08328        EQSLRQQRAF<span class="highlight">H</span><span class="highlight">Q</span>MGMMEQEAWRPQRGLPERSVNILRAWLFEHFLHPYPSDADKHLLARQT
KRH28575        EQSLRQQRAF<span class="highlight">H</span><span class="highlight">Q</span>MGMMEQEAWRPQRGLPERSVNILRAWLFEHFLHPYPSDADKHLLARQT
KRH69971        EQSLRQQRAF<span class="highlight">H</span><span class="highlight">Q</span>MGMMEQEAWRPQRGLPERSVNILRAWLFEHFLHPYPSDADKHLLARQT
Zm00001d033898_ DQSLRQQRAFHHMGMMEQEAWRPQRGLPERSVNILRSWLFEHFLHPYPSDADKHLLARQT
BGIOSGA013510-P DQSLRQQRAFHHMGIMEQEAWRPQRGLPERSVNILRSWLFEHFLHPYPSDADKHLLARQT
Os03t0732100-01 DQSLRQQRAFHHMGIMEQEAWRPQRGLPERSVNILRSWLFEHFLHPYPSDADKHLLARQT
VIT_18s0001g118 DQSLRQQRAL<span class="highlight">H</span><span class="highlight">Q</span>MGMMEPEAWRPQRGLPERSVNILRAWLFEHFLHPYPSDADKHLLSRQT
VIT_04s0023g010 EQSLRQQRAF<span class="highlight">H</span><span class="highlight">Q</span>MGMMEQEAWRPQRGLPERSVNILRSWLFEHFLHPYPSDADKHLLARQT
                :*****:**:*:**:*: **********:****:**:*******************:***

AT4G36870.2     GLSRNQVSNWFINARVRLWKPMVEEMYQQESKEREREEELEENE-EDQETK-NSNDDKST
Bra011713.1-P   GLSRNQVSNWFINARVRLWKPMVEEMYQQESKEREER--LEENEQEDQETKNNSSNDKST
Bra010577.1-P   GLSRNQVSNWFINARVRLWKPMVEEMYQQESKELE------ENE-QDQETK-NSNSDKST
AET04003        GLSRNQVSNWFINARVRLWKPMVEDMYQQELKEAEGSS--EGDQ-EDRENK-NPQSSTSN
AES94832        GLSRNQVSNWFINARVRLWKPMVEEMYQQELNEAEVEA-------EDREMN-QSNSNN--
KRH76894        GLSRNQVSNWFINARVRLWKPMVEEMYQQELKEAESA--------EEREKD-QSN-----
KRH08328        GLSRNQVSNWFINARVRLWKPMVEDMYQQELKEAEGA------E-EEREGN-QSSSNN--
KRH28575        GLSRNQVSNWFINARVRLWKPMVEEMYQQELKEAESA--------EDRENN-NS------
KRH69971        GLSRNQVSNWFINARVRLWKPMVEDMYQQELKEAEGA------E-EDRERN-QSSSNN--
Zm00001d033898_ GLSRNQVSNWFINARVRLWKPMIEEMYQQECRELDGS------S-AGGGGP-ESVNDP--
BGIOSGA013510-P GLSRNQVSNWFINARVRLWKPMIEEMYQQECKELEGS------S-GAG-------DDP--
Os03t0732100-01 GLSRNQVSNWFINARVRLWKPMIEEMYQQECKELEGS------S-GAG-------DDP--
VIT_18s0001g118 GLSRNQVSNWFINARVRLWKPMVEEMYQQDAKEEEAAA----SS-EDREAN-PQN<span class="highlight">H</span><span class="highlight">Q</span>---
VIT_04s0023g010 GLSRNQVSNWFINARVRLWKPMVEEMYQQETKEEEGS--------EEREVN-QLQSSG--
                **********************:*:****: .* :                         

AT4G36870.2     KSNNNESNFTAVRT----TSQTPT--------TTAPDAS---DADAAV--ATGHRLRS--
Bra011713.1-P   KPNNNESNFTAVRT----TSQTPT--------TAAQSAATPPD-------AGHHRLRSAA
Bra010577.1-P   KPNNDDSNFTPVRT----TSQTSM--------TAAQSATTAPDADAAV--ATGHRLRSSS
AET04003        NNNNTSGHLSTTTTTTNIIAQTPTQPS-----TTTSTTTTPQPTS-----NNNQLMRSNN
AES94832        -----SGH----------QTQTPP--------TSTATATEPPPPQ-----------KRSD
KRH76894        ---NISGN----------QAQIPTTPSAAT--TSTATATAPPPPPSTTTKTTKPTGKRSD
KRH08328        -----SG<span class="highlight">H</span><span class="highlight">Q</span>---------LAQTPT-PS-----TTASTATAPPPPT-----TTPPTGKRSD
KRH28575        ---NISGN----------QAQTPTTPSAAT--TS--TATAPPPPPPTTT-ATKPTGKRSD
KRH69971        -----SG<span class="highlight">H</span><span class="highlight">Q</span>---------LAQTPT-PS-----TTASTATAP-----------PPTQRH<span class="highlight">H</span><span class="highlight">Q</span>
Zm00001d033898_ -----SGAD---------DSHSPT---------TTGAAQLPQQ<span class="highlight">Q</span><span class="highlight">H</span>HL---------GTVP
BGIOSGA013510-P -----SGAD---------DTHSPT---------TTAAAH<span class="highlight">H</span><span class="highlight">Q</span><span class="highlight">H</span>RHGQLMVEHGGASSGGGA
Os03t0732100-01 -----SGAD---------DTHSPT---------TTAAAH<span class="highlight">H</span><span class="highlight">Q</span><span class="highlight">H</span>RHGQLMVEHGGASSGGGA
VIT_18s0001g118 ---------------------------------KNSISAQTPRS------STPPASQRSQ
VIT_04s0023g010 ------------------LAQTPT-PSTGAGGAAAATASTTPTT------TTTATGKRSE
                                                     :                      

AT4G36870.2     NINAYENDASS---LLLPSSYSNAAAPAAVSDDLNSRYGGSDAFSA--------------
Bra011713.1-P   DINAYENDPSS-------SSHSNAAA-------VVSSYGGSTAFSA--------------
Bra010577.1-P   DINAYENDPS--------SSYTNVAA--------VSSYAGSTAFSA--------------
AET04003        TINANENDPSL-TPLINRQGFSENQA-------MLLQQPSSTVKTT--VSEVA----PPI
AES94832        -INAAENDPSL-VAINRQQDFSENQA-------MQSTTTINTTVSE----VVV----PP-
KRH76894        TINAPESDPS<span class="highlight">Q</span><span class="highlight">H</span>LAMNNRQGFSENQA-------KKSTASTTTTTI---ASEVA----PPV
KRH08328        ----SESDPSL-APIN-------------------------NTTSTAAMTVTA----SEV
KRH28575        AINAPDSDPS<span class="highlight">Q</span><span class="highlight">H</span>VAMNNRQGFSENQA-------KKSTASTTATTTTAAASEVA----PPV
KRH69971        ----TEKDP---------------------------------------------------
Zm00001d033898_ GVMPHKPDPGA---------------------------AGPSA------ADAAFVGIDPV
BGIOSGA013510-P AMSSHKHEPGV-V-------------------------AGPSSSSAAAVADAAFVGIDPV
Os03t0732100-01 AMSSHKHEPGV-V-------------------------AGPSSSSAAAVADAAFVGIDPV
VIT_18s0001g118 -INAQENDPSL-DTINYRNCFSGTQA------ITQATTTTTTTTTT-------------I
VIT_04s0023g010 -INAADGDPSL-IAIN-RQCFSENQA-------KQATSTIPTTTSTSADVSPP----PPV
                     . :.                                                   

AT4G36870.2     ---------------------VATCQQS------VGGFDDADMD-G-VNVIRFGT-----
Bra011713.1-P   ---------------------IATCQQG------VGGFADADMD-GDNNVIRFGTI----
Bra010577.1-P   ---------------------VATCQKG------IGGFDDTDMD-G-GNVIRFGT-----
AET04003        SD--------------SMAI-DDTCRHGSFVSAEYGTTTGTSADIG-STLIRFGT-----
AES94832        ---F----DSDLPPHRSMAM-DDTCRYGSLVAEDYETGS----DIGSSTLIRFGS-----
KRH76894        SQCF---DDSDLPAHRLMAS-DDTCH---LVTADFGTASASA-DIG-STLIRFGT-----
KRH08328        TQL------SELP--RTMVA-DESCRHGSLVATEFGTASAAS-EIG-STLIRFGT-----
KRH28575        SQCF----DSDLPPHRLMASNDNTCR---LVTADFGTASASA-DIG-STLIRFGT-----
KRH69971        ------------------------------------------------------------
Zm00001d033898_ E----------LLGG---GA-DDL----------YGRFEP-------GVRVRYGPA----
BGIOSGA013510-P E----------LLGGDGAAA-DDL----------YGRFDPAG------------------
Os03t0732100-01 E----------LLGGDGAAA-DDL----------YGRFDPAG-----AVRVRYGPAGAAA
VIT_18s0001g118 SQRFPTTHEAGMH-QRTVAA-DDTWCRGSVFGGEYGTTDA----------ASLGT-----
VIT_04s0023g010 SQCFPTTHDSDL--HHRLPV-DDTCRRGSLVSSDFGTTSTNP-DIG-STLIRFGT-----
                                                                            

AT4G36870.2     ----NPTGDVSLTLGLRHAG----NMP-DKDASFCVREFGGF
Bra011713.1-P   ----NPTGDVSLTLGLRHAG----NIP-DKNASFCVRDFGGF
Bra010577.1-P   ----NPTGDVSLTLGLRHVS----NMP-DQ--RFFLR-----
AET04003        ----NTSGDVSLTLGLRHAG----NLP-DK-TSFSLTDFGGI
AES94832        -----TTGDVSLTLGLRHAG----NVH-DKNNNFLY------
KRH76894        -----TPGDVSLTLGLRHAG----NMP-EK-SPFSVRDFGAI
KRH08328        -----TAGDVSLTLGLRHAG----NMP-EK-TPFSVRDFGGI
KRH28575        -----TPGDVSLTLGLRHAG----NMPSEK-TPFSVREFGAI
KRH69971        -------------------------IP-KA-TLHSHR-----
Zm00001d033898_ -AAGAAAGDVSLTLGL<span class="highlight">Q</span><span class="highlight">H</span>AGAGNAAGP-DSTGRFSLRDYSGC
BGIOSGA013510-P --AAAAAGDVSLTLGL<span class="highlight">Q</span><span class="highlight">H</span>AGAGN-AGP-DGSGRFSLRDYSGC
Os03t0732100-01 GAAAAAAGDVSLTLGL<span class="highlight">Q</span><span class="highlight">H</span>AGAGN-AGP-DGSGRFSLRDYSGC
VIT_18s0001g118 -----PAGDVSLTLGLRHAG----NLP-DK-GRLSLRDFGAC
VIT_04s0023g010 -----TAGDVSLTLGLRHAG----NLP-DK-SPFSVRDFGGC
                                            .             


<a href='HQ_QH-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            