
<!doctype html>
<html>
    <head>
        <title>PQ-QP</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT1G62970.1</h1>
        <h3>AT1G62970.1 pep chromosome:TAIR10:1:23323272:23326188:1 gene:AT1G62970 transcript:AT1G62970.1 gene_biotype:protein_coding transcript_biotype:protein_coding description:Chaperone DnaJ-domain superfamily protein [Source:UniProtKB/TrEMBL;Acc:Q0WQM3]</h3>
        <pre>
length  797
count   23
ratio   0.00
zscore  10.85






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT1G62970.1     MMSHGGGGGVSNGGGGDAERMLAQAEKLLLSGDLNGSKTYAIRACEADHSL---------
Bra009131.1-P   M----------DGNKDDALKCLKIGKDAMGAGDRSRAVKFLEKARRLDPSLPIDDLLS--
PNW79463        M---------------EAQAVLEAIEAARASGKGEGP-----------------------
KRH54891        M----------DGNKDDALKCLRIGKEAMESGDRSRALKFVTKARRLDPTLPVDDLLSTI
KRH63002        M----------DGNKDDALKCLSIGKEALESGDRSRALKFVTKARRLDPTLPVDDLLSTI
                *               :*   *   :    :*. . .                       

AT1G62970.1     -VDHAELILA------------------------------------VADTLVAGESRIRG
Bra009131.1-P   --DLNNQSPSPGSTAP------KPSLRQRGSSSSS------ASSSSSSASFTEEQRAIVR
PNW79463        -------STSAGAQQHQANGHHRPSASKPAAHGGPHLPHRPAKPKPVEDPGTPEQKALVA
KRH54891        EVDAGDQAPAAEAEGP-TKPTD<span class="highlight">Q</span><span class="highlight">P</span>SIRRRATGAAATVSAGPSSASSSSASYTEEQVSIIR
KRH63002        DADAGD<span class="highlight">Q</span><span class="highlight">P</span>AAAEAAES-TKSPD<span class="highlight">Q</span><span class="highlight">P</span>SIRRRAAGAAAAAPKGPSSASSSSASYTEEQVSIIR
                         :                                       . .  :  :  

AT1G62970.1     TTSDLPDWYAVLRLVRLTHNPELVATQYSRLAVLLNPSRNRYPYSEQAFRLISDAWYVLS
Bra009131.1-P   EIKSKKDYYEILGLAK-TSSVEDMRKAYRKLSLKVHPDKNKAPGSEEAFKSVSKAFQCLS
PNW79463        EVLKAKDFYEVLGITK-DATDDDIKKAYRKLALKLHPDKNKALHSDEAFKAVSKAFNCLS
KRH54891        EIKRKKNFYEILGLEK-TCTVEDVRKSYRKLSLKVHPDKNKAPGAEEAFKAVSKAFQCLS
KRH63002        EIKRKKNFYEILGLEK-TCTIEDVRKSYRKLSLKVHPDKNKAHGAEEAFKAVSKAFQCLS
                      ::* :* : :   . : : . * :*:: ::*.:*:   :::**: :*.*:  **

AT1G62970.1     DPSRKTLYDRELHLSQ-----------------------------------------F--
Bra009131.1-P   NDDARRKYDVSGSDEPSYQTR-----RRQHHS-------FYD---DEFDADEIFRS-FFG
PNW79463        DGDKRAYYDRTGHESSAAAAAAAAAQRAGGAGPGPGGAYYYTTSTEDFDPEELFNM-FFG
KRH54891        NEESKRKYDVSGEDEAVYEQRAARPAARGYNG-------YYE---ADIDAEEIFRN-FFG
KRH63002        NEESKRKYDVSGEDEAIYEQRTARPAARGYNG-------YYE---ADIDAEEIFRNFFFG
                : . :  **     .                                          *  

AT1G62970.1     GQLGQLGFQLFNQT<span class="highlight">P</span><span class="highlight">Q</span>SQSQS<span class="highlight">P</span><span class="highlight">Q</span>H--HQQQSQS<span class="highlight">P</span><span class="highlight">Q</span>H--PNQNQ-----TVSFQNQQLP--
Bra009131.1-P   GGGGATQFRSFNFG----------GGNRAATNQASDTTGFNPR-----------------
PNW79463        GGLGAHAFRAQFGVPRQRRHGHAHGGHHHGGGHAQG--GGQAQGGGGGTAAADQQQRAAM
KRH54891        GMAPAANFGGFSFGP---AGFNGFNGHRHAADHGSG--GFNVR-----------------
KRH63002        GMAPAANFGGFSFGP---AGFNGFNGHRHAAEHGSG--GFNVR-----------------
                *      *                  ::      .     : :                 

AT1G62970.1     FQLHQ--P-------HFAQAVQTQSQSQQQRSQFDQQLLQEQQQASWRQ----QVGQEHS
Bra009131.1-P   -VLLQLLPVVFILLLNFMPS<span class="highlight">P</span><span class="highlight">Q</span>-PIYSLSASYQYEHKFTTHR-GVNYFVRSA-KFEQEYP
PNW79463        LGLLQLMPILLILVFTFFQSSQSPPYSLVQESAYKVELLTQRLSIPFYVKSVPELERAYP
KRH54891        -ALIQLLPVLLIVLLNFLPSSE-PLYSLSKSYPYEHRFTTPK-GVNYYVKST-NFVQEYP
KRH63002        -TLIQLLPVLLILLLNFLPSSE-PLYSLSKSYPYEHRFTTPK-GVNYYVKST-NFVQEYP
                  * *  *        *  : :    *      :. .:   :    :      :. : :.

AT1G62970.1     SGSSGDWK--RPVEEVRLINL--NNATERVQSSRRFAEPERPPSRSFEGSSHRTPSTELT
Bra009131.1-P   INSLERQRVEAQVERDYLTILGQNCRYE--LQRQQWGY----------------------
PNW79463        AGSSARYRLERQVESAYYERLEARCQQERLMRHRAWSWGNREQARAM-------------
KRH54891        LGSDERSKIEERVEREYFSILRQNCHFE--LQRRQWGY----------------------
KRH63002        LRSEERSKIEERVEREYFSILRQNCHFE--LQRRQWGY----------------------
                  *    :    **      *  .   *     : :.                       

AT1G62970.1     WASKPTPVSEPVRHSELVPWQYSEPARQYQLSSRSSEAAQLSLLPSVSDSSHAS<span class="highlight">Q</span><span class="highlight">P</span>TRSN
Bra009131.1-P   -----------VRET---------------------------------------------
PNW79463        -----------KLEA---------------------------------------------
KRH54891        -----------IRET---------------------------------------------
KRH63002        -----------IRET---------------------------------------------
                             .:                                             

AT1G62970.1     QSHAVSK<span class="highlight">P</span><span class="highlight">Q</span><span class="highlight">P</span>VSKPHPPFPMS<span class="highlight">Q</span><span class="highlight">P</span>PPTSNPFPLS<span class="highlight">Q</span><span class="highlight">P</span>PSNSKPFPMSQSSQNSKPFPVSQSS
Bra009131.1-P   -------------PH---------------------------------------------
PNW79463        ------------------------------------------------------------
KRH54891        -------------PH---------------------------------------------
KRH63002        -------------PH---------------------------------------------
                                                                            

AT1G62970.1     QKSKPLLVSQSSQRSKPLPVSQSLQNSNPFPVS<span class="highlight">Q</span><span class="highlight">P</span>SSNSKPFPVS<span class="highlight">Q</span><span class="highlight">P</span><span class="highlight">Q</span><span class="highlight">P</span>ASNPFPVS<span class="highlight">Q</span><span class="highlight">P</span>R
Bra009131.1-P   ------------------------------------------------------------
PNW79463        ------------------------------------------------------------
KRH54891        ------------------------------------------------------------
KRH63002        ------------------------------------------------------------
                                                                            

AT1G62970.1     PNS<span class="highlight">Q</span><span class="highlight">P</span>FSMS<span class="highlight">Q</span><span class="highlight">P</span>SSTARPFPAS<span class="highlight">Q</span><span class="highlight">P</span>PAASKSFPIS<span class="highlight">Q</span><span class="highlight">P</span>PTTSKPFVS<span class="highlight">Q</span><span class="highlight">P</span>PNTSKPMPVS<span class="highlight">Q</span><span class="highlight">P</span>PT
Bra009131.1-P   ------------------------------------------------------------
PNW79463        ------------------------------------------------------------
KRH54891        ------------------------------------------------------------
KRH63002        ------------------------------------------------------------
                                                                            

AT1G62970.1     TSKPLPVS<span class="highlight">Q</span><span class="highlight">P</span>PPTFQSTCPS<span class="highlight">Q</span><span class="highlight">P</span>PAASSSLSPLPPVFNSTQSFQSPPVSTTPSAVPEASTI
Bra009131.1-P   -----------------C------------------------------------------
PNW79463        -----------------C------------------------------------------
KRH54891        -----------------C------------------------------------------
KRH63002        -----------------C------------------------------------------
                                 *                                          

AT1G62970.1     PSPPAPAPVA<span class="highlight">Q</span><span class="highlight">P</span>THVFNQTPPPEQTPKSGAARTDSEPEVSSFWTTCPYCYVLYEYPIIYE
Bra009131.1-P   ---------------------------------DM-------------------------
PNW79463        ------------------------------------------------------------
KRH54891        ---------------------------------DM-------------------------
KRH63002        ---------------------------------DM-------------------------
                                                                            

AT1G62970.1     ESVLKCQTKSCRRAYQAVKVPSPPPVAEEDSYYCCWGFYPIGFSEVVTKIPVTGLHTKIP
Bra009131.1-P   -----------MRRFDSAAAA---------------------------------------
PNW79463        EEIERVNEKLGMRR-------------------------PVGY-----------------
KRH54891        -----------LRKFEVVN-----------------------------------------
KRH63002        -----------LRKFEVVN-----------------------------------------
                            *                                               

AT1G62970.1     VSDLPKSAPKKPSPKVYYYDDEEEDDDSYIGSDPSEDDDDDDWLMNGTKRRKNVKQSKAK
Bra009131.1-P   ------------------------------------------------------------
PNW79463        ------------------------------------------------------------
KRH54891        ------------------------------------------------------------
KRH63002        ------------------------------------------------------------
                                                                            

AT1G62970.1     ATTKPKRQKVIEVD
Bra009131.1-P   --------------
PNW79463        --------------
KRH54891        --------------
KRH63002        --------------
                              


<a href='PQ_QP-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            