
<!doctype html>
<html>
    <head>
        <title>QQ-QQ</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT4G25520.2</h1>
        <h3>AT4G25520.2 pep chromosome:TAIR10:4:13032157:13036471:-1 gene:AT4G25520 transcript:AT4G25520.2 gene_biotype:protein_coding transcript_biotype:protein_coding gene_symbol:SLK1 description:Probable transcriptional regulator SLK1 [Source:UniProtKB/Swiss-Prot;Acc:Q0WVM7]</h3>
        <pre>
length  748
count   21
ratio   0.00
zscore  13.04






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT4G25520.2     ------------------------------------------------------------
Bra019159.1-P   ------------------------------------------------------------
Bra019158.1-P   MLILNGRGRVQWDEANIVEIESNKPVRQKITEPKTPYHPPYPMIDDDGGSLCSRVRSFDN
KRH09132        ---------------------------------MTP----------------LRV-----
KRH38604        ---------------------------------MTP----------------SRV-----
KRH09131        M------------------------------PPMTP----------------SRV-----
KRH38603        ---------------------------------MTP----------------SRV-----
KRG88471        ------------------------------------------------------------
VIT_16s0100g005 ---------------------------------MAP----------------SRV-----
                                                                            

AT4G25520.2     ------------------------------------------------------------
Bra019159.1-P   ------------------------------------------------------------
Bra019158.1-P   FVEMHRAEELKNDLNDLASSSKIISHISDSGDWTPSEDEAEPIDQNEEDPEGEKNTSFNE
KRH09132        -----------------------------AGG----------------------------
KRH38604        -----------------------------AGG----------------------------
KRH09131        -----------------------------AGG----------------------------
KRH38603        -----------------------------SGS----------------------------
KRG88471        ------------------------------------------------------------
VIT_16s0100g005 -----------------------------AGS----------------------------
                                                                            

AT4G25520.2     ------------------------------------------------------------
Bra019159.1-P   ------------------------------------------------------------
Bra019158.1-P   HRRVHYDEFRMVKEMRSSGGSFYGEDAEGDDSAKTGKPEATISQNTPGFIDQDIPLALTL
KRH09132        -----------LTQSSSNSGIFYQGDGQSQNVVNSHLSSSFVNSSS------------TV
KRH38604        -----------LTQSSSNSGIFYQGDGQSQNVVDSHLSSSFVNSSS------------TV
KRH09131        -----------LAQSSSHSGIFFQGDGQSQNVVNSDLSSSFVNSSS------------TV
KRH38603        -----------LTRSSSHSGIFFQGDGQSQNIVNSHLSSSFVNSSS------------TV
KRG88471        --------------------------------VNSHLSSSFVNSSS------------TV
VIT_16s0100g005 -----------LAQSSSSSGIFFQGDGQSQAVVNSHMSSSFGNSSN------------SI
                                                                            

AT4G25520.2     --------------MNRTVVSG------AVESSFSLTDAVG--TEALNMQRSSGINNN--
Bra019159.1-P   --------------MNKATVSG------AIESSFSLTD-------ALTMQRNGGISNP--
Bra019158.1-P   LLLSKSNI--VFWFTGNQTEVAALNTIPVRTWSFLVLGTGIGASASPSRSRSRSLRDRKF
KRH09132        SGASRSNLGPVSGDMNNAVLNSVANSAPSVGASSLVTDANSALSGGPHLQRSASVNTDSY
KRH38604        PGAGRSNLGPVSGDINNAVLNTVANSAPSVGASSLVTDANSSLSGGPHLQRSTSVNTDSY
KRH09131        PGAGRSNLGPVSGGMNNAVLNSVPNSAPSVGASSLVTDANSALSGGPHLQRSASVNTDSY
KRH38603        PGAGCSNLGPVSGDINNAVLNTVTNSAPSVGASSLVTDANSALSSGPHLQRSASVNTDSY
KRG88471        PGAGCSNLGPVSGDINNAVLNTVANSAPSVGASSLVTDANSALSGGPHLQRSA-------
VIT_16s0100g005 PGTGRSNLGPVSGDVNNTVLNSVANSGPSVGASSLVTDANSALSGGPHLQRSASINTESY
                               .. .             *  : .       .   .*.        

AT4G25520.2     MRIPTSPMSFSSNSVNIPGSLVLDGSAASMQHLP<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>LL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>TGQ------------
Bra019159.1-P   -RVPTSPMSF-SNNINVPGSLVLDGSAS--------------M<span class="highlight">Q</span><span class="highlight">Q</span>GGQ------------
Bra019158.1-P   VG--------------------FDYSSI--------------LKVCSF------------
KRH09132        LRLPASPMSFTSNNISISGSSVMDVSSVV<span class="highlight">Q</span><span class="highlight">Q</span>SSHQDQNV<span class="highlight">Q</span><span class="highlight">Q</span>L<span class="highlight">Q</span><span class="highlight">Q</span>N<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>PQGASSAMSLSA
KRH38604        LRLPASPMSFTSNNISISGSSVMDGSSVV<span class="highlight">Q</span><span class="highlight">Q</span>SSHQDQNV<span class="highlight">Q</span><span class="highlight">Q</span>L<span class="highlight">Q</span><span class="highlight">Q</span>NK<span class="highlight">Q</span><span class="highlight">Q</span>PQGASSATSLPA
KRH09131        LRLPASPMSFTSNNISISGSSVMDGSSVV<span class="highlight">Q</span><span class="highlight">Q</span>SSHQDQNV<span class="highlight">Q</span><span class="highlight">Q</span>L<span class="highlight">Q</span><span class="highlight">Q</span>N<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>PQGASSATSLPA
KRH38603        LRLPASPMSFTLNNISISGSSVMDGSSV---------------QN<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>PQGASSATSLPA
KRG88471        ---------------------MMDGSSVV<span class="highlight">Q</span><span class="highlight">Q</span>SSHQDQNV<span class="highlight">Q</span><span class="highlight">Q</span>L<span class="highlight">Q</span><span class="highlight">Q</span>N<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>LQGASSATSLPA
VIT_16s0100g005 MRLPASPMSFSSNNISISGSSVMDGSSVV<span class="highlight">Q</span><span class="highlight">Q</span>SSHQDPSS<span class="highlight">Q</span><span class="highlight">Q</span>ANQS<span class="highlight">Q</span><span class="highlight">Q</span>H-QGASSATSLPT
                                      :* *:                :                

AT4G25520.2     ---------------GSVPMRENNYSHVD--KKPRL-EVKQEDML---------------
Bra019159.1-P   ---------------GSVPVRENSYLHVD--KKPKL-EVKQEDML---------------
Bra019158.1-P   ---------------SSVPMGENNYYHVDKKKKPKLEEVEQDDMLHYNQYQDPPTGGQDL
KRH09132        SQTGPSMLQMGAQIPGSFIQDPNNMSHLS--KKPRM-DIKQEDMM---------------
KRH38604        SQTGPSTLQMGAQVPGSFIQDPNNMSHLS--KKNRL-DTKQEDMT---------------
KRH09131        SQTGLSPLQMGAQVPGSFIQDPNNMSHLS--KKPRM-DIKQEDVM---------------
KRH38603        SQTRPSLLQMGAQVPRSFIQDPNNMSHLS--KKPRM-DIKQEDIM---------------
KRG88471        SQTGPSLLQMGAQVPRSFILDPNNMSHLS--KKPRM-DIKQEDIM---------------
VIT_16s0100g005 SQAGQVSLSMNPRVPASFIQEPNNPSQVH--KKARL-DIKQEDIL---------------
                                *.    *.  ::   ** :: : :*:*:                

AT4G25520.2     --------<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>IL----------<span class="highlight">Q</span><span class="highlight">Q</span>LIQRQDPT---GRNPQMQALL----<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>RLR-QHQ
Bra019159.1-P   --------<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>VL----------<span class="highlight">Q</span><span class="highlight">Q</span>YIQRQDPT---GRNPQLQALL----H<span class="highlight">Q</span><span class="highlight">Q</span>RMR<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>
Bra019158.1-P   QLQAWRR<span class="highlight">Q</span><span class="highlight">Q</span>NQFLQSQRPHHLP<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>YIQHQGPT---VRNPQLQALL----HRYRLR-QHQ
KRH09132        --------<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>VI----------<span class="highlight">Q</span><span class="highlight">Q</span>ILQRQDSMQFQGRNPQLQAFL---<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>RL-R<span class="highlight">Q</span><span class="highlight">Q</span>
KRH38604        --------<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>VI----------<span class="highlight">Q</span><span class="highlight">Q</span>LLQRQDSMQFQGRNPQLQAFL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>RL-R<span class="highlight">Q</span><span class="highlight">Q</span>
KRH09131        --------<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>VI----------<span class="highlight">Q</span><span class="highlight">Q</span>ILQRQDSMQFQGRNPQLQALL----<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>RL-R<span class="highlight">Q</span><span class="highlight">Q</span>
KRH38603        --------<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>VI----------<span class="highlight">Q</span><span class="highlight">Q</span>ILQRQDSMQFQGRNPQLQAFL----<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>RL-R<span class="highlight">Q</span><span class="highlight">Q</span>
KRG88471        --------<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>VI----------<span class="highlight">Q</span><span class="highlight">Q</span>ILQRQDPMQFQGRNPQLQAFL----<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>RL-R<span class="highlight">Q</span><span class="highlight">Q</span>
VIT_16s0100g005 --------P<span class="highlight">Q</span><span class="highlight">Q</span>IV----------<span class="highlight">Q</span><span class="highlight">Q</span>ILQRQDPMQLQGHNPQFQSLI----<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>RLR-<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>
                         :*.:          ** :*:*..     :***:*:::    :: :   ::*

AT4G25520.2     QMLQSMS-PSQRLQL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>-----------------------------LR<span class="highlight">Q</span><span class="highlight">Q</span>L<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>GTQ
Bra019159.1-P   QILQSMS-PSQRLQL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>-----------------------------LLR--<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>GAQ
Bra019158.1-P   QTLQ--------LQL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>-----------------------------LRRLLLQ---Q
KRH09132        QMF<span class="highlight">Q</span><span class="highlight">Q</span>MP-QLHRAHL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>MQLR<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>VMQ
KRH38604        QMF<span class="highlight">Q</span><span class="highlight">Q</span>MP-QLHRAHL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>-------------------<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>MQLR<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>VMQ
KRH09131        QIFQSMP-QLQRAHL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>----------------<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>H<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>MQLR<span class="highlight">Q</span><span class="highlight">Q</span>L<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>VMQ
KRH38603        QIFQSMP-QLQRAHL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>--------------------------------R<span class="highlight">Q</span><span class="highlight">Q</span>L<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>VMQ
KRG88471        QIFQSMP-QLQRAHL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>----------------------------------------
VIT_16s0100g005 QMLQSMP<span class="highlight">Q</span><span class="highlight">Q</span>MQRAHL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>HQ---------------------<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>LQLRHHL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>GMQ
                * :*         :****                                          

AT4G25520.2     QIPPNVRPYE---VGVCA-RKLMMYLYHL<span class="highlight">Q</span><span class="highlight">Q</span>RPAENCITYWRKFVAEYFSPRAKQRLCLS
Bra019159.1-P   QIPPSVRPFE---PGVCA-RKLMMYLYHL<span class="highlight">Q</span><span class="highlight">Q</span>RPAENCITYWRKFVAEYFSPRAKQRLCLS
Bra019158.1-P   QIPPNVCPFG---GGMCAHRKFMMFLHHIKQRPEDNCITFWRAFVAEYFSPRAKQRLCFS
KRH09132        PSSVVKRPYDSSVSGVCA-RRLMQYLYHQRQRPNDNSIAYWRKFVAEYYSPRAKKRWCLS
KRH38604        PSSAVKRPYESSVSGVCA-RRLMQYLYHQRQRPNDNSIAYWRKFVAEYYSLRAKKRWCLS
KRH09131        PSSAGKRPYDSGVSGVCA-RRLMQYLYHQRQRPNDNSIAYWRKFVAEYYSPRAKKRWCLS
KRH38603        PSSAGKRPYDSGVSGVCA-RRLMQYLYHQRQRPNDNSIAYWIKFVAEYYSPRAKKRWCLS
KRG88471        ------------------------------QRPNDNSIAYWRKFVAEYYSPRAKKRWCLS
VIT_16s0100g005 PISAMKRPYD---SGVCA-RRLMQYLYHQRQ--PDKTIAYWRKFVAEYYSPRAKKRWCLS
                                              *   :: *::*  *****:* ***:* *:*

AT4G25520.2     QYESAGHHALGMFPQAAPDMWQCDLCGTKSGKGFEATFDVLARLIEIKFASGIIDELLYL
Bra019159.1-P   QYESAGHHALGMVSQAAPDMWQCDLCGTKSGKGFEATFDVLARLNEIKFASGIIDEFLYL
Bra019158.1-P   QYKGA-DHMLGTLPQG---MWQCNHCGTKSGKGVEATFDVLPRLFEIKFASGFVNELLSL
KRH09132        LYSNVGHHALGVFPQASMDAWHCDICGSKSGRGFEATYEVLPRLNEIKFGSGVIDELLFL
KRH38604        LYSNVGHHALGVFPQASMDAWHCDICGSKSGRGFEATYEVLPRLNEIKFGSGVIDELLFL
KRH09131        LYSNVGHHALGVFPQAAMDAWQCDMCGSKSGRGFEATYEVLPRLNEIKFGSGVIDELLFL
KRH38603        LYSNVGHHALGVFPQAAMDAWQCDICGSKSGRGFEATYEVLPRLNEIKFGSGVIDELLFL
KRG88471        LYSNVGHHALGVFPQAAMDAWQCDICGSKSGRGFEATYEVLPRLNEIKLGSGVIDELLFL
VIT_16s0100g005 LYDNVGNHALGVFPQAAMDAWHCEICNSKSGRGFEATFEVLPRLNEIKFGSGVIDELLFL
                 *... .* ** ..*.    *:*: *.:***:*.***::**.** ***:.**.::*:* *

AT4G25520.2     DHPRENRFPNGLMMLEYRKAVQETVHEQFRVVREGHLRIIFSQDLKILSWEFCARRHEEL
Bra019159.1-P   DLPRENRFSNGLMMLEYRKAVQETVHEQFRVIREGHLRIIFSQDLKILSWEFCARRHEEL
Bra019158.1-P   EDPRECRVSSGLIVLKYRKLVQTIEYEQCRVVHEGPFLIIFSQDLKILRWEFCVQRHEEF
KRH09132        DMPREMRFASGAMMLEYGKAVQESVYEQLRVVREGQLRIIFTQDLKILSWEFCARCHEEL
KRH38604        DMPREMRFASGAMMLEYGKAVQESVYEQLRVVREGQLRIIFTQDLKILSWEFCARRHEEL
KRH09131        DLPRETRFPSGVMMLEYAKAIQESVYEQLRVVREGQLRIIFTQDLKILSWEFCARRHEEL
KRH38603        DLPRETRFPSGAMMLEYAKAVQESVYESLRVVREGQLRIIFTQDLKILSWEFCARRHEEL
KRG88471        DLPRETRFPSGAMMLEYAKAVQESVYEPLRVVREGQLRIIFTQDLK-----FCARRHEEL
VIT_16s0100g005 DLPRECRFSSGIMMLEYGKAVQESVYEQLRVVREGQLRIIFTPDLKILSWEFCAQHHEEL
                : *** *...* ::*:* * :*   :*  **::** : ***: ***     **.: ***:

AT4G25520.2     LLRRLIAPQVNQLLQVAQKCQSTISESGSEGVS<span class="highlight">Q</span><span class="highlight">Q</span>DLQSNSNMVLGAGRQLAKFMELQSL
Bra019159.1-P   LLRRLIAPQVNQLLQVALKCQSTISESGSDGVSKQDLQSNSNMVLGAGRQLAKFMELQSL
Bra019158.1-P   LPRRRIAPKVNQLLQV---------------VA<span class="highlight">Q</span><span class="highlight">Q</span>DLQTNSNMVLAARRQLANSMELQPL
KRH09132        LPRRLVAPQVNQLVQVAKKCQSTIAESGSDGVS<span class="highlight">Q</span><span class="highlight">Q</span>DIQTNSNMLLTAGGQLAKILEMQSL
KRH38604        LPRRLVAPQVNQLVQVAKKCQSTIAESGSDGAS<span class="highlight">Q</span><span class="highlight">Q</span>DIQTNSNMLLTAGGQLAKILEMQSL
KRH09131        LPRRLVAPQVNQLVQVAQKCQSTIAESGADGVS<span class="highlight">Q</span><span class="highlight">Q</span>DLQTNSNMVLTAGRQLAKILELQSL
KRH38603        LPRRLVAPQVNQLIQVAQKCQSTIAESGADWVS<span class="highlight">Q</span><span class="highlight">Q</span>DLQTNSNIPPNLS---------QKN
KRG88471        LPRRLVAPQVNQLIQVAQKCQSTIAESGADWVS<span class="highlight">Q</span><span class="highlight">Q</span>DLQTNSNMVLTAGRQLAKILELQSL
VIT_16s0100g005 LPRRLVAPQVNQLVQVAQKCQSTIAESGSDGIS<span class="highlight">Q</span><span class="highlight">Q</span>DLQTNSNMVLTAGRQLARSLESQSL
                * ** :**:****:**                ::**:*:***:              *  

AT4G25520.2     NDLGYPKRYIRTLQISEVVKSMKDLMNFTGEQKIGPIEGLKRLLEQTVTVKLQKQKMQEM
Bra019159.1-P   NDLGYPKRYIRTLQISEVVKSMKDLMNFTGDHKIGPIEGLKRLFEQTATVKLQRQK--EM
Bra019158.1-P   NDLGYPKKYFRALQTYEVVKSMKALMDFTENHKIGPIEGWKRLSEQKERMRLQRQKMQEM
KRH09132        NELGFSKRYVRCLQISEVVNSMKDLIDICADHKIGAIESLKNFPRLATASKVQMQKMQEM
KRH38604        NELGFSKRYVRCLQISEVVNSMKDLIDICAEHKIGAIESLKNYPRLATASKHQMQKMQEM
KRH09131        NDLGFSKRYVRCLQISEVVNSMKDLIDICSEHKIGAIESLKNYPRLATASKGQMQKMQEM
KRH38603        DDLEPVHRWIVMKFDYHVRNPIPNILSVGN---FKIISELREKP----------------
KRG88471        NDLGFSKRYVRCLQV-------------CC------------------------------
VIT_16s0100g005 NDLGFSKRYVRCLQISEVVNSMKDLIDFCRENKVGPIDGLKSYPRHASAVKLEMQKMQEM
                ::*   :::.                                                  

AT4G25520.2     EQFGNNGAI-NGPVQAQMVLTSGTMNGSTGNNTNNHHQIVGRGAMSGPAEGQMVISSGTV
Bra019159.1-P   EQLGNSGAIMNASSQAQMALTQGTMNGLTGNN----------------------------
Bra019158.1-P   EHLWNSGAM-NRSAQAQ-------MDGLTGYN----------------------------
KRH09132        EQLANVQGL------------------PTDRNTLNKLMALNPG-----------------
KRH38604        EQLGNVQCL------------------PTDQNTLNKLMALNPG-----------------
KRH09131        EQLANVQGL------------------PTDRNTLNKLMTLNPG-----------------
KRH38603        ------------------------------------------------------------
KRG88471        ------------------------------------------------------------
VIT_16s0100g005 EQLANVQGL------------------PTDRNTLNKLIALHPG-----------------
                                                                            

AT4G25520.2     SGATANNNSNNHNQIVGRGAMNGSAQAAAALTNYQSMLMRQNAMN-NPNSNTGKQEGFSS
Bra019159.1-P   ---------NNHPQIVGRGAMNGSAQAAAALTNYQSMLMRQNAMN-NPNS---KQEGFSS
Bra019158.1-P   ---------NHH---------HSSAQAAAALTNNQSMLMRLNAVN-NQYSDTSIQEGFSS
KRH09132        ----LNNHINNPHNMVNRGALSGSAQAALALNNYQNLLMRQNSMNSSPGS--LQREG-SS
KRH38604        ----LNNHINNSHNMVNRGALSGSAQAALALNNYQNLLMRQNSTNSSPGS--LQREG-SS
KRH09131        ----LNNHMNNTNNMVGRGALSGSAQAALALNNYQNLLMRQNSMNSSPGS--LQREG-SS
KRH38603        ----------------------------FAL-----------------------------
KRG88471        ------------------------------------------------------------
VIT_16s0100g005 ----LNSHMSNNPHMVNRGALSGSAQAALALTNYQNLLMRQNSMNSNPSS--L<span class="highlight">Q</span><span class="highlight">Q</span>EGPSS
                                                                            

AT4G25520.2     QN-PTPNSNQSPSSS---------S<span class="highlight">Q</span><span class="highlight">Q</span>RHNLVTGGFPN---SPQM<span class="highlight">Q</span><span class="highlight">Q</span>---------<span class="highlight">Q</span><span class="highlight">Q</span>RT
Bra019159.1-P   Q-------SQSPSSS---------SHQRQNLATGGFPS---SPQM-----------<span class="highlight">Q</span><span class="highlight">Q</span>RT
Bra019158.1-P   SQHPTPNSNQ--------------GHQRQNLATGGFLS---SPQM<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>--------<span class="highlight">Q</span><span class="highlight">Q</span>RT
KRH09132        FN----NSNQSPSSALQGAGPALIPGPMQNSSVSGFPSPRLPP<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>HH------L<span class="highlight">Q</span><span class="highlight">Q</span>PS
KRH38604        FN----NSNQSPSSALQGASPALISGSMQNSSVSGFPSPHLPP<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>HH------L<span class="highlight">Q</span><span class="highlight">Q</span>RS
KRH09131        FN----NSNPSPSSALQGTGPALIPGSMQNSPVGGFPSPHLTP<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>-----LL<span class="highlight">Q</span><span class="highlight">Q</span>RT
KRH38603        ------------------------------------------------------------
KRG88471        ------------------------------------------------------------
VIT_16s0100g005 FN----SSNQSPSSTFQGPA-TLISGSMHNLPGSGFSSPHLPP<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>RS
                                                                            

AT4G25520.2     MNGPTNILPQNHPHQ-LQSPHSHGNTPE<span class="highlight">Q</span><span class="highlight">Q</span>MLHQLLQEMSENGGSV<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>AFSGQSGSNS
Bra019159.1-P   MNGAPNTL<span class="highlight">Q</span><span class="highlight">Q</span>NPPHRLLQPPHSHGTNQE<span class="highlight">Q</span><span class="highlight">Q</span>MLHQML<span class="highlight">Q</span><span class="highlight">Q</span>MSENGANV<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>AFSGQSGGNS
Bra019158.1-P   LN----------------------------------------------------------
KRH09132        LS-ANALL<span class="highlight">Q</span><span class="highlight">Q</span>NHSQG-SQGNQAL----<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>MIHQLLQEMSNNNG-GVQPQSLGGPSAN--
KRH38604        LS-SNALL<span class="highlight">Q</span><span class="highlight">Q</span>NHSHG-SQGNQAL---<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>MIHQLLQEMSNNNG-GMQPLSLGGPN----
KRH09131        LS-ANGLL<span class="highlight">Q</span><span class="highlight">Q</span>NHSQG-SQGNQAL---<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>MI<span class="highlight">Q</span><span class="highlight">Q</span>LLQEMSNNNG-GLQSQSLGGHNANGN
KRH38603        ------------------------------------------------------------
KRG88471        ------------------------------------------------------------
VIT_16s0100g005 LN-PSSLL<span class="highlight">Q</span><span class="highlight">Q</span>NPGLS-SQSSQAL----<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>MI<span class="highlight">Q</span><span class="highlight">Q</span>MLQEMTNNCGPGM<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>SLSGQNVNGS
                                                                            

AT4G25520.2     ---------NAERNTTASTSNISGGG--RAPSRNNSFKAASNN-----------------
Bra019159.1-P   ---------NAERNPTASTSNISGGGGVRVPSRNNSFKSASNS-----------------
Bra019158.1-P   -----------------------------------TF<span class="highlight">Q</span><span class="highlight">Q</span>T--------------------
KRH09132        MAKNALGFGGHYPSLSGGSANVTGNNG--PMSRNNSFKTTANSDSSAAGGNNGLNQRTSE
KRH38604        -AKNAMGFGGHTPSLSGGSANVPGNNG--PMSRINSFKTASNSDS---------------
KRH09131        ISKNTMGFGGHTPSLSGGSANVPGNNR--PISRNNSFKTASNSDSSAAGGNNGFNQRTSD
KRH38603        ------------------------------------------------------------
KRG88471        ------------------------------------------------------------
VIT_16s0100g005 MTRSGMGFGNNSAAATVASPNLSGSIGGPPLSKSNSFKGPLNSDSSAGGANSGFNQKASD
                                                                            

AT4G25520.2     ---NLHFSEDISITD--HDFSEDGFFNNNDIYGGL------
Bra019159.1-P   ---NLHLTEDIPITELPHDFSADDFFNNSDIYGSL------
Bra019158.1-P   -----HFPED--ATEIPVEFSDDN-FNDSDKQGSL------
KRH09132        MPQNLHLQDV--VQDIGNEFTDNPFLN-SDLDDNMGFGWKA
KRH38604        -----------------------------------------
KRH09131        M<span class="highlight">Q</span><span class="highlight">Q</span>NLHLQDV--AQDIGNEFLDNPFFN-SDLDDNMGFSWKA
KRH38603        -----------------------------------------
KRG88471        -----------------------------------------
VIT_16s0100g005 LAHNLHLSDEM-VQDIAREFPDNGFFN-SDLEDNMSYGWKA
                                                         


<a href='QQ_QQ-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            