
<!doctype html>
<html>
    <head>
        <title>QQ-QQ</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT1G24300.1</h1>
        <h3>AT1G24300.1 pep chromosome:TAIR10:1:8614363:8620664:-1 gene:AT1G24300 transcript:AT1G24300.1 gene_biotype:protein_coding transcript_biotype:protein_coding description:GYF domain-containing protein [Source:UniProtKB/TrEMBL;Acc:F4I983]</h3>
        <pre>
length  1495
count   18
ratio   0.00
zscore  11.11






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT1G24300.1     MAEGKFDLPDDLILSKSSDQL-------------------------KELASDNSIPLSPQ
Bra010950.1-P   MFESR-------------------------------------------------------
Bra010951.1-P   ------------------------------------------------------------
VIT_01s0146g001 MAESKLDLPDDLISTKPSDQFWTATVVASGGNDDEKALMGLADESKDQLASESSIPLSPQ
                                                                            

AT1G24300.1     WLYTKSSESKMDVRSPTPMPMGNPSDPNLKDAWRLDAPEDKKDWKKIVSENETNRRWREE
Bra010950.1-P   ------------------------------------------------------------
Bra010951.1-P   ------------------------------------------------------------
VIT_01s0146g001 WLYSKPNETKMETRAPNSAALGNSTDPNQKEGWRLDASEDKKDWRKIATDTESNRRWREE
                                                                            

AT1G24300.1     ERETGLLGARKVDRRKTERRIDNVSSRETGEVKTTAASDRWNDVNSRAAV--HEPRRDNK
Bra010950.1-P   -----------------------------------------EDVRSPAPVPIGSRRRDNK
Bra010951.1-P   ------------------------------------------------------------
VIT_01s0146g001 ERETGLLGGRR-NLRKVDRRVDTVSIRESIDSRALPTSERWHDGSNRNSV--HETRRDSK
                                                                            

AT1G24300.1     WSSRWGPDDKEKEARCEKVEINKDKEEPQSESQSVV-SNVRATSERDSDPRDKWRPRHRM
Bra010950.1-P   WSSRWGPDDKEKDAKV-------DKEEP--ETQSVLGSSLRAT---DSDARDKWRPRHRM
Bra010951.1-P   -----------------------------------------------------------M
VIT_01s0146g001 WSSRWGPEEREKESRTEK-RPDVDKEDAHSDNQSFV-GSNRPAPERDSDSRDKWRPRHRM
                                                                           *

AT1G24300.1     E-SQSGVPTSYRTAPGFGLDRGRAEGPNLGFTVGRGRASTIG-----RGSSTSLIGAG--
Bra010950.1-P   EQLQSGGPNSYRAAPGFGLDKGRSEGPNLGFAVGRGRA---------RGTWSTFFGAGGF
Bra010951.1-P   EQVQSGGPASYRAAPGFGLDKGRSEGPNIGFTVGRGRA---------RGTWSAFFGAGGF
VIT_01s0146g001 E-LHSGGPTSYRAAPGFGIERARLEGSHVGFAIGRGRSTALGSTPVLRSSSAGPIGGAQF
                *  :** * ***:*****:::.* **.::**::****:         *.: :  :*..  

AT1G24300.1     -------SASA---PVFRYPRGKLLDMYRKQKPDPSLGRIPTEMDEVASITQVALIEPLA
Bra010950.1-P   LRNERVPGKPA---PMCRYVRGKLLDLYRNQKPD----RMPIDMEDVDSVTQVDLIEPLA
Bra010951.1-P   LRNESVPGKPA---PTCRYVRGKLLDLYRNQKPD----RMPIDMEEVDSVTQVALIEPLA
VIT_01s0146g001 ERNGNVTGKLNLLDDTLCYPRGKLLDIYRRKKLDPSFATMPENMEETPHITLGDFIEPLA
                       .          * ******:**.:* *     :* :*::.  :*   :*****

AT1G24300.1     FIAPDTEEEASINGIWKGRIISSEVYTSSGEESLGENSLLKCRIPESGETKVDGALLGFM
Bra010950.1-P   FIAPDAEEEESLKGIWKGRITGSEAHISPGEESLAEKSL--------GETKVDGGLLGVV
Bra010951.1-P   FIAPGVEEEESLKGIWKGRITSSEAHTSPGEESLAEESM--------GETKVDGALLGVM
VIT_01s0146g001 FVAPDAEEEVILRDIWKGKITSSGVVYNSFRKGRTTENVTGIEDLESPKEK-QGILPSIT
                *:**..***  :..****:* .* .  .. .:.   :.:         : * :* * .. 

AT1G24300.1     NGD----------NGSMKNNDSGLLGSH------------NGGLGAASSVPRLNSVASES
Bra010950.1-P   SGD----------SVSMHNSNSGLLGSH------------GGGLWGASESD---------
Bra010951.1-P   SGD----------SVSMHNSNSGVLGSH------------NGGLWGASESD---------
VIT_01s0146g001 TKEIADTFPEGVNDGAYQDDDSGISFNYNMTKNMIDEMDANQGEGKYSVAGMDDMISTVS
                . :          . : ::.:**:  .:            . *    *            

AT1G24300.1     YGSFGAGYQVSHGSPEA------------VRSVFTKSPVLDGSESVVGSFE--------Q
Bra010950.1-P   --------QVSHGSPEA------------VRSGFTKSSVLNAGEPLV---E--------Q
Bra010951.1-P   --------QVSRGSPEA------------VRSAFDKSSVLDADEPIV-------------
VIT_01s0146g001 KGSSLCGVSEMSGANRTASQLKAVENEHLANSDFTKHDKLDNITSAA-SFDIGCGLPDIS
                        .   *: .:            ..* * *   *:   . .             

AT1G24300.1     DYMGKL<span class="highlight">Q</span><span class="highlight">Q</span>PDV-------EVDQSEAA------MPPEDFLFLYIDPQGVIQGPFIGSDIIS
Bra010950.1-P   DYTGKL<span class="highlight">Q</span><span class="highlight">Q</span>PEI-------EVNHSEGT------LPPEEFMFTYIDPQGVIQGPFIGSDIIS
Bra010951.1-P   --TGKL<span class="highlight">Q</span><span class="highlight">Q</span>PDI-------EVNHSEGT------LPPEEFMFSYIDPQGVIQGPFIGSDIIS
VIT_01s0146g001 NSIFALPSPKHSLSSNMQHLNSTGGTNLLGRGIPPEDFSLHYLDPQGEIQGPFLGVDIIS
                     * .*.        .:: : .:      :***:* : *:**** *****:* ****

AT1G24300.1     WFEQGFFGTDLQVRLANAPEGTPFQDLGRVMSYLKTESAHAHISNQESELEETRLKANSD
Bra010950.1-P   WFEQGFFGTDLQVRLATAPEGTPFQDLGSVMSYLKTESMQAHINDQIIELEQTSRKANSE
Bra010951.1-P   WFEQGFFGTDLQVRLATAPEGTPFQDLGSVMSYLKTESMQAHINDQIIELEQTSRKANSE
VIT_01s0146g001 WFKQGFFGIDLPVRLSDAPEGIPFQDLGEIMPHLKTKDG-ANSTDASSELEHAGILGANL
                **:***** ** ***: **** ****** :*.:***:.  *: .:   ***.:   . . 

AT1G24300.1     TGLSIAP----VAESNDSSSMNGTSRSFSVYNNPSAQDNFQRKSESEFYATPPHTEDRSF
Bra010950.1-P   IGLSFAP----VEDSNGSA---------------LGHDNVQMKSKSEAYVKPPHLDDRSF
Bra010951.1-P   IGLSFAP----VQESNGSA---------------LGHDNVQIQSKSEAYVKPPHVDDRSF
VIT_01s0146g001 EASSPAPGPVPVPDIADTTALNDHHWSLSEFDGLSSQNF<span class="highlight">Q</span><span class="highlight">Q</span>RKSEREGPLQLSYSDGQSF
                 . * **    * :  .::                .::  * :*: *     .: :.:**

AT1G24300.1     LDFSTQDEEIVFPGR--AGVSGYASVKSSTSMHDAFMEVSGQSAIPVESTKAATQKQHEN
Bra010950.1-P   LDYSAQDEEIVFPGR--AGVSGYGSAKSSTSMHDALMGVSGHSAVPVESAKVATQNQNEN
Bra010951.1-P   LDFSAQDEEIVFPGR--ARVSGYGSAKSSTSMHDALMGVSGHPAIPVESAKVATQNQNEN
VIT_01s0146g001 HDFSPQDEEIVFPGRPGSGGGGYPIGKPSRSTQDPLANPITYSSLPNELTEPVMANQNDN
                 *:*.**********  :  .**   *.* * :*.:      .::* * :: .  :*::*

AT1G24300.1     KLHPFGVLWSELESSNVPVNLLPNRSYDAMGE--PTGAIDNRPIDS-------RRNTQVD
Bra010950.1-P   KLHPFGVLWSELESSSTPVDLLPNRSYD--------------------------------
Bra010951.1-P   KLHPFGVLWSELESSSTPVDLLPNRSYD--------------------------------
VIT_01s0146g001 KLHQFGLLWSELEGAH-PTHAQPSNLSSSIGRLGPLGAMAGSTPDAEAFSDVYRRNILSN
                *** **:******.:  *..  *..  .                                

AT1G24300.1     PNMSLDGLASNRMSQFEHLSNRFNLGDQLSSNQHN<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>FQNRDMLSHSHIGDQAQDLDYL
Bra010950.1-P   -------------------------------TGH-DQKYL-----------DQARDLDHL
Bra010951.1-P   -----------------------------------DQKYL-----------DQAQDLDHL
VIT_01s0146g001 PNSYQDATATRHLSHIEQDSNRFDLAEQLM-----R<span class="highlight">Q</span><span class="highlight">Q</span>FQ--------------------
                                                    *::                     

AT1G24300.1     ITLQL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>KIQL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>KIQL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>-QKIQL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>KMQL<span class="highlight">Q</span><span class="highlight">Q</span>HQLEQEHQLHQK--LLQE<span class="highlight">Q</span><span class="highlight">Q</span>
Bra010950.1-P   --LTLKS<span class="highlight">Q</span><span class="highlight">Q</span>----Q-------<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>-QKIQL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>KIQL<span class="highlight">Q</span><span class="highlight">Q</span>RQLEQEYQL<span class="highlight">Q</span><span class="highlight">Q</span>K--LLRE<span class="highlight">Q</span><span class="highlight">Q</span>
Bra010951.1-P   -TLQL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>----<span class="highlight">Q</span><span class="highlight">Q</span>KIQLQH<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>-QKIQL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>KIQL<span class="highlight">Q</span><span class="highlight">Q</span>RQLEQEYQL<span class="highlight">Q</span><span class="highlight">Q</span>K--LLQE<span class="highlight">Q</span><span class="highlight">Q</span>
VIT_01s0146g001 -----<span class="highlight">Q</span><span class="highlight">Q</span>L<span class="highlight">Q</span><span class="highlight">Q</span>RQL<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>NLLSSHAHLNESLLEQVASRNHMHHQRLANQPFHQKQMLLQEQK
                     :. *    *       : : ::  *:*  . :  ::*   :  ::**  **:**:

AT1G24300.1     QSHARQLHY<span class="highlight">Q</span><span class="highlight">Q</span>------ILQGQTPDSRFGQSHDFPRSNNVDQMLLEHQLMNELQKSSGHP
Bra010950.1-P   QSHARNLHF<span class="highlight">Q</span><span class="highlight">Q</span>------ILQGQTPDSRLGQSHDFHRLNNVDQMILE<span class="highlight">Q</span><span class="highlight">Q</span>LMDELQ-SSGHR
Bra010951.1-P   QSHARNLHF<span class="highlight">Q</span><span class="highlight">Q</span>------ILQGQTPDSRLGQSHDFHRLNNVDQMLLE<span class="highlight">Q</span><span class="highlight">Q</span>LMDELQ-SSGHR
VIT_01s0146g001 QAQARQALLEQYNSVPTILRGM--------------------------------------
                *::**:   :*      **:*                                       

AT1G24300.1     SQNFAPYIEQLAAGNFGQLPHEGHQRELLEQLLSTKMQSQYGPMQSPYGQLQSEPTRSLE
Bra010950.1-P   SQNFTPYMEQLAAGNFGQLPHEGHQKELLEQLLSEQMQSQYRQMHSQHGHMQSEPIRSLE
Bra010951.1-P   SQNFTPYMEQLAAGNFGQLHHEGHQKELLEQLLSEQMQSQYRQMHSQHGHMQSEPIRSLE
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G24300.1     YQLL<span class="highlight">Q</span><span class="highlight">Q</span>EQLMQLANG--------------------------------------------V
Bra010950.1-P   YQLL<span class="highlight">Q</span><span class="highlight">Q</span>DQLMRLANGGRQNTLLEELRHIDPQHGQLQSEPIRSLEYQWM<span class="highlight">Q</span><span class="highlight">Q</span>EQLMQLANRA
Bra010951.1-P   YQLL<span class="highlight">Q</span><span class="highlight">Q</span>EQLMRLANGGRQNTLLEELRHIDPQHGQMQSEPIRSLEYQIM<span class="highlight">Q</span><span class="highlight">Q</span>EQLMQLADRA
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G24300.1     RHNTLLEEQRHIDPLWPSDHNDQLLRSHPGIQRSRSSTGFRQLDFH<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>RPPFEDQFGQ
Bra010950.1-P   RNNTLLEEHRHIDPLWPSNRNDQLLGAHPGIHRSHSSAGFRPVDFH<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>GPPFEDHFSH
Bra010951.1-P   RHNTLLEEHRHIDPLWPSNRNDQLLGAHPGIHRSHSSAGFRPVDFH<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>RPPFEDHFSH
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G24300.1     LERNLLY<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>LRQELFEQGLPFERSASLPVSVSGMNLDPVNGLGLSQGLELRDATTHM--
Bra010950.1-P   LERNLSY<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>LNQELFEQGLPFERS------TSGLNLDAVKGLNFS---ELRDAQMQSSG
Bra010951.1-P   LERNLSY<span class="highlight">Q</span><span class="highlight">Q</span><span class="highlight">Q</span>LNQELFEQGLPFERSTSLPRNTSGLNLDAVKGLNLS---ELRDAQMQSSG
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G24300.1     QIGNSTLGFNHQNPRIPIGEPHFSQLESMEGRWSGADTQVVGDWAESQLHRSNIDAEHHK
Bra010950.1-P   RLGNSTPGFSHQNPHIQLGEPHFSEMEPRKERWHGADTQLAGGWAETQFHRPNTEADHHK
Bra010951.1-P   RLGNSTPGFSHQNPHIQLGEPHFSEMEPRKERWHGADTQLAGGWPETQFHRSNTEADHHK
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G24300.1     MRSESRRMGEDSNSWMLGGTTEDRSKQLFMELLHQRPGHQSAESPSMNRGQSFDRMAPSG
Bra010950.1-P   MRSEMRRAGEDSNSWMVDGHTDEKSKQLFMELLHQRPGHQPLESPSMNRGEPYDRMAASG
Bra010951.1-P   MRSEMSRVGEDSNSWMVDGHTDEKSKQLFMELLHQRPGHQPLESPSMNRGEPYDRMAASG
VIT_01s0146g001 -------------------------------LIHL-------------------------
                                               *:*                          

AT1G24300.1     LTPGIQTLGGYSDHGGSHNAPSTFGARAFSDEQINRSSGDRNNMGSLHRNNSLLSGIIDG
Bra010950.1-P   F--------GFADHGGRQNASSSFGSHTSSDEHVNGLPGDRNYMGSLQRDNSLLSGSTDG
Bra010951.1-P   F--------GFADHGGRQNASSSFGSHASSDEHVNGLPGDGNYMGSLQRDNSLLSGSTDG
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G24300.1     GRSTQNETQAFSNMYAMNKDANDIKTWNNVPPKNEGMGRMMSFEAQDRMGKQAVLDSLVH
Bra010950.1-P   GR--KNETKDFSNMLGMSKDVNDIRTWNNAPPKKEGAG-LMSFEAQDRMGKQAVMDSLVQ
Bra010951.1-P   GR--KNETKDFSNMLGMSKDVNDIRTWNNAPPKKEGAG-LMSFEAQDRMGKQAVMDSLVQ
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G24300.1     GELPVVTPG<span class="highlight">Q</span><span class="highlight">Q</span>SSLNISDQYSDNLVGEDRRKDRLVVPSHGQNSVLLKRPPSSHSSSSHEG
Bra010950.1-P   GEVPVATLGRQSSSSISESYSDNLVGE-VRKDRLVVPSHGQVSVLLKRPP-----SSHEG
Bra010951.1-P   GEVPVATLGRQSSSSISGMYLPEF--------RFRFP-----------------------
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G24300.1     LIERISDTASRTAASSYSGIEGGVRRESGAAGNKGSTSEATSFSEMLKKS---NSMKKVA
Bra010950.1-P   LLEQMSDATNRT-----------------AVGNKGSKA---SFSEMLKNSSSNSSMKKVA
Bra010951.1-P   ------------------------------------------------------------
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G24300.1     AE-SNDVTEGSKGGGGKKKGKKGRQIDPALLGFKVTSN----------------------
Bra010950.1-P   AEPSSDPNEGNKGGGGKKKGKKGRQLDPALLGFKKTYKADSQTGKWRSLTTFIKRGFVPE
Bra010951.1-P   ------------------------------------------------------------
VIT_01s0146g001 ------------------------------------------------------------
                                                                            

AT1G24300.1     ------RIMGEIHRADDF
Bra010950.1-P   TTTSLFQIMRTAASGDTA
Bra010951.1-P   ------------------
VIT_01s0146g001 ------------------
                                  


<a href='QQ_QQ-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            