
<!doctype html>
<html>
    <head>
        <title>WC-CW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT1G69330.1</h1>
        <h3>AT1G69330.1 pep chromosome:TAIR10:1:26063662:26065518:-1 gene:AT1G69330 transcript:AT1G69330.1 gene_biotype:protein_coding transcript_biotype:protein_coding description:At1g69330/F10D13.3 [Source:UniProtKB/TrEMBL;Acc:Q9C546]</h3>
        <pre>
length  270
count   3
ratio   0.00
zscore  5.78






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT1G69330.1     -MWGF-------------ASNVIGSMGLKKSPKDSAQASSQCSDDEV-SNISRD------
Bra004373.1-P   -MWSF-------------ASNVIGSIGLKKDP---PQPSSD-NDDEV-SNVSRDDEEEEE
AES96966        MMWNF-------------ASHALTSIGMKRSSTEPILSCAECSDDEVCSNASKD------
Pp3c9_15340V3.2 MTWDLVGLLPSQDSNSIRVAEILFTNGGKKSA-SPKVGVAP-------ASRSLE------
PAC:32927344.CD MSQEL-------GNETTLMAPCECHSGYQPAP-NPKIGTGS-------ASRSLE------
KRH74998        MMWSF-------------ASNALTSIRLKIGSKQPSQTSAECSDDEVCSNASRD------
KRH70858        MMWSF-------------ASNALTSIGLKRGSKQPSQTSAECSDDEVCSNASRD------
                    :              :        :  .       .        :. * :      

AT1G69330.1     EEGLECPI<span class="highlight">C</span><span class="highlight">W</span>ESFNIVENVPYVL<span class="highlight">W</span><span class="highlight">C</span>GHTLCQNCVFGLQSAVLRLSSQDIRIPFFVSCP<span class="highlight">W</span><span class="highlight">C</span>
Bra004373.1-P   EQGLDCPICFESFNIVENVPYVL<span class="highlight">W</span><span class="highlight">C</span>GHTLCQNCVFALQPAVLRLSSQDIKVPFFVSCP<span class="highlight">W</span><span class="highlight">C</span>
AES96966        -EGLECPI<span class="highlight">C</span><span class="highlight">W</span>ESFNIVENVPYVL<span class="highlight">W</span><span class="highlight">C</span>GHTLCKNCVLGLQWAVMKFPTQQIRIPFFVSCP<span class="highlight">W</span><span class="highlight">C</span>
Pp3c9_15340V3.2 -EGLECPV<span class="highlight">C</span><span class="highlight">W</span>ESFDDTDNTPYVL<span class="highlight">W</span><span class="highlight">C</span>GHSLCKNCVLNLEWATVKLSGLPLQLPLFICCP<span class="highlight">W</span><span class="highlight">C</span>
PAC:32927344.CD -EGLECPV<span class="highlight">C</span><span class="highlight">W</span>EILDDRDNTPYVL<span class="highlight">W</span><span class="highlight">C</span>GHSLCKSCVLNLEWATVKLSGLPLQLPLFISCP<span class="highlight">W</span><span class="highlight">C</span>
KRH74998        -EGLECPI<span class="highlight">C</span><span class="highlight">W</span>ESFNIVENVPYVL<span class="highlight">W</span><span class="highlight">C</span>GHTLCKNCVLGLQWAVVKFPTQQIKIPFFISCP<span class="highlight">W</span><span class="highlight">C</span>
KRH70858        -EGLECPI<span class="highlight">C</span><span class="highlight">W</span>ESFNIVENVPYVL<span class="highlight">W</span><span class="highlight">C</span>GHTLCKNCILGLQWAVVKFPTQQIKIPFFISCP<span class="highlight">W</span><span class="highlight">C</span>
                 :**:**:*:* ::  :*.********:**:.*:: *: *.:::.   :::*:*:.****

AT1G69330.1     QLLSFRIVYKGNLKFPRKNFFLLWMVESLNGDRTSHGDRTSHGSLVTDNQQSAPTPRCSM
Bra004373.1-P   QLLSLRVVYNGVLKFPKKNFFLLWMVESLNGDRTRHG-----GSLV--------------
AES96966        HLLSFRFIYKGNMKFPRKNFFLLWMVESLNGDRHKA------VSASVDS-QPIWSPKVN-
Pp3c9_15340V3.2 QFLTVRFKWKGQLRYPCKNFFLLWVVESLQGEHGRS------SVCAEDT-EP--SPLCG-
PAC:32927344.CD QFLTFRFKWKGQLKYPCKNFFLLWVVESLQGEHGRP------PACTNET-EL--APMFG-
KRH74998        HLLSFRIICKGNLKYPCKNFFLLWMVESLNGDRNKL------VSTCTDN-QPIWSPKCN-
KRH70858        HLLSFRIIYKGNLKYPCKNFFLLWMVESLNGDRHKV------FSTCTDN-QPIWSPKCN-
                ::*:.*.  :* :::* *******:****:*::                           

AT1G69330.1     SLGNHSSNNNLVAR---PLLRNQSTDLLPHHD-HSNQP--SRQLF--SFHKSLDFFISFT
Bra004373.1-P   GCENHQYCSNLIAR---PVVV----------D-NQSQPSSSRQHF--SFHKSLDFFISFT
AES96966        LLGSQGTGCSP-SR---SSSSHCSEQLGSNNAVRGSDR--ERHYF--SLHKSLDFFLHFT
Pp3c9_15340V3.2 LGPVSLSSPHA-GRAGSGVVSPGAIVRQDPVIVATNRW--GLHGWRRSGTRLVASFSQFM
PAC:32927344.CD LDLASLITSRA-GRVGFSAASPGAIVTGNSVAAVGDWW--NFHHWQRSGTRVVASLLQFT
KRH74998        LLGSQVTNCNL-RR---ASVSPCSRLLGSNHDVGVNDG--ETHHF--SLHKSLDFFLHFT
KRH70858        LLGSQVTNCNL-RR---ASVSPCSRLLGSNRDVRDNDG--ETHHF--SLHKSLDFFLHFT
                             *                     .      : :  *  : :  :  * 

AT1G69330.1     SKFPFVIIFLLIVFFAIPGSLIILALYFLLTILFAVPAGLVLYFAYPILERLVNEITSS
Bra004373.1-P   SKFPFVIVFLLIVFFAIPGSLVILALYFLLTILLAVPSGLVLYFAYPILERLVNEITS-
AES96966        SKFPLVIIFLLIAFFVIPCSIVILIIYFLLTIIFAIPSFIVLYFAYPTIQRLVREITS-
Pp3c9_15340V3.2 VRVPLVLLFLFILVYVLPFSTLVLAIYCIITVLFAVPSFLVVYFSYPSLDWLVRAIAS-
PAC:32927344.CD ARIPLVLLFLFIVVYVLPFSTLVLALYCLITVLFAIPSFLVVYFSYPSLDWLVRAIAS-
KRH74998        SKFPLVVIFLLIALFLIPCCAVILVLYFLVTILFAIPSFLVLYFAYPTIQRLVREITS-
KRH70858        SKFPLVVIFLLIALFLVPCSAVILVLYFLVTIVFAIPSFLVLYFAYPTIQRLVREITS-
                 :.*:*::**:* .: :* . ::* :* ::*:::*:*: :*:**:** :: **. *:* 


<a href='WC_CW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            