
<!doctype html>
<html>
    <head>
        <title>WD-DW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT3G51580.2</h1>
        <h3>AT3G51580.2 pep chromosome:TAIR10:3:19130768:19133347:-1 gene:AT3G51580 transcript:AT3G51580.2 gene_biotype:protein_coding transcript_biotype:protein_coding description:Transmembrane protein [Source:UniProtKB/TrEMBL;Acc:F4J4F8]</h3>
        <pre>
length  410
count   8
ratio   0.00
zscore  7.85






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT3G51580.2     MDR---SSVLTLLLLILFISNASSFDRFRELVDEETQKDNSNSTTKQESPLINPQPIGDG
Bra012849.1-P   MGRSWVSSLLLLLLLILFFSKASS----RGLADEQPPKDTSNTTTTQSPPL--PPNVGDG
AES79843        MGKHTLLILLLFSLILFHGSHASFLKDLRKLIDSTKDSL-----IPEEKKL-DP------
KRH00444        MRTHALLALLFLFLIMFDVSDASFVLKLRKLIGPDPNDT-----KTK-----DPNHVGE-
KRH40730        MRTHALLTLLLFSRVMFDVSDAFSVREFRKLIASDPKDN-----QPK-----DPNHAGE-
VIT_08s0058g013 MDRNEFLPVIFLLFLVVNVSGGSVAGKFRKLADAEPKED-STATKIDPPP--SPVSVG--
                *       :: :  ::.  * .      * *      .        .      *      

AT3G51580.2     KSNVTSSEPTLPTSNSTNTNPKEPDSMSP---PP--PLIPV--------NEKNDTKVLN-
Bra012849.1-P   KANATSS---LPASNSTSPDPKEPDSASP---PP--PPPPPPPQQLQAEGEKNVTVVSN-
AES79843        --------------NKTAGEEKKNDVNIP---PV--PTPQPLPKVENN-NGENQKETNNK
KRH00444        -------------TNKDNG-------SGPISSPVPQPQPQPLPKVMKN-TNDDIKKDND-
KRH40730        -------------TNKATGGSPSTNVSGPISSPV--PQPQPLPKVMKN-TNDDKKKDNN-
VIT_08s0058g013 --------------NQSE--------------PK--------------------------
                              *.                *                           

AT3G51580.2     ----------TTEP----MSPPPPPANLTDSQDSGKLPANMAPP-PKSLESGKNETEPGK
Bra012849.1-P   ----------TTEA----VS--PPPANQTESVDKGKLPKKMAPP-PKSLESGEKGKVPET
AES79843        ITNTPPPVPATAPPPPVLVTAPPPP-----------VPATAPPPLPKKDEG--KGQ----
KRH00444        ----------SAPP----VSIPPP---------------------PKNDDGGDTGM----
KRH40730        ----------SAPP----VSVPPP---------------------PKNDDGGDKGL----
VIT_08s0058g013 ---------------------PIDPGKL------GKEKSKDAPV-PTTNSSKADSK----
                                                             *.. ..         

AT3G51580.2     ESPPLAKDPAKGKDDKGSSESASVDTCVGKSNICR-TENSLVACTLSIDKGYETFLDIIV
Bra012849.1-P   GNPPPGKDPEKVDDAKGSSESVIVETCAGKSKICR-TENSLVACTLSIEKD---------
AES79843        -----VEEKGKNEG-IKLAHSTTNDTCEG-LHTCR-DDGDMVACISKMDSK---------
KRH00444        -----HKGKDKTEG-MKFSHHSTTETCDG-FKKCTDDDGGMVACISKIDHK---------
KRH40730        -----DKENDKIEG-MKFSHNSTTETCDG-FNKCT-DDGGMVACISKIDPK---------
VIT_08s0058g013 -----GKQKEKNDD-KASSELGTYGTCDGLLEKCTDPKKKMIACIQSFENG---------
                      :   * ..    :.     ** *  . *   .  ::**  .::           

AT3G51580.2     IPQQFARSLLCAANWLILVQNEGETSLKAKIVL--PV-NALQELTLPKHQSQKVNIS-IS
Bra012849.1-P   -----------SSKWSILVQNEGEESLKAKIVL--PV-SSSPDLTLPKHQSQRVNIS-IS
AES79843        -------------NFVVLLQNRGGGTIKVKLRS--DLESNLGDIVVDKNKTEKVTIKQIK
KRH00444        -------------YLVVLLHNGGDGIIKVKLRT--DFDNNLGVVDVDKSKTEKVNITRSS
KRH40730        -------------YLVVLLHNGGDGIIIVKLRT--DFDNNLGDVEVHKSKTEKVNITQSS
VIT_08s0058g013 -------------SKALIVQNEGENTLKVNLTVSDSVKDALEVLVIPKHQTRSMNI--LS
                                ::::* *   : .::     . .    : : * ::. :.*   .

AT3G51580.2     GDTNKIILDTGKGQCALH--MYPS-EESTLPFHFPSYEKLVTPINGAYFLIVSVIIFGGI
Bra012849.1-P   VDTNKIILDAGKGECVLH--MYPP-KENTLSIHLPSYEKLVTPINGAYFLIVSVVIFGGA
AES79843        SESTELTLDAGKGDCVLHVTVVTPVPEASFFLRLPSFDKILTPVNGAYFLIFTVIVFAVT
KRH00444        SESTQLTLNAGKGDCVLH--MVSNIPEENFILRLPSYDKILTPVNGAYFLILMVLVFGGT
KRH40730        SKSTQLTLNAGKGDCVLH--MVSNIPDENFILRLPSYDKILTPVNGAYFLILMVLVFGGT
VIT_08s0058g013 TGSSKLVLNAGNKDCVFENHLI---SEGNFFQWLP-HSELVTPIYGVYFLFLTALVVGAM
                  :.:: *::*: :*.:.  :     : .:   :* ..:::**: *.***:. .::..  

AT3G51580.2     WAFCLCRKNRRA-GSGVPYRELELS-GGPGLENESGVHDVETAD-<span class="highlight">W</span><span class="highlight">D</span>EG<span class="highlight">W</span><span class="highlight">D</span>D<span class="highlight">D</span><span class="highlight">W</span><span class="highlight">D</span>ENNAV
Bra012849.1-P   WGFCLCRKNRRADDNGVPYRELELSGGGTGLES---VHDVETAD-<span class="highlight">W</span><span class="highlight">D</span>EG<span class="highlight">W</span><span class="highlight">D</span>D<span class="highlight">D</span><span class="highlight">W</span><span class="highlight">D</span>ENNAV
AES79843        WACCCIFKKKPR--DEIPYQELEMA-----LPESASATVVESAEG<span class="highlight">W</span><span class="highlight">D</span>QG<span class="highlight">W</span><span class="highlight">D</span>D<span class="highlight">D</span><span class="highlight">W</span><span class="highlight">D</span>DNVAV
KRH00444        WACCACRK-KHR--NEVPYQELEMA-----LPESASATNVESAEG<span class="highlight">W</span><span class="highlight">D</span>Q<span class="highlight">D</span><span class="highlight">W</span><span class="highlight">D</span>D<span class="highlight">D</span><span class="highlight">W</span><span class="highlight">D</span>VNVAV
KRH40730        WACCACRK-KHH--DEVPYQELEMA-----LPESALATNVESAEG<span class="highlight">W</span><span class="highlight">D</span>Q<span class="highlight">D</span><span class="highlight">W</span><span class="highlight">D</span>D<span class="highlight">D</span><span class="highlight">W</span><span class="highlight">D</span>DNVAV
VIT_08s0058g013 WTCCKFRKRRHQ--GGVPYQELEMG-----LPESSPAVNVETAAG<span class="highlight">W</span><span class="highlight">D</span>EG<span class="highlight">W</span><span class="highlight">D</span>D<span class="highlight">D</span><span class="highlight">W</span><span class="highlight">D</span>EDKAV
                *  *   * :    . :**:***:.     * .   .  **:*  **:.****** : **

AT3G51580.2     KSPGSAAKSVSISANGLTARAPNRDG<span class="highlight">W</span><span class="highlight">D</span>H<span class="highlight">D</span><span class="highlight">W</span><span class="highlight">D</span>D
Bra012849.1-P   KSPGGAANSARISANGLTARASNRDG<span class="highlight">W</span><span class="highlight">D</span>N<span class="highlight">D</span><span class="highlight">W</span><span class="highlight">D</span>D
AES79843        KSPVVR-HAGSISANGLTSRSSNKDGWEDN<span class="highlight">W</span><span class="highlight">D</span>D
KRH00444        KSPAA--LAGSISANGLTSRSSNKDGWENN<span class="highlight">W</span><span class="highlight">D</span>D
KRH40730        KSPAA--HAGSISGNGLTSRSSNKDGWENN<span class="highlight">W</span><span class="highlight">D</span>D
VIT_08s0058g013 KSPGGR-YVGSISSNGLTTRSSNREGWEN<span class="highlight">D</span><span class="highlight">W</span><span class="highlight">D</span>D
                ***        **.****:*:.*::**:.:***


<a href='WD_DW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            