
<!doctype html>
<html>
    <head>
        <title>WG-GW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT1G26680.4</h1>
        <h3>AT1G26680.4 pep chromosome:TAIR10:1:9219463:9223749:1 gene:AT1G26680 transcript:AT1G26680.4 gene_biotype:protein_coding transcript_biotype:protein_coding description:transcriptional factor B3 family protein [Source:TAIR;Acc:AT1G26680]</h3>
        <pre>
length  992
count   8
ratio   11.52
zscore  6.71






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT1G26680.4     -------------MADQSLL--------HSPINPHFF---------------QPILTESR
Bra024695.1-P   -------------MADDQSL--------HSPITPHFF---------------QPLLPGFH
Bra024697.1-P   MK--------CSRMAYDQSL--------HSPNNPHFF---------------QPLLPGFH
Bra024696.1-P   -------------MAH-QSL--------HSPTNPHFF---------------QPLLPGFH
Bra012445.1-P   -------------MVDQSLLPASSSRLPHSP----------------------PCLVQ--
Zm00001d004358_ MPLPSDSGSGARAMSKQFKVLLPTSF--HKLVSTAFLDRRSISVADVSLSSRNPLIPQ--
                             *     :        *.                       * :    

AT1G26680.4     THLNIPVAFFSKHVEGRNNQNKTVTLRSDASDKTWLVKM----DGLKLTD<span class="highlight">G</span><span class="highlight">W</span>EDFAFAHD
Bra024695.1-P   THLNIPVAFFSRHVQGNNDHIKTAKLRTDASAKTWLVKI----NGLKLTD<span class="highlight">G</span><span class="highlight">W</span>EDFAVAHD
Bra024697.1-P   TYLNIPVAFFLKHVQGSNDHIKTAKLTTDASAKTWLVKV----DGVRLTD<span class="highlight">G</span><span class="highlight">W</span>EDFAVGHD
Bra024696.1-P   TYLNIPVAFFSKHVEGSNDHIKTAKLRTDASAKTWLVEM----DGLKLTD<span class="highlight">G</span><span class="highlight">W</span>EDFAVAHD
Bra012445.1-P   ---NIPVAFFSTHIKGRNDHIKIAKLRTDASDKTWKVKM----DGSKFTD<span class="highlight">G</span><span class="highlight">W</span>EDFAVAHD
Zm00001d004358_ -RISDELAGCFDAGEGEGAPEPTA-LVVSPFGKVWRVEVVRDGHGASLGR<span class="highlight">G</span><span class="highlight">W</span>AEFLAAHG
                   .  :*      :* .     . *  ..  *.* *::    .*  :  ** :*  .*.

AT1G26680.4     LRTGDIVVFRLEGEMVFHVTALGPSCCEIQYHT-SSHNINDDDRNDQINIASRNSSRVKK
Bra024695.1-P   LRIGDIIIFRYEGEMVFHVTAFGPSCCDIQYTSESSHNINDDSQDQTNN-----------
Bra024697.1-P   LRIGDITIFRHEGEMVFHVTAFGPSCCDIQYTSESSHNINDDSQDQTNNAV---------
Bra024696.1-P   LRIGDIIIFRHEGKMVFHVTAFGPSLCEFQYTSASSHNINDVSDDDHTNK----------
Bra012445.1-P   LRVGDITVFRHEGEMVFHVTALGPSFCEIQYT--SSHNINDDTHDQTNNIVTGNSSREK-
Zm00001d004358_ VGI<span class="highlight">G</span><span class="highlight">W</span>FVVLRYEGGGALTVKVFDTSLCIKEFGAPAA------------------------
                :  * : ::* **  .: *..:..* *  ::   ::                        

AT1G26680.4     NPRKKVESSLDHSRFVAKVSAWCLSNDRLYIPLSFARLNGLNKINSKKIYLQNEEGRSWK
Bra024695.1-P   ------------------------------------------------------------
Bra024697.1-P   ------------------------------------------------------------
Bra024696.1-P   ------------------------------------------------------------
Bra012445.1-P   ------------------------------------------------------------
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT1G26680.4     LVLRHDKSGMQTFVQS<span class="highlight">G</span><span class="highlight">W</span>RRFCSENGIRQGQYTFKLVRKSAPPVIRLCRAKAKPKQRSVA
Bra024695.1-P   ------------------------------------------------------------
Bra024697.1-P   ------------------------------------------------------------
Bra024696.1-P   ------------------------------------------------------------
Bra012445.1-P   ------------------------------------------------RKRVKKNPKPVP
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT1G26680.4     EYSSDHSCFEGSVTPSSLRNDLLYLPRSFVNSNRLDKRCSEIVLKNEQGVKWPLVLKRFK
Bra024695.1-P   ------------------------------------------------------------
Bra024697.1-P   ------------------------------------------------------------
Bra024696.1-P   ------------------------------------------------------------
Bra012445.1-P   QASLHSSCYVGSVSASSLKHNKLYLGREFVTSNGLNKGCSEIVLKNEGGGRWTLPLKHYK
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT1G26680.4     SV--TYLPK<span class="highlight">G</span><span class="highlight">W</span>TSFCQVNRIKAGDSFKFKLVGTWKKPVLSLCPTQSNNH-KTPLECSEGN
Bra024695.1-P   ------------------------------------------------------------
Bra024697.1-P   ------------------------------------------------------------
Bra024696.1-P   ------------------------------------------------------------
Bra012445.1-P   SINHTYLGP<span class="highlight">G</span><span class="highlight">W</span>TTFCQVNGIKAEDAFMFKLVRTGEKPVLCLCPEESSHRDKTPVECLE--
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT1G26680.4     KSEESEEDCLEVKKKKYWSRCRASVENMDDDQTNIGNSSRKKRVSKNP----REKVESSS
Bra024695.1-P   ----------------------------------TGNSSGEKR-----------------
Bra024697.1-P   ----------------------------------TGNSSGEKR-----------------
Bra024696.1-P   ----------------------------------TGNSSGEKR-----------------
Bra012445.1-P   ----------------------------DSDDVNPLSSSSKSKESKEESLGDKRASSSYS
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT1G26680.4     DHSSFVGSVNPSSLYKDQLYLPRNFVSSNFLDKRCSEIVLK-NERGEKRTLVLKHFKKDL
Bra024695.1-P   ------------------------------------------------------------
Bra024697.1-P   ------------------------------------------------------------
Bra024696.1-P   ------------------------------------------------------------
Bra012445.1-P   EDRFLTLTLTQKAVKRYQLTIPVAFFLKHVQGRNDHIKIAKLTTAASDKTWTV---KMDG
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT1G26680.4     TFLKK<span class="highlight">G</span><span class="highlight">W</span>TSFCQVNRIKAGDSFKFKLVGTWNKPVLSLCPTETNYHKTPLACSEGNKSEES
Bra024695.1-P   ------------------------------------------------------------
Bra024697.1-P   ------------------------------------------------------------
Bra024696.1-P   ------------------------------------------------------------
Bra012445.1-P   FKLTH<span class="highlight">G</span><span class="highlight">W</span>EDFAVAHDLRVGDMIVFRHEG------------ELAFHVTIMGPS--------
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT1G26680.4     EEEGTEDKNTSQDCLEVKKRKYWSTCRASAENIDDDQTNIGNSSKEKRVKKNPVKKAESS
Bra024695.1-P   ----------------------------------------------KRVKKTPRTKEESS
Bra024697.1-P   ----------------------------------------------KRVKKNPRTKEDSS
Bra024696.1-P   ----------------------------------------------KRVKKNPRTKEESS
Bra012445.1-P   -------------CCEIQ----------YTHNINHNHTN--------------NIIESSS
Zm00001d004358_ --------------------------------------------------VMPSKSCKSI
                                                                         .* 

AT1G26680.4     SDHSSFVANVTASSLNYDRLYLPLSFVSSNGLDKMNGKE-IVLLNEEGLSWKFNLKYNQA
Bra024695.1-P   SDHSHFMAHVSPSSLRFDRLYVPISFARSNGLDNMSGKE-IVLLNQEGRSWNLNITYTKA
Bra024697.1-P   SDHSQFVAHVSPSSLSSDRLYIPTGFARSNGLDNMSGKEIIVLLNEEGRSWSLDMTCNKV
Bra024696.1-P   SDHSHFVAHVTPSSLSCDRLYVPKGFAMSNGLDNMSGKE-IVLLNEEGKSWNLNITYNES
Bra012445.1-P   SDNSLFVAKVSASNLRLDRLYLPMSFARTNGLDKMSGEQ-IILLNGEGRSWSLKLKHDKS
Zm00001d004358_ FCKPQFI-RIFHSNLS-EKMMLPARFVKDYVSEECLNSRIAIIFSPIAKFWRVELKNDQS
                  :. *: .:  *.*  ::: :*  *.     ::  ...  :::.  .  * ..:.  : 

AT1G26680.4     GKHTYVRP<span class="highlight">G</span><span class="highlight">W</span>SRFCDANGMSQGQ---------------------------------QFTF
Bra024695.1-P   SVQTLVGP<span class="highlight">G</span><span class="highlight">W</span>RRFCAENGMNKGH---------------------------------HYTF
Bra024697.1-P   GMQTFVRP<span class="highlight">G</span><span class="highlight">W</span>RRFCAENGMKKGH---------------------------------HYTF
Bra024696.1-P   GMHTFVRP<span class="highlight">G</span><span class="highlight">W</span>RRFCAENGISQGH---------------------------------HCTF
Bra012445.1-P   DMHTFVRP<span class="highlight">G</span><span class="highlight">W</span>RRFCAENGMSHG----------------------------------HYTL
Zm00001d004358_ GI--FFTG<span class="highlight">G</span><span class="highlight">W</span>SQFLDFHGIKNGEVLLLKYEGNMVFKFKAFGLSGCQKDFKNQNAGIQLNT
                .    .  ** :*   :*:.:*                                  : . 

AT1G26680.4     KLVQKHGPPVMYLSLSEH-RPK-------------S-ESSSHRSYFVGSVTASSIKKDK-
Bra024695.1-P   KLVGKSAPPVIRLSLSE----------------PAP-ESSLHHSNYVGSVSSNSLRKDK-
Bra024697.1-P   KLVRRSAPLIIRLSHAEH-EPE-----------PAT-ESSLNHSYYVGSVSPNSLSTDK-
Bra024696.1-P   ELVRKSAPPVLRLSRAEH-DPK-----------PAP-ESSLHHSYYVGSASSNSLRIDK-
Bra012445.1-P   KLIRNSGPPVIKLCGQVHNRPK-----------PVPVVSSLHHSCFVGSVTSNSLKTDK-
Zm00001d004358_ KKQQETPPIRKRKSNDEKSSSEENKRLKTPVTSPSPSDPSLEKPYQIGTSSWIKKRINTY
                :   .  *     .                     .  .* .:.  :*: :  .   :. 

AT1G26680.4     -----LYLWKSFVSSNGLDKGCKKIILKNK<span class="highlight">W</span><span class="highlight">G</span>REWKLVLKHYKSNCFTI-IKR<span class="highlight">G</span><span class="highlight">W</span>TSFCQ
Bra024695.1-P   -----LYIQRKFVNANGL-KGLSEIVLKNE<span class="highlight">W</span><span class="highlight">G</span>GSWNLGLRYYESLDHTY-LGP<span class="highlight">G</span><span class="highlight">W</span>KTFCQ
Bra024697.1-P   -----L---RHFVTANGLEKGLSEITLKNE<span class="highlight">W</span><span class="highlight">G</span>GSWILGFRHYEPQNHTY-LGP<span class="highlight">G</span><span class="highlight">W</span>KTFCQ
Bra024696.1-P   -----LYLQMKFVTANGMKKGFSEIILKNE<span class="highlight">W</span><span class="highlight">G</span>GRWSVRLRQYASSNRTY-LGP<span class="highlight">G</span><span class="highlight">W</span>KTFCQ
Bra012445.1-P   -----LYLGKKFVIENGLDRGCSEMVLKNE<span class="highlight">W</span><span class="highlight">G</span>GRWSLALRFYESAYQTY-LGP<span class="highlight">G</span><span class="highlight">W</span>RTFCQ
Zm00001d004358_ ALEQFLALSVKFCNSIGFRMACT-ITLKTATNSTRSWQVRGTAYKWHRYIIGV<span class="highlight">G</span><span class="highlight">W</span>KSFCQ
                     *     *    *:  . . : **.  .      .:          :  ** :***

AT1G26680.4     GNGLKAGDSFKFKLVGTG------EKPVLSLCPAESSHEKIPLECPEGIDDVNSLSSNPS
Bra024695.1-P   VNRIKTGDSFMFKVVETG------DKPVLLLC--TTNRGKTPLDCPEDSGDINSLSSDTG
Bra024697.1-P   VNGIKAEDSFMFKLVETG------DKPVLLLC--TSNRGKTPLECSEDRDDVNSLSSDTS
Bra024696.1-P   VNGIKAGDPFMFKLVETG------DKPVLLLC--SSNRGETPLECS---DDVNSLSSDTS
Bra012445.1-P   VNEIKVGDSFKFKLVGTG------DNPVLLLC--TR---KMPLECPGE-----NSGSDTS
Zm00001d004358_ DNRLKAGDLCTFNIIETTLWHVTITRSALAAD--TFNKQKKP-PCSSSRDKKGSSS----
                 * :*. *   *::: *        ...*          : *  *.       . .    

AT1G26680.4     SGDDS-SRSEESEEENMEDKN-------ISQDCLETKKRKYC----SSSSYSQNRFVTLT
Bra024695.1-P   S-----RESQESEEESLEDDD------SSSEESFEMVKRENSSRCRASSSYSQDRFVKLT
Bra024697.1-P   SEDDSKEESQESDKESIEDAD-------RSQQCIVMEKENNSLRCICSSPYSKHRFVRLS
Bra024696.1-P   SG----EESQESEEESLEDDS-------RSKDCVEMEKRKSSLRCKCSSSYSKHRFVRLT
Bra012445.1-P   SGDDS-SEPQESEEESLGDKN-------ISQECVKMEKNEEF----VEMQNTKRRSL---
Zm00001d004358_ ------REGAKRPKPSMAHFSKVQVPPSYARRCAYDVGPPPWIRKQMNASTIRNRLVRMV
                       .  :  : .: . .        :. .              .    : * :   

AT1G26680.4     -LTRSAFQTYKLFLPRGFTQVNGINKPRKITLLGQDGVKK----VVDLYQESSSRIMRFG
Bra024695.1-P   -LTPRALKTYKLMLPLDFTRVNGINKPGKRM-----------------------------
Bra024697.1-P   -FTQSALKTSVLYLPLAFTRMNCINKPIKIMLLGKDGVKK---QVVDLLKNKSTGAMRFG
Bra024696.1-P   -LTQNALKTSVQHLPLGFTRVNGINRSRKIMLLGKDGVKK---WVVDLLKNKSTGAMRIR
Bra012445.1-P   ------------------------------------------------------------
Zm00001d004358_ ILAFDSRKLVIGCLAPAF--CNQIELRRRCEIVLKTSINSNHSWRVGGLSQKK-GSYIIG
                                                                            

AT1G26680.4     K<span class="highlight">G</span><span class="highlight">W</span>REFFDAQGVKIDDSFVLELIW-EKEASPVLKFCTKVPK
Bra024695.1-P   --------------------------ERVL-----------
Bra024697.1-P   K<span class="highlight">G</span><span class="highlight">W</span>REFCDAHGVKVGESFQLELFREDEEAIPVLKFCTTT--
Bra024696.1-P   K<span class="highlight">G</span><span class="highlight">W</span>RELCDAHGVKVGESFLLELIR-DEEAIPVLKFCTKL--
Bra012445.1-P   -----------------------------------------
Zm00001d004358_ P<span class="highlight">G</span><span class="highlight">W</span>KKFCTENRVKVGDVCTFNVIETQLWHVVITRRC-----
                                                         


<a href='WG_GW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            