
<!doctype html>
<html>
    <head>
        <title>WH-HW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT1G70830.1</h1>
        <h3>AT1G70830.1 pep chromosome:TAIR10:1:26709947:26711462:-1 gene:AT1G70830 transcript:AT1G70830.1 gene_biotype:protein_coding transcript_biotype:protein_coding gene_symbol:MLP28 description:MLP-like protein 28 [Source:UniProtKB/Swiss-Prot;Acc:Q9SSK9]</h3>
        <pre>
length  335
count   4
ratio   0.00
zscore  7.07






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT1G70830.1     MADVATKHPMEDEVKKTEASSLVGKLETDVEIKASADKFHHMFAGKPHHVSKASPGNIQG
Bra007946.1-P   MA----------EAAPTEASSLVGKLETEVEIKASAGKFHHMFAGRPHHVSKASP-----
Bra003939.1-P   MA----------EAA-TTKATLVGKLETDVEIKASAGKFHHMFAGRPHHVSKASPGKIKA
AET01440        M-----------------ATSAIEKIEMDVPIKATPQQFYEVLCNKTHHISNVCPDIVKG
KRH13947        M-----------------AYSQLQKVEANVHIKASADQFHNVLCNRTHHIANIFPGKVQS
KRH13946        M-----------------AYSQLQKLEANVSIKASAEQFYDVLCHKTHQIPKIFPEKALS
KRH37979        M-----------------AYSQLQKVETSLQIKASAEQFYDVFCNKPHTIANISPENIQS
Zm00001d003379_ M---------------------ASKVELVVEVKSPADKLWAALRDSTELFPKIFPEQYKS
BGIOSGA016576-P M---------------------ASKVELVVEVKSPADKLWTALRESTELFPKIFPEQYKS
Os04t0465600-01 M---------------------ASKVELVVEVKSPADKLWTALRDSTELFPKIFPEQYKS
VIT_01s0011g051 M-------------------GLVGKLQAETLIVAPADKFHEMFSCRPHHISNVCPEKIQK
VIT_01s0011g051 M--------------------------------ASADKFHEIFWCRPHHISNVCPAKIQK
VIT_01s0011g051 M-------------------GLVGKLEAEILILAPADKFHEVWGGRPHHMSSVSPGKIQK
                *                                :.. ::       .. ...  *     

AT1G70830.1     CDLHEGDWGTVGSIVFWNYVHDG-EAKVAKERIEAVEPDKNLITFRVIEGDLMKEYKSFL
Bra007946.1-P   ---------------------DG-EAKVAKERIEAVEPEKNLITFRVIEGDLMKEYKSFL
Bra003939.1-P   CDLHEGDWGKVGSIVYWNYVHGKWRGKGAKERIEAVEPEKNLITFRVIEGDLMKEYKSFL
AET01440        IELHEGDWGTEGCIISWNYVFAG-KTCISKQRIEDIDKENNKITFKVLGGDLLEDYKSFK
KRH13947        VEIHKGEWGTEGSVISWNYLHEG-KVCVAKEMVEGIDMKNNKMTFKVIEGDLLGHYKSFK
KRH13946        VEILKGAWGTEGSIISWNYLYEG-KVCVAKEVIEGIDKKNKKMSFKVIEGDVLGHYKSFK
KRH37979        VEVHKGEWGKEGSIVSWNYLHEG-TVCVAKQVLEGIDKENNKMTMKVIEGDVLGLYKSFK
Zm00001d003379_ IETVEGDGKSAGTVRLLKYTEAVPMLTFAKEKLETADDENKVVSYSVVDGELADFYKNFK
BGIOSGA016576-P IETVEGDGKSAGTVRLLKYTEGVPMVTFAKEKVEVADDEKKVVSYSVVDGELVSFYKNFR
Os04t0465600-01 IETVEGDGKSAGTVRLLKYTEGVPMVTFAKEKVEVADDEKKVVSYSVVDGELVSFYKNFR
VIT_01s0011g051 VNLHEGEWGTSGYVIEWNYNIDG-NDHVAKQIVEITDGENKSITYKFIEGDLLKKYKSFI
VIT_01s0011g051 VHLHEGEWGTSGSVIQ<span class="highlight">W</span><span class="highlight">H</span>YNIDG-NDHVAKEIVEAIDEENKSVTFKVIEGDLMKEYKSFR
VIT_01s0011g051 VDLHEGDWGTVGSVIEWSYVIDG-KNHVAKEIVEAIDKENKAVTFKVIEGDLLKEYKSFK
                                            :*: :*  : .:: ::  .: *::   **.* 

AT1G70830.1     LTIQVTPK-----PGGPGSIV<span class="highlight">H</span><span class="highlight">W</span><span class="highlight">H</span>LEYEKISEEVAHPETLLQFCVEVSKEIDEHLLAEEE
Bra007946.1-P   LTIQVTPK-----HGGPGSIV<span class="highlight">H</span><span class="highlight">W</span><span class="highlight">H</span>LEYEKISEEVAHPETLLQFCVEVSKEIDEHLLSEE-
Bra003939.1-P   ITIQVTPK-----HGGPGSIV<span class="highlight">H</span><span class="highlight">W</span><span class="highlight">H</span>LEYEKISDEVAHPETLLQFCVEVSKEIDEHLLSEE-
AET01440        FIMQIVPQ-------REGSVVRWIVEYEKLNNNVPNPHSLFHLSVEVLKYVDAHLAPEDK
KRH13947        FLLQVTPK-------EKGSVVNWVLEYEKQNDNTPDPYTLLELTVEMSKEIGAYLSQDHN
KRH13946        FIMQVTPK-------EKGSVVQCVVEYEKQKDHIPDPHTLLQSTVELCKKINAYLTQDNN
KRH37979        SNLQVTPK-------GKGSVVLWAMEYEKQEDHIPDAHTLLQLAVVVSKKIDAYLTQGHN
Zm00001d003379_ ITLKVTP---AKAEGEGGAVVSWAMEFDKANDQVPDPDVIKETATKTFHDLDDYLLKN--
BGIOSGA016576-P VTLQVTPKGGAAAPAADGAVVSWTMDFDKASEEVPDPDVIKETAAKTFHDLDDYLLKN--
Os04t0465600-01 VTLQVTPKGGAAAPAADGAVVSWTMDFDKASEEVPDPDVIKETAAKTFHDLDDYLLKN--
VIT_01s0011g051 IVVQTIPK-------GDATWV<span class="highlight">H</span><span class="highlight">W</span>TLEYEKLNKDIPTPIKELEFLVLVGEGVDDHLVQA--
VIT_01s0011g051 IIVQAIPK-------GDATYV<span class="highlight">H</span><span class="highlight">W</span>TLEYEKLNKDVLAPIKILGFVVHVSEDIDDHLVQA--
VIT_01s0011g051 AIVQTISK-------GETTWVRWTFEYEKLNKEISTPVKLLGFVIHMNEELDDHLIQA--
                  ::  .           : *   .:::* ...   .           . :. :*     

AT1G70830.1     EVKTPETPSLVGKLETDVEIKASAEKFHHMFAGKPHHVSKASPGNIQGCDLHEGDWGQVG
Bra007946.1-P   ------------------------------------------------------------
Bra003939.1-P   ------------------------------------------------------------
AET01440        K-----------------------------------------------------------
KRH13947        ------------------------------------------------------------
KRH13946        ------------------------------------------------------------
KRH37979        ------------------------------------------------------------
Zm00001d003379_ ------------------------------------------------------------
BGIOSGA016576-P ------------------------------------------------------------
Os04t0465600-01 ------------------------------------------------------------
VIT_01s0011g051 ------------------------------------------------------------
VIT_01s0011g051 ------------------------------------------------------------
VIT_01s0011g051 ------------------------------------------------------------
                                                                            

AT1G70830.1     SIVFWNYVHDREAKVAKERIEAVEPNKNLITFRVIDGDLMKEYKSFLLTIQVTPKLGGPG
Bra007946.1-P   ------------------------------------------------------------
Bra003939.1-P   ------------------------------------------------------------
AET01440        ------------------------------------------------------------
KRH13947        ------------------------------------------------------------
KRH13946        ------------------------------------------------------------
KRH37979        ------------------------------------------------------------
Zm00001d003379_ ------------------------------------------------------------
BGIOSGA016576-P ------------------------------------------------------------
Os04t0465600-01 ------------------------------------------------------------
VIT_01s0011g051 ------------------------------------------------------------
VIT_01s0011g051 ------------------------------------------------------------
VIT_01s0011g051 ------------------------------------------------------------
                                                                            

AT1G70830.1     SIV<span class="highlight">H</span><span class="highlight">W</span><span class="highlight">H</span>LEYEKISEEVAHPETLLQFCVEVSKEIDEHLLAEE
Bra007946.1-P   -----------------------------------------
Bra003939.1-P   -----------------------------------------
AET01440        -----------------------------------------
KRH13947        -----------------------------------------
KRH13946        -----------------------------------------
KRH37979        -----------------------------------------
Zm00001d003379_ -----------------------------------------
BGIOSGA016576-P -----------------------------------------
Os04t0465600-01 -----------------------------------------
VIT_01s0011g051 -----------------------------------------
VIT_01s0011g051 -----------------------------------------
VIT_01s0011g051 -----------------------------------------
                                                         


<a href='WH_HW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            