
<!doctype html>
<html>
    <head>
        <title>WH-HW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT5G27730.1</h1>
        <h3>AT5G27730.1 pep chromosome:TAIR10:5:9819256:9823250:1 gene:AT5G27730 transcript:AT5G27730.1 gene_biotype:protein_coding transcript_biotype:protein_coding description:Heparan-alpha-glucosaminide N-acetyltransferase-like protein (DUF1624) [Source:UniProtKB/TrEMBL;Acc:Q94CC1]</h3>
        <pre>
length  472
count   4
ratio   0.00
zscore  7.07






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT5G27730.1     ----------------------------MAEIK----------V--ERSHDQ-----HLL
Bra020578.1-P   ----------------------------MSEIK----------VVDVMSHDQ-----GLL
AES82432        ----------------------------MEEII-----------------GE-----HSV
KRH68502        ----------------------------MAEIK-----------------GE-----HSL
KRG96791        ----------------------------MAEIK-----------------GE-----HSL
Zm00001d011929_ ----------------------------MDDGSSNKVVE---DVAEEDPGRR-----GGP
BGIOSGA000194-P ----------------------------M-DGGGQGVARRHGAVAEEDPDRRRGDRSGKE
BGIOSGA000195-P ----------------------------M-DGGGQGVARRHGAVAEEDPDRRRGDRSGKE
Os01t0931100-02 ----------------------------M-DGGGQGVARRHGAVAEEDPDRRRGDRSGKE
VIT_14s0060g022 MTKQRFPQYLSQPFLPYILRHLCLRRPHMADTS----------IKDDAPDNQ-----HRL
                                            * :                    .        

AT5G27730.1     EPKEDTSSSYTRRSLA---GNRPRLASLDIFRGLTVALMILVDDAGGDWPMIAHAPWNGC
Bra020578.1-P   VHEEAASTLNLNE--------KRRLASLDIFRGLTVALMILVDDAGGDWPVIAHAPWEGC
AES82432        HVVSEVEPV-SAKELP---KKVKRVASLDIFRGLTVALMILVDDAGGEWPAIGHAPWNGC
KRH68502        NVSQELPEV-SDKNLP----KTKRVASLDIFRGLTVALMILVDDAGEQWPMIGHAPWNGC
KRG96791        NVSEELP-L-SDKNLP----KTKRVASLDIFRGLTVALMILVDDAGGQWPMIGHAPWNGC
Zm00001d011929_ WRTDEADA---NEKAP---RPSRRVASLDVFRGLTVALMILVDGAGGEWPVIGHAPWNGC
BGIOSGA000194-P GDDDGVDV---DEKAPPPRRTSRRVASLDVFRGLTVALMILVDGAGGEWPVIGHAPWNGC
BGIOSGA000195-P GDDDGVDV---DEKAPPPRRTSRRVASLDVFRGLTVALMILVDGAGGEWPVIGHAPWNGC
Os01t0931100-02 GDDDGVDV---DEKAPPPRRTSRRVASLDVFRGLTVALMILVDGAGGEWPVIGHAPWNGC
VIT_14s0060g022 IISDSGFP---PEERP---QKTKRLASLDIFRGLTVALMILVDDAGGEWPMIGHAPWNGC
                   .        .          *:****:*************.** :** *.****:**

AT5G27730.1     NLADFVMPFFLFIVGVSIALSLKRISNKFEACKKVGFRTCKLLFWGLLLQGGFSHAPDEL
Bra020578.1-P   NLADFVMPFFLFIVGVSIALAFKRVSNKFEACKKVGFRTCKLLFWGLLLQGGFSHAPDEL
AES82432        NLADFVMPFFLFIVGMAIPLSLKKIPNKLLAVKKVIVRTLKLLFWGLLLQGGYSHAPDHL
KRH68502        NLADFVMPFFLFIVGMAIPLALKRIPNRLLAVKKVIVRTLKLLFWGLLLQGGFSHAPDNL
KRG96791        NLADFVMPFFLFIVGMAIPLALKRIPNRLLAVKKVIVRTLKLLFWGLLLQGGFSHAPDNL
Zm00001d011929_ NLADFVMPFFLFIVGMAVPLALKRIPDRGRAVRRVVVRTLKLLFWGILLQGGYSHAPDEL
BGIOSGA000194-P NLADFVMPFFLFIVGMAIPLSLKRIPDRGRAVRRVVLRTLKLLLWGILLQGGYSHAPDDL
BGIOSGA000195-P NLADFVMPFFLFIVGMAIPLSLKRIPDRGRAVRRVVLRTLKLLFWGILLQGGYSHAPDDL
Os01t0931100-02 NLADFVMPFFLFIVGMAIPLSLKRIPDRGRAVRRVVLRTLKLLFWGILLQGGYSHAPDDL
VIT_14s0060g022 NLADFVMPFFLFIVGVAIALALKRIPDRLMAIKKVTLRTLKLLFWGLLLQGSFTQDPDKL
                ***************:::.*::*::.::  * ::* .** ***:**:****.::: **.*

AT5G27730.1     TYGVDVTMMRFCGILQRIALSYLVVALVEIFTKD-SHE-E-NL---STGRFS-IFKSYYW
Bra020578.1-P   IYGVDVTMMRFCGILQRIALSYLVVALVEILTKN-SHE-E-SL---STGTFS-IFKSYYW
AES82432        SYGVDMKHIRWCGILQRIALAYLVVALVEIISRS-RQD-RDDP---EPTNLS-IFTLYYW
KRH68502        TYGVDMKHIRWCGILQRIALAYLVVALVEIFSRS-TQA-R-DP---EPTHLS-IFNLYYW
KRG96791        TYGVDMKHIRWCGILQRIALAYLVVALVEIFSRS-AQA-R-DP---EPTHLS-IFKLYYW
Zm00001d011929_ AYGVDMRHVRWGGILQRIALAYLVVAVLEMVTKDGAKVHQ-DQPPGSSGRFSRVFRMYLS
BGIOSGA000194-P SYGVDMKHVRWCGILQRIALAYLVVAVLEIVTKN-AKI-Q-DQ---SSSGFS-IFRMYFS
BGIOSGA000195-P SYGVDMKHVRWCGILQRIALAYLVVAVLEIVTKN-AKV-Q-DQ---SSSGFS-IFRMYFS
Os01t0931100-02 SYGVDMKHVRWCGILQRIALAYLVVAVLEIVTKN-AKV-Q-DQ---SSSGFS-IFRMYFS
VIT_14s0060g022 TYGVDMKKIRWCGILQ-------------------AQA-K-DL---SPGQFS-IFKLYCW
                 ****:  :*: ****                    :  . .    ..  :* :*  *  

AT5G27730.1     <span class="highlight">H</span><span class="highlight">W</span>IVAASVLVIYLATLYGTYVPDWEFVVYDKDSVLYGKILSVSCGVRGKLNPPCNAVGYV
Bra020578.1-P   <span class="highlight">H</span><span class="highlight">W</span>IVGASVLVIYLATLYGTYVPDWEFVVSDKDSILYGKIKSVSCGVRGKLNPPCNAVGYV
AES82432        <span class="highlight">H</span><span class="highlight">W</span>LVAACILVVYMPLLYGIHVPDWQFTVHNPDSIYNGTTFTVTCGVRGKLDPPCNAVGYI
KRH68502        <span class="highlight">H</span><span class="highlight">W</span>LVGACILVVYLALLYGIHVPDWGFTVHNPDSIYNGTTLTVTCGVRGKLDPPCNAVGYI
KRG96791        <span class="highlight">H</span><span class="highlight">W</span>LVGACILAVYLALLYGIHVPDWQFTVHNPDSIYNGTTLTVTCGVRGKLDPPCNAVGYI
Zm00001d011929_ QWIVACCILVVYLSLAYGVYVPDWEFRVRNADSPDYGKVLTVRCGTRGALDPPCNAVGYI
BGIOSGA000194-P QWIVACCILVIYLSLVYGIYVPDWDFRVSDVKNPNFGKILTVTCGTRGKLSPPCNAVGYI
BGIOSGA000195-P QWIVACCILVIYLSLVYGIYVPDWDFRASDVKNRNFGKILTVTCGTRGKLSPPCNAVGYI
Os01t0931100-02 QWIVACCILVIYLSLVYGIYVPDWDFRVSDVKNPNFGKILTVTCGTRGKLSPPCNAVGYI
VIT_14s0060g022 <span class="highlight">H</span><span class="highlight">W</span>LMGACVLIVYMAVSYGTYVPD<span class="highlight">W</span><span class="highlight">H</span>FTVHDRDSADYGKVLTVACGARGKLDPPCNVVGYI
                :*::...:* :*:.  ** :**** * . : ..   *.  :* **.** *.****.***:

AT5G27730.1     DRQVLGINHMYHHPAWRRSKACTDDSPYEGAIRQDAPSWCRAPFEPEGILSSISAILSTI
Bra020578.1-P   DRQVLGINHMYHHPAWRRSKACTDDSPYEGSLRQDAPSWCHAPFEPEGVLSSISAILSTI
AES82432        DREVLGINHVYKKPASRRSEACTVKPPYEGPFKKTAPAWCYAPFEPEGILSSISAILSTI
KRH68502        DREVLGINHMYKRPAWRRSEACTENSPYEGPFKKNAPSWCYAPFEPEGILSSISAILSTI
KRG96791        DREVIGINHMYKRPAWRRSEACTENSPYEGPFKKNAPSWCYAPFEPEGILSSISAILSTI
Zm00001d011929_ DRRVLGINHMYQKPAWRRHRACTDDSPHEGPFREDAPAWCVAPFEPEGILSSLSAVLSTV
BGIOSGA000194-P DRKVLGINHMYHRPAWRRHKDCTDDSPHEGPFKTDSPAWCYAPFEPEGLLSSLSAVLSTI
BGIOSGA000195-P DRKVLGINHMYHRPAWRRHKDCTDDSPHEGPFKTDSPAWCYAPFEPEGLLSSLSAVLSTI
Os01t0931100-02 DRKVLGINHMYHRPAWRRHKDCTDDSPHEGPFKTDSPAWCYAPFEPEGLLSSLSAVLSTI
VIT_14s0060g022 DREILGMNHMYQHPAWTRSKACNEYSPDKGPFRKDAPSWCYAPFEPEGILSSISAILSTI
                **.::*:**:*::**  * . *.  .* :*.::  :*:** *******:***:**:***:

AT5G27730.1     IGVHFGHIILHLKGHSARLK<span class="highlight">H</span><span class="highlight">W</span>ISTGLVLLALGLTLHFTHLMPLNKQLYSFSYICVTSGA
Bra020578.1-P   IGVHFGHTILHFKEHSARLK<span class="highlight">H</span><span class="highlight">W</span>ISTGLALLSLGLTLHFTHLMPLNKQLYTFSYTCVTSGA
AES82432        IGLHYGHVLIHLQDHLSRLKQWILLGLALLTLGFILHFSHVIPLNKQLYTLSYVCVTSGA
KRH68502        IGLHFGHVLIHLQDHPSRLK<span class="highlight">H</span><span class="highlight">W</span>LLLGLALLTSGLILHFTHAIPLNKQLYTLSYVCVTSGA
KRG96791        IGLHFGHVLIHLQDHPSRLK<span class="highlight">H</span><span class="highlight">W</span>LLLGLALLTSGLILHFTHAIPLNKQLYTLSYVCVTSGA
Zm00001d011929_ VGVHYGHVLVHMKSHTDRLRQWVTMGVALLVLGIILHFSHAIPLNKQLYTFSYICVTAGA
BGIOSGA000194-P IGVHYGHVLVHMKSHKDRLKQWSIMGITLLILGLTLHFSHAIPLNKQLYTFXYICVTAGA
BGIOSGA000195-P IGVHYGHVLVHMKSHTDRLKQWSIMGITLLILGLTLHFSHAIPLNKQLYTFSYICVTAGA
Os01t0931100-02 IGVHYGHVLVHMKSHTDRLKQWSIMGITLLILGLTLHFSHAIPLNKQLYTFSYICVTAGA
VIT_14s0060g022 IGVHFGHVLMHLKGHSDRLK<span class="highlight">H</span><span class="highlight">W</span>VVMGFALLVLGITLHFTGAIPLNKQLYTFSYVCVTSGA
                :*:*:** ::*:: *  **::*   *..**  *: ***:  :*******:: * ***:**

AT5G27730.1     AALVFSSLYSLVDILEWKHMFLPLKWIGMNAMLVYVMGAEGILAAFFNGWYYRHPHNTLI
Bra020578.1-P   AALVFSALYSLVDVLEWKHVFLPLEWIGMNAMLVYVMGAEGVLAAFFNGWYYRHPSNTLI
AES82432        AALVFSAFYVMVDIWGLKLLFIPFKWIGMNAMLVYVMAAEGIFAGFINGWYYDDPRNTLT
KRH68502        AALLFSAFYITVDIWGLTFLFLPLKWIGMNAMLVYVMAAEGIFAGFINGWYYGDPHNTLI
KRG96791        AALLFSAFYIMVDIWGLTFLFLPLKWIGMNAMLVYVMAAEGIFAGFINGWYYGDPHNTLV
Zm00001d011929_ AGVVFSALYFLVDVVSLRYAFAPLRWVGMNAMLVYVMAAEGVFEGFLNGWYYESTNNTLV
BGIOSGA000194-P AGIVFCMFYFLVDILNLHYPFAPLKWTGMNAMLVYVMAAAGIFEGFLNGWYYEGTNNTLV
BGIOSGA000195-P AGIVFCMXXXXXXXXXXXXXXXXXX-----------------------------------
Os01t0931100-02 AGIVFCMFYFLVDILNLHYPFAPLKWTGMNAMLVYVMAAAGIFEGFLNGWYYEGTNNTLV
VIT_14s0060g022 AALVFSFFYILVDVWGMRFLCLPLEWIGMNAMLVYVMAAEGVFAKFINGWYYGDPHNTLI
                *.::*.                                                      

AT5G27730.1     NWIREHVFIRV<span class="highlight">W</span><span class="highlight">H</span>SRRVGVLMYVIFAEILFWGLVTGVFHRFKIYWKL
Bra020578.1-P   NWIKEHVFVRV<span class="highlight">W</span><span class="highlight">H</span>SRRVGVLMYVIFGEILFWGLVTGVCHRFKIYWKL
AES82432        YWIQEHVFIRV<span class="highlight">W</span><span class="highlight">H</span>SRSAGILLYVIFAEILFWAVIAGILHQLGIYWKL
KRH68502        YWIQKHVFIKV<span class="highlight">W</span><span class="highlight">H</span>STRVGILLYVILAEILFWAVVAGILHRLGIYWKL
KRG96791        YWIQKHVFIKV<span class="highlight">W</span><span class="highlight">H</span>STRVGILLYVIFAEILFWAVVAGILHRLGIYWKL
Zm00001d011929_ YWVRKHVFVKV<span class="highlight">W</span><span class="highlight">H</span>STRVGILLYVLFAQILFWALVSGVLHRARLYWKL
BGIOSGA000194-P YWVRKHVFVKV<span class="highlight">W</span><span class="highlight">H</span>STRVGILLYVLFAQILFWALVAGLLHRARLYWKL
BGIOSGA000195-P -----------------------------------------------
Os01t0931100-02 YWVRKHVFVKV<span class="highlight">W</span><span class="highlight">H</span>STRVGILLYVLFAQILFWALVAGLLHRARLYWKL
VIT_14s0060g022 NWIQQHIFIQV<span class="highlight">W</span><span class="highlight">H</span>SRKVGILLYVIFAEILFWAIVAGMLHQLGL<span class="highlight">H</span><span class="highlight">W</span>KL
                                                               


<a href='WH_HW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            