
<!doctype html>
<html>
    <head>
        <title>WK-KW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT2G24680.6</h1>
        <h3>AT2G24680.6 pep chromosome:TAIR10:2:10493750:10496991:-1 gene:AT2G24680 transcript:AT2G24680.6 gene_biotype:protein_coding transcript_biotype:protein_coding description:transcriptional factor B3 family protein [Source:TAIR;Acc:AT2G24680]</h3>
        <pre>
length  621
count   8
ratio   0.00
zscore  6.76






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT2G24680.6     M---------------------VLNSSD--------LGPS-RCDI---------------
Bra000545.1-P   MTIVHVAEAKAACATAIACPIPVLEASTRNKPGEEGISPTQRCSLVQDPLPVTHTPTTRI
Zm00001d004358_ M-----------------------------------------------------------
                *                                                           

AT2G24680.6     ----------RDLPAPSSTNDQGKTELARKKKVK----RSNTEIEADASSSDNSCFVALV
Bra000545.1-P   RQTWVSYISRKKHPITTEEVDDGQDNTG-NKKVK----KNNSQAEADSSSSDNSCFVAPV
Zm00001d004358_ -------------PLPS---DSGSGARAMSKQFKVLLPTSFHKLVSTAFLDRRSISVADV
                             * .:   *.*.   . .*:.*     .  :  : :  . .*  ** *

AT2G24680.6     TASNLRKDALYLPQDLTSSVG-------LERKYREIVVTDERERRSWALDLRFNKSSDTF
Bra000545.1-P   TASNLEEDSLCLPPDFTISNG-------FKRKCRKIVLTDERE-RSWALDLRFNKSSDTF
Zm00001d004358_ SLSSRNP---LIPQRISDELAGCFDAGEGEGAPEPTALVVSPFGKVWRVEVVRDGHGAS-
                : *. .     :*  :: . .        :   .  .:. .   : * :::  :  . : 

AT2G24680.6     YISRGWRSFCDENGKKPGGVFVFKLVGNRETPVLSFCSTESIND-GTQGHKNNKYNCMEL
Bra000545.1-P   CITRGWRNFCEENGRKEGSLFKFKLMRNGETPLLSVCPTESISD-GTQGGDENIPQI---
Zm00001d004358_ -LGRGWAEFLAAHGVGIGWFVVLRYEGGGALTVKVFDTSLCIKEFGAPAAVMPSKSCKSI
                 : *** .*   :*   * .. ::   .   .:  . .: .*.: *: .      .    

AT2G24680.6     KSKKKRMRCRDSTSPSQNRFMTLTLTHDNLIKSRRQLPLPFMRE---NGLVKPGIINLLG
Bra000545.1-P   --------CRDSTLPNPNRFVTLTLTNDSLKSSRLYLPLPFLKE---NGMDKPGMVTLLG
Zm00001d004358_ FCKPQFIR----------------IFHSNL-SEKMMLPARFVKDYVSEECLNSRIAIIFS
                                        : :..* ..:  **  *:::   :   :. :  ::.

AT2G24680.6     KDGT<span class="highlight">K</span><span class="highlight">W</span>EVNLRREANGTMYLGKG<span class="highlight">W</span><span class="highlight">K</span>DFTIANGLKTGESFTLEAVLENGTRMLSLVSTQST
Bra000545.1-P   KDGTRMLANLLRENSGRMSLGKGLKGFARANGLNMGESFTLELIWENATPVLSLLSTEFR
Zm00001d004358_ PIAKFWRVELKNDQSGIFFTG-GWSQFLDFHGIKNGEVLLLKYEGNMVFKFKAFGLSGCQ
                  ..   .:* .: .* :  * * . *   :*:: ** : *:   :    . ::  :   

AT2G24680.6     SDRNQQGECLKDSEKESNSTEPSSGKRKAMINRGVRRKSFPASQNRCVTLKLTHDNLIKS
Bra000545.1-P   ------------------------------------------SQNRFVTLTLTQDSFKNS
Zm00001d004358_ KDFKNQNAGIQLNTKKQQETPPIR-KRKSNDEKS------SSEENKRLKTPVTS------
                                                          .:*: :.  :*       

AT2G24680.6     RRVSSSINPCFRGYVSFISCTHIRYVIRSFML-QYLPLS--FTRDNGLDKPGMIFLLGKT
Bra000545.1-P   R--------------------------------LGLPLP--FMRENGMNEPGTIALLGKD
Zm00001d004358_ ---PSPSDPSLEKPYQIGTSSWIKKRINTYALEQFLALSVKFCNSIGFRMACTITLKTAT
                                                   *.*.  * .. *:  .  * *    

AT2G24680.6     G--R<span class="highlight">K</span><span class="highlight">W</span>EANLLREASGRIVLTGKG<span class="highlight">W</span><span class="highlight">K</span>EFAMANGLKSGELFT---LEAILEKGTPMLSLVN
Bra000545.1-P   G--T<span class="highlight">K</span><span class="highlight">W</span>MTNLLRESQGRMSL-GKG<span class="highlight">W</span><span class="highlight">K</span>DFARGNGLEIGESFT---LESIWEDATPMLSL--
Zm00001d004358_ NSTRSWQVRGTAY<span class="highlight">K</span><span class="highlight">W</span>HRYII-GVG<span class="highlight">W</span><span class="highlight">K</span>SFCQDNRLKAGDLCTFNIIETTLWHVTITRSALA
                .   .* ..       *  : * ***.*.  * *: *:  *   :*:   . *   *   

AT2G24680.6     TQSTNYRSQQGECSRDSEKES-SICAEPSRGNK<span class="highlight">K</span><span class="highlight">W</span><span class="highlight">K</span>ATNNRKE-----------------
Bra000545.1-P   ------------SSRESKRETVHVSAEPTIGNETKKDENNIEE-----------------
Zm00001d004358_ ADTFN-KQKKPPCSSSRDKKG-SSSREGAKRPKPSMAHFSKVQVPPSYARRCAYDVGPPP
                            .* . .::    . * :   :      .  :                 

AT2G24680.6     ---RRDSSSAIQNRYVTLTLTPEDVRACTL-ILPSQFMKANGINKLGK---KTLLGQNRK
Bra000545.1-P   ------------------------------------------------------------
Zm00001d004358_ WIRKQMNASTIRNRLVRMVILAFDSRKLVIGCLAPAFCNQIELRRRCEIVLKTSINSNHS
                                                                            

AT2G24680.6     <span class="highlight">K</span><span class="highlight">W</span>FAYLLSKSGFVALGSG<span class="highlight">W</span><span class="highlight">K</span>GFCEANGVKTGESFNLEYIDEQDTTPVFKFCSNSVEYVKF
Bra000545.1-P   ------------------------------------------------------------
Zm00001d004358_ WRVGGLSQKKGSYIIGPG<span class="highlight">W</span><span class="highlight">K</span>KFCTENRVKVGDVCTFNVIETQ------------LWHVVI
                                                                            

AT2G24680.6     TSLP
Bra000545.1-P   ----
Zm00001d004358_ TRRC
                    


<a href='WK_KW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            