
<!doctype html>
<html>
    <head>
        <title>WK-KW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT1G23480.2</h1>
        <h3>AT1G23480.2 pep chromosome:TAIR10:1:8333472:8336414:1 gene:AT1G23480 transcript:AT1G23480.2 gene_biotype:protein_coding transcript_biotype:protein_coding gene_symbol:CSLA3 description:Glycosyltransferase (Fragment) [Source:UniProtKB/TrEMBL;Acc:W8Q7B4]</h3>
        <pre>
length  556
count   8
ratio   0.00
zscore  6.76






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT1G23480.2     MSPFLKFFLFLYDYLSPSSFFLVQRNTLGASLDTTDGVVRSGIIGEIIYI<span class="highlight">W</span><span class="highlight">K</span>QTRIFVFI
Bra024603.1-P   MSPFMKFFNFLYDSLPPSSWFLVQRHNL-ASSSETDEVVRSGIIGELIYI<span class="highlight">W</span><span class="highlight">K</span>ETRIFVFI
Pp3c14_12270V3. MA------------------------------DSFSEGTFPVLVDQLINLWLSVRGPIVA
Pp3c14_13750V3. MA------------------------------ESARGGTFSVLVDQLLQLWLDVRSPIVA
Pp3c17_19630V3. MA------------------------------ESGSTGGLPMLVDQLVKIWLEVRGPVVA
Pp3c17_19670V3. MA------------------------------ESGSTGGLPMLVDQLVKIWLEVRGPVVA
                *:                              .       . ::.::: :* ..*  :. 

AT1G23480.2     PILKCLVTICLVMSLLLFIERVYMSIVVVFVKLLRRTPEKVH<span class="highlight">K</span><span class="highlight">W</span>EPINDDDLELANTNYP
Bra024603.1-P   PILKFLVTMCLVMSFLLFIERVYMSVVVVIVKLLRRTPEKVH<span class="highlight">K</span><span class="highlight">W</span>EAINDDDLELAHTNYP
Pp3c14_12270V3. PILQFAINVCLVMVTMLFIERIFMCGVMIFVKLLGRTPETQFKFEPIG-DDLELANTAFP
Pp3c14_13750V3. PLLQFAINVCLVMVTMLFVERIFMCGVMVFVKLLRRTPETQFKFEPLR-DDLEFGNSSFP
Pp3c17_19630V3. PVLQFAINVCLVMVTMLFVERIFMCGVMVFVKLLRRTPETQFKFEAIQ-DDLEFGNSSYP
Pp3c17_19670V3. PVLQFAINVCLVMVTMLFVERIFMCGVMVFVKLLRRTPETQFKFEAIQ-DDLEFGNSSYP
                *:*:  :.:****  :**:**::*. *:::**** ****. .*:*.:  ****:.:: :*

AT1G23480.2     MVLIQIPMYNEKEVCQLSIGAACRLSWPLDRMIVQVLDDSTDPASKELVNAECD<span class="highlight">K</span><span class="highlight">W</span>ARKG
Bra024603.1-P   MVLIQIPMYNEKEVCQLSIGAACRLSWPLDRMIVQVLDDSTDPASKELVNAECD<span class="highlight">K</span><span class="highlight">W</span>ARKG
Pp3c14_12270V3. MVLVQIPMFNEREVYQLSIQAACGLSWPGDRMIIQVLDDSTDQTIRELVQMECQRWASKG
Pp3c14_13750V3. MVLVQIPMFNEREVYQLSIQAACGLSWPGDRMIIQVLDDSTDQTTRDLVQMECQRWASKG
Pp3c17_19630V3. MVLVQIPMFNEREVYQLSIQAACGLSWPQDRMIIQVLDDSTDQTTRELVQVEVQRWASKG
Pp3c17_19670V3. MVLVQIPMFNEREVYQLSIQAACGLSWPQDRMIIQVLDDSTDQTTRELVQVEVQRWASKG
                ***:****:**:** **** *** **** ****:******** : ::**: * ::** **

AT1G23480.2     INIMSEIRDNRIGYKAGALKAGMMHNYVKQCEFVAIFDADFQPDPDFLERTIPFLIHNHE
Bra024603.1-P   INIMSEIRDNRIGYKAGALKAGMMHNYVKQCEFVAIFDADFQPDPDFLARTIPFLIHNRE
Pp3c14_12270V3. INIKYETRPNRKGYKAGALRQGMKHPYVSMCDYVAIFDADFQPEPEFLHRTVPFLVHNSN
Pp3c14_13750V3. INIRYETRPNRKGYKAGALRQGM<span class="highlight">K</span><span class="highlight">W</span>PYVQTCDYVAIFDADFQPEPEFLQRTVPFLVHNSN
Pp3c17_19630V3. INIKYETRPNRKGYKAGALRQGMRHPYVQTCDYVAIFDADFQPEPEFLQRTVPFLVHNSN
Pp3c17_19670V3. INIKYETRPNRKGYKAGALRQGMRHPYVQTCDYVAIFDADFQPEPEFLQRTVPFLVHNSN
                ***  * * ** *******: **   **. *::**********:*:** **:***:** :

AT1G23480.2     ISLVQCR<span class="highlight">W</span><span class="highlight">K</span>FVNANECLMTRMQEMSLNYHFVAEQESGSSIHAFFGFNGTAGVWRIAALNE
Bra024603.1-P   ISLVQCR<span class="highlight">W</span><span class="highlight">K</span>FVNANECLMTRMQEMSLNYHFLAEQESGSSIHAFFGFNGTAGVWRIAALNE
Pp3c14_12270V3. LALVQAR<span class="highlight">W</span><span class="highlight">K</span>FVNANECLMTKMQEVSLNYHFSVEQRVGSATYGFFGFNGTAGVWRIRAMEE
Pp3c14_13750V3. LALVQAR<span class="highlight">W</span><span class="highlight">K</span>FVNANECLMTKMQEVSLNYHFSVEQRVGSATYGFFGFNGTAGVWRIRAMEE
Pp3c17_19630V3. LALVQAR<span class="highlight">W</span><span class="highlight">K</span>FVNANECLMTKMQEVSLNYHFSVEQRVGSATYGFFGFNGTAGVWRIRAMEE
Pp3c17_19670V3. LALVQAR<span class="highlight">W</span><span class="highlight">K</span>FVNANECLMTKMQEVSLNYHFSVEQRVGSATYGFFGFNGTAGVWRIRAMEE
                ::***.*************:***:****** .**. **: :.************* *::*

AT1G23480.2     AGG<span class="highlight">W</span><span class="highlight">K</span>DRTTVEDMDLAVRACLHG<span class="highlight">W</span><span class="highlight">K</span>FVYVHDVEVKNELPSTFKAYRFQQHRWSCGPANLW
Bra024603.1-P   AGG<span class="highlight">W</span><span class="highlight">K</span>DRTTVEDMDLAVRACLHG<span class="highlight">W</span><span class="highlight">K</span>FVYVHNVEVKNELPSTFKAYRFQQHRWSCGPANLW
Pp3c14_12270V3. AGGWNDRTTVEDMDLAVRASMCG<span class="highlight">W</span><span class="highlight">K</span>FVYIHDLEVKNELPSTFKAFRFQQHRWSCGPANLF
Pp3c14_13750V3. AGGWNDRTTVEDMDLAVRASLCG<span class="highlight">W</span><span class="highlight">K</span>FVYIHDLEVKNELPSTFKAFRFQQHRWSCGPANLF
Pp3c17_19630V3. AGGWNDRTTVEDMDLAVRASLCG<span class="highlight">W</span><span class="highlight">K</span>FVYIHDLEVKNELPSTFKAFRFQQHRWSCGPANLF
Pp3c17_19670V3. AGGWNDRTTVEDMDLAVRASLCG<span class="highlight">W</span><span class="highlight">K</span>FVYIHDLEVKNELPSTFKAFRFQQHRWSCGPANLF
                ****:**************.: ******:*::************:**************:

AT1G23480.2     RKMTMEILQNKKVSA<span class="highlight">W</span><span class="highlight">K</span>KLYLIYNFFFIRKIVVHIFTFVFYCLILPTTVLFPELQVP<span class="highlight">K</span><span class="highlight">W</span>A
Bra024603.1-P   RKMTMEILQNKKVSA<span class="highlight">W</span><span class="highlight">K</span>KLYLIYNFFFIRKIVVHIFTFVFYCLILPATVLFPELQVP<span class="highlight">K</span><span class="highlight">W</span>A
Pp3c14_12270V3. RKVLPSILKNQKLKL<span class="highlight">W</span><span class="highlight">K</span>RLHMIYAFFFVRKIVAHIVTFTFYCLVIPASVLVPEVDLPFWG
Pp3c14_13750V3. RKVLPSILKNQNLKL<span class="highlight">W</span><span class="highlight">K</span>RLHMIYAFFFVRKVVAHIVTFTFYCVVIPASVLVPEVDLPFWG
Pp3c17_19630V3. RKVLFTILKNQNLRL<span class="highlight">W</span><span class="highlight">K</span>RLHMIYAFFFVRKIVAHIVTFTFYCVVIPASVLVPEVDLPFWG
Pp3c17_19670V3. RKVLFTILKNQNLRL<span class="highlight">W</span><span class="highlight">K</span>RLHMIYAFFFVRKIVAHIVTFTFYCVVIPASVLVPEVDLPFWG
                **:   **:*:::  **:*::** ***:**:*.**.**.***:::*::**.**:::* *.

AT1G23480.2     TVYFPTTITILNAIATPRSLHLLVFWILFENVMSMHRTKATFIGLLEAGRVNEWVVTEKL
Bra024603.1-P   TVYVPSTITLLNALATPRSLHLLVFWILFENVMSMHRTKATFIGLLEAGRVNEWVVTEKL
Pp3c14_12270V3. AVYVPSIITLLNAITTPKSLHLLVFWILFENVMSLHRTKATIIGLFDIGNVNEWVVTEKL
Pp3c14_13750V3. AVYVPSIITLLNAITTPKSLHLLVFWILFENVMSLHRTKATIIGLFDIGNVNEWVVTEKL
Pp3c17_19630V3. AVYVPSIITLLNAITTPKSLHLLVFWILFENVMSLHRTKATIIGLFDIGNVNEWVVTEKL
Pp3c17_19670V3. AVYVPSIITLLNAITTPKSLHLLVFWILFENVMSLHRTKATIIGLFDIGNVNEWVVTEKL
                :**.*: **:***::**:****************:******:***:: *.**********

AT1G23480.2     GDTLKSKLIGKATTKLYT-------------RFGQRLNWRELVVGLYIFFCGCYDFAY-G
Bra024603.1-P   GDTLKSKLITKATTKLYT-------------RFGQRINWRELVVGLYIFICGCYDFAY-G
Pp3c14_12270V3. GNLMKYR--SAKYGKKYTLQRMASNVLARGWRMSERMHILELWTGGFLMFCAAYDFYFQG
Pp3c14_13750V3. GNLMKYR--SSKYGKKYTLQRMASNMLARG<span class="highlight">W</span><span class="highlight">K</span>MSERMHLLEVWTGGFLMFCAVYDFYFQG
Pp3c17_19630V3. GNLMKYR--SAKYGKKYTLQRMASNMLARG<span class="highlight">W</span><span class="highlight">K</span>MSERLHILELWTGGFLMFCAVYDFYFQG
Pp3c17_19670V3. GNLMKYR--SAKYGKKYTLQRMASNMLARG<span class="highlight">W</span><span class="highlight">K</span>MSERLHILELWTGGFLMFCAVYDFYFQG
                *: :* :       * **             ::.:*::  *: .* ::::*. *** : *

AT1G23480.2     GSYFYVYLFLQSCAFFVAGVGYIGTFVP-TV
Bra024603.1-P   GSYFYVYLFLQACAFFVAGVGYIGTIVP-IG
Pp3c14_12270V3. KNHFYVYLFLQSFAFIIMGLGYVGTFVPQYS
Pp3c14_13750V3. KNHFYVYLFLQSCAFLVMGFGYVGTFVPHYS
Pp3c17_19630V3. KNHFYVYLFLQSCAFLIMGFGYVGTFVPHYS
Pp3c17_19670V3. KNHFYVYLFLQSCAFLIMGFGYVGTFVPHYS
                 .:********: **:: *.**:**:**   


<a href='WK_KW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            