
<!doctype html>
<html>
    <head>
        <title>WK-KW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT2G24690.2</h1>
        <h3>AT2G24690.2 pep chromosome:TAIR10:2:10498972:10503188:-1 gene:AT2G24690 transcript:AT2G24690.2 gene_biotype:protein_coding transcript_biotype:protein_coding description:Transcriptional factor B3 family protein [Source:TAIR;Acc:AT2G24690]</h3>
        <pre>
length  832
count   9
ratio   0.00
zscore  7.68






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT2G24690.2     M-----------------------------------------------------------
Bra000564.1-P   MAKSTTLLSPQFFHTLLPGFQTHLTIPSDFFSNYIKDLETTTAELKSDSSLDIT<span class="highlight">W</span><span class="highlight">K</span>VKLT
Bra000563.1-P   ------------------------------------------------------------
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT2G24690.2     -------------ANSLILGSPI-------------------------------------
Bra000564.1-P   GRILNDGWGDFAVANNLHIGNVVLVRYEGGIVFHVSDLGPCFSQIQHINPPRQNI-----
Bra000563.1-P   ------------------------------MMFQVSDVGHRDVETPSSNNGQDNIGSMNK
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT2G24690.2     -NPEGEMMIQFSNNHDNNIEFQRRKKMKKNNPETETDSSSDLSCFVAQ--TGALLH----
Bra000564.1-P   ----------------DEEDDVERER-KGDESLIKKKLKP--------------------
Bra000563.1-P   KHPYARQEAGFSSNHGDNQDNDELPR-KRNYSEAEKGSSSKHSCIVADATTASSLHTYTP
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT2G24690.2     --LPQGFTSSNGSNRECRK--SPRPI-ISSYSSSEKQFVTFTLAPVDGRHCRLRLPMQFT
Bra000564.1-P   ---------------------KQRPVSYSSYSPCHKRFVTFTLPPEYATLCRLTLPNLFV
Bra000563.1-P   VFLTQDFTSLGGLKGKCDKKCSPKRI-ISSMGT---RFVTFTLAPVDVRHCRLRLPMQFS
Zm00001d004358_ ----------------------------------------------------MPLP----
                                                                    : **    

AT2G24690.2     RENGINKPGKIYLVGKDGS<span class="highlight">K</span><span class="highlight">W</span>LANLLLENNRGRMTLGDG<span class="highlight">W</span><span class="highlight">K</span>SFVKANGLKTGDTYTFKLL
Bra000564.1-P   KENGISKPAEIRVLGKDGT<span class="highlight">K</span><span class="highlight">W</span>PITLLLD-KKGIMSFGKG<span class="highlight">W</span><span class="highlight">K</span>EFVKANGLETG--FTLKLM
Bra000563.1-P   RENGINKPGKIYLLGNDGS<span class="highlight">K</span><span class="highlight">W</span>LANLLQE-SRGRMTVGDG<span class="highlight">W</span><span class="highlight">K</span>SFVKANGLKIGESYTLELN
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT2G24690.2     WEDTTPVLSLCFEEYNT-----DTRVGEESSKESLPAEPSSQEKIVKDDNNKDESST<span class="highlight">W</span><span class="highlight">K</span>R
Bra000564.1-P   WEDTTPSFSLCCPESAS--DRDEKECLESIKKQSLSIDRRIRDKPGKDDNNKEEKRSWER
Bra000563.1-P   WEDTTPVLSLCPTDHHSIGIRAEGECSEASGNESLPIRPSSGTEVIKYDNRKDESSSWER
Zm00001d004358_ ------------SDSGS-GAR-------AMSKQFKVLLPTSFHKLV---------STAFL
                             :  :              ::          :            :   

AT2G24690.2     EGNHLRCKD----------------------STSPSQNCTLTVTITPDSLEHGRLRLPLQ
Bra000564.1-P   QKIHLRGRDVGDFEEVPINKKVKTDSHGTEAAFSSLDNSCFVAFVTDSSLDSDTLYLPRY
Bra000563.1-P   EKNHLRSRD----------------------SSSASQNRFLTLTITPDSLRHGRLRLPLE
Zm00001d004358_ DRRSISVAD----------------------VSLSSRNPLIPQRISDE--------LAGC
                :   :   *                         .  *  :   :: .        *.  

AT2G24690.2     FMTENSMNKP-GEITLLGTD-GA<span class="highlight">K</span><span class="highlight">W</span>MASLLL-EKKGRMSLGKG<span class="highlight">W</span><span class="highlight">K</span>DFAKANGLKTGDSIT
Bra000564.1-P   FTSSNGLTRKCRKIVLIDGG-GRSWAMDLSFNESSDIFYISPGWRSFCDKNGLQAGGFFT
Bra000563.1-P   FMTENSMNRP-GDITLLGKD-GA<span class="highlight">K</span><span class="highlight">W</span>LASLLL-ERRGRMCLGKG<span class="highlight">W</span><span class="highlight">K</span>DFAKANGLKTGDSIT
Zm00001d004358_ FDAGEGEGAP-EPTALVVSPFGKVWRVEVVR-DGHG-ASLGRGWAEFLAAHGVGIGWFVV
                * : :.        .*:    *  *  .:   :  .   :. ** .*   :*:  *  ..

AT2G24690.2     LEPIWEDRTPVLSI--------KSSSGKGQSEFSKESLSIKPSSGNMTKKVENNREASRK
Bra000564.1-P   FKLVGNEETPVLSFCPMESIDSRSQK-DCFEAHSTEV---------------SNEEE---
Bra000563.1-P   LESIWEDSTPVLRLLHIESSDDRRQQGECSKARDKESAFTEPSSGNKTRKLESNREESRI
Zm00001d004358_ LRY---EGGGALTVKVFDTSLCIKEFGAPAAVMPSKSCKSIFCKPQFIRIFHSNLSEKMM
                :.    :   .* .          .         .:                .* .    

AT2G24690.2     YPPRSRE----------------SSSAIQNQFMALTPLRDIVSQVA-------HDLSIGE
Bra000564.1-P   -----------------------------------------------------------E
Bra000563.1-P   YPLTRLD----------------SSLVNQNRFEISTLSPEIVRHGA-------HDYQIGE
Zm00001d004358_ LPARFVKDYVSEECLNSRIAIIFSPIAKFWRVELKNDQSGIFFTGGWSQFLDFHGIKNGE
                                                                           *

AT2G24690.2     VINFRHKGDNMLRVSDLGSNCCGVQDLLAPSSNYDHDNISNISMKINPHIRKEAVTFSSY
Bra000564.1-P   IERERNEEETLMDIEE--------------------------------------------
Bra000563.1-P   IVVFRDERDMMFQVSDLGPRFSVIQNVLAPSSNSDQENI---------------------
Zm00001d004358_ VLLLKYEGNMVFKFKAFGLSGC-------------QKDFKNQNAGIQLNTKKQQ------
                :   : : : :: ..                                             

AT2G24690.2     DGYAHDNFEPPSKKKVKKNNPEMEADYLSDHSCFVSHVTTSSLHTNALSLEV--------
Bra000564.1-P   ----------------KKSKPEQRPVS---------------------------------
Bra000563.1-P   --------ELPRKKKAKKNNPGTEAYSSPERSCFVAHVTASNLHTDAPHLPQDFTSSNSP
Zm00001d004358_ --------ETPPIRK-RKSNDE----------------KSSSEENKRLKTPV--------
                                :*.:                                        

AT2G24690.2     ------------------------------------------------------------
Bra000564.1-P   ---YSSYSPCYKRFITFTLPPDYATLDKLTLPKLFVKENGFNKPGEIYLLGKDGT<span class="highlight">K</span><span class="highlight">W</span>LTR
Bra000563.1-P   TQIVSSYLSSEKQVATFTLAPVDVRNCRLRLPMQFTRENGINKPGKIYLLGKDGS<span class="highlight">K</span><span class="highlight">W</span>LAN
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT2G24690.2     ------------------------------------------------------------
Bra000564.1-P   LLLDKKGLMRLGKG<span class="highlight">W</span><span class="highlight">K</span>DFVKANGLESG--FTLKLIWEDATPSFSLCCAESTSDKNE-EEY
Bra000563.1-P   LLLESRGRMTLGDG<span class="highlight">W</span><span class="highlight">K</span>SFVKANGLKTGESFTLKLNWEDTTPVLSLCPPEYSIDSREGGEC
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT2G24690.2     -----KESLPIEPSIKPNISEDDNNKEEDIIEENSSFEREKNHWICIDSISSSQNRFLTL
Bra000564.1-P   LETIMKQSLSIDRRIRDKISKDENDK-----EEKRSWEREKINMRGRDSTTSRQKQFLTL
Bra000563.1-P   SETIEKEPLPIVLSCGKEINKEDENRK----RESRSWERETNLLIWRDSTVLSQNRCLTL
Zm00001d004358_ -----TSPSPSDPSLE------------------KPYQIGTSSWIKKRINTYALEQFLAL
                     ... .                         .::  .             :: *:*

AT2G24690.2     TITPDSLKHGRLRLPLQFMIENSMNKPGEITVLGKDGA<span class="highlight">K</span><span class="highlight">W</span>LVSLLLERRGRMSLGKG<span class="highlight">W</span><span class="highlight">K</span>D
Bra000564.1-P   AITPTSLRCNRMRLPIPFLRKSCMDKPGVIYLLGKDDT<span class="highlight">K</span><span class="highlight">W</span>MANLIQEGDGRMKLGKGWTA
Bra000563.1-P   TITPDSLKHGRLRLPLEFMTDNSMNKPGEITLLGKDGV<span class="highlight">K</span><span class="highlight">W</span>LVSLLLEKRGRMSLGKG<span class="highlight">W</span><span class="highlight">K</span>D
Zm00001d004358_ SVK--FCNSIGFRMACTITLKTATNSTRSWQVRGT-AY<span class="highlight">K</span><span class="highlight">W</span>---------HRYIIGVG<span class="highlight">W</span><span class="highlight">K</span>S
                ::.    .   :*:.  :  ... :..    : *.   **          *  :* **. 

AT2G24690.2     FAKANGLKTGDSITLESVWEGASPVLKLLRIESSSFRGQSEFSKESHFTDASS----GQE
Bra000564.1-P   FAKENEFKAGESVTLESIWEDETPMIRFLRTESES-----------------S-------
Bra000563.1-P   FAKANGLKTGDSITLESIWENATPVLSLLRVESSN---DSEVSKQ-------S----GNK
Zm00001d004358_ FCQDNRLKAGDLCTFNII----ETTLWHVTITRSALAADT-FNKQKKPPCSSSRDKKGSS
                *.: * :*:*:  *:: :     . :  :    .                  *       

AT2G24690.2     TREAEKNREESSSLWELEKRTYCP----------------RKRDSSSDVKNRFLTLT---
Bra000564.1-P   ----KANKKESIYMEDTES---------------------RTRDSSTEIHARFVTLT---
Bra000563.1-P   TRK-EENR---SSSWELEK---------------------RRSSSSSEIQNRTVILT---
Zm00001d004358_ SR--EGAKRPKPSMAHFSKVQVPPSYARRCAYDVGPPPWIRKQMNASTIRNRLVRMVILA
                    :  :       . ..                     *   .:: :: * : :.   

AT2G24690.2     ----------LAPEDVKDGNLHLPCQFMRINGINKPGQITLLGRGGM<span class="highlight">K</span><span class="highlight">W</span>FAYLLSGDGTV
Bra000564.1-P   ----------LKHEDVKACMLILPSQFLKANGINKLGKITLLDENEVELSAYLLSREGIV
Bra000563.1-P   ----------LTPEGVRDCKLQLPSQFMRSNGIMNPGKITLLGRSGM<span class="highlight">K</span><span class="highlight">W</span>FAYLLSKDGTI
Zm00001d004358_ FDSRKLVIGCLAPAFCNQIELRRRCEIVLKTSINSNHSWRVGG---------LSQKKGSY
                          *     .   *   .:::  ..* .  .  : .         * . .*  

AT2G24690.2     VVGNG<span class="highlight">W</span><span class="highlight">K</span>G----------------------------------------------------
Bra000564.1-P   AVESGWESQSQKKRSTSLSCVLSLSNSTQTYKKTVTTSLPQSSSTKMVQSAMEKHTLSMP
Bra000563.1-P   ALENG<span class="highlight">W</span><span class="highlight">K</span>G----------------------------------------------------
Zm00001d004358_ IIGPG<span class="highlight">W</span><span class="highlight">K</span>K----------------------------------------------------
                 :  **:                                                     

AT2G24690.2     ------------------------------------------------------------
Bra000564.1-P   DWSLLPEELLHMISEMLVRIRLSLSFLPITPRESEDNMALPYRFQCSVKVELLFIVLNFQ
Bra000563.1-P   ------------------------------------------------------------
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT2G24690.2     -----FCEANGV--MLGETFVL-EFIPKDDTNHVFK-----------FYTK---------
Bra000564.1-P   NLPKSFVRANGLETRCGDIVLMNEKVPMDSRLEISSLSNSSKEEEHCFYVSLPPTENPKK
Bra000563.1-P   -----FCEANGV--MVGESFVL-ELIPTQDADHVFK-----------FYSNYG-------
Zm00001d004358_ -----FCTENRV--KVGDVCTF-NVIETQLWHVV--------------------------
                     *   * :    *:   : : :  :    :                          

AT2G24690.2     --------
Bra000564.1-P   MRAQKLKK
Bra000563.1-P   -YHIHNKC
Zm00001d004358_ ---ITRRC
                        


<a href='WK_KW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            