
<!doctype html>
<html>
    <head>
        <title>WN-NW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT1G16480.1</h1>
        <h3>AT1G16480.1 pep chromosome:TAIR10:1:5625702:5629013:-1 gene:AT1G16480 transcript:AT1G16480.1 gene_biotype:protein_coding transcript_biotype:protein_coding description:Tetratricopeptide repeat (TPR)-like superfamily protein [Source:UniProtKB/TrEMBL;Acc:F4I4G1]</h3>
        <pre>
length  937
count   7
ratio   0.00
zscore  7.37






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT1G16480.1     ------------------------------------------------------------
Bra016637.1-P   MAKPSRGRRSPSVSGSSSRSSSSSRSRSSGSRSLSRSRSLSSSSRSVSSGSRSPPPRGKR
AES69403        MN----------------------------------------------------------
KRH55426        ------------------------------------------------------------
KRH26029        M-----------------------------------------------------------
Zm00001d030143_ ------------------------------------------------------------
BGIOSGA035691-P ------------------------------------------------------------
VIT_19s0085g002 MK----------------------------------------------------------
                                                                            

AT1G16480.1     ------------------------------------------------------------
Bra016637.1-P   KAAPIQESLVLHVDSLSRNVNEGHLKEIF----GNYGEVVHVELAMDRAVNLPKGYAYVE
AES69403        -------VLLCYINTLIYDISVVEVGQHFYWQIGDFQ--VHAQ-----------GLEFVE
KRH55426        ------------------------------------------------------------
KRH26029        ------------------------------------------------------------
Zm00001d030143_ ------------------------------------------------------------
BGIOSGA035691-P ------------------------------------------------------------
VIT_19s0085g002 ------------------------TKTHF-------------------------------
                                                                            

AT1G16480.1     ------------------------------------------------------------
Bra016637.1-P   FKARADAEKALLFMDGGQIDGNVVKAKFTLPSRQKLSPPPKPVSTAPKREAPKPDNAAAG
AES69403        FEAR---------------NGNT-------------------------------------
KRH55426        ------------------------------------------------------------
KRH26029        ------------------------------------------------------------
Zm00001d030143_ ------------------------------------------------------------
BGIOSGA035691-P ------------------------------------------------------------
VIT_19s0085g002 ------------------------------------------------------------
                                                                            

AT1G16480.1     ------------------------------------------------------------
Bra016637.1-P   IEKDRPMRSRETSPRRRSPLPRRLPDSPPRRRPSPPIRRRGDTPPRRRAASPPRGRSPSS
AES69403        -----------------------------------------------RIAS---------
KRH55426        ------------------------------------------------------------
KRH26029        ------------------------------------------------------------
Zm00001d030143_ ------------------------------------------------------------
BGIOSGA035691-P ------------------------------------------------------------
VIT_19s0085g002 ------------------------------------------------------------
                                                                            

AT1G16480.1     ------------------------------------------------------------
Bra016637.1-P   PPPRRQRSPLRGSPRRIRGSPVRRRSPPPLRRRSPPRRLRSPLRRSPIRRRSRSPIRRPG
AES69403        ---RKRNCGLKNTFFAV------------------------------LSHKCRLFVL---
KRH55426        ------------------------------------------------------------
KRH26029        ------------------------------------------------------------
Zm00001d030143_ ------------------------------------------------------------
BGIOSGA035691-P ------------------------------------------------------------
VIT_19s0085g002 ------------------------------------------------------------
                                                                            

AT1G16480.1     ------------------------------------------------------------
Bra016637.1-P   RSRSRSISPRRGRGPAGRRGRSSSSYSSSPSPRRIPRKISRSRSPKRSC-----LRIQKK
AES69403        ----SCLLPWLG--------------------------LLQLLELARNCRTLCTLNVVHA
KRH55426        ------------------------------------------------------------
KRH26029        ------------------------------------------------------------
Zm00001d030143_ ------------------------------------------------------------
BGIOSGA035691-P ------------------------------------------------------------
VIT_19s0085g002 -----------------------------------------LFSVSRSV-----FNTIPK
                                                                            

AT1G16480.1     ------------------------------------------------------------
Bra016637.1-P   LKVGELATCHRKLLSRHGGESRLLRNLSSRRFK--------------TRRFSTLQCLK-N
AES69403        PPTPY---------------SPLLN------------------------------CQN-Q
KRH55426        ------------------------------------------------------------
KRH26029        ------------------------------------------------------------
Zm00001d030143_ ------------------------------------------------------------
BGIOSGA035691-P ------------------------------------------------------------
VIT_19s0085g002 TPVGK--TRQWAPVSTITTASALINETPVENFAEQVKDDDLKTSNAGSRRWG---CLDGD
                                                                            

AT1G16480.1     ------------------------------------------------------------
Bra016637.1-P   VEEEGEGRKVTSLSGLFSLSDHWSPEVSSFHENGFSQISKETTGRAVHALCVKGLVRTSV
AES69403        LETCVKENEFLSYGIHTFIRNHSNPQVSRFLQKGFSEISEGNVGKALHALCVKDVIQQNT
KRH55426        ------------------------------------------------------------
KRH26029        -------------------------------------VPIGTPTPKFHAFLKRGVIHLGT
Zm00001d030143_ ------------------------------------------------------------
BGIOSGA035691-P ------------------------------------------------------------
VIT_19s0085g002 IAKVFLQQQHTDYGIRCL-------NAVNFPLKGFSEITSQMAGKALHAFCIVGSVNLGI
                                                                            

AT1G16480.1     --------MYTKFGRVKPARHLFDIMPVRNEVS<span class="highlight">W</span><span class="highlight">N</span>TMMSGIVRVGLYLEGMEFFRKMCDL
Bra016637.1-P   LHINTLINMYTKFGRVKPARHLFDEMPVRNEAS<span class="highlight">W</span><span class="highlight">N</span>TMMSGLVRVGMYREGVGFFKEMCGL
AES69403        FYTNTLVNMYSKFGSIKYAQHVFDKMYDRNDAS<span class="highlight">W</span><span class="highlight">N</span>NMISGFVRVGWYHKAMQFFCHMFEN
KRH55426        ------------------------------------------------------------
KRH26029        FQANTLISMYSKFGSIEHAQHVFDKMPERNEAS<span class="highlight">W</span><span class="highlight">N</span>NLMSGFVRVGWYQKAMQFFCHMLEH
Zm00001d030143_ -------------------------MPHRTSSSWYTAVSGCARCGLESTAFTLLRVMRER
BGIOSGA035691-P -------------------------MPDRTPSTWYTAVSGCVRCGRDVAAFELLRGMRER
VIT_19s0085g002 FQTNTLINMYSKFGNIEHARYVFDEMRHRNEASWSTMLSGYVRVGLYEEAVGLFCQMWGL
                                                                            

AT1G16480.1     GIKPSSFVIASLVTACGRSGSM--FREGVQVHGFVAKSGLLSDVYVSTAILHLYGVYGLV
Bra016637.1-P   GVRPSGFVIASLVTACGRGGCM--FSEGVQVHGFVAKSGLMSDVYVSTAVLHLYGVYGLV
AES69403        GVTPSSYVIASMVTACDRSGCM--TEGARQIHGYVVKCGLMSNVFVGTSLLHFYGTHGSV
KRH55426        ------------------------------------------------------------
KRH26029        GVRPSSYVAASLVTACDRSGCM--TEGAFQVHAHVIKCGLACDVFVGTSLLHFYGTFGWV
Zm00001d030143_ DVPLSGFALASLVTACEHRGWQEGAACGAAIHALTHRAGLMGNVYIGTALLHLYGSRGLV
BGIOSGA035691-P GVPLSGFALASLVTACERRGRDEGIACGAAIHALTHRAGLMGNVYIGTALLHLYGSRGIV
VIT_19s0085g002 GVEPNGFMVASLITACSRSGYM--ADEGFQVHGFVVKTGILGDVYVGTALVHFYGSIGLV
                                                                            

AT1G16480.1     SCSRKVFEEMPDRNVVSWTSLMVGYSDKGEPEEVIDIYKGMRGEGVGCNENSMSLVISSC
Bra016637.1-P   SCSRKVFEEMPVRNVVSWTSLMVGYSDKGEAEEVIGIYKGMRGEGVGCNENSMSLVISSC
AES69403        SEANKLFEEIEEPNIVSWTSLMVCYADNGHTKEVLNIYRHLRHNGLICTGNTMATVIRTC
KRH55426        ------------------------------------------------------------
KRH26029        AEVDMVFKEIEEPNIVSWTSLMVGYAYNGCVKEVMSVYRRLRRDGVYCNENAMATVIRSC
Zm00001d030143_ LNAQRLFWEMPQRNVVSWTAIMVALSSNGCMEEALVAYRRMRKEGVMCNANALATVVSLC
BGIOSGA035691-P SDARRLFWEMPERNVVSWTALMVALSSNGYLEETLRAYRQMRREGVPCNANAFATVVSLC
VIT_19s0085g002 YNAQKLFEEMPDHNVVSWTSLMVGYSDSGNPGEVLNVYQRMRQEGVSGNQNTFATVTSSC
                                                                            

AT1G16480.1     GLLKDESLGRQIIGQVVKSGLES-KLAVENSLISMLGSMGNVDYANYIFDQMSERDTISW
Bra016637.1-P   GLLRDESLGCQVIGQVIKSGLER-KLAVENSLVSMFGNVGKVDCAKYIFDQMSERDTISW
AES69403        GMFGDKTMGYQILGDVIKSGLDTSSVSVANSLISMFGNYDSVEEASRVFNNMQERDTISW
KRH55426        ------------------------------------------------------------
KRH26029        GVLVDKMLGYQVLGSVIKSGLDT-TVSVANSLISMFGNCDSIEEASCVFDDMKERDTISW
Zm00001d030143_ GALEDEVAGLQVTAHVVVSGLLT-HVSVANSLITMFGNLRRVQDAERLFDRMEERDRISW
BGIOSGA035691-P GSLENEVPGLQVASHVIVSGLQN-QVSVANSLITMFGNLGRVQDAEKLFDRMEEHDTISR
VIT_19s0085g002 GLLEDQVLGYQVLGHIIQYGFED-SVSVANSLISMFSSFSSVEEACYVFDHMNECDIISW
                                                                            

AT1G16480.1     NSIAAAYAQNGHIEESFRIFSLMRRFHDEVNSTTVSTLLSVLGHVDHQKWGRGIHGLVVK
Bra016637.1-P   NSIAAAYAQNGHCEESLWVFHLMRHVHDEVNSTTVSTLLSVLGHVDHQKWGRGIHALVFK
AES69403        NSIITASAHNGRFEESLGHFFWMRRTHPKTDYITISALLPACGSAQHLKWGRGLHGLITK
KRH55426        ------------------------------------------------------------
KRH26029        NSIITASVHNGHCEKSLEYFSQMRYTHAKTDYITISALLPVCGSAQNLRWGRGLHGMVVK
Zm00001d030143_ NAMISMYSHEEVYSKCFIVLSDMRHGEVKPDVTTLCSLVSVCASSDLVALGSGIHSLCVS
BGIOSGA035691-P NAMISMYSHQGICSKCFLVFSDMRHHGLRPDATTLCSLMSVCASADHFSHGSGIHSLCLR
VIT_19s0085g002 NAMISAYAHHGLCRESLRCFHWMRHLHNETNSTTLSSLLSVCSSVDNLKWGRGIHGLVVK
                                                                            

AT1G16480.1     MGFDSVVCVCNTLLRMYAGAGRSVEANLVFKQMPTKDLIS<span class="highlight">W</span><span class="highlight">N</span>SLMASFVNDGRSLDALGL
Bra016637.1-P   MGFDSVVCVCNTLLRMYAGAGRSEEAELVFNQIPAKDLIS<span class="highlight">W</span><span class="highlight">N</span>SLMACFVEDGRSLDALGL
AES69403        SGLESNVCVCNSLLSMYAQAGSSEDAELVFHTMPARDLIS<span class="highlight">W</span><span class="highlight">N</span>SMMASHVEDGKYSHAILL
KRH55426        ------------------------------------------------------------
KRH26029        SGLESNVCVCNSLLSMYSQAGKSEDAEFVFHKMRERDLIS<span class="highlight">W</span><span class="highlight">N</span>SMMASHVDNGNYPRALEL
Zm00001d030143_ SGLHCSVPLINALVNMYSTAGKLDEAESLFRNMSRRDVIS<span class="highlight">W</span><span class="highlight">N</span>TMISSYVQSNSCVEALET
BGIOSGA035691-P SSLDSSVTVINALVNMYSAAGKLSDAEFLF<span class="highlight">W</span><span class="highlight">N</span>MSRRDLIS<span class="highlight">W</span><span class="highlight">N</span>TMISSYVQNCNSTDALKT
VIT_19s0085g002 LGLDSNVCICNTLLTLYSEAGRSEDAELVFQAMTERDLIS<span class="highlight">W</span><span class="highlight">N</span>SMMACYVQDGKCLDGLKI
                                                                            

AT1G16480.1     LCSMISSGK-SVNYVTFTSALAACFTPDFFEKGRILHGLVVVSGLFYNQIIGNALVSMYG
Bra016637.1-P   LCSMIRTGK-SANYVSFTSALAACFSPEFLGKGRIIHGLVMVTGLFDNQIIGNALVSMYG
AES69403        LVEMLKTRK-AMNYVTFTTALSACYNLEKL---KIVHAFVIHFAVHHNLIIGNTLVTMYG
KRH55426        ----------------------------------------MTSGDHGRMNTANQLPAMLS
KRH26029        LIEMLQTRK-ATNY--------------------IVHAFVILLGLHHNLIIGNALVTMYG
Zm00001d030143_ LGQLLQTDEGPPNSMTFSSALGACSSPEALMNGRTIHAMILQRSLQNVLLIGNSLLTMYS
BGIOSGA035691-P LGQLFHTNE-IPNHLTFSSALGACSSPGALIDGKMVHAIVLQLSLQRNLLVGNSLITMYG
VIT_19s0085g002 LAELLQMGK-VMNHVTFASALAACSNPECLIESKIVHALIIVAGFHDFLIVGNALVTMYG
                                                        :  .       .* * :* .

AT1G16480.1     KIGEMSESRRVLLQ--MPRRDVVAW----------NALIGGYAEDEDPDKALAAFQTM--
Bra016637.1-P   KIGKMSESRRVLLQ--MPRRDEVAW----------NALIGGYAEDEDPDKALETFRTM--
AES69403        KFGLMDEAQKVCKI--MPERDVVTW----------NALIGGHADDKDPNATIQAFNLM--
KRH55426        HVISWLSIDEAVRSSILPKRWKPLWKHASCLDFDVSSVIKSLSQLENSSSRVKSVQVLTY
KRH26029        KF------------------DEVTW----------NALIGGHADNKEPNAAIEAFNLL--
Zm00001d030143_ KCNSMEDTERVFES--MPCYDVVSC----------NVLTGGYAALEDVANAMRVFSWM--
BGIOSGA035691-P KCNSMEDAEKVFQS--MPTHDVVSY----------NVLIGGYAVLEDGTKAMQVFSWI--
VIT_19s0085g002 KLGMMMEAKKVLQT--MPQPDRVTW----------NALIGGHAENEEPNEAVKAYKLI--
                :                                  . :  . :  ::    :     :  

AT1G16480.1     --------RVEGVSSNYITVVSVLSACLLPGDLLERGKPLHAYIVSAGFESDEHVKNSLI
Bra016637.1-P   --------RLEGVPANYITVVSVLGACLTPGDLLERGKPLHAYIVSAGFESDEHVKNSLI
AES69403        --------RREGLLSNYITIVNLLGTCMSPDYLLKHGMPIHAHIVVAGFELDTYVQSSLI
KRH55426        SQEHTRDYRRCEDPCSL--SLSKLIACLSPDDLLEHGMPIHAHIVVAGFELDTFVQSSLI
KRH26029        --------REEGVPVNYITIVNLL------------------------------------
Zm00001d030143_ --------RGTGIKPNYITMINLQGTCKSLGDLHSYGMPLHAYVTQTGLLSDEYITNSLI
BGIOSGA035691-P --------RSAGIKPNYITMINIHGSFTSSNDLHNYGRPLHAYIIRTGFLSDEYVANSLI
VIT_19s0085g002 --------REKGIPANYITMVSVLGACSAPDDLLKHGMPIHAHIVLTGFESDDYVKNSLI
                        *      .    :.                                      

AT1G16480.1     TMYAKCGDLSSSQDLFNGLDNRNIIT<span class="highlight">W</span><span class="highlight">N</span>AMLAANAHHGHGEEVLKLVSKMRSFGVSLDQF
Bra016637.1-P   TMYAKCGDLSSSHDLFNRLDNRNIIT<span class="highlight">W</span><span class="highlight">N</span>AMLAANAHQGHGEEVLKLVSKMRSFGLSLDQF
AES69403        TMYAQCGDLNTSSYIFDVLANKNSST<span class="highlight">W</span><span class="highlight">N</span>AIFSANAHYGPGEEALKFIARMRNDGVDLDQF
KRH55426        TMYAQCGDLNTSNYIFDVLANKNSST<span class="highlight">W</span><span class="highlight">N</span>AILPANAHYGPGEEALKLIIKMKNDGVHLDQF
KRH26029        ------------------------------------------------------------
Zm00001d030143_ TMYATCGDLESSTGIFSRINNKSVIS<span class="highlight">W</span><span class="highlight">N</span>AIIAANVRHGRGEEAIKLFMDSQHAGNKLDRF
BGIOSGA035691-P TMYAKCGNLESSTNIFNSITNKNIVS<span class="highlight">W</span><span class="highlight">N</span>AIIAANAQLGHGEEALKLFIDMQHAGNKLDRV
VIT_19s0085g002 TMYAKCGDLNSSNYIFDGLGNKSPIT<span class="highlight">W</span><span class="highlight">N</span>AMVAANAHHGCGEEALKIFGEMRNVGVNLDQF
                                                                            

AT1G16480.1     SFSEGLSAAAKLAVLEEGQQLHGLAVKLGFEHDSFIFNAAADMYSKCGEIGEVVKMLPPS
Bra016637.1-P   SFSEGLSAAAKLAVLEEGQQLHGLAVKLGFEQDCFIFNAAADMYNKCGEVDEAVKMLPPS
AES69403        SFSVALATIGNLTVLDEGQQLHSWIIKLGFELDEYVLNATMDMYGKCGEIDDVFRILPIP
KRH55426        SFSVALAIVGNLTLLDEGQQLHSLIIKHGFESNDYVLNTTMDMYGKCGEIDDGFRILPQP
KRH26029        ----THAIIGNLTLLDEGQQLHSLIIKHGFESNDYVLNATMDMYGKCGEIDDVFRILPQP
Zm00001d030143_ CLAECLSSSANLASLEEGMQLHGLSVKNGLDCDSHVVNATMDMYGKCGKMDCMLKTLPDP
BGIOSGA035691-P CLAECLSSCASLASLEEGMQLHGLGMKSGLDSDSYVVNAAMDMYGKCGKMNEMLQMVPDQ
VIT_19s0085g002 SFSGGLAATANLAVLEEGQQLHGLVIKLGFESDLHVTNAAMDMYGKCGEMHDVLKMLPQP
                      :  ..*: *:** ***.  :* *:: : .: *:: ***.***::   .: :*  

AT1G16480.1     VNRSLPS<span class="highlight">W</span><span class="highlight">N</span>ILISALGRHGYFEEVCATFHEMLEMGIKPGHVTFVSLLTACSHGGLVDKGL
Bra016637.1-P   VNRSLPS<span class="highlight">W</span><span class="highlight">N</span>ILISAFGRHGYFEKVCETFHEMLESGVKPGHVTFVSLLTACSHGGLVDQGL
AES69403        KIRSKRS<span class="highlight">W</span><span class="highlight">N</span>ILISALARHGFFRQATEAFHEMLDLGLKPDHVTFVSLLSACSHGGLVDEGL
KRH55426        RS----------------------------------------------------------
KRH26029        RSRSQRS<span class="highlight">W</span><span class="highlight">N</span>ILISALARHGFFQQAREAFHEMLDLGLRPDHVTFVSLLSACSHGGLVDEGL
Zm00001d030143_ AHRPTQC<span class="highlight">W</span><span class="highlight">N</span>TLISGYARYGYFKEAEDTFKHMVSVGQKPDYVTFVALLSACSHAGLIDKGM
BGIOSGA035691-P AIRPQQC<span class="highlight">W</span><span class="highlight">N</span>TLISGYAKYGYFKEAEETFKQMVAMGRKPDYVTFVALLSACSHAGLVDKGI
VIT_19s0085g002 INRSRLS<span class="highlight">W</span><span class="highlight">N</span>ILISAFARHGCFQKARETFHEMLKLGPKPDHVTFVSLLSACNHGGLVDEGL
                                                                            

AT1G16480.1     AYYDMIARDFGLEPAIEHCICVIDLLGRSGRLAEAETFISKMPMKPNDLVWRSLLASCKI
Bra016637.1-P   AYYDMIARDFGIKPAIEHCVCVIDLLGRSGRLAEAETFISNMPMKPNDLVWRSLLASCKI
AES69403        VYFSSMTSEFGVPTAIEHCVCIIDLLGRSGRLAEAEGFIDKMPVPPNEFVWRSLLAACKV
KRH55426        -------------------SCLVSLLVR--------------------------------
KRH26029        AYFSSMSTKFGVPTGIEHCVCIIDLLGRAGKLTEAENFINKMPVPPTDLVWRSLLAACKI
Zm00001d030143_ DYYNSMAPTFGVSPGIKHCVCIVDLLGRLGKFAEAEKFIDEMPVLPNDLIWRSLLSSSRT
BGIOSGA035691-P DYYNSMASSFGVSPGIKHCVCIVDLLGRLGRFAEAERFIEEMPVLPNDLIWRSLLSSSRT
VIT_19s0085g002 AYYDSMTREFGVFPGIEHCVCIIDLLGRSGRLSHAEGFIKEMPVPPNDLAWRSLLAACRI
                                    *::.** *                                

AT1G16480.1     HGNLDRGRKAAENLSKLEPEDDSVYVLSSNMFATTGRWEDVENVRKQMGFKNIKKKQACS
Bra016637.1-P   HGDLDRGRRAAEHLSKLEPEDDSVYVLSSNMFATTGRWEDVENVRNQMGFKNIKKKQACS
AES69403        HGNLELGRKAADRLFELNSSDDSAYVLYSNVCASTQRWGDVENVRKQMESQSLKKKPACS
KRH55426        --------------------------------------------------NVITHSPCVV
KRH26029        HGNLELARKAADRLFELDSSDDSAYVLYSNVCASTRRWRDVENVRKQMESHNIKKKPACS
Zm00001d030143_ HKNLDIGRKAAKNLLELDPFDDSAYVLLSNLYATNARWVDVDKLRSHMKTIKLNKRPACS
BGIOSGA035691-P HKNLEIGRKAAKKLLELDPFDDSAYVLLSNLYATNARWVDVDKLRSHMKTININKRPACS
VIT_19s0085g002 HGNLELARKTAEHLLELDPSDDSAYVLYSNVCATSGKWEDVENLRKEMGSNNIKKQPACS
                                                                    :.:  .  

AT1G16480.1     WVKLKDKVSSFGIGDRTHPQTMEIYAKLEDIKKLIKESGYVADTSQALQDTDEEQKEHNL
Bra016637.1-P   WVKQRDRVSTFGIGDRTHPQTLEIYEKLEDIKKLIKESGYVADTSGALQDTDEEQKEQNL
AES69403        WIKLKNKVMTFGMGDQFHPQSAQIYAKLEELRKMTREEGHMPDTSYALQDTDEEQKEHNL
KRH55426        CIKLKNQV-TIGMGDQYLPQNAEIYAKLEELKKMISEASYV-HTSYSLQDTDEEQKEHIL
KRH26029        WVKLKNQVTTFGMGDQYHPQNAEIYAKLEELKKIIREAGYMPDTSYSLQDTDEEQKEHNL
Zm00001d030143_ WLKLKNEVSTFGIGDRSHMHAEKIYVKLDEILLKLREVGYVADTSSALHDTDEEQKEHNL
BGIOSGA035691-P WLKLKNEVSTFGIGDRGHKHAEKIYAKLDEMLLKLREVGYIADTSSALHDTDEEQKEQNL
VIT_19s0085g002 WVKLKDKVHSFGMGEKYHPQASRISAKLGELMKMTKEAGYVPDTSFALHDMDEEQKEYNL
                 :* ::.* ::*:*::   :  .*  ** ::     * .:: .** :*:* ******  *

AT1G16480.1     <span class="highlight">W</span><span class="highlight">N</span>HSERLALAYALMSTPEGSTVRIFKNLRICSDCHSVYKFVSRVIGRRIVLRDQYR----
Bra016637.1-P   <span class="highlight">W</span><span class="highlight">N</span>HSERLALAYALMSTPEGCTVRIFKNLRICSDCHSVYKFVSKVVGRRIVLRDQYR----
AES69403        <span class="highlight">W</span><span class="highlight">N</span>HSERIALAFGLINSAEGSPLRIFKNLRVCGDCHSVFKLVSKIVGRKIVVRDSYR----
KRH55426        <span class="highlight">W</span><span class="highlight">N</span>HSERIAHAFG-----------ICKNFSFSGDCLSVFKMVSQIIGRKIIIRNAYR----
KRH26029        <span class="highlight">W</span><span class="highlight">N</span>HSERIALAFGLINSSEGSPLRIFKNLRVCGDCHSVFKMVSQIIGRKIILRDAYR----
Zm00001d030143_ <span class="highlight">W</span><span class="highlight">N</span>HSEKLALAYGLLVVPEGSTIRIFKNLRVCADCHLVFKLVSMVFHREIVLRDPYRCIMC
BGIOSGA035691-P <span class="highlight">W</span><span class="highlight">N</span>HSEKLALAYGLIVVPEGSTCQMGEIDQLIAR---FFSTVGVFVRFHGSQQKRGSSIIL
VIT_19s0085g002 <span class="highlight">W</span><span class="highlight">N</span>HSERLALAFGLINTPESSTLRIFKNLRVCGDCHSVYKFVSGIVGRKIVLRDPYR----
                *****::* *:.           : :   . .    .:. *. ..  .   :.       

AT1G16480.1     ------------------------------------------------------------
Bra016637.1-P   ------------------------------------------------------------
AES69403        ------------------------------------------------------------
KRH55426        ------------------------------------------------------------
KRH26029        ------------------------------------------------------------
Zm00001d030143_ DLSLAKQGKLL----DVVQASTGAAAPHPTVREVVANYYSPNRHRRWLPSSVRGRGVAAA
BGIOSGA035691-P DERRGGPGSSITERRQYEQETTGFIKDHEEIRR------RPRCRRIKHDITFLFSPLGQA
VIT_19s0085g002 ------------------------------------------------------------
                                                                            

AT1G16480.1     -------FHHFER---GLCSCKDYW
Bra016637.1-P   -------FHHFES---GMCSCKDYW
AES69403        -------FHHFHG---GKCSCSDYW
KRH55426        -------FHHFRS---GKCSCSDYW
KRH26029        -------FHHFSS---GKCSCSDYW
Zm00001d030143_ RQIVLRSYDHDEAHCVAAVAGARQW
BGIOSGA035691-P ESTMIRKLKH------RRLMNSEPT
VIT_19s0085g002 -------FHHFSG---GKCSCGDYW
                        .*               


<a href='WN_NW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            