
<!doctype html>
<html>
    <head>
        <title>WQ-QW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT1G19620.1</h1>
        <h3>AT1G19620.1 pep chromosome:TAIR10:1:6784020:6785075:1 gene:AT1G19620 transcript:AT1G19620.1 gene_biotype:protein_coding transcript_biotype:protein_coding description:unknown protein; Ha. [Source:TAIR;Acc:AT1G19620]</h3>
        <pre>
length  317
count   8
ratio   0.00
zscore  11.31






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT1G19620.1     MEAEKAEASWTYENLF--QNCKYTVGLSAVVAF-GYCCLPVIVNQLHGK-----------
Bra025752.1-P   MEAENDEATKGHDNTFSLHSILFIVQVSAGLAFAGYCGYPAIVNVLEGFILWFDWFRLVL
Bra016500.1-P   MEKENDEPTKGHDNTFSLQNVLFIVKVSAVLAFTGYCGYPTIINVLDGVDLWFGWFRLVL
                ** *: *.:  ::* *  :.  : * :** :** ***  *.*:* *.*            

AT1G19620.1     ---------------------IQTTEKKEPDLYDQYVAAVPPPSTVTPFDSYNMSKFEDV
Bra025752.1-P   RNLFFIFIILNALIASIYFTFLKATEKKKPSLRDEYITAVPSDVRSSPEQSFSPMEVGDY
Bra016500.1-P   RNVFFIFIILNALIGAIYFSFLKTTEKKKQDLYDEYIAAAPSAVRSSPEPSSALMEVGDY
                                     :::****: .* *:*::*.*.    :*  *    :. * 

AT1G19620.1     -----GYYNNSTYRENVPAFQAVKAVEESSYSYDQEVVEMSSKSYRRTSRSEKKMTE---
Bra025752.1-P   GGGGGGHYNNSCYPESVQQFQAVNTVVESS-CCERVVMETNSKSYKRMMSSEKKMKKTRS
Bra016500.1-P   ---GGEYYNKSSYYKTVPPFQAVTTVEESK-CYERLVMETSSKSYRKTTSLEKKKKR---
                      :**:* * :.*  ****.:* **. . :: *:* .****::    *** ..   

AT1G19620.1     --RMVEY-RKTETERV-TKTGSWRSQ--SMDGLSSGEEFRRTIQTFVSEQKKMHIIRASG
Bra025752.1-P   STTTVEYHQRMESERV-MKTASCRSSRDMMDELSN-EEFRMTIETFLMEKKMIYH<span class="highlight">W</span><span class="highlight">Q</span>-NG
Bra016500.1-P   --STVEY-RRTESERVLMKTASWRSH--AMDELSS-EEFRMKVETFITGYKKKMNCQ-NG
                    *** :: *:***  **.* **    ** **. **** .::**:   *     : .*

AT1G19620.1     VDDQLENGYIHRASGVDDRVHYWEHGAVP<span class="highlight">Q</span><span class="highlight">W</span><span class="highlight">Q</span>NDDVPQLQNGGVPHW<span class="highlight">W</span><span class="highlight">Q</span>NGSVSQLQNSD
Bra025752.1-P   VVDHRQNGGPQLQYGDADQCH------------NGVPQLQN-GVPYHWKYG-FPQFQNGA
Bra016500.1-P   VADH--------------------------------------------------------
                * *:                                                        

AT1G19620.1     VP<span class="highlight">Q</span><span class="highlight">W</span><span class="highlight">Q</span>NGGVLQLQNGDVP<span class="highlight">Q</span><span class="highlight">W</span><span class="highlight">Q</span>TGGVSQLQNGVHEPQNGIV---<span class="highlight">Q</span><span class="highlight">W</span>KGSRDDGSGRHRHRH
Bra025752.1-P   VH<span class="highlight">Q</span><span class="highlight">W</span><span class="highlight">Q</span>N-GFPQLQNGGVH<span class="highlight">Q</span><span class="highlight">W</span><span class="highlight">Q</span>N-GYPQLQG---QPDNDET------GRRLVGGSGSTSSH
Bra016500.1-P   ---WGN-GVSRFQNGSVD<span class="highlight">Q</span><span class="highlight">W</span><span class="highlight">Q</span>N-CVPQLQD---QPYYDGTDRRRHRSRRLVGGSGSTGSG
                   * * *. ::***.* ***.   .***.   :*  . .      . *  *..      

AT1G19620.1     GHRSRKH---RQ
Bra025752.1-P   G----PHLAILD
Bra016500.1-P   G----PYLAISN
                *     :    :


<a href='WQ_QW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            