
<!doctype html>
<html>
    <head>
        <title>WS-SW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT5G04290.1</h1>
        <h3>AT5G04290.1 pep chromosome:TAIR10:5:1196069:1202653:1 gene:AT5G04290 transcript:AT5G04290.1 gene_biotype:protein_coding transcript_biotype:protein_coding gene_symbol:RDM3 description:Protein RNA-directed DNA methylation 3 [Source:UniProtKB/Swiss-Prot;Acc:F4JW79]</h3>
        <pre>
length  1493
count   36
ratio   0.00
zscore  27.34






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT5G04290.1     MDRKG---KGKQVA-----------------GSDSY----------SGGQKRKNS--VEF
Bra009470.1-P   MDRKG---KGKQIA-----------------GAGS-----------SSGKKRKGGGGVEF
KEH44273        MTNKG---KAPAVKGKAPA------------GKGS-----------SG--KRKN----NF
Pp3c16_360V3.2  MAKHYDDDEAEEVDEEEAELEEVEEEEEEEEGEDLEEEEEEEEEDEARGRKRKYKG-SQF
KRH35991        MTNKGKG-KGKEVA-----------------GKGS-----------AG--KRKA----VF
KRG90810        ------------------------------------------------------------
KRG90770        MTNKGKG-KGKEVA-----------------GKGS-----------AG--KRKG----VF
Zm00001d038644_ MAVKG---KGKQVAPESPGP---------GGGGASG----------AGGKRRKASG----
Zm00001d010719_ MAVKG---KGKQVAAESPGP-------GAGAGGASG----------SGGKRRMASS----
BGIOSGA020147-P MAVKG---KGKAVAS----------------ASASG----------GGVKRRKGPG----
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 MSLCHK--FASEVK--------F--------GYGA-------------------------
VIT_13s0019g049 M-----------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     RDEGLR------------IKKRKNPEVLQFFEESAEVGYYGGSSDEDDDGLGFLN-DME-
Bra009470.1-P   RDEGLR-----------INSKRKKPGVLQFFEESAEVGYYGGSSDEECDMDDLFN-DME-
KEH44273        DDDKTG--------------KRKNRGVVQFFEDEAA------DVDSDDSEFSDFS-DES-
Pp3c16_360V3.2  IDDVAEEDDDEDEEDEAPRKRRQHRRGSEFIDDTAAVASDEEEEEEEEEE------DFI-
KRH35991        YDDKTG-----------VGRKRNKPGVLQFFDDAADV---------EESDFSDFS-DDDS
KRG90810        ---MIL-----------SGCLR--------------------------------------
KRG90770        QDDKTG-----------SGCKRNNRSVLQFFEDAADV---------EESDFSDFSDDDDS
Zm00001d038644_ -DTGEGPSSSS------AAKRRRRAGVLQFVDDVAGVDDDNEDEDEDELE----S-EDD-
Zm00001d010719_ -DAGAGPSSSS------AAKRRRRAGVLQFVDDVAGVDDDYE---EDELE----S-EDD-
BGIOSGA020147-P --DAAGPSSSSSAAAE-AAKRRRRSGVLQFFDDAAFVGDDEEDEDEEEEE-EMFA-SDG-
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     ----DEPEVEE-SSKAGKGEKGKS--SFVFP-----KEEDLNEEEFDRIMEERYKPGSGF
Bra009470.1-P   ----DEPEVET-SGKDDKGGKGKGSSSFVFP-----KEEEMNEDDYDRMMEERYKPGSGF
KEH44273        ----YEDEFETLSTTRNESAKGSSSVPR-VP-----KQELVDEEEFDRLMEERVN--SRF
Pp3c16_360V3.2  ----ERGDISPGAEDVRRPHHRAAIRDD--------AQEDV--EGLERYIQQRY----GR
KRH35991        DFDFSDDELNAGPRRMDMPDNGQSSLPRVVP-----KEEMVDEEEWDRILEERYKDPSRF
KRG90810        ------------------------------------------------------------
KRG90770        DFDFSDDELNAAPRRMDMPDKGQSSLPRVVP-----KEEMLDEEEWDRILEERYKDPSRF
Zm00001d038644_ ----PDDGFFTGGEHAEKLSHKRTERSHPLPFL--VKEEELSGDELEEFIKNRY---SNR
Zm00001d010719_ ----TDDGFFTGGEHAQNLSHKRTERSHPLPFL--VKEEELSGDELEEFIKNRY---SNR
BGIOSGA020147-P ----DDDGFFTEGK-AENVNLKRTERSHPLPFLGIVKEEELSGDELEEFIKDRY---SSR
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------MVEECYKD----
VIT_19s0093g001 ---------------------------------------------------------SNA
VIT_13s0019g049 RVDFLEDGFNTGLKVKNEP--GKAHNLPFFP-----KEEELSEEELEKMLEERYKDGSKF
VIT_00s0131g004 ------------------------------------------------MLDERYKDGSKF
                                                                            

AT5G04290.1     LRYAD--DD--IKDAIEMDALAPTSKD--PPIWKVKCAIGRERRSVFCLMHKFVELRKIG
Bra009470.1-P   VRYAA--DD--VKSSIEMDALVPTAHD--PPIWKIKCAIGREKHSVFCLMHKFVELRKIG
KEH44273        FRFAEDGDEFEDKPMEDNSSLHHALKESIPTIWRVKCTVGRERLSAICLMQKFVHLKGLD
Pp3c16_360V3.2  QEYET-YDEA-ETTEVEQQALLPSVKD--PKLWMVKCNLGHEREAAICLMQKYIDQEQIN
KRH35991        IRFA---DEFGDDKGMDPSSIHEGVDELMPYIWKVKCTVGRERLSALCLMQKFADLDSLG
KRG90810        ------------------------------------------------------------
KRG90770        IRFS---DEFGDDKGMDPSSIHEGVDELTPSIWKVKCTVGRERLSALCLMQKFADLYSLG
Zm00001d038644_ VKYAA--DRSYSREDDDIFPMDCALKE--PTIWRVKCMVGRERQMAFCFMQKFVDLRKIG
Zm00001d010719_ VKYAA--DRSYSREDDDIFSMDCALKE--PTIWRVKCMVGRERQMAFCFMQKFVDLRKIG
BGIOSGA020147-P VKHTP--FDGSTNVQDDEFTEDGLLKE--PVIWRIKCMVGRERQIAFCLMQKYVDLQKFG
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ----------------------------------VKCMVGRERLSAFCLMNKYVDLQSLG
VIT_19s0093g001 LVASTRGHCFDHSTSLLSNSLIPSIKD--PTIWKVKCMVGRERLSAFCLMNNYVDLQSLG
VIT_13s0019g049 VTYAE--DDYETKRSVQRNSLIPSIKD--PTIWKVKCMVGRERLSAFCLMQKYVDLQSLG
VIT_00s0131g004 VTYAE--DDYETKRSVQRNSFIPSIKD--PTIWKVKCMVGRERLSAFYLMQKYVDLQSLG
                                                                            

AT5G04290.1     TKLEIISVFSVDHVKGFIFIEADKEHDVLEACKSLVGIYATRMVLLPKAETPNLLTVQKK
Bra009470.1-P   TKLKILSVFFVEHVKGFIFIEADKEQDVLEACMSLNGIYATRMMLLPKSEAPHLLTVQRK
KEH44273        TKLQLISAFAVDHVKGFIYVEAERQNDINQACYGITGIYATRVQPVPRNEVFHLFSVKIK
Pp3c16_360V3.2  QPLLIKSAIALDHLKGYLYIESEKEAYVRQACRGMRMIYSQKVTLVPIKEMTDVLSVEKK
KRH35991        TKLKIKSAFAVDHMKGFVYIEAEKQYDINEACQGIPGIYVTRVAPVPNSEVYHLFSVRSR
KRG90810        VQLRVNKALLSTFIEGLSFYMME-------------------------------------
KRG90770        TKLKIKSAFSVDHMKGFVYIEAEKQYDINEACQGIPGIYVTRVAPVPNSEVYNLFSVRSR
Zm00001d038644_ TKVPIITAFALDHIRGFVFVEAEKAGYVTEACKGFCSVYTSRITSVPATEVPSLLSSRTK
Zm00001d010719_ TKVPIITAFALDHIRGFVFVEAEKACDVTEACKGFCSVYTSRITSVPAAEVPSLLSSRTK
BGIOSGA020147-P TKVPIISAFALDHVRGFVFVEAEKACDVTEACKGFCSVYISRINSVPVAEVPSLLSSRAK
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 TKLQIIYAFSVDHVKGFIYIEADKQCDINEACKGLCIIYTSRVAPVLKNEVTHLLSVRSK
VIT_19s0093g001 TKLQIISVFLVDHVKGCIYIEADKQCDINEACKGLCTIYTSRVAPVPKNEVTKFIKYDKK
VIT_13s0019g049 TKLQIISAFSVEHVKGFIYIEADKQCDINEACKGLCSIYTSRVAPVPKNEVTHLLSVRSK
VIT_00s0131g004 TKLQIISVFSVEHVKGFIYIEADKQCDINESCKGLCTIYTSRVAPVTKNEVTHLLSIRSK
                                                                            

AT5G04290.1     TKK-VSEGTWARVK------NGKYKGDLAQIVAVSDTRNKALIKLIPRIDIQALTQKYGG
Bra009470.1-P   TKT-FSEGTWARIK------SGKYKGDLAQVVAVSDTRGKALIKLIPRIDIAALTQKYGG
KEH44273        KPE-ISEGMWGRVK------GGNYKGDLAQVVAVNNTHKKVTVKLVPRIDLFALAAKFGG
Pp3c16_360V3.2  AVE-IDQDTWVRVK------IGIYKGDLAKVVDVDHVRQRAQIKLIPRVDLQALAAKLEG
KRH35991        TPE-ISEGMWARIK------GGNYKGDLAQVVSVNNTRKKVTVKLIPRIDLQALAAKFGG
KRG90810        ------------------------------------MRKK--------------------
KRG90770        TPE-ISEGMWARIK------GGNYKGDLAQVVSVNNTRKKVTVKLIPRIDLQALAAKFGG
Zm00001d038644_ PFE-ISRGTWIRMK------NGNYKGDLAQVVNMDSGRKKVMIKIIPRVDLHAISKKFGG
Zm00001d010719_ PFE-ISRGTWIRMK------NGNYKGDLAQVVSVDDGRKKVMIKIVPRVDLHAISKKFGG
BGIOSGA020147-P PFA-ISPGTWVRMK------TGIYKGDLAQVVSADEGRKRVMIKLIPRVDLRAISKKIGG
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 CIE-SSEGTWARMK------NGKYKGDLAQ------------------------------
VIT_19s0093g001 IYTSLSINILIKLEFPCPQCGAEYAGFLCNPNTVH-------------------------
VIT_13s0019g049 CNE-ISEGTWARMK------NGKYKGDLAQIVVVSDAQKKATVKLIPRIDLQAMAEKFGG
VIT_00s0131g004 CNE-SSEGTWAPMK------NGKYKGDLAQIVVVSDA-----------------------
                                                                            

AT5G04290.1     GVTVQKGQTPAPRLISSSELEEFRPLIQVRRDRDTGITFEHLDSLMLKDGYLYKKVSLDS
Bra009470.1-P   GVAVQKGLNPAPRLISSSELEEFRPIIQVRRDRDTGMTFEHLDSLMLKDGFLYKKVSLDS
KEH44273        GCR-QKAAVPAPRLISSTELEEFRPLIQIKHDRETGKVFQVLDGMLLKDGFLYKKVSPDS
Pp3c16_360V3.2  RDDMKKRPRPAPRFISIQDVKDLRIPVERKRDGSTGELFDQFGGMMFKDGYLYKYVSLKT
KRH35991        GYSRQKMAVPAPRLISSSELEEFRPLIQIKRDRDTGKVFEVLDGLMLKDGYVYKKVSLDS
KRG90810        ------------------------------------------------------------
KRG90770        GYSRQKMAVPAPRLISSSELEEFRPLIQIKRDRDTGKVFEVLDGLMLKDGYVYKKISPDS
Zm00001d038644_ AVSLKEAAVPAPRLISSQELEFFRPHIEIKRDRQTGEVFEVLDGLMFKDGFLYKKVALSS
Zm00001d010719_ AVSLKGAAVPAPRLISSQELEFFRPHIEIKRDRQTGEVFEVLDGLMFKDGFLYKKVALSS
BGIOSGA020147-P AIPLKEAAIPAPRLISSQELEFFGSHIERKHDRQTNDFYEVLDGLMFKDGFLYKRVALSS
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 --------------------ELFSQIV---------------------------------
VIT_19s0093g001 ---------------------------------------------------------LQC
VIT_13s0019g049 GVSAKKRNNPAPRLISSSELEEFRPLIQYRRDRDTGKLFEILDGQMLKDGYLYKKVSIDS
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     IS<span class="highlight">S</span><span class="highlight">W</span>GVIPTKDELLKF--TPVD--------------------------------------
Bra009470.1-P   LTSCGVKPSKEEILKF--APLE--------------------------------------
KEH44273        LSFWGVVPTEEELLKF--GSSE--------------------------------------
Pp3c16_360V3.2  IDAKGIEPSLDELQRF-QKPGEDGMDALGLPPSAIKNRGQFMKGDAVVVIEGDLRNLMGV
KRH35991        LSLWGVVPTEEELLKF--GPCE--------------------------------------
KRG90810        ------------------------------------------------------------
KRG90770        LSLWGVVPTEDELLKF--GPSE--------------------------------------
Zm00001d038644_ LIYWGIQPTETELLKFSSSPSN--------------------------------------
Zm00001d010719_ LIYWGIQPTETELLKFSSSPSN--------------------------------------
BGIOSGA020147-P LIYWGIQPTETELLKFSSTPTI--------------------------------------
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 HACGGMMPSRNDI----GVPQH----CLGC------------------------------
VIT_13s0019g049 LSCWGVTPSEEELHKF--TPSS--------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     -RKETGDVEWISEIYGEERKKKILPTCREGGKGEGSGGGKGEGSG------GGKGEGS--
Bra009470.1-P   -EKETGDVEWISEIYGEEKKKKSIPA--------GKGVEKGEGSG------GGKGEGS--
KEH44273        -NNDSNDMEWLSNIYGDTKKKRVTIA--------DK------------------------
Pp3c16_360V3.2  VEKVDDDNVYIVPKYKDLKETLVFK---------EKQLQKFFKTGDHVKVIAGNHEGATG
KRH35991        -NNESNDLEWLSQLYGDKKKKRVIRP--------DKG-----------------------
KRG90810        ------------------------------------------------------------
KRG90770        -NNESNDLEWLSQLYGDKKKKRVIRP--------EKG-----------------------
Zm00001d038644_ -RASADDLDWLSGMYGSKKRN--LPA--------ERDMK--SSSS------KIKSSKA--
Zm00001d010719_ -RASADDLDWLSGMYGSKKRN--LPE--------KRDME--SSSS------KIRSSKA--
BGIOSGA020147-P -TSSTDDLDWVSSMYGPKKRN--VPK--------EPDME--PSSS------KASSSKGK-
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 -DRAFCGAYWHTQRFGGSDSHFVVL-----------------------------------
VIT_13s0019g049 -NEESVDLEWLSQLYGERKQKRTTKS--------DK------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     ---------------------------RGGKGEGSSDFKSESS-YELYNLVCFSRKDFGL
Bra009470.1-P   ---------------------------GGGKGEGSSESKSESS-HELYNLVCFSRKDFGL
KEH44273        ---------------------------GDGKGEGSSASGAGND-FELYALVRFRKKDFGV
Pp3c16_360V3.2  MIVKVQNNVITILSDSTREDLRVFAHNIVESSEVTSGITKLGD-YELHDLVALDQSTVGL
KRH35991        ---------------------------GGGKGESSSGSGVGNG-FELYDLVCFGKKDFGV
KRG90810        ------------------------------------------------------------
KRG90770        ---------------------------GGGKGESSSGSGVGNG-FELYDLVCFGKKDFGV
Zm00001d038644_ ---------------------------PDLKGSTSTENYDDNDEFNLHDLVLYGRKDFGV
Zm00001d010719_ ---------------------------SDLKASTSTENYDENDEFNLHDLVLYGRKDFGV
BGIOSGA020147-P ------------------------YSKASSKASTSTEDYEDKG-FNLHDLVLFGRRDFGV
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 -------------------------------------------------LVCSHHR----
VIT_13s0019g049 ---------------------------GGEKGEGSSGSSMVNS-FELHDLVCFGRKDFGI
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     IVGVDDKGDGYKVLKEGIDGPVVVTVGKKEMQNGPFDSKFTALDLNKKQISVNDVVKISK
Bra009470.1-P   IVGVDDKGDGYKVLKEGSDGPVVVSVGKKEMQE-PFDSKFSALDLNNKQISIGDVVKIAK
KEH44273        IIST-DKDGTYKILKESSEGPVAVTVQRNEIKSALCDLKLSAQDLHRKTIVVNDNVKVLD
Pp3c16_360V3.2  IVRV-EK-DAFQILKGNPERTELLMVKPREIRRKVFDRKVCAQDRDMNVVSMKDIVRVLD
KRH35991        IVGM-DKDDIYKILKEGSDGPAAVTIERHEIKSGLFDLKLTALDQQSKTILVNDTVRVLE
KRG90810        ---M-D------------------------------------------------------
KRG90770        IVGM-DKDDIYKILKEGSDGPDAVTVDRHEIKSGLFDLKLTALDQHSKTILVNDTVRVLE
Zm00001d038644_ IIAV-EK-DGFRILKGGPEGS-AVTVRKQDIKKGCVDKMFTAVDHHKKTISINDTVNVVE
Zm00001d010719_ IVAV-EK-DGFRILKGGPEGS-AVKVRKQDIKKGCVDKMFTAVDHQKKTISINDTVNVVE
BGIOSGA020147-P VIAF-EK-DGLRILKGGPEGS-AVTVRKQDIKDVCADKMLTAVDHKKKIICINDTVNVLE
Os05t0506200-01 --------------------------------------MLTAVDHKKKIICINDTVNVLE
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ---------------------------------------------------LND------
VIT_13s0019g049 VIGM-EKDDNYKILKDGPEGPVVQTVVLHELKNPLFENKFTALDQHMKTISINDTLKVLE
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     GPSE--------------------------------------------------------
Bra009470.1-P   GPSE--------------------------------------------------------
KEH44273        GPSQ--------------------------------------------------------
Pp3c16_360V3.2  GHCK--------------------------------------------------------
KRH35991        GPSK--------------------------------------------------------
KRG90810        ------------------------------------------------------------
KRG90770        GPTK--------------------------------------------------------
Zm00001d038644_ GPFQ--------------------------------------------------------
Zm00001d010719_ GPFQ--------------------------------------------------------
BGIOSGA020147-P GPFQRDGAMGKQPGMALENLHRSKRDVRRGLRERWGDGMEIEVSRVPKSSTVTVLRTGHD
Os05t0506200-01 GPFQ--------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 GPLK--------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     --GKQGVVRQVYRGIIFLYDESEEENGGYFCCKSQSCEKVKLFTEESN--EKTGGFDGTA
Bra009470.1-P   --GKQGVVRQVYRGIIFLYAEGEEENGGYLCCKSQSCEKVKLFTEESN--DKTGGFDASA
KEH44273        --GKQGIVRHIYQGIVFLYDGNEEENGGYVTAKAIMCEKVKLAVSDFSAIGKDSEPGPLV
Pp3c16_360V3.2  --GKQGPVEHIHRGLLFIHDRHHLDNGGYVCVKARQCSALGGSRNGGD--RRNGGPVSSL
KRH35991        --GKQGIVKHIYRGIVFLYDGNEEENGGYLTCKSNMCEKVKVAVGDCS--GKDSEPAPLV
KRG90810        -----------------------------LTCKTNMCEKVKLAVGDCS--GKDSEPGPLV
KRG90770        --GKQGIVKHIYRGIVFLYDGNEEENGGYLTCKSNKCEKVKLAVGDCS--GKDSEPGPLV
Zm00001d038644_ --GKQGLVKHLYLGILFIFNESESENCGFFCAQSGSCEKIRKGLGSST--TENSDFP---
Zm00001d010719_ --GKQGVVKHLYLGILFIYNESESENCGFFCAQSGSCEKIRKGLGSST--TESSDFT---
BGIOSGA020147-P GEGKQGVVKHLYMGILFIHNESESENSGFFCAQCSSCENVK----------KRKELASFT
Os05t0506200-01 --GKQGVVKHLYMGILFIHNESESENSGFFCAQCSSCENVK----------KRKELASFT
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ----------------------------------QAC-----------------------
VIT_13s0019g049 --GRQGLVKKIYRGVIFLYDENETENNGYFCSKSQMCEKIKLYGDACN--EKGGESGPSG
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     FEDFVSSPKSPLSPEKEW--QPRERY-----------------------NSSNQGDIGST
Bra009470.1-P   FGDSASSPKSPLSPEKEWQQQPKEKY-----------------------INSNQGDKGSM
KEH44273        FDDQPSSPRSPLSPKKPW--QAKE-------------------------KDFNRGDSDSL
Pp3c16_360V3.2  ---------SSLSLNGNVLQSPRRESPYGGHGGGGGRGGAHGGGGFGGGRAGGRGRREDS
KRH35991        FEDQPSSPRSPLSPKKPW--QARENN-----------------------REFNRGDNNNM
KRG90810        FEDQPSSPRSPLSPKKPW--QARENN-----------------------REFNRGDNNNM
KRG90770        FEDQPSSPRSPLSPKKPW--QARENN-----------------------REFNRGDNNNM
Zm00001d038644_ ---------TPMFSEPAYEQNEHRNT-----------------------EGPYRSTREQL
Zm00001d010719_ ---------TPMFPEPAYEQNGHRDT-----------------------ERPYRSTREQL
BGIOSGA020147-P FGNSDNNP-IPMFS---YEQNEQRDN-----------------------ERPYRSTREQL
Os05t0506200-01 FGNSDDNP-IPMFS---YEQNEQRDN-----------------------ERPYRSTREQL
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 FEDFTSSPQSPLSPKKPW--QARENN-----------------------RDFNRGDKDGM
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     YSIGQKLRIRVGPLKGYLCRVIALRYSDVTVKLDSQHKIFTVKSEHLAEVRDRNTVL---
Bra009470.1-P   YSIGQKLRIRVGPLKGYLCRVIALRYSDITVKLDSQHKIFTVKSEHLAEVRDRNTVP---
KEH44273        FAIGQTLRIRIGPLKGYLCRVTGIRRSDVTVKLDSQQKFLTVKSEHLSVVQGKSTAV---
Pp3c16_360V3.2  I-VGRSVKIRLGPFKGYRGRVVDATDSTVRIELESQMKVVTVRREQLSDPGESQTVFFNN
KRH35991        FSIGQTLRIRIGPLKGYICRVIALRRADVTVKLDSQQKVLTVKCEHLSEVQGKSTAI---
KRG90810        FTIGQTLRIRIGPLKGYICRVIALRRADVTVKLDSQQKVLTVKCEHLSEVQGKSTAI---
KRG90770        FTIGQTLRIRIGPLKGYICRVIALRRADVTVKLDSQQKVLTVKCEHLSEVQGKSTAI---
Zm00001d038644_ FSIGQMLRIRKGPLKGYLCRVVRIFRNDVTVKLDSLLKIVTVQADLLSVPANRGDNL---
Zm00001d010719_ FSIGQMLRIRKGPLKGYLCRVVRIFRNDVTVKLDSLLKIVTVQADLLSVPANRGDNL---
BGIOSGA020147-P FSIGEMLRIRKGPMKGYLCRVVRIFRNDVTVKLDSLLKIVTVQAEFLSVPAKRGDNS---
Os05t0506200-01 FSIGEMLRIRKGPMKGYLCRVVRIFRNDVTVKLDSLLKIVTVQAEFLSVPAKRGDNS---
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 -----------------LCHLL--------------------------------------
VIT_13s0019g049 FSVGQTLRIRVGPLKGYLCRVLAIRYSDVTVKLDSQHKVLTVKCEHLSEVRGKGFSV---
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     -STSGDAGTGSFQPFGMLGTESSTGDWA-------------IGAGTSSEGGNWNIGGPST
Bra009470.1-P   -SMSGDAGMGSFQQFDMLGTEGNNGDWA-------------KGAGTS-------------
KEH44273        -SASGDPDSSSLNPFDLLGAEGSSGGWL------------NNSAGTSTGSGGWNAGETS-
Pp3c16_360V3.2  YRESPRYGAGSETPMHPSRTPMHHPAYMTPMRDPNFATPSHDGMRTPMRDRAWNPHTPMT
KRH35991        -SSSGDPDSSSSKPFDLLGTEGSSGGWL-------------NGVGTSTGGGGWNAGRASS
KRG90810        -SSSGDPDSSSSKPFDMLGTEGSSGGWL-------------NGVGTSTGAGGWNAGGASS
KRG90770        -SSSGDPDSSSSKPFDMLGTEGSSGGWL-------------NGVGTSTGAGGWNAGGASS
Zm00001d038644_ -SGAPASKFGS-QDTPFFGSEAGKT<span class="highlight">S</span><span class="highlight">W</span>D-------------RGL-PSFGSD<span class="highlight">S</span><span class="highlight">W</span>QPFSSST
Zm00001d010719_ -SGAPASKFGS-QDTSFFGSEAGKT<span class="highlight">S</span><span class="highlight">W</span>D-------------NGL-PSFGSD<span class="highlight">S</span><span class="highlight">W</span>QPFSSST
BGIOSGA020147-P ----------SGAPSGPFGSEADKP<span class="highlight">S</span><span class="highlight">W</span>D-------------NGL-PSFGSD<span class="highlight">S</span><span class="highlight">W</span>QPFSSAA
Os05t0506200-01 ----------SGAPSGPFGSEADKP<span class="highlight">S</span><span class="highlight">W</span>D-------------NGL-PSFGSD<span class="highlight">S</span><span class="highlight">W</span>QPFSSAA
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ----------------------------------------LAGM----------------
VIT_13s0019g049 -SISDNPESSSLKSFGLLGTQDSARDWV-------------DGAGTSAESDRWNTGETSA
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     DSHESL---------NIERNMVQLCREKN------P-----------W-----GGSKPTS
Bra009470.1-P   ------------------------------------------------------------
KEH44273        ------------------------------------------------------------
Pp3c16_360V3.2  PHRGNN-----------------------------------------WD-----------
KRH35991        T----------------------------------------------------GG-----
KRG90810        T----------------------------------------------------GGS----
KRG90770        T----------------------------------------------------GGSG---
Zm00001d038644_ LPVQNAAGCGTCTFSGAESEADPWCKKTSAEDDSDP-----------WGKKTDGDSDPWG
Zm00001d010719_ LSVQNA---------GGESEADP<span class="highlight">W</span><span class="highlight">S</span>KKTSAKDDSDPWGKKTDGDSDRWGKKTDGDSDPWG
BGIOSGA020147-P LPVQNA---------DGESEVDPWCKKTS-------------------------------
Os05t0506200-01 LPVQNA---------DGESEVDPWCKKTS-------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 E-----------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     DVSPTVADDNTSAWANA--AAENKPASASDQPGGWNPWGKTPA-SEAGTVSGWGD-----
Bra009470.1-P   -------EDNTSTWGST--AAENKPDSAGDQSGGWNAWGKPPA-PEASTVGAWGD-----
KEH44273        --------------------------------GERSAWG---------------------
Pp3c16_360V3.2  -------DANPSTWDTHTSTPQYEPGTPGGR-----------------------------
KRH35991        --------------------------------GGWNAGG-----ASSNGGGGWNA-----
KRG90810        ---------------------------------GWNAGG-----ASATGGGGWNV-----
KRG90770        -------------WNAG------GASSTGG--GGWNAGG-----ASSTGGGGWNV-----
Zm00001d038644_ KEAVSAADGDCNPWGK-----ETVPSAAGDS----DPWGKKVF-SADGDSDPWGKKAISS
Zm00001d010719_ KEAVSAADGDSNPWGK-----ETVPSAAGDS----DPWGKKVVSSADGDSD<span class="highlight">S</span><span class="highlight">W</span>GKKATSS
BGIOSGA020147-P -------------------------SSANDS----DPWGTK---TKSASVDVWNN-----
Os05t0506200-01 -------------------------SSANDS----DPWGTK---TKSASVDVWNN-----
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     ---------TSASNVEAS<span class="highlight">S</span><span class="highlight">W</span>EKQGASTSNVADLG<span class="highlight">S</span><span class="highlight">W</span>GTHGGSSGGNKQDEDSVWGKLCEA
Bra009470.1-P   ---------ASTPKVEA-<span class="highlight">S</span><span class="highlight">W</span>GKQGASTSNVEDSG<span class="highlight">S</span><span class="highlight">W</span>GKHGGSSDGNKQNDDSVWGKLVE-
KEH44273        -------------------------------------------------NPSAPSSV-PG
Pp3c16_360V3.2  ---------SFEAPTPGNG<span class="highlight">W</span><span class="highlight">S</span>AQ-------TPGASFSEAGTPTE--PAQSYAAPSPYLPG
KRH35991        ---------GGSSSTGGGGWNAGGGSS---TRGGGWNAGGASSERDAESNHSAPSLLNTE
KRG90810        ---------GGASSTGGGGWNAGGASS---TGGGGWNAGGPSSKRDAGSNHSAPSLLNTE
KRG90770        ---------GGASSTGGGGWNAGGASS---TGGGGWNAGGPSSKRDAGSNHSAPSLLNTE
Zm00001d038644_ AVEVWNTSATQKEGSSGNAWDKQAGVGGSDAAGS<span class="highlight">S</span><span class="highlight">W</span>DRAVVNKESEKSDN---WGEACRV
Zm00001d010719_ AVD------TEKEGSGGTAWDKQAGVGGSDAAGS<span class="highlight">S</span><span class="highlight">W</span>DRAVVN-ESEKSDN---WGEACRV
BGIOSGA020147-P -------STTQVENSSDNAWDKQPGGSGPNIGGS<span class="highlight">S</span><span class="highlight">W</span>DRPATDKESEKSDN---WGEACKE
Os05t0506200-01 -------STTQVENSSDNAWDKQPGGSGPNIGGS<span class="highlight">S</span><span class="highlight">W</span>DRPATDKESEKSDN---WGEACKE
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     S----ESSQK--KEES<span class="highlight">S</span><span class="highlight">W</span>GKKGGSD-------GES<span class="highlight">S</span><span class="highlight">W</span>G---NKDGNSSASKKD------G
Bra009470.1-P   -----ESSQK--KEES<span class="highlight">S</span><span class="highlight">W</span>GKKTGSDSDLGKGNGESTWG---NKDGNSSASNKE------G
KEH44273        STSDPLSSEG-NAEDNAWGTKSTST-------ATPAWGAAV-NTGTTSETDQSGGWGTKS
Pp3c16_360V3.2  TPGGPPMTPG--VPSYLPGTPGGQP-------MTPGTG------------------GLDP
KRH35991        SISNPFSSKG--AEDSAWETKSNSN-------QNS<span class="highlight">S</span><span class="highlight">W</span>GVAVEKTGIASNPDQSDGWGS-G
KRG90810        STSNPFSSKG--AEDSAWETKSNSN-------KTS<span class="highlight">S</span><span class="highlight">W</span>GAAVDKTGIASNADQSGGWGSGG
KRG90770        STSNPFSSKG--AEDSAWETKSNSN-------KTS<span class="highlight">S</span><span class="highlight">W</span>GAAVDKTGIASDADQSGGWGSGG
Zm00001d038644_ M----DMGTG--ADTDPWGSKVKAVD----MEGPN<span class="highlight">S</span><span class="highlight">W</span>E----KATMPPDNKLE------D
Zm00001d010719_ M----DMGTGAEADTDPWGSKVKAVD----MEGPNNWE----KATVPPNNKLE------D
BGIOSGA020147-P T-----EKTG--SDTDPWGSKVKEID----LKETD<span class="highlight">S</span><span class="highlight">W</span>G----KASMQPEKKLE------D
Os05t0506200-01 T-----EKTG--SDTDPWGSKVKEID----LKETD<span class="highlight">S</span><span class="highlight">W</span>G----KASMQPEKKLE------D
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     VS-----WGQ----QDKGSDESKGG-----SA<span class="highlight">W</span><span class="highlight">S</span>NQCGDFGS---GKKKDGSSGWNKSAE
Bra009470.1-P   VS-----WGQ----HDKGSDGDQGG-----S<span class="highlight">S</span><span class="highlight">W</span>------------GKKND-------SVK
KEH44273        TSTATPAWGAAV-NTGTTSEPDQS------G<span class="highlight">S</span><span class="highlight">W</span>------------GTKST----------
Pp3c16_360V3.2  TSPAI-------------------------------------------------------
KRH35991        GS-----WGQAEHKTGSMGDGNQN------SNW------------NDKPNNLNGNQSSGR
KRG90810        GS-----WGQAEHKTGSVGDGNQN------------------------------------
KRG90770        GS-----WGQAEHKTGSVGDGNQN------------------------------------
Zm00001d038644_ VSQ---GWNQ----HKGNSTEDQAK--------DNV---------SKDED----------
Zm00001d010719_ VSQ---GWGQ----PKGNSTEDQAI--------DNV---------SKDVD----------
BGIOSGA020147-P DSQ---GWGQ----PVGKSNQDQEKGADKCGAWDTVIAGSSSSVPGRGDD----------
Os05t0506200-01 DSQ---GWGQ----PVGKSNQDQEKGADKCGAWDTVIAGSSSSVPGRGDD----------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     DSNANSKGVPDWGQPNDGS<span class="highlight">S</span><span class="highlight">W</span>GKKGDGAA<span class="highlight">S</span><span class="highlight">W</span>GKKDDGG-<span class="highlight">S</span><span class="highlight">W</span>GKK-DDGNKDDGGS<span class="highlight">S</span><span class="highlight">W</span>GKK
Bra009470.1-P   D--------------DGGS<span class="highlight">S</span><span class="highlight">W</span>GKKDDGGSTWGKKDDGGS<span class="highlight">S</span><span class="highlight">W</span>GKKVDDVIKDDGGS<span class="highlight">S</span><span class="highlight">W</span>DKK
KEH44273        --------------STPTPAWGAAANAGTT--SEPDQSG<span class="highlight">S</span><span class="highlight">W</span>GTKSTSTPTPAWGAAANTE
Pp3c16_360V3.2  -----------------------------------------------------GSG----
KRH35991        DSK---------------SNWNTTKASEGE-----------------------SSGWNSV
KRG90810        ------------------SNWNTTKASEGE-----------------------SSGWNSI
KRG90770        ------------------SNWNTTKASEGE-----------------------SSGWNSI
Zm00001d038644_ NNRA<span class="highlight">W</span><span class="highlight">S</span>SSLP----VTEDGTWGKSKDDNCD-----------------------GAG----
Zm00001d010719_ NNRACDSSLP----VTEDGTWGKSKDNSCG-----------------------GAG----
BGIOSGA020147-P D<span class="highlight">S</span><span class="highlight">W</span>IKTDTLP----VAQDDAWGNSKDCSDG-----------------------AAGWNET
Os05t0506200-01 D<span class="highlight">S</span><span class="highlight">W</span>IKTDTLP----VAQDDAWGNSKDCSDG-----------------------AAGWNET
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     DDGQKDDGGS<span class="highlight">S</span><span class="highlight">W</span>EKKF---------DGGS<span class="highlight">S</span><span class="highlight">W</span>GK-----KDDGGS<span class="highlight">S</span><span class="highlight">W</span>GK-----KDDGGSL
Bra009470.1-P   VEGNKDDGGS<span class="highlight">S</span><span class="highlight">W</span>GKKV----DCNKDDGGS<span class="highlight">S</span><span class="highlight">W</span>GKKVEGNKDDGGS<span class="highlight">S</span><span class="highlight">W</span>GKKVDCNKDDGGSS
KEH44273        TTSEPDKSG-GWGTKS-------TSTPTPAWGAAV------------NTITTSEPVQSGG
Pp3c16_360V3.2  ------------------------------------------------------------
KRH35991        KKSNQTSSI-GWGGGNGFK-----------------------------------------
KRG90810        QKSNETSSA-GWGGGNGFKSGSDEGNLNST<span class="highlight">W</span><span class="highlight">S</span>G------------W----------KSGS
KRG90770        QKSNETSSA-GWGGGNGFKSGSDEGNLNST<span class="highlight">W</span><span class="highlight">S</span>G------------W----------KSGS
Zm00001d038644_ -------GAGGWDASV--------ANWNKSSVL--------------------ADAQDEG
Zm00001d010719_ -------GAGGWDVSV--------GNWNKSSVV--------------------ADAQDEG
BGIOSGA020147-P RTSKQSHSTGGWDAAA--------ANLNESSDV---------------------DAHKDS
Os05t0506200-01 RTSKQSHSTGGWDAAA--------ANLNESSDV---------------------DAHKDS
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     WGKKDDGGS<span class="highlight">S</span><span class="highlight">W</span>-----GKEDDGGSLWGKKDDGES<span class="highlight">S</span><span class="highlight">W</span>GKKDDGES<span class="highlight">S</span><span class="highlight">W</span>G-KKDDGGS---<span class="highlight">S</span><span class="highlight">W</span>
Bra009470.1-P   WGKKDDGGP<span class="highlight">S</span><span class="highlight">W</span>NKKDDGNKDDGGSAWGKKVESGS<span class="highlight">S</span><span class="highlight">W</span>RNKDGG-S<span class="highlight">S</span><span class="highlight">W</span>G-NK-DGGS---<span class="highlight">S</span><span class="highlight">W</span>
KEH44273        WGTKSTSTPTP-------------AWGAAVNIE-TTSEPDQS-GGWG-TKSTSTPT-PAW
Pp3c16_360V3.2  ----------------------------------GAFDNEGG------------------
KRH35991        SGVKEVGNPDGTSDIDANQDVG---WKSQ-----PTKDGSGS-SGWG-TK-------NNW
KRG90810        SGVKQAGNPAGTSDIDANQDAG---WKNK-----PNKDGSES-SGWE-TK-------NNW
KRG90770        SGVKQAGNTAGTSDIDANQDAG---WKNK-----PNKDGSES-SGWE-TK-------NNW
Zm00001d038644_ WGKGN<span class="highlight">W</span><span class="highlight">S</span>SAKS---EETNNEIGN--WNKA-----GASDQAG-------PKSFGGD---GW
Zm00001d010719_ WGKGN<span class="highlight">W</span><span class="highlight">S</span>SAES---EETNNQSGN--WNKT-----GASDQAGY-NNWDKPKSFGGN---GW
BGIOSGA020147-P WGKAKDTTANA---EEKNSESGN--WNKA-----GPLDKVCG-SDWGSPKFSSGDGPS<span class="highlight">S</span><span class="highlight">W</span>
Os05t0506200-01 WGKAKDTTANA---EEKNNESGN--WNKA-----GPLDKVCG-SDWGSPKFSSGDGPS<span class="highlight">S</span><span class="highlight">W</span>
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     GKKDEGGY--SEQTFDRGGRGFGGRRGGGRRGGRDQFGRGSSFGNSEDPAP<span class="highlight">W</span><span class="highlight">S</span>KP-----
Bra009470.1-P   AKKDDGGY--SEQTFDRGGRGFGGRRGGGRRGGRDQFGRGRSFGHSEDQAP<span class="highlight">W</span><span class="highlight">S</span>KPGG---
KEH44273        GAAVNTGT--TSEPDQSGGWGKGGGSSGQ-----------------------SEP---ND
Pp3c16_360V3.2  ------------------------------------------------------------
KRH35991        NAPVSSSHDKQGKGNDQGGWNAGKASGGS---------------------------AADF
KRG90810        NAPVSSSNDKVEKGNDQGGWNAGKASGGS---------------------------AVDF
KRG90770        NAPVSSSNDKVEKGNDQGRWNAGKASGGL---------------------------AADF
Zm00001d038644_ NKGGD-----QNS<span class="highlight">S</span><span class="highlight">W</span>NSPG-NFGGGRGFGRGRGRVRGRESGDFNGRNDQGNWKN<span class="highlight">S</span><span class="highlight">W</span>GGDN
Zm00001d010719_ NKGDD-----QNS<span class="highlight">S</span><span class="highlight">W</span>NRPE-NFGGGRGFGRGQGRGRGQQSGYFNGRNDQG<span class="highlight">S</span><span class="highlight">W</span>KK<span class="highlight">S</span><span class="highlight">W</span>GSDN
BGIOSGA020147-P NKGDKIGGDSQNG<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>RPGGNFEGGRGFGRGRGRGRGRESGDFGGRNDQG<span class="highlight">S</span><span class="highlight">W</span>KNSGTSDS
Os05t0506200-01 NKGDKIGGDSQNG<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>RPGGNFEGGRGFGRGRGRGRGRESGDFGGRNDQG<span class="highlight">S</span><span class="highlight">W</span>KNSGTSDS
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     SGGS<span class="highlight">S</span><span class="highlight">W</span>----------------------GKQDGDGGGSS----W------------GKEN
Bra009470.1-P   GGGS<span class="highlight">S</span><span class="highlight">W</span>----------------------GKQDSGGGGSS----W------------GKDN
KEH44273        NQNTAW----------------------GTKSTSTPKPS----W------------GAAV
Pp3c16_360V3.2  -------------------------------------------WVMPDIVVTIRKLGEET
KRH35991        CQASGWKG--------------------GLSENAQEGSN----W-------GIKKFGSGT
KRG90810        SQASGWKG--------------------GLSEHTQEGSN----W-------GDKKFGSCD
KRG90770        SQASGWKG--------------------GLSEHTQEGSN----W-------GDKKFGSCD
Zm00001d038644_ AEKS<span class="highlight">S</span><span class="highlight">W</span>RSDNQVDNEAGDSGGYRGRGRGGRGQYGGRGRGRDNGWRN--------------
Zm00001d010719_ AERP<span class="highlight">S</span><span class="highlight">W</span>RSDNQVDNEAGESGGYRGRGRGGRGQYG--GRGKDNGWRNDD--RSNSGFGREN
BGIOSGA020147-P SGRP<span class="highlight">S</span><span class="highlight">W</span>RSDSQVEKEVGDSGGYRGRGR-GRGQYGGQGRG-DNGWRNGG--QSNSEFGRSD
Os05t0506200-01 SGRP<span class="highlight">S</span><span class="highlight">W</span>RSDSQVEKEVGDSGGYRGRGR-GRGQYGGQGRG---------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     DAGGGSS------WG-----------KQDN------------------------------
Bra009470.1-P   DAGGGSS------WG-----------KQNNAGGGGGSS-------------<span class="highlight">W</span><span class="highlight">S</span>KDNDAGG
KEH44273        NTGTTSEPDQSGGWG---------------K--VGGSS-------------GQTELNDNQ
Pp3c16_360V3.2  QTAVIREVMPVRVIGFLTFF---AYI----------------------------------
KRH35991        VSGDSGGNLG-NDWG-----------PKSN-----WNS-------------GSRSGNENQ
KRG90810        ACGDSSGNQGSNGWG-----------QKSN-----WNS-------------GSRSGNETQ
KRG90770        VSGDSSGNQGSNGWG-----------QKSN-----WNS-------------GSRSGNENQ
Zm00001d038644_ --GDGPK------WG-----------GGDN-----WNATNPPSNQP<span class="highlight">W</span><span class="highlight">S</span>SSGGTKSYGQNQ
Zm00001d010719_ NSGDGPK------WG-----------GGGN-----WNATNPP-----SSSVGTKSYGQNQ
BGIOSGA020147-P --ADAPN------WGNKGVSNKG<span class="highlight">S</span><span class="highlight">W</span>DSGDN-----WNA--------------PKSSDENQ
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     --------------------------------------GVGS<span class="highlight">S</span><span class="highlight">W</span>GKQNDGSGGGS<span class="highlight">S</span><span class="highlight">W</span>GK-
Bra009470.1-P   GS<span class="highlight">S</span><span class="highlight">W</span>GKQNNAGGGGS<span class="highlight">S</span><span class="highlight">W</span>G--QEKDAGGGS-SLGKQGSDGGGS<span class="highlight">S</span><span class="highlight">W</span>GKKNDTGGGGS<span class="highlight">S</span><span class="highlight">W</span>GQ-
KEH44273        NTAWGTKSSST----------PAPAWGATVNTGTTSEPDQSGGWGTKS-TSTPKP<span class="highlight">S</span><span class="highlight">W</span>GA-
Pp3c16_360V3.2  ------------------------------------------------------------
KRH35991        NSH<span class="highlight">W</span><span class="highlight">S</span>SGHTEPRNQDSNL--DKKNNWISV-NGANLASDPKSSKWNSGSGNSNENSNWGN-
KRG90810        NSH<span class="highlight">W</span><span class="highlight">S</span>SGRNEPGNQDSNL--DKKSNWNSG-NSGNLASDPKSSNWNSGY----ENSNWGT-
KRG90770        NSH<span class="highlight">W</span><span class="highlight">S</span>SGRNEPGNQDSNL--DKKSNWNSG-NSGNLASDPKSSNWNSGSGNSNENSNWGT-
Zm00001d038644_ PSTW---NNPEDNKPSVG--EQGDPWASK-VSSTEGKGQQND<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>SKMTSRGAEDTSGGW
Zm00001d010719_ PFTW---NNSEDNKPSVG--EQGDPWASK-VSSTEDTEQQND<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>SKMISGGAEDTSGGW
BGIOSGA020147-P TSAW---NSSEDKRPSGGQEQQSGAWASK-MTSTAGAEDKSDAWGTKA-EGNSGSTGGKW
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     -----QNDAGGGSS------WG-----KQDSGGDGS<span class="highlight">S</span><span class="highlight">W</span>GKQ-DGGGDSGSAWGKQNNTSG
Bra009470.1-P   -----DNNAGGGSS------WG-----KQASDGGGS<span class="highlight">S</span><span class="highlight">W</span>GKKNDAGGGGGS<span class="highlight">S</span><span class="highlight">W</span>GKDNDAGG
KEH44273        -----AVNTGTTSEPDQSSGWG----AKSTSDVNNSNW--------------------NS
Pp3c16_360V3.2  --------------------<span class="highlight">W</span><span class="highlight">S</span>--------------PW----------------------
KRH35991        -------NGNNKSS------WG------AGNENKNSN<span class="highlight">W</span><span class="highlight">S</span>SGRNGPEDQESNQGKKSNWNS
KRG90810        -------NVNNKSS------WG------TGNENKNS<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>SGHSDPGNQDANQGKKSNGNS
KRG90770        -------NVNNKSS------WG------TGNENKNS<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>SGHSDPGNQDANQGKKSNWNS
Zm00001d038644_ DSKEKGSSCNGGGEGLQNDPWANKIGSNEGKEQEADPWASKMSSAVGADDKNGTWDTTTK
Zm00001d010719_ DTKVKDSSCNDGGEGLQNDPWANKIGSNEGKEQETDPWASKMSSTVCADDKKG<span class="highlight">S</span><span class="highlight">W</span>NTTAK
BGIOSGA020147-P EN-------ASSGEEQQSDPWGSKICSNKGKEQETDPWTSKVTSTADADDNNNVWNTSAS
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     GS<span class="highlight">S</span><span class="highlight">W</span>GKQSDAGGGS<span class="highlight">S</span><span class="highlight">W</span>GKQ-DGGGGG-----S<span class="highlight">S</span><span class="highlight">W</span>GKQ--DG----------GGGSGSAWG
Bra009470.1-P   GS<span class="highlight">S</span><span class="highlight">W</span>GKQGSDGGGS<span class="highlight">S</span><span class="highlight">W</span>GKKNDAGGGG-----S<span class="highlight">S</span><span class="highlight">W</span>GKQGSDGGGS<span class="highlight">S</span><span class="highlight">W</span>GKKSDGGGGGS<span class="highlight">S</span><span class="highlight">W</span>G
KEH44273        GDS--NKTSNS-GKNSNWNNSSGGAWNENKTSNW--------------------SSGGAS
Pp3c16_360V3.2  -----------------------------DIAAW--------------------------
KRH35991        GDS-DNQASDPNNSNWNSNKS<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>AGNENRNSN<span class="highlight">W</span><span class="highlight">S</span>SGDPGNKDSNWGKKSNWNSGSGDAN
KRG90810        GNS-GNQPSDP-NSNWNSNKS<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>AGNENKKSN<span class="highlight">W</span><span class="highlight">S</span>SGDPGNKDSNWGNKNNCISGSGDAN
KRG90770        GNS-GNQPSDP-NSNWNSNKS<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>AGNENKKSN<span class="highlight">W</span><span class="highlight">S</span>SGDPGNTDSNWGNKNNCISGSGDAN
Zm00001d038644_ GTLSNEKADDP------WNSKRGNDDDSKKTDAWG------------------AGSSGGN
Zm00001d010719_ GIPSKEKADDP------WNSKGGN-DDSKKTDTWG------------------AGSSGGD
BGIOSGA020147-P DTASGSES------------KWGNAGAEEKADAWKSK-------------VGSENSGGWN
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     KQNETSNG--------S<span class="highlight">S</span><span class="highlight">W</span>GKQN-DSG------GGS<span class="highlight">S</span><span class="highlight">W</span>GKQD--------GGGGGS<span class="highlight">S</span><span class="highlight">W</span>--
Bra009470.1-P   QENNAGGG--------S<span class="highlight">S</span><span class="highlight">W</span>GKQG-SDG------GGS<span class="highlight">S</span><span class="highlight">W</span>GKKNDA------GGGGGS<span class="highlight">S</span><span class="highlight">W</span>--
KEH44273        EGTNWRSN-----------------------------EGPNETS------DDGGGG--R-
Pp3c16_360V3.2  ------------------------------------------------------------
KRH35991        QNTTWGSN--------S<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>TANASSD------AGVNEGSNENS------DGVGGGNWR-
KRG90810        QNTTWRSN--------S<span class="highlight">S</span><span class="highlight">W</span>NTANASSD------DG-NEGSNENS------DGVGGGNWR-
KRG90770        QNT<span class="highlight">S</span><span class="highlight">W</span>RSN--------S<span class="highlight">S</span><span class="highlight">W</span>NTANASSD------DG-NEGSNENS------DGVGGGNWR-
Zm00001d038644_ QES<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>KPIFSLGDQEPTWNNPRFGDDDNVNNRGGFGRGNQGRGRGRSFGGGGGGS<span class="highlight">S</span><span class="highlight">W</span>N-
Zm00001d010719_ QEL<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>KPNFSLGDQESTWNNPRFSDDDNANNRGGFGRGNRGRGRGRNF-GDGGGS<span class="highlight">S</span><span class="highlight">W</span>N-
BGIOSGA020147-P QKS<span class="highlight">S</span><span class="highlight">W</span>GKPSFSGGEQEPA<span class="highlight">W</span><span class="highlight">S</span>NPKNGDDNSGYGRGGFGRGNRGRGRGRNF--GDSGS<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>G
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     -GKQ------------------NDGGG--GS<span class="highlight">S</span><span class="highlight">W</span>GKQGD-----------GGSKPWNEHSG
Bra009470.1-P   -GQQ------GGGGSAWANKQNNDGGG--GQ<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>KQDD-----------GGSKPWGEQS-
KEH44273        GGYR------GRGG--------SDSGGYRGRGFRGRVER----------GGFGGRGEQ-G
Pp3c16_360V3.2  --LR------------------DDGSCRVALGPTGDGDTLLV------------------
KRH35991        GGYQ------GRGS--------SDRGGFRGRGFRGRGERGGFGGRAERGGGFGGRWERGG
KRG90810        GGYR------GRGG--------SDRGGFRGRGFRGKGERGGFSGRGEL-GGFGGRGEQ-G
KRG90770        GGYR------GRGG--------SDRGGFRGRGFRGRGERGGFGGRGEH-GGFGGRGER-G
Zm00001d038644_ ESNKNGESGGGRSGEPW-NRRGFD--GGRGRGRFGRGDR---------------------
Zm00001d010719_ GGNRNDESGGGRSGEPW-NRRDFDGGGGRGRGRFGRGDR---------------------
BGIOSGA020147-P GSYRNDESRGERSEDRW-NTRDSDGGRGRGRGHFGRGDR---------------------
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 --KR--------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     --GGRGFGERRGGG-----GFR-GGRNQSGR--GGRSFD-------------GGR-----
Bra009470.1-P   --GGRGFGGRRGGGF--RGGFR-GGRNQLGRDGGGRSFE---------------------
KEH44273        GFGGRGRSDGEGFGG--RWGSDRGSGGGRGR---GRNDR------------PGG<span class="highlight">W</span><span class="highlight">S</span>NSRD
Pp3c16_360V3.2  -----------------------------GQ-----------------------------
KRH35991        GFGGRGRSDREGCGG--RWGSE-GGRGGRGR---GRNDQ------------SGGWNNRRD
KRG90810        GFGGRGRSDGEGFGG--RWGSE-GGRGGRGR---GRNDQ------------SGGWNNRRD
KRG90770        GFGGRGRSDREGFGG--RWGSE-GGRGGRGR---GRNDQ------------SGGWNNRRD
Zm00001d038644_ ---NQGNNLGSGDGG--<span class="highlight">S</span><span class="highlight">W</span>GSGRGNGGGRGG---YRNWNDNNERRPFG--QGGGW-----
Zm00001d010719_ ---NQGNNFGSGDGG--<span class="highlight">S</span><span class="highlight">W</span>GSGRGN-GGRGG---YRNWNDNNERS-FG--EGGG<span class="highlight">W</span><span class="highlight">S</span>----
BGIOSGA020147-P ---NQGNNYGSGDNNDRTWGSGRGNRDQDG----CKNWNQNDDRRPFGQDRGGDW-----
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     -----SS<span class="highlight">S</span><span class="highlight">W</span>KTDNQE-----------NTWKS-DQSGG-------SDWKKGWGEDSNNSK-
Bra009470.1-P   -----SSGWKRDNQE----------NTTWKSNNQSGG-------SDWKK<span class="highlight">S</span><span class="highlight">W</span>GEDSNTAK-
KEH44273        SGEDGSSDWKKGSDN----------VEGWKSNNGSG-------------<span class="highlight">S</span><span class="highlight">W</span>NQNSSDKK-
Pp3c16_360V3.2  --------------------------------------------ADMELVLPKKTDKIKI
KRH35991        SGEDGPSDWKKGAEN-----------GGWKNGNGSQ-------------AWNQDTGDKDR
KRG90810        SGEDGPSDWKKGADN-----------GGWKNSNGSQ-------------AWNQDTGDKDR
KRG90770        SGEDGPSDWKKGADN-----------GGWKNSNGSQ-------------AWNQDTGDKDR
Zm00001d038644_ ---SQSSDWNANKGSGEGNQAFSKSKP<span class="highlight">S</span><span class="highlight">W</span>DAQNASGGDDQAGKKSDTDN<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>QNRSSPSI
Zm00001d010719_ --QSQSSDWNANKGNGEGNQAFSKSKP<span class="highlight">S</span><span class="highlight">W</span>EAQNTSGRDGQAG-KGDANN<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>KNRSSPS-
BGIOSGA020147-P ---SQSSDWNANKGQ-----------S<span class="highlight">S</span><span class="highlight">W</span>GAPKPSGGDDQAG-KNNGNNPWGAPADSNK-
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     ------------PSG-SSAGGCA--GNWP<span class="highlight">S</span><span class="highlight">W</span>DTNS-KKETN-------------------
Bra009470.1-P   ------------PSD-SSSAG----GNWG<span class="highlight">S</span><span class="highlight">W</span>DANS-KKETNAGGGWGA-NSKNE------
KEH44273        ----------DGGNW-TSASG----GASG<span class="highlight">S</span><span class="highlight">W</span>NS---------------------------
Pp3c16_360V3.2  VS--------------GEHRGCT--GKL--------------------------------
KRH35991        Q<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>QGNADKERP<span class="highlight">S</span><span class="highlight">W</span>-NQGS-----GSNQ<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>SASAGANDN----WN-------------
KRG90810        Q<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>QGNADKEHP<span class="highlight">S</span><span class="highlight">W</span>-NQGS-----GRNQ<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>SAS-GANDNGSQAWNQGNADKEHPS---
KRG90770        Q<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>QGNADKEHP<span class="highlight">S</span><span class="highlight">W</span>-NQGS-----GRNQ<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>SAS-GANDNGSQAWNQGNADKEHP<span class="highlight">S</span><span class="highlight">W</span>NQ
Zm00001d038644_ LGQPSGDANNKSSAWDGAPNGGAGRGGSG<span class="highlight">S</span><span class="highlight">W</span>GKSD-ED------GWNS------------
Zm00001d010719_ ------------SAW-GAPSGGA--GGRG<span class="highlight">S</span><span class="highlight">W</span>GKSN-ED------GWN-------------
BGIOSGA020147-P -GHPRSVV--------AAAPG---------------------------------------
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------VGKS--------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     -------------------------------DKPGD--DSK--SAWGT-SNDQVNTDNNN
Bra009470.1-P   --------TSTGGD---WGSNSKKETNAVDGDKPSN--ENKAAAAWGTPANDQENAGNNN
KEH44273        -------------------------------DRPGQVAEAKDGAGWAK-GTDS------G
Pp3c16_360V3.2  ----------MGID-----------------------------------GADG-------
KRH35991        ------------------------------NNGSTQTAE----A<span class="highlight">S</span><span class="highlight">W</span>KK-STTTEETNIQD
KRG90810        ------------------------------------------------------------
KRG90770        AGNKQ<span class="highlight">S</span><span class="highlight">W</span><span class="highlight">S</span>SASGGGNNNW-----------NNNGSSQTAE----A<span class="highlight">S</span><span class="highlight">W</span>KK-S-TTEGTNIQG
Zm00001d038644_ ----------SSGG------------------------SAREKP<span class="highlight">S</span><span class="highlight">W</span>GG-GAEAPPRKEDD
Zm00001d010719_ ----------SAGG------------------------SAKEKPAWGG-GSEASTKKED-
BGIOSGA020147-P --------TRQMVD-----------------------------------GTAAARVGPQA
Os05t0506200-01 ------------------------------------------------------------
VIT_14s0006g021 ------------------------------------------------------------
VIT_19s0093g001 ------------------------------------------------------------
VIT_13s0019g049 ------------------------------------------------------------
VIT_00s0131g004 ------------------------------------------------------------
                                                                            

AT5G04290.1     D<span class="highlight">S</span><span class="highlight">W</span>NKKPNNDV-----GTSGEAD------NAWGGKTNAVAPSPSGSAAWGTGDKKTGW
Bra009470.1-P   DGWGKKPSDDV-----KTSGAAD------NAWGGKTNAGASSSSGSA----------W
KEH44273        AGWAK-----------GTDSASDKG---QS<span class="highlight">S</span><span class="highlight">W</span>NGSAADGAAG<span class="highlight">S</span><span class="highlight">W</span>GKK--NDGGSKGGW
Pp3c16_360V3.2  --------------IVKLDDTLDIRILDMSSLSKIA---------------------T
KRH35991        SGWNKGSSSNTTSRGWGQSSAADKG--QAS<span class="highlight">S</span><span class="highlight">W</span>KESAIDDAAG<span class="highlight">S</span><span class="highlight">W</span>GKK--NDRGDKGGW
KRG90810        ----------------------------------------------------------
KRG90770        GGWNKGSSSNTTSKDWGQSSAADKG--QAS<span class="highlight">S</span><span class="highlight">W</span>KEAA-DGASG<span class="highlight">S</span><span class="highlight">W</span>GKK--NDGGDKGGW
Zm00001d038644_ GAWGKGGEGSR-----SLGGGGEGG--SSS<span class="highlight">S</span><span class="highlight">W</span>DKPADGA<span class="highlight">W</span><span class="highlight">S</span>GNNKGGS-----GSGGW
Zm00001d010719_ ----KGGEGNG-----SQGGGG------VS<span class="highlight">S</span><span class="highlight">W</span>DKGADDAWNGNTGSDI-----GSGGW
BGIOSGA020147-P MTYNAIISNNVAML-------------------------------------------L
Os05t0506200-01 ----------------------------------------------------------
VIT_14s0006g021 ----------------------------------------------------------
VIT_19s0093g001 ---------------------------------------------------------L
VIT_13s0019g049 ---------------------------------------------------------R
VIT_00s0131g004 ----------------------------------------------------------
                                                                          


<a href='WS_SW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            