
<!doctype html>
<html>
    <head>
        <title>WT-TW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT1G26680.4</h1>
        <h3>AT1G26680.4 pep chromosome:TAIR10:1:9219463:9223749:1 gene:AT1G26680 transcript:AT1G26680.4 gene_biotype:protein_coding transcript_biotype:protein_coding description:transcriptional factor B3 family protein [Source:TAIR;Acc:AT1G26680]</h3>
        <pre>
length  992
count   6
ratio   0.00
zscore  6.19






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT1G26680.4     -------------MADQSLL--------HSPINPHFF---------------QPILTESR
Bra024695.1-P   -------------MADDQSL--------HSPITPHFF---------------QPLLPGFH
Bra024697.1-P   MK--------CSRMAYDQSL--------HSPNNPHFF---------------QPLLPGFH
Bra024696.1-P   -------------MAH-QSL--------HSPTNPHFF---------------QPLLPGFH
Bra012445.1-P   -------------MVDQSLLPASSSRLPHSP----------------------PCLVQ--
Zm00001d004358_ MPLPSDSGSGARAMSKQFKVLLPTSF--HKLVSTAFLDRRSISVADVSLSSRNPLIPQ--
                             *     :        *.                       * :    

AT1G26680.4     THLNIPVAFFSKHVEGRNNQNKTVTLRSDASDK<span class="highlight">T</span><span class="highlight">W</span>LVKM----DGLKLTDGWEDFAFAHD
Bra024695.1-P   THLNIPVAFFSRHVQGNNDHIKTAKLRTDASAK<span class="highlight">T</span><span class="highlight">W</span>LVKI----NGLKLTDGWEDFAVAHD
Bra024697.1-P   TYLNIPVAFFLKHVQGSNDHIKTAKLTTDASAK<span class="highlight">T</span><span class="highlight">W</span>LVKV----DGVRLTDGWEDFAVGHD
Bra024696.1-P   TYLNIPVAFFSKHVEGSNDHIKTAKLRTDASAK<span class="highlight">T</span><span class="highlight">W</span>LVEM----DGLKLTDGWEDFAVAHD
Bra012445.1-P   ---NIPVAFFSTHIKGRNDHIKIAKLRTDASDK<span class="highlight">T</span><span class="highlight">W</span>KVKM----DGSKFTDGWEDFAVAHD
Zm00001d004358_ -RISDELAGCFDAGEGEGAPEPTA-LVVSPFGKVWRVEVVRDGHGASLGRGWAEFLAAHG
                   .  :*      :* .     . *  ..  *.* *::    .*  :  ** :*  .*.

AT1G26680.4     LRTGDIVVFRLEGEMVFHVTALGPSCCEIQYHT-SSHNINDDDRNDQINIASRNSSRVKK
Bra024695.1-P   LRIGDIIIFRYEGEMVFHVTAFGPSCCDIQYTSESSHNINDDSQDQTNN-----------
Bra024697.1-P   LRIGDITIFRHEGEMVFHVTAFGPSCCDIQYTSESSHNINDDSQDQTNNAV---------
Bra024696.1-P   LRIGDIIIFRHEGKMVFHVTAFGPSLCEFQYTSASSHNINDVSDDDHTNK----------
Bra012445.1-P   LRVGDITVFRHEGEMVFHVTALGPSFCEIQYT--SSHNINDDTHDQTNNIVTGNSSREK-
Zm00001d004358_ VGIGWFVVLRYEGGGALTVKVFDTSLCIKEFGAPAA------------------------
                :  * : ::* **  .: *..:..* *  ::   ::                        

AT1G26680.4     NPRKKVESSLDHSRFVAKVSAWCLSNDRLYIPLSFARLNGLNKINSKKIYLQNEEGRSWK
Bra024695.1-P   ------------------------------------------------------------
Bra024697.1-P   ------------------------------------------------------------
Bra024696.1-P   ------------------------------------------------------------
Bra012445.1-P   ------------------------------------------------------------
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT1G26680.4     LVLRHDKSGMQTFVQSGWRRFCSENGIRQGQYTFKLVRKSAPPVIRLCRAKAKPKQRSVA
Bra024695.1-P   ------------------------------------------------------------
Bra024697.1-P   ------------------------------------------------------------
Bra024696.1-P   ------------------------------------------------------------
Bra012445.1-P   ------------------------------------------------RKRVKKNPKPVP
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT1G26680.4     EYSSDHSCFEGSVTPSSLRNDLLYLPRSFVNSNRLDKRCSEIVLKNEQGVKWPLVLKRFK
Bra024695.1-P   ------------------------------------------------------------
Bra024697.1-P   ------------------------------------------------------------
Bra024696.1-P   ------------------------------------------------------------
Bra012445.1-P   QASLHSSCYVGSVSASSLKHNKLYLGREFVTSNGLNKGCSEIVLKNEGGGR<span class="highlight">W</span><span class="highlight">T</span>LPLKHYK
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT1G26680.4     SV--TYLPKG<span class="highlight">W</span><span class="highlight">T</span>SFCQVNRIKAGDSFKFKLVG<span class="highlight">T</span><span class="highlight">W</span>KKPVLSLCPTQSNNH-KTPLECSEGN
Bra024695.1-P   ------------------------------------------------------------
Bra024697.1-P   ------------------------------------------------------------
Bra024696.1-P   ------------------------------------------------------------
Bra012445.1-P   SINHTYLGPG<span class="highlight">W</span><span class="highlight">T</span>TFCQVNGIKAEDAFMFKLVRTGEKPVLCLCPEESSHRDKTPVECLE--
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT1G26680.4     KSEESEEDCLEVKKKKYWSRCRASVENMDDDQTNIGNSSRKKRVSKNP----REKVESSS
Bra024695.1-P   ----------------------------------TGNSSGEKR-----------------
Bra024697.1-P   ----------------------------------TGNSSGEKR-----------------
Bra024696.1-P   ----------------------------------TGNSSGEKR-----------------
Bra012445.1-P   ----------------------------DSDDVNPLSSSSKSKESKEESLGDKRASSSYS
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT1G26680.4     DHSSFVGSVNPSSLYKDQLYLPRNFVSSNFLDKRCSEIVLK-NERGEKRTLVLKHFKKDL
Bra024695.1-P   ------------------------------------------------------------
Bra024697.1-P   ------------------------------------------------------------
Bra024696.1-P   ------------------------------------------------------------
Bra012445.1-P   EDRFLTLTLTQKAVKRYQLTIPVAFFLKHVQGRNDHIKIAKLTTAASDK<span class="highlight">T</span><span class="highlight">W</span><span class="highlight">T</span>V---KMDG
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT1G26680.4     TFLKKG<span class="highlight">W</span><span class="highlight">T</span>SFCQVNRIKAGDSFKFKLVG<span class="highlight">T</span><span class="highlight">W</span>NKPVLSLCPTETNYHKTPLACSEGNKSEES
Bra024695.1-P   ------------------------------------------------------------
Bra024697.1-P   ------------------------------------------------------------
Bra024696.1-P   ------------------------------------------------------------
Bra012445.1-P   FKLTHGWEDFAVAHDLRVGDMIVFRHEG------------ELAFHVTIMGPS--------
Zm00001d004358_ ------------------------------------------------------------
                                                                            

AT1G26680.4     EEEGTEDKNTSQDCLEVKKRKYWSTCRASAENIDDDQTNIGNSSKEKRVKKNPVKKAESS
Bra024695.1-P   ----------------------------------------------KRVKKTPRTKEESS
Bra024697.1-P   ----------------------------------------------KRVKKNPRTKEDSS
Bra024696.1-P   ----------------------------------------------KRVKKNPRTKEESS
Bra012445.1-P   -------------CCEIQ----------YTHNINHNHTN--------------NIIESSS
Zm00001d004358_ --------------------------------------------------VMPSKSCKSI
                                                                         .* 

AT1G26680.4     SDHSSFVANVTASSLNYDRLYLPLSFVSSNGLDKMNGKE-IVLLNEEGLSWKFNLKYNQA
Bra024695.1-P   SDHSHFMAHVSPSSLRFDRLYVPISFARSNGLDNMSGKE-IVLLNQEGRSWNLNITYTKA
Bra024697.1-P   SDHSQFVAHVSPSSLSSDRLYIPTGFARSNGLDNMSGKEIIVLLNEEGRSWSLDMTCNKV
Bra024696.1-P   SDHSHFVAHVTPSSLSCDRLYVPKGFAMSNGLDNMSGKE-IVLLNEEGKSWNLNITYNES
Bra012445.1-P   SDNSLFVAKVSASNLRLDRLYLPMSFARTNGLDKMSGEQ-IILLNGEGRSWSLKLKHDKS
Zm00001d004358_ FCKPQFI-RIFHSNLS-EKMMLPARFVKDYVSEECLNSRIAIIFSPIAKFWRVELKNDQS
                  :. *: .:  *.*  ::: :*  *.     ::  ...  :::.  .  * ..:.  : 

AT1G26680.4     GKHTYVRPGWSRFCDANGMSQGQ---------------------------------QFTF
Bra024695.1-P   SVQTLVGPGWRRFCAENGMNKGH---------------------------------HYTF
Bra024697.1-P   GMQTFVRPGWRRFCAENGMKKGH---------------------------------HYTF
Bra024696.1-P   GMHTFVRPGWRRFCAENGISQGH---------------------------------HCTF
Bra012445.1-P   DMHTFVRPGWRRFCAENGMSHG----------------------------------HYTL
Zm00001d004358_ GI--FFTGGWSQFLDFHGIKNGEVLLLKYEGNMVFKFKAFGLSGCQKDFKNQNAGIQLNT
                .    .  ** :*   :*:.:*                                  : . 

AT1G26680.4     KLVQKHGPPVMYLSLSEH-RPK-------------S-ESSSHRSYFVGSVTASSIKKDK-
Bra024695.1-P   KLVGKSAPPVIRLSLSE----------------PAP-ESSLHHSNYVGSVSSNSLRKDK-
Bra024697.1-P   KLVRRSAPLIIRLSHAEH-EPE-----------PAT-ESSLNHSYYVGSVSPNSLSTDK-
Bra024696.1-P   ELVRKSAPPVLRLSRAEH-DPK-----------PAP-ESSLHHSYYVGSASSNSLRIDK-
Bra012445.1-P   KLIRNSGPPVIKLCGQVHNRPK-----------PVPVVSSLHHSCFVGSVTSNSLKTDK-
Zm00001d004358_ KKQQETPPIRKRKSNDEKSSSEENKRLKTPVTSPSPSDPSLEKPYQIGTSSWIKKRINTY
                :   .  *     .                     .  .* .:.  :*: :  .   :. 

AT1G26680.4     -----LYLWKSFVSSNGLDKGCKKIILKNKWGREWKLVLKHYKSNCFTI-IKRG<span class="highlight">W</span><span class="highlight">T</span>SFCQ
Bra024695.1-P   -----LYIQRKFVNANGL-KGLSEIVLKNEWGGSWNLGLRYYESLDHTY-LGPGWKTFCQ
Bra024697.1-P   -----L---RHFVTANGLEKGLSEITLKNEWGGSWILGFRHYEPQNHTY-LGPGWKTFCQ
Bra024696.1-P   -----LYLQMKFVTANGMKKGFSEIILKNEWGGRWSVRLRQYASSNRTY-LGPGWKTFCQ
Bra012445.1-P   -----LYLGKKFVIENGLDRGCSEMVLKNEWGGRWSLALRFYESAYQTY-LGPGWRTFCQ
Zm00001d004358_ ALEQFLALSVKFCNSIGFRMACT-ITLKTATNSTRSWQVRGTAYKWHRYIIGVGWKSFCQ
                     *     *    *:  . . : **.  .      .:          :  ** :***

AT1G26680.4     GNGLKAGDSFKFKLVGTG------EKPVLSLCPAESSHEKIPLECPEGIDDVNSLSSNPS
Bra024695.1-P   VNRIKTGDSFMFKVVETG------DKPVLLLC--TTNRGKTPLDCPEDSGDINSLSSDTG
Bra024697.1-P   VNGIKAEDSFMFKLVETG------DKPVLLLC--TSNRGKTPLECSEDRDDVNSLSSDTS
Bra024696.1-P   VNGIKAGDPFMFKLVETG------DKPVLLLC--SSNRGETPLECS---DDVNSLSSDTS
Bra012445.1-P   VNEIKVGDSFKFKLVGTG------DNPVLLLC--TR---KMPLECPGE-----NSGSDTS
Zm00001d004358_ DNRLKAGDLCTFNIIETTLWHVTITRSALAAD--TFNKQKKP-PCSSSRDKKGSSS----
                 * :*. *   *::: *        ...*          : *  *.       . .    

AT1G26680.4     SGDDS-SRSEESEEENMEDKN-------ISQDCLETKKRKYC----SSSSYSQNRFVTLT
Bra024695.1-P   S-----RESQESEEESLEDDD------SSSEESFEMVKRENSSRCRASSSYSQDRFVKLT
Bra024697.1-P   SEDDSKEESQESDKESIEDAD-------RSQQCIVMEKENNSLRCICSSPYSKHRFVRLS
Bra024696.1-P   SG----EESQESEEESLEDDS-------RSKDCVEMEKRKSSLRCKCSSSYSKHRFVRLT
Bra012445.1-P   SGDDS-SEPQESEEESLGDKN-------ISQECVKMEKNEEF----VEMQNTKRRSL---
Zm00001d004358_ ------REGAKRPKPSMAHFSKVQVPPSYARRCAYDVGPPPWIRKQMNASTIRNRLVRMV
                       .  :  : .: . .        :. .              .    : * :   

AT1G26680.4     -LTRSAFQTYKLFLPRGFTQVNGINKPRKITLLGQDGVKK----VVDLYQESSSRIMRFG
Bra024695.1-P   -LTPRALKTYKLMLPLDFTRVNGINKPGKRM-----------------------------
Bra024697.1-P   -FTQSALKTSVLYLPLAFTRMNCINKPIKIMLLGKDGVKK---QVVDLLKNKSTGAMRFG
Bra024696.1-P   -LTQNALKTSVQHLPLGFTRVNGINRSRKIMLLGKDGVKK---WVVDLLKNKSTGAMRIR
Bra012445.1-P   ------------------------------------------------------------
Zm00001d004358_ ILAFDSRKLVIGCLAPAF--CNQIELRRRCEIVLKTSINSNHSWRVGGLSQKK-GSYIIG
                                                                            

AT1G26680.4     KGWREFFDAQGVKIDDSFVLELIW-EKEASPVLKFCTKVPK
Bra024695.1-P   --------------------------ERVL-----------
Bra024697.1-P   KGWREFCDAHGVKVGESFQLELFREDEEAIPVLKFCTTT--
Bra024696.1-P   KGWRELCDAHGVKVGESFLLELIR-DEEAIPVLKFCTKL--
Bra012445.1-P   -----------------------------------------
Zm00001d004358_ PGWKKFCTENRVKVGDVCTFNVIETQLWHVVITRRC-----
                                                         


<a href='WT_TW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            