
<!doctype html>
<html>
    <head>
        <title>WT-TW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT1G56570.1</h1>
        <h3>AT1G56570.1 pep chromosome:TAIR10:1:21195644:21198487:1 gene:AT1G56570 transcript:AT1G56570.1 gene_biotype:protein_coding transcript_biotype:protein_coding gene_symbol:PCMP-E64 description:Putative pentatricopeptide repeat-containing protein At1g56570 [Source:UniProtKB/Swiss-Prot;Acc:Q9FXA9]</h3>
        <pre>
length  611
count   6
ratio   0.00
zscore  6.19






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT1G56570.1     MSI---TKLARSNAFKPIPNFVRSSLRNAGVESSQNTEYPPYKPKKHHILATNLIVSYFE
Bra027929.1-P   MGF---TKLVRSNAFKPIPNFVRSSLRIACVESDQNTE-SPYKPRKHHILATNLIISYFE
AES61474        MPMACSRKLLSSTNFRPIPFSVQNSLR--CIQND-----TPFNPKDLTGLTTDLIKSYFD
KRG90505        ------------------------------------------------------------
KRH33792        ------------------------------------------------------------
BGIOSGA003326-P MS------------------------------------------QKH---ATTFISSLCS
Os01t0299200-00 ------------------------------------------------------------
Os01t0299150-00 MS------------------------------------------QKH---ATTLISSLCS
VIT_14s0060g008 MST---RKLLSTTHFHPIPLIVRNSIQ--LVQNCTTPPNPPFIPKGPSVLATTLIKSYFG
VIT_05s0049g016 ------------------------------------------------------------
VIT_12s0035g005 ------------------------------------------------------------
                                                                            

AT1G56570.1     KGLVEEARSLFDEMPDRDVVA<span class="highlight">W</span><span class="highlight">T</span>AMI-----TGYASSNYNARAWECFHEMVKQGTSPNEF
Bra027929.1-P   KGLVEEARSLFDEMPERDVVA<span class="highlight">W</span><span class="highlight">T</span>AMI-----KGYASSIHNARAWECFREMMRQGTNPNEF
AES61474        KGSFEEAHTLFDEMPHRDVIA<span class="highlight">W</span><span class="highlight">T</span>SMI-----TGYTSCNHHSRAWNVFTNMLRDGVKPNAF
KRG90505        -------------MPQRNVV<span class="highlight">T</span><span class="highlight">W</span><span class="highlight">T</span>AMI-----TSNNSRNNHMRAWSVFPQMLRDG------
KRH33792        -------------MSHRNVV<span class="highlight">T</span><span class="highlight">W</span><span class="highlight">T</span>AMIWQQPYECMECVSSNAQRWG----------EPHCF
BGIOSGA003326-P RGAVCHARALFDEMPERDVVA<span class="highlight">W</span><span class="highlight">T</span>AML-----SGYASNGLRREALDVFRRMVAAGAAPNEY
Os01t0299200-00 ------------------------------------------------------------
Os01t0299150-00 RGAVCHARALFDEMPERDVVA<span class="highlight">W</span><span class="highlight">T</span>AML-----SGYASNGLRREALDVFRRMVAAGAAPNEY
VIT_14s0060g008 KGLIGEARTLFDEMPERDVVA<span class="highlight">W</span><span class="highlight">T</span>VMI-----AGYTSCNNHTHAWMVFCEMMNEELDPNAF
VIT_05s0049g016 ------------------------------------------------------------
VIT_12s0035g005 ------------------------------------------------------------
                                                                            

AT1G56570.1     TLSSVLKS-CRNMKVLAYGALVHGVVVKLGMEG-SLYVDNAMMNMYATCSVTMEAACLIF
Bra027929.1-P   TLSSVLTS-CRDMKVLAYGRLVHGVVIKLGMEG-SIYVDNALMNMYATCSATMDAACLIF
AES61474        TVSAVLKA-CKSLKALLCGKLVHGLAIKIGTQGSSIYVDNALMDMYATCCDSMDNARLVF
KRG90505        ------------VKALSCGQLVHSLAIKIGVQGSSVYVDNSLMDMYATCCDSMDRARMVF
KRH33792        HHIIFFEGLCKGLKALPCGQLVHALAFKIGVQGSSIYIDNALMDMYATCCNRMDHARMVF
BGIOSGA003326-P TLSSVLTA-CRGPCAPAMAMPLHAVAVRRGVDR-MPYVVNALIDSYASLAEGVVDARRLF
Os01t0299200-00 -----MII-C-------------------------PF-----------------------
Os01t0299150-00 TLSSVLTA-CRGPCAPAMAMPLHAVAVRRGVDR-MPYVVNALIDSYASLAEGVVDARRLF
VIT_14s0060g008 TISSVLKA-CKGMKCLSYGRLVHGLAIKHGLDG-FIYVDNALMDMYATCCVSMDDACMVF
VIT_05s0049g016 ------------------------------------------------------------
VIT_12s0035g005 ------------------------------------------------------------
                                                                            

AT1G56570.1     RDIKV-KNDV<span class="highlight">T</span><span class="highlight">W</span><span class="highlight">T</span>TLITGFTHLGDGIGGLKMYKQMLLENAEVTPYCITIAVRASAS---I
Bra027929.1-P   RFIKV-KNEV<span class="highlight">T</span><span class="highlight">W</span><span class="highlight">T</span>TLITGFTHLGDGIGGLKMYKQMLLENEDVTPHCITIAVKASAS---I
AES61474        EDIGT-KNAVS<span class="highlight">W</span><span class="highlight">T</span>TLITGYTHRRDAFGGLRVFRQMFMEEGELSPFSFSIAVSACAS---I
KRG90505        DDITT-KTDVC<span class="highlight">W</span><span class="highlight">T</span>TLITGYTHRGDAYGGLRVFRQMFLEEGALSLFSFSIAARACAS---I
KRH33792        DDITT-KTD----------------------------DEGALSLFSFSIAAKACAS---I
BGIOSGA003326-P DALGSGRTAAS<span class="highlight">W</span><span class="highlight">T</span>SMIAGYARWGQERTGLRLFKTMLKDGVELSTFACSIALHACTL---V
Os01t0299200-00 ----------SFYTICA------------------VQDGVELSTFACSIALHACTL---V
Os01t0299150-00 DALGSGRTAAS<span class="highlight">W</span><span class="highlight">T</span>SMIAGYARWGQERTGLRLFKTMLK--VWISPTLRKSSMLRCFVQFPV
VIT_14s0060g008 RGIHL-KNEVS<span class="highlight">W</span><span class="highlight">T</span>TLIAGYTHRDDGYGGLRVFRQMLLEEVELNPFSFSIAVRACTS---I
VIT_05s0049g016 ------------------------------------------------------------
VIT_12s0035g005 ------------------------------------------------------------
                                                                            

AT1G56570.1     DSVTTGKQIHASVIKRGFQSNLPVMNSILDLYCRCGYLSEAKHYFHEMEDKDLI<span class="highlight">T</span><span class="highlight">W</span>NTLI
Bra027929.1-P   DSVTTGKQIHSSVVKRGFHSNLPVMNSILDLYCRCGYLTEAKRYFHEMVDRDLI<span class="highlight">T</span><span class="highlight">W</span>NTLI
AES61474        GSSNLGKQVHAAVINHGFESNLPVMNAILDMYCRCRCASEAKQLFGEMTQKDTI<span class="highlight">T</span><span class="highlight">W</span>NTLI
KRG90505        GSGILGKQVHAEVVKHGFESNLPVMNSILDMYCKCHCESEAKRLFSVMTHKDTI<span class="highlight">T</span><span class="highlight">W</span>NTLI
KRH33792        GSGILGKQVNAAVVKHGFESNLPVMNSILDMYCKCHCASEEKQLFSEMTHKDTI<span class="highlight">T</span><span class="highlight">W</span>NTLI
BGIOSGA003326-P IDLCLGQQLHLQCIKKALDVNLAVVNSLIDMYCTCARILDARSLFDGTPERNLI<span class="highlight">T</span><span class="highlight">W</span>NTMI
Os01t0299200-00 IDLCLGQQLHLQCIKKALDVNLAVVNSLIDMYCTCARILDARSLFDGTPERNLI<span class="highlight">T</span><span class="highlight">W</span>NTMI
Os01t0299150-00 IVMCL--------VKTG-------------------------------------------
VIT_14s0060g008 GSHTFGEQLHAAVTKHGFESNLPVMNSILDMYCRCSCFSEANRYFYEMNQRDLI<span class="highlight">T</span><span class="highlight">W</span>NTLI
VIT_05s0049g016 ------------------------------------------------------------
VIT_12s0035g005 ------------------------------------------------------------
                                                                            

AT1G56570.1     SELERSDSSEALLMFQRFES-QGFVPNCYTFTSLVAACANIAALNCGQQLHGRIFRRGFN
Bra027929.1-P   SELERSDSSEALLMFQRFESQQGFVPNCYTFTSLVAACANIAALNCGQQLHGRIYRRGFN
AES61474        AGFETLDSYESLCIFSQMVS-EGFSPNCFTFTSVIAACANLAILYCGQQLHGGIIHRGLD
KRG90505        AGFEALDSRESLCIFSRMVS-E------------VGACANLAVLYCGQQLHGVIVRSGLD
KRH33792        AGFEALDSRESLCIF-----------------SAVGACANLAVLYCGQQLHGVI------
BGIOSGA003326-P AGYSQCDPLMALQLLLEMND-E---PNCFTLTSITSACADLAALRCGKQVHGAVLRRSYS
Os01t0299200-00 AGYSQCDPLMALQLLLEMND-E---PNCFTLTSITSACADLAALRCGKQVHGAVLRRSYS
Os01t0299150-00 ------------------------------------------------------------
VIT_14s0060g008 AGYERSNPTESLYVFSMMES-EGFSPNCFTFTSIMAACATLAFLNCGQQIHGRIIRRGLD
VIT_05s0049g016 ------------------------------------------------------------
VIT_12s0035g005 ------------------------------------------------------------
                                                                            

AT1G56570.1     KNVELANALIDMYAKCGNIPDSQRVFGEIVDRRNLVS<span class="highlight">W</span><span class="highlight">T</span>SMMIGYGSHGYGAEAVELFDK
Bra027929.1-P   KNVELANALIDMYAKCGDVPDSERVFAEIADRRNLVS<span class="highlight">W</span><span class="highlight">T</span>SMMIGYGSHGYGAEAVELFDE
AES61474        NNLELSNALIDMYAKCGNVADSHKIFSGM-RHTNLVS<span class="highlight">W</span><span class="highlight">T</span>SMMIGYGAHGHGKEAVDLFNE
KRG90505        NYLEISNALIYMYAKCGNIADSRKIFSKM-PCTNLVS<span class="highlight">W</span><span class="highlight">T</span>SMINGYGDHGYGKDAVELFNE
KRH33792        -----QNAM-----------------------HKFVS<span class="highlight">W</span><span class="highlight">T</span>SMMIEYGDHGYGKEAVELFNE
BGIOSGA003326-P DDLQMGNALVDMYSKCGSITNAKNVFDRM-GCKDKFS<span class="highlight">W</span><span class="highlight">T</span>SMIAGYGMNGYGNEAVQLFSS
Os01t0299200-00 DDLQMGNALVDMYSKCGSITNAKNVFDRM-GCKDKFS<span class="highlight">W</span><span class="highlight">T</span>SMIAGYGMNGYGNEAVQLFSS
Os01t0299150-00 ------------------------------------------------------------
VIT_14s0060g008 GNLALSNALIDMYSKCGNIADSHQVFGGM-SRRDLVS<span class="highlight">W</span><span class="highlight">T</span>AMMIGYGTHGYGEEAVELFDK
VIT_05s0049g016 ------------------------------------------------------------
VIT_12s0035g005 ------------------------------------------------------------
                                                                            

AT1G56570.1     MVSSGIRPDRIVFMAVLSACRHAGLVEKGLKYFNVMESEYGINPDRDIYNCVVDLLGRAG
Bra027929.1-P   MVSVGIRPDRIVFMAVLSACRHAGLVERGLKYFNAMESEYGIEPDRDIYNCVVDLLGRAG
AES61474        MVGSGIKPDKIVFMAVLSACSHAGLVDEGLRYFRLMTSYYNVAPDRDIYACVVDLLSRAG
KRG90505        M----IRSDKMVFMAVLSACSHAGLVDEGLRYFRLMTSYYNITPDIEIYGCVVDLFGRAG
KRH33792        IVRSGIKPDKMVFMAVLSACSHTGLVDEGMRYFILMTSYYDVTPDIEVYECVVDLLGRAG
BGIOSGA003326-P MIHAGVHPDHVVFLSLISSCSHAGLVDEGWNFFRSMINEYNLQPNKEVYGSVVNLLARAG
Os01t0299200-00 MIHAGVHPDHVVFLSLISSCSHAGLVDEGWNFFRSMINEYNLQPNKEVYGSVVNLLARAG
Os01t0299150-00 ------------------------------------------------------------
VIT_14s0060g008 MVRSGIRPDRVVFMAILSACSHAGLVDEGLRYFKLMVGDYNISPDQEIYGCVVDLLGRAG
VIT_05s0049g016 -----------------------------------MVGDYNISPDQEIYGCEVNLLGRAG
VIT_12s0035g005 -------------MGILSVCSHAGLVDEGLRYFKLMVGDYNISPDQEIYGCEVDLLGHVG
                                                                            

AT1G56570.1     KIGEAYELVERMPFKPDES<span class="highlight">T</span><span class="highlight">W</span>GAILGACKAHKHNGLISRLAARKVMELKPKMVGTYVMLS
Bra027929.1-P   RIGEAYELVETMPFKPDES<span class="highlight">T</span><span class="highlight">W</span>GAILGACKAHKHTGLISRLAASRVMELKPRMMGTYVMLS
AES61474        RVKEAYELIENMPFKPDESIWVALLGACKKYKQPS-IQKLAALKVLEMKPNKAGTYVLLS
KRG90505        RVKEAYQLIENMPFNPDESIWAALLGACKVHNQPS-VAKFAALRALDMKPISAGTYALIS
KRH33792        RVKEAYQLIESMPFNPNESIWAALLGACKVHKQPT----------LDMKPNSAGTYALLS
BGIOSGA003326-P RLREALDLIDTMPFAPDEYVWGALLGASKMHNNVE-MGRLAARKITEINPDDVKNYIMLA
Os01t0299200-00 RLREALDLIDTMPFAPDEYVWGALLGASKMHNNVE-MGRLAARKITEINPDDVKNYIMLA
Os01t0299150-00 ------------------------------------------------------------
VIT_14s0060g008 KVEEAYELIESMPFKPDECVWGPFLGACKAHTFPN-LGKLAAHRILDLRPHMAGTYVMLS
VIT_05s0049g016 KVEEAYELIESMPFKPDECVWGPFLRAYKAHRFPN-SRKLAAHRILDLRPNMAGTYVMLS
VIT_12s0035g005 KVEEAYELIESMPFKPDECVWGPFLGACKAHRFPN-SRKLAAHRILDLRPNMAGTYVMLS
                                                                            

AT1G56570.1     YIYAAEGKWVDFARVRKMMRMMGNKKEAGMSWILVENQVFSFAVSDKMCPNASSVYSVLG
Bra027929.1-P   YIYAAERKWGDYARVRKMMRMMGNKKEAGVSWILVENQVCSFAVSDKACPCAVSVYSVLG
AES61474        NFSAAEGNWADFASLRKLMRSTKSKKEVGRSWIELKNQVCSFIVGDIFDSSNKEVCEVLE
KRG90505        NIYAAEGNWDDFASSTKLRRGIKNKSDSGRSWIELKDQICSFVVGDRFVSSNEQVCEVLK
KRH33792        NIYAAEGNWADFASSTKL-RSIKNKSESGRSGLN--------------------------
BGIOSGA003326-P SIYAAGSKWGEYAFTRRSLRGIGSRKEAGISWIEVMDKMYSFTAADSSSPQVCLADEVLH
Os01t0299200-00 SIYAAGSKWGEYAFTRRSLRGIGSRKEAGISWIEVMDKMYSFTAADSSSPQVCLADEVLH
Os01t0299150-00 ------------------------------------------------------------
VIT_14s0060g008 NIYAADGKWGEFARLRKLMKRMGNKKETGRSWVEVGNHVYSFVVGDEVGSKIEGIYQVLE
VIT_05s0049g016 NIYAADGKWGSL------------------------------------------------
VIT_12s0035g005 NIYAADGKWGSL------------------------------------------------
                                                                            

AT1G56570.1     LLIEETREAGYVPELDSLVNDQEVGT
Bra027929.1-P   LVIEETREADF-----SLV--QEAGT
AES61474        LLIRHMKDAGYVLDLDCSAHDLEDET
KRG90505        LLMVHMKDA----DMDLI--------
KRH33792        --------------------------
BGIOSGA003326-P ILSQHMDDVGS-----EFCHIIFKAT
Os01t0299200-00 ILSQHMDDVGS-----EFCHIIFKAT
Os01t0299150-00 --------------------------
VIT_14s0060g008 NLIGHMKESGYVPDLDCLIYDLEDGT
VIT_05s0049g016 --------------------------
VIT_12s0035g005 --------------------------
                                          


<a href='WT_TW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            