
<!doctype html>
<html>
    <head>
        <title>WV-VW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT5G23800.2</h1>
        <h3>AT5G23800.2 pep chromosome:TAIR10:5:8022264:8024795:-1 gene:AT5G23800 transcript:AT5G23800.2 gene_biotype:protein_coding transcript_biotype:protein_coding gene_symbol:ATDUF10 description:DOMAIN OF UNKNOWN FUNCTION 724 10 [Source:TAIR;Acc:AT5G23800]</h3>
        <pre>
length  566
count   8
ratio   0.00
zscore  7.36






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT5G23800.2     MGRQVPSESLSLSEGCEVEISYKNNGNES<span class="highlight">V</span><span class="highlight">W</span>YKAI-LE----AKPNSIFKEE------LS
Bra033167.1-P   M-RSLPSERLSLMEGCQVEISYTNNRFKK<span class="highlight">V</span><span class="highlight">W</span>YSAT-IELERQTKPKSRFRQR-------C
VIT_07s0005g018 M----ESGTEIFSKGSLVEVSSDEDGFKGAWYVATILE----SPPKSASKKRSRALVEYQ
                *     *    : :*. **:*  ::  : .** *  :*    : *:*  ::.        

AT5G23800.2     VRLLKDDFSTPLNELRHKVLIRPIPPTNVQACIDIEIGTFVDADYKDAWWAGFVVKVIDD
Bra033167.1-P   VRVLKDDSLAPLTVFTHKASFRPIPTDRYEDRVEIKEGSIVDADHKDEW<span class="highlight">W</span><span class="highlight">V</span>GLVVKQVGD
VIT_07s0005g018 DLLVDDVGSKPLTEVVDTSFLRPLPPP--EADTNFCVNDIVDAFYRDGWWTGVITRISED
                  ::.*    **. . ..  :**:*.   :   ::  . :*** ::* **.*.:.:   *

AT5G23800.2     DKF<span class="highlight">W</span><span class="highlight">V</span>CFDSPPDIIQFDRNHLRPTLE<span class="highlight">W</span><span class="highlight">V</span>DEKIYSWWIIGSTRNSEFLKRLAEEPMFSPGT
Bra033167.1-P   DKFLVLFDTPSDIVLFKREQLRVHLD<span class="highlight">W</span><span class="highlight">V</span>DET---W<span class="highlight">W</span><span class="highlight">V</span>LPGRS-----VQLHEKPMFISGA
VIT_07s0005g018 SKCTVFFQNPPDEIQFDRSDLRVHKE<span class="highlight">W</span><span class="highlight">V</span>DGK----WIRPEKQRTTGL-------IFSPGA
                .*  * *:.*.* : *.*..**   :*** .    *:                 :* .*:

AT5G23800.2     IVELCSKRDEGEV<span class="highlight">V</span><span class="highlight">W</span><span class="highlight">V</span>PALVYKEF-----------------KENDEYRYIVK--DKPLIG
Bra033167.1-P   MVEVSDKIDKGEV<span class="highlight">V</span><span class="highlight">W</span><span class="highlight">V</span>PAIIIKEIVDDEDEDEEEKEEEEEEEEEDEKKYFVKVCVNPSSF
VIT_07s0005g018 AVEVTLNEQSSRDAWFPAIVRREI--------------------GLGSYVVQ--CQSLKN
                 **:  : :... .*.**:: :*:                    .   *.*:   :.   

AT5G23800.2     RSYKSRPSKTVDLRSLRPIPPPIRVKEYRLDEYIEVYHDGIGWRQGRVVKSEGGVMGSLF
Bra033167.1-P   EGIKKRPNKEVDMRSIRPRPPPFSAEELKLVDYIEVFH-GTSWRQGRVI-------GRVF
VIT_07s0005g018 SGEAGVLDVTVDNLHIRPSPPRLEGRKFGLLEKVDAFY-DYGWWNGIVTKVLTG------
                 .     .  **   :** ** :  .:  * : ::.:: . .* :* *            

AT5G23800.2     QNWCTLLLEATKKQLMFKQSDLRPLR<span class="highlight">V</span><span class="highlight">W</span>EDG<span class="highlight">V</span><span class="highlight">W</span>---------------------------
Bra033167.1-P   RGRCKVLLEATNKLLSFKISDIRPSK<span class="highlight">V</span><span class="highlight">W</span>EDG<span class="highlight">V</span><span class="highlight">W</span>---------------------------
VIT_07s0005g018 -RRYVVFFKHTNMEKEFIHSDLRPHME<span class="highlight">W</span><span class="highlight">V</span>DGK<span class="highlight">W</span><span class="highlight">V</span>GATQEVLGTTYSDEQLGLVLNNSNNT
                     :::: *:    *  **:**   * ** *                           

AT5G23800.2     --------------------------------------------------KTRES-----
Bra033167.1-P   --------------------------------------------------KPRES-----
VIT_07s0005g018 SVGMQLESSGTVIDDAGDKISHSTRFEKDRLEQPATYLENSTSVMTSNRKKVKETTSGDD
                                                                  * :*:     

AT5G23800.2     -----SLTQGSGDKTANES-----------------------------------------
Bra033167.1-P   -----PSTQGSVDEMSDSSSTFS----------------------PASNSPRENPMESPL
VIT_07s0005g018 ATPSRPSKKLKEGDVADAPISHAVGQLRMAPIETLIQEVPCGFANPTTGGTGSNQTEQPV
                     . .: . .. :: .                                         

AT5G23800.2     ------------------------------------------------------------
Bra033167.1-P   T-----------------------------------------------------------
VIT_07s0005g018 AGNQSSTKTGSASQGMKVGNEQKSSGLGNQTPNSVKRKGRPPKSQVKIAQPFPAGEEVNT
                                                                            

AT5G23800.2     ------------------------------------------------------------
Bra033167.1-P   ------------------------------QGLVDE-------------ISDSGNTF---
VIT_07s0005g018 VQNAEGTVEKEYTTNNVDMDMAVGLPSNAEEGTIDENPSDHPNEESLKVMRDQKRHFDAT
                                                                            

AT5G23800.2     -------------------------------------------------------DPPVT
Bra033167.1-P   --------------------------------------------------SPVSNSPPVT
VIT_07s0005g018 ARHKSKEIGKKEEGTVSTQLKRRGRPPKKLENRNPEASSEGRAPRVSFKRSPKDSSAGRA
                                                                       ..  :

AT5G23800.2     PHPGITITPLK-------------------------------------------------
Bra033167.1-P   PSPSISATPLM-------------------------------------------------
VIT_07s0005g018 PKLPVKRSPRTSMREPKLKRQRASAVGTKIKGPKATSEGSNLNNQMELEAGLDLNSLYAS
                *   :. :*                                                   

AT5G23800.2     ------------------------------------------QTEVETQ--RKTFPKKT-
Bra033167.1-P   ------------------------------------------QTR---------------
VIT_07s0005g018 AAGKDDNEVGGVAYEMAIKDCKMNEVELPMSTVVESSAKRGSQTEIPVRHSRRAYRKRTT
                                                          **.               

AT5G23800.2     --------------------------LPRNQNGSGNDSTLENENSNRKRKREENLCS---
Bra033167.1-P   ------------------------------ENGT-------REDNNRKRKREEKL-----
VIT_07s0005g018 NQNLLGSRQKNVEGIKIRTPRTRKQKLPDESIGQALSKQLVKSSSKRGRRRSININSAPP
                                              . *        ....:* *:*. ::     

AT5G23800.2     ------------------------------------------------------------
Bra033167.1-P   ------------------------------------------------------------
VIT_07s0005g018 TQGSQDALGEKTAPLAEIDSKTKQVEMAVSGVHSVPDDQPLKMWFEGMHSPGVADNSKLS
                                                                            

AT5G23800.2     -GSSVEETN----------------------------ILFEKKLP<span class="highlight">V</span><span class="highlight">W</span>KILESMEVFKTIP
Bra033167.1-P   --SSVEETE-AR--------------------DITMVLPFEKKLPIWKTVESMEVFKTFP
VIT_07s0005g018 PGQTVNQWNEARERPSEVTQSPRIDPTGEIMLDLNQSLPFVKSSPIWNTLETLEVFQRMP
                  .:*:: :                            : * *. *:*: :*::***: :*

AT5G23800.2     QSPHFRPLAEIREDSREMLAVGMMLTFSCLLEQVKALQHDEARSSFISLSNSFAELEKHG
Bra033167.1-P   QSPHFTPLLEIREDAREMSAVGMLLTFSGLLEEVKSLKLNNPISSLNSLSDSFTELEKHG
VIT_07s0005g018 QKPHFRPLENCKEERREGLAIGNMVTFSTLIEKVAKLRFDDPRSIFGSSLEALVELEMHG
                *.*** ** : :*: **  *:* ::*** *:*:*  *: ::. * : *  :::.*** **

AT5G23800.2     FNAQVAQLRINKLLTLRGMQSRKMDELKGAKKVTAE--KESVKVENE-----RKILELQR
Bra033167.1-P   FDVKVPMLRISKLLSLIDRQAKKMEELEDCEKVTAE--KESTKVENE-----RKILELQK
VIT_07s0005g018 FDTKPVQSRINELVFIKDQQEQLKGRTKEVENQIMEHTHEKTKIDEEIYEIDKKMIELQE
                *:.:    **.:*: : . * :   . :  ::   *  :*..*:::*     :*::***.

AT5G23800.2     L-------NEEMAKEIAQSMSCEGKILQQLDNMKLEFQATASAPW
Bra033167.1-P   L-------NEEADKEIAQSKSCEATIGQQLDDVKLQFHTTASAPW
VIT_07s0005g018 KRALAVSNKESKDSEIAALLSSVDAMNESIQSARQDFERVATSPW
                        :*.  .***   *.   : :.::. : :*. .*::**


<a href='WV_VW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            