
<!doctype html>
<html>
    <head>
        <title>WV-VW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT3G15850.1</h1>
        <h3>AT3G15850.1 pep chromosome:TAIR10:3:5358601:5361185:1 gene:AT3G15850 transcript:AT3G15850.1 gene_biotype:protein_coding transcript_biotype:protein_coding gene_symbol:ADS3 description:JB67 [Source:UniProtKB/TrEMBL;Acc:A0A178VBZ6]</h3>
        <pre>
length  371
count   7
ratio   0.00
zscore  6.36






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT3G15850.1     MASLLT-----KPKPVFL-----------------------CSPSLSPRTL---------
Bra001609.1-P   MASLLTSPLT-KPKPFCL--------------------------SSSTRIL---------
PNW78440        MASLQL------PAR-------TGLRLGQSQQTVVRPVVRPLGVVQQRRAVITCAG----
PNW83820        MSDS-------QPRP---------------------------------------------
PNW83844        MQALRAQTSR-SPQT-------YSNTCPILSRELTRCTTRRPGCVPTRATVVPCAQ----
PNW83836        ------------------------------------------------------------
AES86572        MALITS-----QPKNPILHNHFTTLHCSIPQFS--RSILN-LGPFSNNA-----------
Pp3c17_23740V3. MATASRMSMGVTP-------------CGLQSLSLTRNVRK-VSASVERPGACCSRCYRSV
Pp3c19_17220V3. MGEMVGVVAG-SP-------------CSLQALS--RGVSN-RGPSVVRNAVVGCRSWRRV
KRH44467        MALLTS-----QPKNPILH-RFTTLHCPLPRLH--RTVLG-LGSFSHNSA----------
KRH47440        MALLTS-----QPKKPILH-RFTTLHRPLPRLD--RTVLG-LGSFSHNT-----------
VIT_09s0002g069 MALVIS-----PPNH-----RSHTFSALTPTTP--RTL---VGPILRRCIL------KRI
                                                                            

AT3G15850.1     ---NTATPSLNFTRISFTHHQKLAPFK---PPSLVVAFSE----KGLKRDVT--------
Bra001609.1-P   ----TTSPSLNFTRVSFTHSQKLAPFK---LPSLLSAFSE----KSQKRDVT--------
PNW78440        ----------NDRDVSTRVAEVVAPTT-TLGDAAAAAANDAVRWAQMSYSQQYDELYKNK
PNW83820        -----------SQQQGVKASRAVTSVT-LDAPEPVVERVSNSGGSRKAFAKAFTSLLGLQ
PNW83844        ----------RQSQQGVEASRAVTSAT-PDAPEPVVERVSNSSGSRKAFAKAFTSLLGLQ
PNW83836        -----------------------------------MERVSNSGGSRKAFAKAFTSLLGLQ
AES86572        ----KAIKLCNFNSKSRNFPLKNNPFN---FNPKIIAKSN----AAIGEATV--------
Pp3c17_23740V3. --------------NAGSALKRIDVPRKELFGASLKGLVP----GNVGRAGA--------
Pp3c19_17220V3. GRIGDTNIRLRNNSNLGGSLRRLESLK---LGAVIVAERD----SYTRRVRE--------
KRH44467        ----LSTKKLNFNPTQTKISLQNNPFT---YNLAVTAKAN--ANAAFGEGFK--------
KRH47440        ----LSIKKFNFNPKQTKISLKNNPFT---YNLAVTAKAN--ANAAFGEGLE--------
VIT_09s0002g069 ---KLSVTYTTSLQSRIRYVIKCQSSE---DSQALVQRLDNVANANLGSAEQ--------
                                                                            

AT3G15850.1     -----------TAAAAT-EGDYRRIM--LSDVLVKKKE-KV<span class="highlight">V</span><span class="highlight">W</span>WEREWKAMDFGAVAVVL
Bra001609.1-P   -----------ASAADRESGDYRRIM--LSDVMVKKKKDENPWWEREWSPMDFGAVAVVL
PNW78440        PINPKKVVPKPSKTAQMEEPGVGRVF--LSDVATKGKR--PSFLNRDWNSTDKAFLAFIG
PNW83820        LAKP-----DDLDPDTLRNPPFPRYTPALSELSVQQLR--PYFLNRTYTDTDVGYLAFFS
PNW83844        LAKP-----EDLDPDTLRNPPFPRYTPALSELSVQQLR--PYFLNRTYTDTDVGYLAFFS
PNW83836        LAKP-----EDLDPDTLRSPPFPRYTPALSELSVQQLR--PYFLNRTYTDTDVGYLAFFS
AES86572        ------------EPEEEVPLKHRKIF--LSDVEVKREK--EVFFGRKWNSLDIGTFVIVL
Pp3c17_23740V3. -AGPVSCVLEEAAAEQLVESPPTKIW--LSDVVVKRRR--NYFKNRQWTSKDLTYAGFMI
Pp3c19_17220V3. -CSASLRDGEEVLTGESSTPSEPGIW--MSDVVVKKRP--NYFKDRSWTSKDLTYAGYMA
KRH44467        ---------------EEGASRHRKIL--LSEVEVKRER--RVFFGRKWNSLDFGTAGIVL
KRH47440        --------------AGEVTLKHRKIL--LSEVEVKRQR--RVFFGRKWNSLDVGTAGIVL
VIT_09s0002g069 ------------------EGGLGNIV--VSNEAVRRR--------RKWNSLDVGTAGVIL
                                            :*:  .:          * :.  *      . 

AT3G15850.1     SMHLLSLL-APFQFNWRAVSVAFGLYIVTGLLGITLSFHRNLSHKAFKLPKWLEYLFAYC
Bra001609.1-P   SMHVLSLL-APFHFNWRA<span class="highlight">V</span><span class="highlight">W</span><span class="highlight">V</span>AFGLYVVTGLLGVTLSFHRNLSHKSFKLPKWLEYLFAYF
PNW78440        GMHLLACC-APFTYSPQMVALFLASYFVTGCLGITLSYHRMLSHKSFTVPKWLEYILAYC
PNW83820        AMHLVALVGGPLTYSPECLQLCLAGYVVTGMLGISMSYHRQLSHKSFRCVKPLEYLLAYC
PNW83844        AMHLVALVGGPLTYSPECLQLCLAGYVVTGMLGISMSYHRQLSHKSFRCVKPLEYLLAYC
PNW83836        AMHLVALVGGPLTYSPECLQLCLAGYVVTGMLGISMSYHRQLSHKSFR<span class="highlight">W</span><span class="highlight">V</span>KPLEYLLAYC
AES86572        SMHLLCLF-APFNFNWPAF<span class="highlight">W</span><span class="highlight">V</span>SVALYVVTGLFGITTSFHRNLSHRSFKLPKWLEYFFAYC
Pp3c17_23740V3. FMHGLCLF-APATFSWEAVGVFMGLYVITGMLGITLSYHRNLSHKSFKVPKWLEYTFAYC
Pp3c19_17220V3. FMHGLCLL-APATFSWDALGVSLGLYVITGLFGITLSFHRNLSHKSFKLPKWLEYSFAYC
KRH44467        AMHLLCVF-APFHFNWPAF<span class="highlight">W</span><span class="highlight">V</span>AVALYIVTGLFGITLSFHRNLSHRSFKLPKWLEYFFAYC
KRH47440        AMHLLCLF-APFHFNWPAFRVAVALYIVTGLFGITLSFHRNLSHRSFKLPKWLEYFFAYC
VIT_09s0002g069 AMHFLSVF-APFVFNWSAFWLAMALYAVTGLLGITLSYHRNLSHKSFKLPKWLEYSFAYC
                 ** :.   .*  :.   . : .. * :** :*:: *:** ***::*   * *** :** 

AT3G15850.1     GAQALQGNPID<span class="highlight">W</span><span class="highlight">V</span>STHRYHHQFCDSDRDPHSPLDGFWFSHMNWMFDTNTITQRCGEPNNV
Bra001609.1-P   GAQALQGNPID<span class="highlight">W</span><span class="highlight">V</span>STHRYHHQFCDSDRDPHSPLDGFWFSHMNWMFDANTITQRVGERNNV
PNW78440        GVLAVQGDPLEWSSSHRYHHLHTDTPLDPHSPYEGFWWCHMGWLLDNNATMERVSNRSNA
PNW83820        GALAFEGDPIEWPSKSVNGH-----------PTHSMWASYMGW--------------GSA
PNW83844        GALAFEGDPIEWSKMHRWHHLHSDTPADRHSPRDGLWHSHMGWLFDEALNSTRRDAQGNM
PNW83836        GALAFEGDPIEWPFKSVNGH-----------PTRSMWASYMGC--------------GSA
AES86572        GVLALQGNPID<span class="highlight">W</span><span class="highlight">V</span>STHRYHHQFCDSEKDPHSPIEGFWFSHMSWLFDTNSIAERCGEPTNV
Pp3c17_23740V3. GVQAVQGDPTE<span class="highlight">W</span><span class="highlight">V</span>SAHRYHHQHCDTENDPHTPYEGFWHSHMGWLLDDAQTQERVGARNNV
Pp3c19_17220V3. GVQAVQGDPLE<span class="highlight">W</span><span class="highlight">V</span>SSHRYHHQHCDTEKDPHSPYEGFWHSHMGWLLDDRRTQERVGARNNV
KRH44467        GVLALQGNPID<span class="highlight">W</span><span class="highlight">V</span>STHRYHHQFCDSERDPHSPTEGFWFSHMSWLFDTNSVLERCGEANNV
KRH47440        GVLALQGNPID<span class="highlight">W</span><span class="highlight">V</span>STHRYHHQFCDSERDPHSPTEGFWFSHMSWLFDTSSVIERCGEANNV
VIT_09s0002g069 GVQALQGNPTD<span class="highlight">W</span><span class="highlight">V</span>STHRYHHQFCDSERDPHTPMRGFWHSHMNWLFDHNSVTEKCGKPNNV
                *. *.:*:* :*       *           *  .:* .:*.                . 

AT3G15850.1     GDLEKQP--------FYRFLRTTYILHPLALAVALYAMGGFPFI<span class="highlight">V</span><span class="highlight">W</span>GMGVRI<span class="highlight">V</span><span class="highlight">W</span><span class="highlight">V</span>YHITW
Bra001609.1-P   GDLEKQP--------FYQFLRSTYIWHPLALAVALYALGGFPFV<span class="highlight">V</span><span class="highlight">W</span>GMGVRI<span class="highlight">V</span><span class="highlight">W</span><span class="highlight">V</span>YHITW
PNW78440        DDLESQW--------YYRWLRDTYPFHVAAQLAVLFLVGGFPAV<span class="highlight">V</span><span class="highlight">W</span>GGALRMC<span class="highlight">W</span><span class="highlight">V</span><span class="highlight">W</span>HITW
PNW83820        PGRQPPP------AAAITPGRAGSSTTAIRSGV---AASAPPAGSRGPA-----------
PNW83844        KDSLSPPWFYKESPAFYGWLRDTYMYHQIGQGVFFYLWGGVPYLIWGFVIRVLFTMHMTW
PNW83836        PGRQPPP------APSITPGRAGSSTTTISTGA---AASGP---IKGPA-----------
AES86572        GDLEKQS--------FYRFLRSTYMVHPFALGALLYAIGGFPFL<span class="highlight">V</span><span class="highlight">W</span>GMGVRV<span class="highlight">V</span><span class="highlight">W</span><span class="highlight">V</span>YHITW
Pp3c17_23740V3. SDMEKDP--------FYNFIEKTYPIHPMLLAAALYAVGGFPYI<span class="highlight">V</span><span class="highlight">W</span>GMAVRT<span class="highlight">V</span><span class="highlight">W</span><span class="highlight">V</span>YHITW
Pp3c19_17220V3. GDMDKDP--------FYQFIQRTYPIHPIMMAVALYVMGGFPYLIWGMALRV<span class="highlight">V</span><span class="highlight">W</span><span class="highlight">V</span>YHITW
KRH44467        GDLEKQS--------FYRFLRSTYLAHPFALGALLYAAGGFPFL<span class="highlight">V</span><span class="highlight">W</span>GMGVRI<span class="highlight">V</span><span class="highlight">W</span><span class="highlight">V</span>YHITW
KRH47440        GDLEKQS--------FYRFLRSSYLVHPFALGALLYAAGGFPFL<span class="highlight">V</span><span class="highlight">W</span>GMGVRI<span class="highlight">V</span><span class="highlight">W</span><span class="highlight">V</span>YHITW
VIT_09s0002g069 GDLEKQP--------FYRFISKTYILHPVALGALLYALGGFPFL<span class="highlight">V</span><span class="highlight">W</span>GMGVRI<span class="highlight">V</span><span class="highlight">W</span><span class="highlight">V</span>YHITW
                 .                              .     ..      *             

AT3G15850.1     LVNSACH<span class="highlight">V</span><span class="highlight">W</span>GKQAW------NTGDLSKNN-W<span class="highlight">W</span><span class="highlight">V</span>AALAFGEGWHNNHHAFEFSARHGLEWW
Bra001609.1-P   LVNSACH<span class="highlight">V</span><span class="highlight">W</span>GEQAW------NTGDLSKNN-WMVAALAFGEGWHNNHHAFEFSARHGLEWW
PNW78440        FVNSASHCWGYQDY------NTGDLSKNN-W<span class="highlight">W</span><span class="highlight">V</span>GILAFGEGWHNNHHAFEFSARHGFEWW
PNW83820        ---------GSRRQCPTRTSSSSSSSRSTPCWIQRWRTCRGRTGTRRAGGTSQTDGL---
PNW83844        LVNSAVHIWGEQRY------QSDDSSRNN-WLVGLLVFGDGHHNNHHAFEYSAAHGLEWW
PNW83836        ---------GSRRR----------------------------------------------
AES86572        LVNSACH<span class="highlight">V</span><span class="highlight">W</span>GKQPW------NTRDLSRNN-W<span class="highlight">W</span><span class="highlight">V</span>ALLAFGEGWHNNHHAFEYSARHGLEWW
Pp3c17_23740V3. FVNSASH<span class="highlight">V</span><span class="highlight">W</span>GSQKW------NTGDLSRNN-W<span class="highlight">W</span><span class="highlight">V</span>ALLAFGEGWHNNHHAFEYSARHGLEWW
Pp3c19_17220V3. FVNSASH<span class="highlight">V</span><span class="highlight">W</span>GSQKW------NTGDLSRNN-W<span class="highlight">W</span><span class="highlight">V</span>ALLAFGEGWHNNHHAFEYSARHGLEWW
KRH44467        FVNSACH<span class="highlight">V</span><span class="highlight">W</span>GNQAW------NTRDLSRNN-W<span class="highlight">W</span><span class="highlight">V</span>ALLAFGEGWHNNHHAFEYSARHGLEWW
KRH47440        FVNSACH<span class="highlight">V</span><span class="highlight">W</span>GNQAW------NTRELSRNN-W<span class="highlight">W</span><span class="highlight">V</span>ALLAFGEGWHNNHHAFEYSARHGLEWW
VIT_09s0002g069 LVNSACH<span class="highlight">V</span><span class="highlight">W</span>GNQAW------NTGDLSRNN-W<span class="highlight">W</span><span class="highlight">V</span>GLLAFGEGWHNNHHAFEYSARHGLEWW
                         * :                                                

AT3G15850.1     QLDMTWYVVKFLQAIGLATDVKLPSEAQKQRMAFTSD-------
Bra001609.1-P   QLDMTWYVVRFLQAIGLATDVKLPSEAQKQRMTLTND-------
PNW78440        QVDMTWYIILALQKVGLATNVKLPTERQKAKLAFKKPAEAA--A
PNW83820        -------ILTIRPCTRGAYTVQQPRGRRSLRARLFTRTL-----
PNW83844        EVDMSYYMIRILELTGLAWDVKRPSATAMAAKRLPGK-------
PNW83836        --------------------------------------------
AES86572        ELDMTWYLVKFLQAIGLATDVKVPSESQKQRMALNNNNNDVVAT
Pp3c17_23740V3. QLDPTW<span class="highlight">W</span><span class="highlight">V</span>IRGLEAVGLATKVKLPRKDHMEKLAFKPTL------
Pp3c19_17220V3. QLDPTWWIICALEAVGVATKVKLPKKEHMEKLAL----------
KRH44467        QIDMTWYVVRFLQAIGLATDVKLPTESQKQKMAFNSDSIAT---
KRH47440        QLDMTWYVVRFLEAIGLATDVKLPTESHKQKMALNSDPIDT---
VIT_09s0002g069 QIDMTWYVVRFLQALGLATDVKLPTELHKQRMAFNNSTLLT---
                                                            


<a href='WV_VW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            