
<!doctype html>
<html>
    <head>
        <title>WW-WW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT4G27730.1</h1>
        <h3>AT4G27730.1 pep chromosome:TAIR10:4:13833807:13836661:-1 gene:AT4G27730 transcript:AT4G27730.1 gene_biotype:protein_coding transcript_biotype:protein_coding gene_symbol:OPT6 description:Oligopeptide transporter 6 [Source:UniProtKB/Swiss-Prot;Acc:Q9T095]</h3>
        <pre>
length  736
count   12
ratio   0.00
zscore  16.95






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT4G27730.1     M------GEIATEFTPVMDDDDDRCVVPEVELTVPKTDDSTLPVLTFRMWVLGIGACIVL
Bra026311.1-P   M------GEIAEEF-PVVITDDDRCVVPEVELTVPKTDDPSLPVLTFRMWVLGIGACIVL
KRH64375        MASTFEGGSDLEKENNNIAESEEECPVKQVELTVPKTDDPTMQLLTFRMWVLGVLSCVLL
KRH53552        MASTFEGGSDSEKE-NTVESEEEECPVKQVELTVPKTDDPTMQLLTFRMWVLGVLSCVLL
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT4G27730.1     SFINQFFWYRTMPLSITGISAQIAVVPLGHLMARVLPTKRFLEGTRFQFTLNPGAFNVKE
Bra026311.1-P   SFINQFFWYRTMPLSITGISAQIAVVPLGHLMARVLPTRRFLEGTRFQFTLNPGAFNVKE
KRH64375        SFVNQFFWYRTQPLIVTSISAQIAVVPIGHFLARTLPTRVFFKDTRFEFSLNRGPFNIKE
KRH53552        SFVNQFFWYRTQPLIVTSISAQIAVVPIGHFMARTLPTRVFFKDSRFEFSLNRGPFNIKE
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT4G27730.1     HVLITIFANSGAGSVYATHILSAIKLYYKRSLPFLPAFLVMITTQILGFGWAGLFRKHLV
Bra026311.1-P   HVLITIFANSGAGSVYATHILSAIKLYYKRSLPFLPAFLVMITTQILGFGWAGLFRKHLV
KRH64375        HVLITIFANSGAGTVYATHILSAVKLMYKRRLDFLPALLVMLTTQVLGFGWAGLFRKFLV
KRH53552        HVLITIFANSGAGTVYATHILSAVKLMYKRRLDFLPALLVMLTTQILGFGWAGLFRKFLV
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT4G27730.1     EPGEM<span class="highlight">W</span><span class="highlight">W</span>PSNLVQVSLFGALHEKEKKSRGGMSRTQFFLIVLVASFAYYIFPGYLFTMLTS
Bra026311.1-P   EPGEM<span class="highlight">W</span><span class="highlight">W</span>PSNLVQVSLFGALHEKEKKSKGHMSRTQFFLIVLVASFAYYILPGYLFTMLTS
KRH64375        EPGEM<span class="highlight">W</span><span class="highlight">W</span>PSNLVQVSLFSALHEKSKRPKGGTTRTQFFLLALVLGLAYYVFPGYLFSMLTS
KRH53552        EPGEM<span class="highlight">W</span><span class="highlight">W</span>PSNLVQVSLFSALHEKSKRPKGGTTRTQFFLLVLVSGLAYYVFPGYLFSMLTS
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT4G27730.1     ISWVCWLNPKSILVNQLGSGEHGLGIGSIGFDWVTISAYLGSPLASPLFASVNVAIGFVL
Bra026311.1-P   ISWVCWFNPKSILVNQLGSGEHGLGVGSIGFDWVTISAYLGSPLASPLFASVNVAIGFVL
KRH64375        FSWMCWLAPKSILVQQLGSGLRGLGIAAFGIDWSTISSYLGSPLASPWFATANIAVGFFL
KRH53552        FSWMCWLAPKSVLVQQLGSGLRGLGIAAFGIDWSTISAYLGSPLASPWFATANIAVGFFL
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT4G27730.1     VMYIVTPVCYWLNIYDAKTFPIFSSQLFMGNGSRYDVLSIIDSKFHLDRVVYSRTGSINM
Bra026311.1-P   VMYIVTPICYWLNIYEAKTFPIFSSQLFMANGSRYDVLSIIDSKFHLDRAVYSNTGSINM
KRH64375        VMYVMTPIAYWSNAYEAKTFPIFSSKLFMGNGSLYDISTIVNSEFHLDRQAYSINGPVHL
KRH53552        VMYVMTPIAYWSNAYEAKTFPIFSSKLFMGNGSLYDISTIVNSEFHLDRQAYSINGPVHL
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT4G27730.1     STFFAVTYGLGFATLSATIVHVLVFNGSDLWKQTRGAFQKNKKMDIHTRIMKKNYREVPL
Bra026311.1-P   STFFAVTYGLGFATLSATIVHVLVFNGSDLWKQTRGAFQKDKKMDIHTRIMKKNYREVPM
KRH64375        STFFAMTYGLGFAALSATVVHVLLFHGREILMQSKRAFGNSKKIDIHTRLMRR-YKSVPM
KRH53552        STFFAMTYGLGFAALSATVVHVLLFHGREILMQSKRAFGNSKKIDIHTRLMRR-YKSVPT
VIT_00s0508g000 ----------------------------RIWQQAKASLQ-DQFDDVHTRLMKKNYDPVPQ
                                             :  *:: ::  .:  *:***:*:: *  ** 

AT4G27730.1     <span class="highlight">W</span><span class="highlight">W</span>FLVILLLNIALIMFISVHYNATVQLP<span class="highlight">W</span><span class="highlight">W</span>GVLLACAIAISFTPLIGVIAATTNQAPGLN
Bra026311.1-P   <span class="highlight">W</span><span class="highlight">W</span>FLVILLLNIALIMFISVHYNATVQLP<span class="highlight">W</span><span class="highlight">W</span>GVLLACAIAISFTPLIGVIAATTNQAPGLN
KRH64375        <span class="highlight">W</span><span class="highlight">W</span>FYIILAANIALIIFICEYYNESLQLP<span class="highlight">W</span><span class="highlight">W</span>GVLLACAISIFFTLPIGIINATTNQQPGLN
KRH53552        <span class="highlight">W</span><span class="highlight">W</span>FYIILAANIALIIFICEYYNESLQLP<span class="highlight">W</span><span class="highlight">W</span>GVLLACAISIFFTLPIGIINATTNQQPGLN
VIT_00s0508g000 <span class="highlight">W</span><span class="highlight">W</span>FYTLLTLVIGLAMAACEGFNRQLQLPYWGILLAISLALLFILPVGVITATTNQQPGLN
                ***  :*   *.* :  .  :*  :***:**:*** :::: *   :*:* ***** ****

AT4G27730.1     IITEYVIGYIYPERPVANMCFKVYGYISMTQALTFISDFKLGHYMKIPPRSMFMAQVAGT
Bra026311.1-P   IITEYVIGYIYPERPVANMCFKVYGYISMTQALTFISDFKLGHYMKIPPRSMFMAQVVGT
KRH64375        IITEYIIGYMYPERPVANMCFKVYGYISMVQALSFLQDFKLGHYMKIPPRTMFMAQVVGT
KRH53552        IITEYIIGYMYPERPVANMCFKVYGYISMVQALTFLQDFKLGHYMKIPPRTMFMAQVVGT
VIT_00s0508g000 VITEMILGYIYPGKPLANLAFKTYGSVSMGQAIMFLSDFKLGHYMKIPPKSMFVVQLVGT
                :*** ::**:** :*:**:.**.** :** **: *:.************::**:.*:.**

AT4G27730.1     LVAVVVYTGTA<span class="highlight">W</span><span class="highlight">W</span>LMEEIPHLCDTSLLPSDSQWTCPMDRVFFDASVIWGLVGPRRVFGDL
Bra026311.1-P   LVAVMVYTGTA<span class="highlight">W</span><span class="highlight">W</span>LMEEIPHLCDTSLLPSDSQWTCPMDRVFFDASVIWGLVGPRRVFGDL
KRH64375        ILSVFIYTITA<span class="highlight">W</span><span class="highlight">W</span>LMRTIPHLCDTTMLDPDSPWTCPMDNVFFDASVIWGLLGPRRIFGDL
KRH53552        ILSVFIYTVTA<span class="highlight">W</span><span class="highlight">W</span>LMRTIPHLCDTTMLDPDSPWTCPMDNVFFDASVIWGLLGPRRIFGDL
VIT_00s0508g000 VIASSVYFGTAWLLLTSIDNICHPSLLPEGSQWTCPGDRVFYSASLILGVVGPLRMFGRL
                :::  :*  *** *:  * ::*..::*  .* **** *.**:.**:* *::** *:** *

AT4G27730.1     GEYSNVNWFFLVGAIAPLLVWLATKMFPAQTWIAKI------HIPVLVGATAMMPPATAV
Bra026311.1-P   GEYSSVNWFFLVGAIAPLLVWLAAKAFPAQTWISKI------HIPVLVGATAMMPPATAV
KRH64375        GEYAKVNLFFFGGAIAPFLVWLAHKAFPGQRWITLI------HMPVLLGATSMMPPATAV
KRH53552        GEYAKVNLFFLGGAIAPFLVWLAHKAFPEQRWITLI------HMPVLLGATSMMPPATAV
VIT_00s0508g000 GLYAKTNYFFLLGILAPVPVWLLFHIFPEKKWMKLIIGELCESLPYLGGAGA-MPIAKAV
                * *:..* **: * :**. ***  : ** : *:  *       :* * ** : ** *.**

AT4G27730.1     NFTSWLIVAFIFGHFIFKYRRV<span class="highlight">W</span><span class="highlight">W</span>TKYNYVLSGGLDAGSAFMTILLFLALGRKGIE-VQW
Bra026311.1-P   NFTSWLIVAFVFGHFIFKYRRE<span class="highlight">W</span><span class="highlight">W</span>KKYNYVLSGGLDAGSAFMTILLFLALGRKGIE-VQW
KRH64375        NFTSWLLVGFLSGYVAYRYRQE<span class="highlight">W</span><span class="highlight">W</span>KRCNYVLSGGLDAGTAFMTILLFLALNNNNIM-LNW
KRH53552        NFTSWILVGFLSGYIAYRYRQE<span class="highlight">W</span><span class="highlight">W</span>KRCNYVLSGGLDAGTAFMTILLFLALNNNNIM-LNW
VIT_00s0508g000 NYWCWGAMGIFFNIFVYRRFKG<span class="highlight">W</span><span class="highlight">W</span>ARHNYVLSAGLDAGVAFMAILSYFTLQDKDINGMQW
                *: .*  :.:. . . ::  : ** : *****.***** ***:** :::*  :.*  ::*

AT4G27730.1     WGNSGDRDTCPLASCPTAKGVVVKGCPVF-
Bra026311.1-P   WGNSGDRDTCPLASCPTAKGVVVKGCPVF-
KRH64375        WGN--NPEGCPLATCPTEKGIVVKGCPVH-
KRH53552        WGN--NPEGCPLANCPTEKGIVVKGCPVH-
VIT_00s0508g000 WGL-GIDDHCPLAHCPTAPAIKVEGCPVFN
                **     : **** ***  .: *:****. 


<a href='WW_WW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            