
<!doctype html>
<html>
    <head>
        <title>WW-WW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT1G50430.1</h1>
        <h3>AT1G50430.1 pep chromosome:TAIR10:1:18681932:18685652:-1 gene:AT1G50430 transcript:AT1G50430.1 gene_biotype:protein_coding transcript_biotype:protein_coding gene_symbol:DWF5 description:7-dehydrocholesterol reductase [Source:UniProtKB/Swiss-Prot;Acc:Q9LDU6]</h3>
        <pre>
length  432
count   6
ratio   0.00
zscore  8.35






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT1G50430.1     ----------------------------------MAETVHSPIVTYAS-MLSLLAFCPPF
Bra014252.1-P   ----------------------------------MAATVHSPIVTYAS-MLSLLAFCPPF
Bra041028.1-P   ----------------------------------MAETVHSPIVAFAL-MLSLLALCPPF
PNW86199        ------------------MGKGGTQHF-------EFFGPHGPAVL----VLVLPAVCYAL
KEH28529        ------------------------------------------------------------
KEH31329        ------------------------------------------------------------
KEH31330        -----------MVE------------------SKKTETVHSPLVTYAS-MISLLTLAPPF
Pp3c24_10460V3. ------------------MGAQGDQKLTRSQYAKEPISVHSKMVTYVS-MTALITLCPAF
KRH44662        ----------------------------------MGATVHSPLVTYAS-VISLLTLCPPF
Zm00001d016463_ MPSQIGIGNTHSVTPLQRCF----LSQVASPYCRPLLLLSSHLTTHRSPMLALLSLCPPF
BGIOSGA006507-P -----------MAKP--RASAAAAAAAAKAPASTPPKTVHSALVTYAS-MLSLLSLCPPF
BGIOSGA006508-P ------------------------------------------------------------
Os02t0465400-01 -----------MAKP--RAS----AAAAKAPASTPPKTVHSALVTYAS-MLSLLSLCPPF
VIT_09s0054g009 -----------MA---------------------EAKTVHSPALTYFS-MISLLTLCPPF
                                                                            

AT1G50430.1     VILLWYTMVHQ-DGSVTQTFGFFWENGV--QGLINIWP---RPTL---------IAWKII
Bra014252.1-P   VILLWYTMVHQ-EGSVVQTGSFLWENGV--QGLINIWP---RPTA---------IAWKII
Bra041028.1-P   VILLWFTRMVL-LFRPLASCGRMEFKG---SSTYGQDP---LPSLGR----LSFATEHLR
PNW86199        II------------------------GCNKESCLKLWPEFKLPSLPQTVSFYSNEAMLVY
KEH28529        ------------------------------------------------------------
KEH31329        -------MTTTL------------------------------------------------
KEH31330        VNLLWYTMTTAMDGSILKTFDYLNNNGL--QGFLNLWP---KPTL---------LSFQII
Pp3c24_10460V3. VMFLWHTMVNL-DGSTTKYLELYKVEGI--DGVLKTWP---MPTM---------NALKII
KRH44662        VILLWYTMTLA-DGSVSETFHYLRQNGL--QGLLHIWP---TPTP---------TACKII
Zm00001d016463_ VILLWYTMVHA-DGSVVRTYEHLRDHGV-LEGLKAIWP---MPTL---------VAWKII
BGIOSGA006507-P VILLWYTMVHA-DGSVVRAYEHLREHGV-LEGLKAIWP---MPTM---------AAWKII
BGIOSGA006508-P -----------------------------------------MTNL---------------
Os02t0465400-01 VILLWYTMVHA-DGSVVRAYEHLREHGV-LEGLKAIWP---MPTM---------AAWKII
VIT_09s0054g009 VILLWYTMVHA-DGSVLQTWDYLKQHGV--QGFIDIWP---RPTA---------IAWKLI
                                                                            

AT1G50430.1     FCYGAFEAILQLLLPGKRVEGPISPAGNRPVYKANGLAAYFVTLATYLGL<span class="highlight">W</span><span class="highlight">W</span>F---GIFN
Bra014252.1-P   FCYGAFEAALQLLLPGKRVEGPISPTGNRPVYKANGMAAYFVTLATYLGL<span class="highlight">W</span><span class="highlight">W</span>F---GIFN
Bra041028.1-P   LLFSFFFLVKEL--------------------KAQFLQR-ETDQFTRSDRWLYLSSGIFN
PNW86199        LAWFFGVALLHVILPGQKAEGVVLPNGKRLTYKLN---AFRIFLITYGAALYF---GFVS
KEH28529        ------------------------------------------------------------
KEH31329        ------------------------------------------------------------
KEH31330        VVYAAFEALLQLLLPGETVYGPISPTGNRPFYKANGVAAYLVTLVTYVALWRF---GIFN
Pp3c24_10460V3. LSFAAFEAFLQLYIPGEVHVGPVSPAGNRPVYKNNGFSCYIITLVSYYGLYKS---GYFN
KRH44662        AVYAAFEAALQLLLPGKTVYGPISPTGHRPVYKANGLQAYFVTLITYFAL<span class="highlight">W</span><span class="highlight">W</span>F---GIFN
Zm00001d016463_ FGFGLFEAVLQLLLPGKRFEGPISPAGNVPVYKANGLQAYAVTLITYLGL<span class="highlight">W</span><span class="highlight">W</span>F---GIFN
BGIOSGA006507-P FGFGLFEAALQLLLPGKRFEGPVSPSGNVPVYKANGLQAYAVTLITYLSL<span class="highlight">W</span><span class="highlight">W</span>F---GIFN
BGIOSGA006508-P ------------------------------------------------------------
Os02t0465400-01 FGFGLFEAALQLLLPGKRFEGPVSPSGNVPVYKANGLQAYAVTLITYLSL<span class="highlight">W</span><span class="highlight">W</span>F---GIFN
VIT_09s0054g009 ACYAAFEAALQLFLPGKTVEGPISPCGNRPVYKANGMQAYAVTLITYLSL<span class="highlight">W</span><span class="highlight">W</span>F---GIFN
                                                                            

AT1G50430.1     PAI----VYDHLGEIFSALIFGSFIFCVLLYIKGHVAPS-SSDSGSCGNLIIDFYWGMEL
Bra014252.1-P   PAI----VYDHLGEIFSALIFGSLVFCVLLYIKGHVAPS-SSDSGSCGNLIIDFYWGMEL
Bra041028.1-P   PAI----VYDHLGAIFSALIFGCFIFCVLLYVKVHVAPS-SSDSGSCGNLIIDFYWGMEL
PNW86199        RQLDLGWLYDNFAQLLTAAIIFSSALSVYLYASSFRKGALLSGHGATGYPAYDFWMGREL
KEH28529        --------------------------------MGHLAPS-STDSGSSGNIIIDFYWGMEL
KEH31329        ---------------------------------GHLAPS-STDSGSSGNIIIDFYWGMEL
KEH31330        PTI----VYDHLGEIYSTLSFGSFIFCIFLYIKGHLAPS-STDSGSSGNIIIDFYWGMEL
Pp3c24_10460V3. PAV----VYDHLGEIYFTLVVGSFFVCILLYIKGHVAPS-SSDCGSSGNVLVDFYWGMEL
KRH44662        PTI----VYHHLGEIYSALIFGSFLFCVFLYIKGHLAPS-STDSGSSGNLIIDFYWGMEL
Zm00001d016463_ PAI----VYDHLGEIYSALVFGSFVFCIFLYIKGHVFPS-SSDSGSSGNVIIDFYWGMEL
BGIOSGA006507-P PAI----VYDHLGEIYSALVFGSFVFCIFLYIKGHLAPS-SSDSGSSGNVIIDFYWGMEL
BGIOSGA006508-P -----------------------------------------------------------M
Os02t0465400-01 PAI----VYDHLGEIYSALVFGSFVFCIFLYIKGHLAPS-SSDSGSSGNVIIDFYWGMEL
VIT_09s0054g009 PAI----VYDHLGEIYSALIFGSLIFCIFLYIKGHVAPS-STDSGSSGNIIIDFYWGMEL
                                                                           :

AT1G50430.1     YPR-IGKSFDIKVFTNCRFGMMSWAVLAVTYCIKQYEINGKVSDSMLVNTILMLVYVTKF
Bra014252.1-P   YPR-IGKNFDIKVFTNCRFGMMSWAVLAVTYCIKQYEINGKVSDSMLVNTILMLVYVTKF
Bra041028.1-P   YPR-IGKNFDIKVFTNCRFGMMSWAVLAVTYCIKQYEINGRVSDSMLVNTILMHVYVTKF
PNW86199        NPRLLGGAFDLKEFCELYPGMMGWALINLGMAHKQWAATGSLTTSMLLVNVFQLYYVVDA
KEH28529        YPR-IGKYFDIKVFTNCRFGMMSWAVLALTYCIKQYEENGKVADSMLVNTTLMLVYVTKF
KEH31329        YPR-IGKYFDIKVFTNCRFGMMSWAVLALTYCIKQYEENGKVADSMLVNTTLMLVYVTKF
KEH31330        YPR-IGKYFDIKVFTNCRFGMMSWAVLALTYCIKQYEENGKVADSMLVNTTLMLVYVTKF
Pp3c24_10460V3. YPR-IGQNFDIKVFTNCRFGMMSWAVLVVTYAIKQIELYGKLSDSMTVSATLMLVYITKF
KRH44662        YPR-IGKHFDIKVFTNCRFGMMSWAVLALTYCIKQYEENGKVADSMLVNTALMLVYVTKF
Zm00001d016463_ YPR-IGKYFDIKVFTNCRFGMMSWAVLAVTYCIKQYEMNGRVADSMLVNTALMLIYITKF
BGIOSGA006507-P YPR-IGKHFDIKVFTNCRFGMMSWAVLAVTYCIKQYEMNGRVADSMLVNTALMLIYVTKF
BGIOSGA006508-P MPR---------------------------------------------------------
Os02t0465400-01 YPR-IGKHFDIKVFTNCRFGMMSWAVLAVTYCIKQYEMNGRVADSMLVNTALMLIYVTKF
VIT_09s0054g009 YPR-IGKNFDIKVFTNCRFGMMSWAVLAVTYCIKQYEENGKVADSMLVNTILMLVYVTKF
                 **                                                         

AT1G50430.1     F<span class="highlight">W</span><span class="highlight">W</span>EAGYWNTMDIAHDRAGFYICWGCLVWVPSVYTSPGMYLVNHPVELGTQLAIYILVAG
Bra014252.1-P   F<span class="highlight">W</span><span class="highlight">W</span>EAGYWNTMDIAHDRAGFYICWGCLVWVPSIYTSPGMYLVNHPVQLGTQLAIYILVAG
Bra041028.1-P   FGWEAGYWNSMDIAHDRAGFYICWGCLVWVPSIYTSPGMYLVNHPVQLGTQLAIYILVAG
PNW86199        LWCEKAILTTMDITTDGFGFMLAFGDLAWVPFTFTSCARYLVDYPQHLSVAATCLVLAVK
KEH28529        F<span class="highlight">W</span><span class="highlight">W</span>EAGYWNTMDIAHDRAGFYICWGCLVWVPSVYTSPGMYLVNHPVNLGTQLVLFILVAG
KEH31329        F<span class="highlight">W</span><span class="highlight">W</span>EAGYWNTMDIAHDRAGFYICWGCLVWVPSVYTSPGMYLVNHPVNLGTQ---------
KEH31330        F<span class="highlight">W</span><span class="highlight">W</span>EAGYWNTMDIAHDRAGFYICWGCLVWVPSVYTSPGMYLVNHPVNLGTQ---------
Pp3c24_10460V3. F<span class="highlight">W</span><span class="highlight">W</span>EAGYWNSMDIAHDRAGFYIVWGCLVWVPSVYTSPALYLVNHPVELGMPLAAAITAAG
KRH44662        F<span class="highlight">W</span><span class="highlight">W</span>EAGYWSTMDIAHDRAGFYICWGCLVWVPSVYTSPGMYLVNHPVNLGIKLALSILVAG
Zm00001d016463_ F<span class="highlight">W</span><span class="highlight">W</span>ESGYWCTMDIAHDRAGFYICWGCLVWVPSIYTSPGMYLVNHPLNLGPQLALSILLAG
BGIOSGA006507-P F<span class="highlight">W</span><span class="highlight">W</span>ESGYWCTMDIAHDRG------------------------------------------
BGIOSGA006508-P ---------------------------------------YFI---LIIHEDLALSILLAG
Os02t0465400-01 F<span class="highlight">W</span><span class="highlight">W</span>ESGYWCTMDIAHDRAGFYICWGCLVWVPSIYTSPGMYLVNHPVNLGPQLALSILLAG
VIT_09s0054g009 F<span class="highlight">W</span><span class="highlight">W</span>EAGYWNTMDIAHDRAGFYICWGCLVWVPSIYTSPGMYLVNHPVNLGTQLAIYILAAG
                                                                            

AT1G50430.1     ILCIYINYDCDRQRQEFRRTNGKCLVWGRAPSKIVASYTTTSGETKTSLLLTSG<span class="highlight">W</span><span class="highlight">W</span>GLAR
Bra014252.1-P   CLCIYINYDCDRQRQEFRRTNGKCSVWGKAPSKIVATYTTTAGETKTSLLLTSG<span class="highlight">W</span><span class="highlight">W</span>GLAR
Bra041028.1-P   ILCIYINYDCDRQRQEFRRTNGKCSVWGKAPSKIVATYKTTASETKTSLLLTSGW-----
PNW86199        LLGYAVFRGANSQKDMFR----------ANPSDPRVAHLRTLKTERGTQLIISG<span class="highlight">W</span><span class="highlight">W</span>GTAR
KEH28529        IMCIYINYDCDRQSQEFRRTNGKSLVWGKAPSKIEASYTTASGETKRSLLLTSG<span class="highlight">W</span><span class="highlight">W</span>GLAR
KEH31329        ---------------------------------IEASYTTSSGETKRSLLLTSG<span class="highlight">W</span><span class="highlight">W</span>RLAR
KEH31330        ---------------------------------IEASYTTSSGETKRSLLLTSG<span class="highlight">W</span><span class="highlight">W</span>RLAR
Pp3c24_10460V3. LLCIWINYDADLQRQTFRKTDGKAKVWGKVPNKIVATYVTEKGEKKQSLLLVSG<span class="highlight">W</span><span class="highlight">W</span>AMAR
KRH44662        ILCIYINYDCDRQRQEFRRTNGKGTVWGKAPSKIEATYTTTSGETKRSLLLTSG<span class="highlight">W</span><span class="highlight">W</span>GLSR
Zm00001d016463_ MLCIYINYDCDRQRQEFRRTNGKCSVWGKAPSKIVASYQTTKGETKTSLLLTSG<span class="highlight">W</span><span class="highlight">W</span>GLSR
BGIOSGA006507-P ------------------------------------------------------------
BGIOSGA006508-P ILCIYINYDCDRQRQEFRRTNGKCSIWGKAPSKIVASYQTTNGETKSSLLLTSG<span class="highlight">W</span><span class="highlight">W</span>GLSR
Os02t0465400-01 ILCIYINYDCDRQRQEFRRTNGKCSIWGKAPSKIVASYQTTNGETKSSLLLTSG<span class="highlight">W</span><span class="highlight">W</span>GLSR
VIT_09s0054g009 ITCIYINYDCDRQRQEFRRTNGKCLVWGKAPSKIVASYTTTSGETKSSLLLTSG<span class="highlight">W</span><span class="highlight">W</span>GLSR
                                                                            

AT1G50430.1     HFHYVPEILSAFFWTVPALFDNF---LAYFYVIFLTLLLFDRAKRDDDRCRSKYGKYWKL
Bra014252.1-P   HFHYVPEISSAFFWTVPALFNNF---LPYFYVIFLTILLFDRAKRDDDRCRSKYGKYWKL
Bra041028.1-P   ------------------------------------------------------------
PNW86199        HINYFGDWIMGVAWCMPAGLQGLASVVPYFYCIYFASLLIHRERRDEHACRLKYGKDWDK
KEH28529        HFHYVPEILAAFFWTVPALFNHF---LPYFYVIFLIILLFDRAKRDDDRCRSKYGKYWKL
KEH31329        HFHYVPEILAAFFWTVPALFNHF---LPYLYVIFLIILLFDRAKRDDDRCRSKYGKYWKL
KEH31330        HFHYVPEILAAFFWTVPALFNHF---LPYLYVIFLIILLFDRAKRDDDRCRSKYGKYWKL
Pp3c24_10460V3. HFHYLPEISAAFFWTVPALFSHP---LPYFYVVFLTILLVDRAERDDKRCQTKYKKFWDE
KRH44662        HFHYVPEILAAFFWTVPALFEHF---LPYFYVIFLTILLFDRAKRDDDRCRSKYGKYWKL
Zm00001d016463_ HFHYVPEILSAFFWTVPALFNHF---LPYFYVIFLTILLFDRAKRDDDRCSSKYGKYWKI
BGIOSGA006507-P ------------------------------------------------------------
BGIOSGA006508-P HFHYVPEILSAFFWTVPALFDHF---LPYFYVIFLTILLFDRAKRDDDRCSSKIR-----
Os02t0465400-01 HFHYVPEILSAFFWTVPALFDHF---LPYFYVIFLTILLFDRAKRDDDRCSSKYGKYWKM
VIT_09s0054g009 HFHYVPEILAAFFWTVPALFGHF---LPYFYVLFLTILLFDRAKRDDDRCRSKYGKYWKK
                                                                            

AT1G50430.1     YCEKVKYRIIPGIY-----------------------------
Bra014252.1-P   YCEKVKYRIIPGIY-----------------------------
Bra041028.1-P   -------------------------------------------
PNW86199        YCALVRYRIVPYLY-----------------------------
KEH28529        YCDRVPYRIIPGIY-----------------------------
KEH31329        YCDRVAYRIIPGIY-----------------------------
KEH31330        YCDRVAYRIIPGIY-----------------------------
Pp3c24_10460V3. YKRTVPCKIIPHIF-----------------------------
KRH44662        YCDKVPYRIIPGIY-----------------------------
Zm00001d016463_ YCNKVPYRVIPGIY-----------------------------
BGIOSGA006507-P -------------------------------------------
BGIOSGA006508-P -FGLIGFNLLSLLYPTLTQMSQRCRQIVIFLVSSLIGEMIAGS
Os02t0465400-01 YCNKVPCRVIPGIY-----------------------------
VIT_09s0054g009 YCEKVRYRIIPGIY-----------------------------
                                                           


<a href='WW_WW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            