
<!doctype html>
<html>
    <head>
        <title>WW-WW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT1G09320.1</h1>
        <h3>AT1G09320.1 pep chromosome:TAIR10:1:3009890:3012407:-1 gene:AT1G09320 transcript:AT1G09320.1 gene_biotype:protein_coding transcript_biotype:protein_coding description:Agenet domain-containing protein [Source:UniProtKB/TrEMBL;Acc:Q500V5]</h3>
        <pre>
length  517
count   6
ratio   0.00
zscore  8.35






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT1G09320.1     MLRPRR-SLGVSSPAKQRKKAAPKNSMATRANRKRLPSYLKPGSAVEISSDEIGFRGSWY
Bra018567.1-P   MSSLRRQSPMVTYPTNQRKK-APSNN-------NRLPSFLKPGVAVEISTNEEGFRGAWF
Bra031661.1-P   MSSRRRQS-------TQRKT-APESSVTP----IRLPPFLKPGAAVEISSDEAGFRGSWY
KRG90506        MAR------------------TPKPKGGPSAAASAATAFFKPGTAVEVSSEDDGFRGSWF
Zm00001d010944_ MRSPRI-------PGRRDAA-SPQT-----------PKGFNPGDPVEVVPDEAGLHGAHF
BGIOSGA018865-P MRSPRLRR--NPPPATTAAA-AAPA-----------PDQFRPGDLVEVLPDEPGYRGAHF
Os05t0132400-01 ------------------------------------------------------------
VIT_14s0060g008 M------------------------------------DYFKKGAEIEISSDEDGFRGAWF
                                                                            

AT1G09320.1     MGKVITIPSSSDKDSVKCQVEYTTLFFDKEGTKPLKEVVDMSQLRPPAPPMSEIEKKKKI
Bra018567.1-P   MGKVVTVPSP-DKDPLNCQVEYATI-SELDGSKPLKEFVHVDHLRPALPPMSEMEKKRDI
Bra031661.1-P   VGEVVAVPSS-DS-TTKCEVEYTTLFFDKEGRKRLKEVVDAAQLRPPVPAMPEREKERDV
KRG90506        TGTVIRRLAS-ER----FLVEYDNLLADDKTTKKLREVLGLRHLRPLPP----TETDREF
Zm00001d010944_ AAVVVGPSAK-PRG---YTVEYDSLLESEGSDRPLREALPARSLRPRPPPPPAPASEAPP
BGIOSGA018865-P PASVTASHAN-PRG---YTVAYD------GSGCPLGEVVAASQLRPRPPDAP---RGVPP
Os05t0132400-01 ------------------------------------------------------------
VIT_14s0060g008 AGTVVKPPAK-KKNKTLVVVEYKTIMADESGANPLRETMDVLQLRPPPP----RERSRTF
                                                                            

AT1G09320.1     VVGEEVDAFYNDG<span class="highlight">W</span><span class="highlight">W</span>EGDV---TEVL-DDGKFSVFFRSSKEQIRFRKDELRFHREWVDGA
Bra018567.1-P   LAGEDVDAFYKDV<span class="highlight">W</span><span class="highlight">W</span>EGTV---TEVRGGDGKLSVYFRGSQELIQFRRDELRFHREWINDT
Bra031661.1-P   AVGEDVDAFYSDG<span class="highlight">W</span><span class="highlight">W</span>EGTV---TEVL-GDGRFSVYFRASKEQIRFRRDELRFHREWVNGA
KRG90506        KFGDEVDAFHNDG<span class="highlight">W</span><span class="highlight">W</span>EGHI---TQEL-ENERFAVYFRVSKEQLVFSKEQLRLHREWLNHD
Zm00001d010944_ AEHAAVDALIDDA<span class="highlight">W</span><span class="highlight">W</span>LGVAL--ARAG-APSKVRVCFPETREVMEFDAADVRPHLEWVDGE
BGIOSGA018865-P AEHAVVDAFKGGA<span class="highlight">W</span><span class="highlight">W</span>VGVALGGGRAA-AGGRVAVCFPETREVVEFDAADVRPHLEWVAGE
Os05t0132400-01 ------------------------------------------------------------
VIT_14s0060g008 QISEEVDAYYNDG<span class="highlight">W</span><span class="highlight">W</span>EGVI---TEAH-ENSRFAVFFRTSREQLEFSENDLRLHREWINGN
                                                                            

AT1G09320.1     WKPPLEETEEEEDESEEDKLDDSEDEEDILARVDLETTRAIAKQM-FSSGTVVEVSSDEE
Bra018567.1-P   WQPPLDEAVEEEEESEEDEVDDYT-EEYLVPQVNLETTKAVAKEM-FSIGTVVEVSSDEE
Bra031661.1-P   WKPPIEEREEEEEEEEE--------EEDLLSRVDPETAREIAKQM-FSIGTIVEVSSDEE
KRG90506        WVPPLQQKQQRQQGNGE--------SKKVLLTPNVKSVETVKG-----------------
Zm00001d010944_ WRSP--------------------------------DNMEIPKTMPYAKGMQIEVSKLED
BGIOSGA018865-P WLSP--------------------------------ENMGISKTMPYKKGTQVEVAKLEG
Os05t0132400-01 ------------------------------------------------------------
VIT_14s0060g008 WVPPLEEEKVKTE------------EKSAVDVSSV-------------------------
                                                                            

AT1G09320.1     GFQGCWFAAKVVEPVG-EDKFLVEYRDLREKDGIEPLKEETDFLHIRPPPPRDEDIDFAV
Bra018567.1-P   GFVGCWFAAKVVERIG-DDEYRVEYKDLREVNGVEPLKEVADFLHIRPPPPSDEDIDFAV
Bra031661.1-P   GFKGSWFSAKVIEHVDEEDKYLVEYRDLREENGIEPLKEEADFLHIRPPPPADEDIDFAV
KRG90506        ------------------------------------------------------------
Zm00001d010944_ NSVVAWSPAVVAKTIW-KNNLLVDY-TLSKCYGTTLSEEIVDVKHVRPCPPQASEISFCI
BGIOSGA018865-P NSVVAWFSAAVEKAIW-KNSLLVDY-NCSKNDGSVLPKEIVDLKHIRPHPQHASAIIFCI
Os05t0132400-01 -------------------------------------------------PQHASAIIFCI
VIT_14s0060g008 ------------------------------------------------------------
                                                                            

AT1G09320.1     GDKINAFYNDG<span class="highlight">W</span><span class="highlight">W</span>VGVVIDGMKHGTVGIYFRQSQEKMRFGRQGLRLHKDWVDGTWQLPLK
Bra018567.1-P   GDKIDAFYNDG<span class="highlight">W</span><span class="highlight">W</span>VGEVIESMKDGSVGIFFRESAEKMRFGRHGLRLHKDWVNGNWELPLK
Bra031661.1-P   RDKIDAFYNDA<span class="highlight">W</span><span class="highlight">W</span>VGDVIESMKEGIVGVCFRYTGEKMRFGRQGLRLHRDWINGTWQLPLK
KRG90506        ------------------------------------------------------------
Zm00001d010944_ NDEVEAFQGDC<span class="highlight">W</span><span class="highlight">W</span>LGVITNVLPEFKYTFKSTHLGTEVEVNQKFLRLRYDWVGDQWEQESK
BGIOSGA018865-P NDEVEGFQGNG<span class="highlight">W</span><span class="highlight">W</span>LGVITDVHPEFRYTFKAAHSGKEIQLDQKALRLRYDWTHDQWQQVTQ
Os05t0132400-01 NDEVEGFQGNG<span class="highlight">W</span><span class="highlight">W</span>LGVITDVHPEFRYTFKAAHSGKEIQLDQKALRLRYDWTHDQWQQVTQ
VIT_14s0060g008 ------------------------------------------------------------
                                                                            

AT1G09320.1     GGKIKRE-KTVS-CNRNVRPKKATEKQAFSIGTPIEVSPEEEGFEDSWFLAKLIEYRGKD
Bra018567.1-P   RGEVKRA-KKVP-FERNVRRKIAIDKQYFSIGTPVEVSSIEEGFEDSWFLAKLIEYRGTD
Bra031661.1-P   RGEMKRATKKVP-CDRNVRPKKAIDKEHFSGGDPVEVSSVEEGFEDSWFLAKVIEYRGTD
KRG90506        --------------------------KGIGVGAIVEVSSDEDGFSGAWFAATVVEALGKD
Zm00001d010944_ ----------------NVP------KAKFTQGVKVEVCSDDEGFRGAWFEATVVKAVG-S
BGIOSGA018865-P MIPGLFY-MLLPTFHNNVT-------KKFAKGSKVEVTSNDDGFHGAWFQGTALKYVN-N
Os05t0132400-01 ----------------NVT-------KKFAKGSKVEVTSNDDGFHGAWFQGTALKYVN-N
VIT_14s0060g008 ----------------------------FSEGTLVEVSSDEDGFQGAWFAATIVKVIGKD
                                            :  *  :** . ::** .:** .. ::  . .

AT1G09320.1     KCLVEYDNLKAEDGKEPLREEVNVSRIRPLPLESVMVSPFERHDKVNALYNDG<span class="highlight">W</span><span class="highlight">W</span>VGVIR
Bra018567.1-P   KCLVEYDKLKAEDGKEPLSEEVNEFQIRPQPPEMVMVNPFEKLDKVNALYNDG<span class="highlight">W</span><span class="highlight">W</span>IGVVK
Bra031661.1-P   KWLVEYEKLKAEDGKEPLREEVNVFQIRPRPQETVMVNPLEKLDEVDALYNDG<span class="highlight">W</span><span class="highlight">W</span>FGVVK
KRG90506        KFLVEYHDLLADDDSQ-LREEIDALHIRPHPLDTDVDGQFSILDEVDAFYNDG<span class="highlight">W</span><span class="highlight">W</span>VGVIS
Zm00001d010944_ KFLVEYATLKADDGTKPLKESVEARHIRPCPPDIPATDGFKLLDEVDAFCNDA<span class="highlight">W</span><span class="highlight">W</span>VGVVS
BGIOSGA018865-P KILVEYDALKADDEITPLTEAIEVQHVRPCPPDIPVTSGFNLLDEVDACWNDG<span class="highlight">W</span><span class="highlight">W</span>VGVIS
Os05t0132400-01 KILVEYDALKADDEITPLTEAIEVQHVRPCPPDIPVTSGFNLLDEVDACWNDG<span class="highlight">W</span><span class="highlight">W</span>VGVIS
VIT_14s0060g008 KLMIEYKSLRTDDDTDFLREEVDALHIRPYPPETVVVDRFNLLEEVDALYNDG<span class="highlight">W</span><span class="highlight">W</span>VGVIS
                * ::**  * ::*    * * ::  ::** * :    . :.  ::*:*  **.**.**: 

AT1G09320.1     KVLAKSSYLVLFKNTQELLKFHHSQLRLHQEWIDGKWITSFKSQKV--
Bra018567.1-P   KVLAKSSYLVHFSKTDEMLKFHHSQLRLHQDWSDGKWITSSK---VCF
Bra031661.1-P   KVLAKSNYLVHFPKTDEVLKFHVSRLRPHQEWIDGKWIASSK---VCF
KRG90506        KALADSRYVVYFRSSNEELEFENSQLRLHQDWIGGKWVMPCKALKL--
Zm00001d010944_ KVLGEKRCTVYFRPWKEEMEFEHAQLRLHCDWMGGRWMRASPALEM--
BGIOSGA018865-P KVNSGDRYMVYFRSSTEETEFGHEQLRLHCDWVGGRWMRASLGLEA--
Os05t0132400-01 KVNSGDRYMVYFRSSTEETEFGHEQLRLHCDWVGGRWMRASLGLEA--
VIT_14s0060g008 KVLSRSCYVVYFRSTNEEMKFDHSDLRLHQDWIDGKWVKASQALEL--
                *. . .   * *    *  :*    ** * :* .*:*: .        


<a href='WW_WW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            