
<!doctype html>
<html>
    <head>
        <title>WW-WW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT5G53510.1</h1>
        <h3>AT5G53510.1 pep chromosome:TAIR10:5:21738849:21742082:1 gene:AT5G53510 transcript:AT5G53510.1 gene_biotype:protein_coding transcript_biotype:protein_coding gene_symbol:OPT9 description:Oligopeptide transporter 9 [Source:UniProtKB/Swiss-Prot;Acc:Q9FJD2]</h3>
        <pre>
length  741
count   12
ratio   0.00
zscore  16.95






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT5G53510.1     MDEI-----VEDSSRVMEIEGQNDDLDRCVVEEVELTVPKTDDPTLPVLTFRMWTLGLGA
Bra029065.1-P   MAATVKGSTVKPSTAAMEIEEGVDELDRCAVEEVELTVPKTDDPTLPVLTFRMWVLGLGA
Bra003050.1-P   ----------------MEIEEAVDELDRCAVEEVELTVPKTDDPTLPVLTFRMWVLGLAA
KRH64375        MASTFEGG----SDLEKENNNIAESEEECPVKQVELTVPKTDDPTMQLLTFRMWVLGVLS
KRH53552        MASTFEGG----SDSEKE-NTVESEEEECPVKQVELTVPKTDDPTMQLLTFRMWVLGVLS
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT5G53510.1     CIILSFINQFFWYRQMPLTISGISAQIAVVPLGHLMAKVLPTRMFLEGSKWEFSMNPGPF
Bra029065.1-P   CIILSFINQFFWYRRMPLSITGISAQIAVVPLGHLMARVLPNKKYLEGSRWEFNMNPGPF
Bra003050.1-P   CIILSFVNQFFWYRQMPLTITGISAQIAVVPLGHLMARVLPNKKYLEGSRWEFNMNPGPF
KRH64375        CVLLSFVNQFFWYRTQPLIVTSISAQIAVVPIGHFLARTLPTRVFFKDTRFEFSLNRGPF
KRH53552        CVLLSFVNQFFWYRTQPLIVTSISAQIAVVPIGHFMARTLPTRVFFKDSRFEFSLNRGPF
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT5G53510.1     NVKEHVLITIFANSGAGTVYATHILSAIKLYYKRSLPFLPAFLLMITTQFLGFGWAGLFR
Bra029065.1-P   NVKEHVMITIFANSGAGTVYATHILSAIKLYYKRSLPFLPAFLIMITTQFLGFGWAGLFR
Bra003050.1-P   NIKEHVMITIFANSGAGTVYATHILSAIKLYYKRSLPFLPAFLIMITTQFLGFGWAGLFR
KRH64375        NIKEHVLITIFANSGAGTVYATHILSAVKLMYKRRLDFLPALLVMLTTQVLGFGWAGLFR
KRH53552        NIKEHVLITIFANSGAGTVYATHILSAVKLMYKRRLDFLPALLVMLTTQILGFGWAGLFR
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT5G53510.1     KHLVEPGEM<span class="highlight">W</span><span class="highlight">W</span>PSNLVQVSLFSALHEKEKKKKGGMTRIQFFLIVLVTSFAYYILPGYLFT
Bra029065.1-P   KHLVEPGEM<span class="highlight">W</span><span class="highlight">W</span>PSNLVQVSLFSALHEKEKKKKGGMTRIQFFLTVLVTSFAYYILPGYLFT
Bra003050.1-P   KHLVEPGEM<span class="highlight">W</span><span class="highlight">W</span>PSNLVQVSLFSALHEKEKKKKGGMTRIQFFIIVLVTSFAYYILPGYLFT
KRH64375        KFLVEPGEM<span class="highlight">W</span><span class="highlight">W</span>PSNLVQVSLFSALHEKSKRPKGGTTRTQFFLLALVLGLAYYVFPGYLFS
KRH53552        KFLVEPGEM<span class="highlight">W</span><span class="highlight">W</span>PSNLVQVSLFSALHEKSKRPKGGTTRTQFFLLVLVSGLAYYVFPGYLFS
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT5G53510.1     MITSISWICWLGPKSVLVHQLGSGEQGLGIGAIGIDWATISSYLGSPLASPLFATINVTI
Bra029065.1-P   MITSISWVCWLSPKSVLVNQLGSGEQGLGIGAVGIDWATISSYLGSPLASPIVATINVTV
Bra003050.1-P   MITSISWVCWLSPKSVLANQLGSGEQGLGIGAIGIDWATIGSYLGSPLASPIFATVNVTI
KRH64375        MLTSFSWMCWLAPKSILVQQLGSGLRGLGIAAFGIDWSTISSYLGSPLASPWFATANIAV
KRH53552        MLTSFSWMCWLAPKSVLVQQLGSGLRGLGIAAFGIDWSTISAYLGSPLASPWFATANIAV
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT5G53510.1     GFVVIMYVATPICYWLNIYKAKTYPIFSSGLFMGNGSSYDVLSIIDKKFHLDRDIYAKTG
Bra029065.1-P   GFVVVVYVVTPICYWLNLYNAKTYPIFSSGLFMGNGSSYDVLSIIDNKFHLDRDIYAKTG
Bra003050.1-P   GFVIIMYVATPICYWLNLYRAKTYPIFSSGLFMGNGSSYDVLSIIDDKFHLDRAVYAKTG
KRH64375        GFFLVMYVMTPIAYWSNAYEAKTFPIFSSKLFMGNGSLYDISTIVNSEFHLDRQAYSING
KRH53552        GFFLVMYVMTPIAYWSNAYEAKTFPIFSSKLFMGNGSLYDISTIVNSEFHLDRQAYSING
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT5G53510.1     PINMSTFFAVTYGLGFATLSATIVHVLLFNGRDLWKQTRGAFQRNKKMDFHTRIMKKNYR
Bra029065.1-P   PIQMSTFFAVTYGLGFATLSATMVHVLLFHGSDLWKQTRGAFKRNKKMDIHTRIMKKNYK
Bra003050.1-P   PIHMSTFFAVTYGLGFATLSATIVHVLLFNGRDLWKQTRGAFKRNKKMDIHTRIMKKNYR
KRH64375        PVHLSTFFAMTYGLGFAALSATVVHVLLFHGREILMQSKRAFGNSKKIDIHTRLMRR-YK
KRH53552        PVHLSTFFAMTYGLGFAALSATVVHVLLFHGREILMQSKRAFGNSKKIDIHTRLMRR-YK
VIT_00s0508g000 --------------------------------RIWQQAKASLQ-DQFDDVHTRLMKKNYD
                                                 :  *:: ::  .:  *.***:*:: * 

AT5G53510.1     EVPM<span class="highlight">W</span><span class="highlight">W</span>FYVILVLNIALIMFISFYYNATVQLP<span class="highlight">W</span><span class="highlight">W</span>GVLLACAIAVFFTPLIGVIAATTNQE
Bra029065.1-P   EVPM<span class="highlight">W</span><span class="highlight">W</span>FLVILVINIAVIVFISVYYNATVQLP<span class="highlight">W</span><span class="highlight">W</span>GVLLACAIAVVFTPLIGVIVATTNQA
Bra003050.1-P   EVPM<span class="highlight">W</span><span class="highlight">W</span>FLVILVLNIALIVFISVYYNATVQLP<span class="highlight">W</span><span class="highlight">W</span>GVLLACAIAVFFTPLIGVILATTNQA
KRH64375        SVPM<span class="highlight">W</span><span class="highlight">W</span>FYIILAANIALIIFICEYYNESLQLP<span class="highlight">W</span><span class="highlight">W</span>GVLLACAISIFFTLPIGIINATTNQQ
KRH53552        SVPT<span class="highlight">W</span><span class="highlight">W</span>FYIILAANIALIIFICEYYNESLQLP<span class="highlight">W</span><span class="highlight">W</span>GVLLACAISIFFTLPIGIINATTNQQ
VIT_00s0508g000 PVPQ<span class="highlight">W</span><span class="highlight">W</span>FYTLLTLVIGLAMAACEGFNRQLQLPYWGILLAISLALLFILPVGVITATTNQQ
                 ** ***  :*.  *.: :  .  :*  :***:**:*** ::::.*   :*:* ***** 

AT5G53510.1     PGLNVITEYVIGYLYPERPVANMCFKVYGYISMTQALTFIQDFKLGLYMKIPPRSMFMAQ
Bra029065.1-P   PGLNVITEYVIGYIYPERPVANMCFKVYGYISMTQALTFIQDFKLGLYMKIPPRSMYMAQ
Bra003050.1-P   PGLNVITEYVIGYIYPERPVANMCFKVYGYISMTQALTFIQDFKLGLYMKIPPRSMYLAQ
KRH64375        PGLNIITEYIIGYMYPERPVANMCFKVYGYISMVQALSFLQDFKLGHYMKIPPRTMFMAQ
KRH53552        PGLNIITEYIIGYMYPERPVANMCFKVYGYISMVQALTFLQDFKLGHYMKIPPRTMFMAQ
VIT_00s0508g000 PGLNVITEMILGYIYPGKPLANLAFKTYGSVSMGQAIMFLSDFKLGHYMKIPPKSMFVVQ
                ****:*** ::**:** :*:**:.**.** :** **: *:.***** ******::*::.*

AT5G53510.1     VVGTLVSVVVYTGTA<span class="highlight">W</span><span class="highlight">W</span>LMVDIPHLCDKSLLPPDSEWTCPMDRVFFDASVIWGLVGPRRM
Bra029065.1-P   VLGTLVAVIVYTGTA<span class="highlight">W</span><span class="highlight">W</span>LMAEIPHLCDKSLLPPDSEWTCPMDRVFFDASVIWGLVGPRRM
Bra003050.1-P   VVGTLVAVLVYTGTA<span class="highlight">W</span><span class="highlight">W</span>LMVDIPHLCDKSLLPDDSQWTCPMDRVFFDASVIWGLVGPRRM
KRH64375        VVGTILSVFIYTITA<span class="highlight">W</span><span class="highlight">W</span>LMRTIPHLCDTTMLDPDSPWTCPMDNVFFDASVIWGLLGPRRI
KRH53552        VVGTILSVFIYTVTA<span class="highlight">W</span><span class="highlight">W</span>LMRTIPHLCDTTMLDPDSPWTCPMDNVFFDASVIWGLLGPRRI
VIT_00s0508g000 LVGTVIASSVYFGTAWLLLTSIDNICHPSLLPEGSQWTCPGDRVFYSASLILGVVGPLRM
                ::**:::  :*  *** *:  * ::*. ::*  .* **** *.**:.**:* *::** *:

AT5G53510.1     FGNLGEYAAINWFFLVGAIAPFFVWLATKAFPAHKWISKI------HFPVILGATSMMPP
Bra029065.1-P   FGDLGEYSNINWFFLLGAIAPFLVWLATKAFPAQKWISNI------HFPVILGATAMMPP
Bra003050.1-P   FGDLGEYSAINWFFLVGAITPFFVWLATKAFPAQKWISQI------HFPVILGATSMMPP
KRH64375        FGDLGEYAKVNLFFFGGAIAPFLVWLAHKAFPGQRWITLI------HMPVLLGATSMMPP
KRH53552        FGDLGEYAKVNLFFLGGAIAPFLVWLAHKAFPEQRWITLI------HMPVLLGATSMMPP
VIT_00s0508g000 FGRLGLYAKTNYFFLLGILAPVPVWLLFHIFPEKKWMKLIIGELCESLPYLGGAGA-MPI
                ** ** *:  * **: * ::*. ***  : ** ::*:. *       :* : ** : ** 

AT5G53510.1     AMAVNFTSWCIVAFVFGHFLYKYKRQ<span class="highlight">W</span><span class="highlight">W</span>KKYNYVLSGGLDAGTAFMTILIFLSVGRKGI-
Bra029065.1-P   AMVVNFTSWCIVAFIFGHFVFKYKRE<span class="highlight">W</span><span class="highlight">W</span>KKYNYVLSGGLDAGTAFMTILIFLALGRKGI-
Bra003050.1-P   AMAVNFTSWCIVAFIFGHFVFKYKRE<span class="highlight">W</span><span class="highlight">W</span>TKYNYVLSGGLDAGTAFMTILIFLALGRRGI-
KRH64375        ATAVNFTSWLLVGFLSGYVAYRYRQE<span class="highlight">W</span><span class="highlight">W</span>KRCNYVLSGGLDAGTAFMTILLFLALNNNNI-
KRH53552        ATAVNFTSWILVGFLSGYIAYRYRQE<span class="highlight">W</span><span class="highlight">W</span>KRCNYVLSGGLDAGTAFMTILLFLALNNNNI-
VIT_00s0508g000 AKAVNYWCWGAMGIFFNIFVYRRFKG<span class="highlight">W</span><span class="highlight">W</span>ARHNYVLSAGLDAGVAFMAILSYFTLQDKDIN
                * .**: .*  :.:. . . ::  : ** : *****.*****.***:** ::::  ..* 

AT5G53510.1     GLL<span class="highlight">W</span><span class="highlight">W</span>GNADDSTNCSLASCPTAKGVIMHGCPVL-
Bra029065.1-P   GLV<span class="highlight">W</span><span class="highlight">W</span>GNADDSTNCSLASCPTAKGVIMNGCPVY-
Bra003050.1-P   GLA<span class="highlight">W</span><span class="highlight">W</span>GNADDSTNCGLASCPTAKGVVTQGCPVF-
KRH64375        MLN<span class="highlight">W</span><span class="highlight">W</span>GN--NPEGCPLATCPTEKGIVVKGCPVH-
KRH53552        MLN<span class="highlight">W</span><span class="highlight">W</span>GN--NPEGCPLANCPTEKGIVVKGCPVH-
VIT_00s0508g000 GMQ<span class="highlight">W</span><span class="highlight">W</span>GLGID-DHCPLAHCPTAPAIKVEGCPVFN
                 : ***   :   * ** ***  .:  .****  


<a href='WW_WW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            