
<!doctype html>
<html>
    <head>
        <title>WW-WW</title>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <link rel="stylesheet" href="styles.css">
    </head>
    <body>
        <h1>AT5G53520.1</h1>
        <h3>AT5G53520.1 pep chromosome:TAIR10:5:21743366:21746061:1 gene:AT5G53520 transcript:AT5G53520.1 gene_biotype:protein_coding transcript_biotype:protein_coding gene_symbol:OPT8 description:Oligopeptide transporter 8 [Source:UniProtKB/Swiss-Prot;Acc:Q9FJD1]</h3>
        <pre>
length  733
count   12
ratio   0.00
zscore  16.95






CLUSTAL format alignment by MAFFT L-INS-i (v7.450)


AT5G53520.1     MKD-FTDTISESE-------CDDEISIVPQVELTVPKTDDPTSPTVTFRMWVLGITACVL
Bra029053.1-P   MRD-MTEIRSELESE-SDSYANDEISIVPQVELTVPKTDDPDAPTVTFRMWVLGIAACVL
Bra029064.1-P   MRD-MTEIRSGLESE-SDPYADDEISIVPQVELTVPKTDDPTAPTVTFRMGVLGITACVL
Bra022658.1-P   MRDNITEMRSDSESD-YDPYGDDEMSIVPQVELTVPKTDDPTSPTVTFRMWVLGITACVL
KRH64375        MAS-TFEGGSDLEKENNNIAESEEECPVKQVELTVPKTDDPTMQLLTFRMWVLGVLSCVL
KRH53552        MAS-TFEGGSDSEKE-NTVESEEEECPVKQVELTVPKTDDPTMQLLTFRMWVLGVLSCVL
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT5G53520.1     LSFLNQFFWYRTNPLTISSVSAQIAVVPIGHLMAKVLPTRRFFEGTRWSFTMNPGPFSTK
Bra029053.1-P   LSFVNQFFWYRTNPLTISSVSAQIAVVPIGHLMAKVLPTRRFFQGTRWSFTMNPGPFSTK
Bra029064.1-P   LSFLNQFFWYRTNPLTISSVSAQIAVVPIGHLMARVLPTRIFFEGTRWSFTMNPGPFSTK
Bra022658.1-P   LSFLNQFFWYRTNPLTISSVSAQIAVVPIGHLMARVLPTRIFFEGTRWSFTMNPGPFSTK
KRH64375        LSFVNQFFWYRTQPLIVTSISAQIAVVPIGHFLARTLPTRVFFKDTRFEFSLNRGPFNIK
KRH53552        LSFVNQFFWYRTQPLIVTSISAQIAVVPIGHFMARTLPTRVFFKDSRFEFSLNRGPFNIK
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT5G53520.1     EHVLITVFANSGSGAVYATHILSAVKLYYKRRLDFLPALLVMITTQVLGFGWAGLYRKHL
Bra029053.1-P   EHVLITVFPKSCSGAVYASHILSAVKLFYKRRLDFLPALLIMITT---------------
Bra029064.1-P   EHVLITVFANPGSGAVYASHILSAVKLFYKRRLDFFPALLIMITTQVLGFGWAGLYRKHL
Bra022658.1-P   EHVLITVFANSGSGAVYASHILSAVKLYYKRRLDFIPALLVMITTQVLGFGWAGLYRKHL
KRH64375        EHVLITIFANSGAGTVYATHILSAVKLMYKRRLDFLPALLVMLTTQVLGFGWAGLFRKFL
KRH53552        EHVLITIFANSGAGTVYATHILSAVKLMYKRRLDFLPALLVMLTTQILGFGWAGLFRKFL
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT5G53520.1     VEPGEM<span class="highlight">W</span><span class="highlight">W</span>PSNLVQVSLFRALHEKENKSKWGISRNQFFVITLITSFSYYLLPGYLFTVLT
Bra029053.1-P   ------------------------------------------------------------
Bra029064.1-P   VEPGEM<span class="highlight">W</span><span class="highlight">W</span>PSNLVQVSLFRALHEKEVKSKWGITRNQFFVIALVTSFSYYILPGYLFSLLT
Bra022658.1-P   VEPGEM<span class="highlight">W</span><span class="highlight">W</span>PSNLVQVSLFRALHEKEKRSEWGVTRNQFFVITLIASFSYYLLPGYLFTLLT
KRH64375        VEPGEM<span class="highlight">W</span><span class="highlight">W</span>PSNLVQVSLFSALHEKSKRPKGGTTRTQFFLLALVLGLAYYVFPGYLFSMLT
KRH53552        VEPGEM<span class="highlight">W</span><span class="highlight">W</span>PSNLVQVSLFSALHEKSKRPKGGTTRTQFFLLVLVSGLAYYVFPGYLFSMLT
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT5G53520.1     TVSWLCWISPKSILVNQLGSGSAGLGIGSFGLDWSTIASYLGSPLASPFFASANIAAGFF
Bra029053.1-P   ------------------------------------------------------------
Bra029064.1-P   TVSWLCRIRPKSILVNQLGSGSVGLGIGAFGLDWATIASYLGSPLASPFFATANIAVGFF
Bra022658.1-P   TISWLCWISPKSILVNQLGSGSAGLGIGSFGLDWSTIASYLGSPLASPFFATANIAVGYF
KRH64375        SFSWMCWLAPKSILVQQLGSGLRGLGIAAFGIDWSTISSYLGSPLASPWFATANIAVGFF
KRH53552        SFSWMCWLAPKSVLVQQLGSGLRGLGIAAFGIDWSTISAYLGSPLASPWFATANIAVGFF
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT5G53520.1     LVMYVITPLCYYLDLYNAKTFPIYSGKLFVASGKEYKVTSIIDANFRLDRQAYAETGPVH
Bra029053.1-P   ------------------------------------------------------------
Bra029064.1-P   LLMYVITPLSYYLDFFHARTFPIYSGKLFVSNGQEYKVRNIINDEFRLDHKAYAEAGPVH
Bra022658.1-P   IVMYVITPLCYYLDFFDAKTFPIYSGKLFVANGQEYNVTSIINDEFRLDHKAYAETGPVH
KRH64375        LVMYVMTPIAYWSNAYEAKTFPIFSSKLFMGNGSLYDISTIVNSEFHLDRQAYSINGPVH
KRH53552        LVMYVMTPIAYWSNAYEAKTFPIFSSKLFMGNGSLYDISTIVNSEFHLDRQAYSINGPVH
VIT_00s0508g000 ------------------------------------------------------------
                                                                            

AT5G53520.1     MSTFFAVTYGLGFATLSASIFHVLIFNGKDLWTQTRGAFGKNKKMDIHTKIMKRNYKEVP
Bra029053.1-P   ------------------------------------------------------------
Bra029064.1-P   MSTFFAVSYGLGFATLTASVVHVLLFNGKDLWNQSKGVLRGNKKMDIHTKIMKRNYKEVP
Bra022658.1-P   MSTFFAVTYGLGFATLTASVVHVFLFYGKDLWNQTKGALRKDKKMDIHTKIMKRNYKEVP
KRH64375        LSTFFAMTYGLGFAALSATVVHVLLFHGREILMQSKRAFGNSKKIDIHTRLMRR-YKSVP
KRH53552        LSTFFAMTYGLGFAALSATVVHVLLFHGREILMQSKRAFGNSKKIDIHTRLMRR-YKSVP
VIT_00s0508g000 -----------------------------RIWQQAKASL-QDQFDDVHTRLMKKNYDPVP
                                                                            

AT5G53520.1     L<span class="highlight">W</span><span class="highlight">W</span>FLSIFAVNLAVIVFICIYYKTQIQLP<span class="highlight">W</span><span class="highlight">W</span>GAFLACLIAIFFTPLVGVIMATTNQAPGL
Bra029053.1-P   -------------------------------------------------------QAPGL
Bra029064.1-P   L<span class="highlight">W</span><span class="highlight">W</span>FLSIFAVNIAVVIFMCFYYKTQIQLP<span class="highlight">W</span><span class="highlight">W</span>GAFLACLIAIFFTPLVGVIKATTNQAPGL
Bra022658.1-P   L<span class="highlight">W</span><span class="highlight">W</span>FLSIFVVNIAVIVFICVFYEKQIQLP<span class="highlight">W</span><span class="highlight">W</span>GAFLACLIALFFTPLVGVIMATTNQAPGL
KRH64375        M<span class="highlight">W</span><span class="highlight">W</span>FYIILAANIALIIFICEYYNESLQLP<span class="highlight">W</span><span class="highlight">W</span>GVLLACAISIFFTLPIGIINATTNQQPGL
KRH53552        T<span class="highlight">W</span><span class="highlight">W</span>FYIILAANIALIIFICEYYNESLQLP<span class="highlight">W</span><span class="highlight">W</span>GVLLACAISIFFTLPIGIINATTNQQPGL
VIT_00s0508g000 Q<span class="highlight">W</span><span class="highlight">W</span>FYTLLTLVIGLAMAACEGFNRQLQLPYWGILLAISLALLFILPVGVITATTNQQPGL
                                                                       * ***

AT5G53520.1     NIITEYIIGYAYPERPVANICFKTYGYISMSQSLTFLSDLKLGTYMKIPPRTMFMAQVVG
Bra029053.1-P   NIITEYIIGYAYPERPVANICFKTYGYISMSQSLTFRADLKLGAYIKIESRIFFKFDTSM
Bra029064.1-P   NIITEYIIGYAYPERPVANICFKTYGYISMSQSLTFLADLKLGSYMKIPPRTMFMAQVVG
Bra022658.1-P   NIITEYIIGYAYPERPVANICFKTYGYISMSQSLTFLADLKLGSYMKIPPRTMFMAQVVG
KRH64375        NIITEYIIGYMYPERPVANMCFKVYGYISMVQALSFLQDFKLGHYMKIPPRTMFMAQVVG
KRH53552        NIITEYIIGYMYPERPVANMCFKVYGYISMVQALTFLQDFKLGHYMKIPPRTMFMAQVVG
VIT_00s0508g000 NVITEMILGYIYPGKPLANLAFKTYGSVSMGQAIMFLSDFKLGHYMKIPPKSMFVVQLVG
                *:*** *:** ** :*:**:.**.** :** *:: *  *:*** *:** .: :*  :   

AT5G53520.1     TLVAVIAYAGTA<span class="highlight">W</span><span class="highlight">W</span>LMAEIPNLCDTNLLPPGSQWTCPSDRVFFDASVIWGLVGPRRMFGD
Bra029053.1-P   GARAPLAMA--------------------------LPSPHVY------------------
Bra029064.1-P   TLVAVLVYASTA<span class="highlight">W</span><span class="highlight">W</span>LMAEIPNLCDTSLLPPGSQWTCPTDRVFFDASVIWGLVGPRRMFGD
Bra022658.1-P   TLVAVFVYALTA<span class="highlight">W</span><span class="highlight">W</span>LMAEIPNLCDTSLLPPESQWTCPTDRVFFDASVIWGLVGPRRMFGD
KRH64375        TILSVFIYTITA<span class="highlight">W</span><span class="highlight">W</span>LMRTIPHLCDTTMLDPDSPWTCPMDNVFFDASVIWGLLGPRRIFGD
KRH53552        TILSVFIYTVTA<span class="highlight">W</span><span class="highlight">W</span>LMRTIPHLCDTTMLDPDSPWTCPMDNVFFDASVIWGLLGPRRIFGD
VIT_00s0508g000 TVIASSVYFGTAWLLLTSIDNICHPSLLPEGSQWTCPGDRVFYSASLILGVVGPLRMFGR
                   :                                *  .*:                  

AT5G53520.1     LGEYSNINWFFVGGAIAPALVYLASRLFPNKKWISDI------HIPVLIGATAIMPPASA
Bra029053.1-P   -------------GRVAPALVYLATRIFPKKKWISNI------HMPVLIGATAIMPPATA
Bra029064.1-P   LGEYANINWFFLGGAIAPALVYLATRIFPKKKWISNI------HMPVLIGATAIMPPATA
Bra022658.1-P   LGEYSNINWFFLGGAIAPTLVYLATRVFPNKKWISSI------HMPVLIGATAIMPPATA
KRH64375        LGEYAKVNLFFFGGAIAPFLVWLAHKAFPGQRWITLI------HMPVLLGATSMMPPATA
KRH53552        LGEYAKVNLFFLGGAIAPFLVWLAHKAFPEQRWITLI------HMPVLLGATSMMPPATA
VIT_00s0508g000 LGLYAKTNYFFLLGILAPVPVWLLFHIFPEKKWMKLIIGELCESLPYLGGAGA-MPIAKA
                             * :**  *:*  : ** ::*:. *       :* * ** : ** *.*

AT5G53520.1     VNFTSWLVMAFVFGHFVFKYRRE<span class="highlight">W</span><span class="highlight">W</span>QRYNYVLSGGMDAGTGFMSVLLFLALQRS-EIA-I
Bra029053.1-P   VNFTSWLVMVFVSGHFVFKYKRE<span class="highlight">W</span><span class="highlight">W</span>QRYNYVLSGGMDAGTGFMSVLLFLALQRSDDIM-L
Bra029064.1-P   VNFTSWLVMAFVFGHFVFKYKRE<span class="highlight">W</span><span class="highlight">W</span>QRYNYVLSGGMDAGTGFMSVLLFLALQRS-DIM-L
Bra022658.1-P   VNFTSWLVMAFVFGHFLFKYKTE<span class="highlight">W</span><span class="highlight">W</span>QRYNYVLSGGMDAGTGFMSVLLFLALQRS-DIM-L
KRH64375        VNFTSWLLVGFLSGYVAYRYRQE<span class="highlight">W</span><span class="highlight">W</span>KRCNYVLSGGLDAGTAFMTILLFLALNNN-NIM-L
KRH53552        VNFTSWILVGFLSGYIAYRYRQE<span class="highlight">W</span><span class="highlight">W</span>KRCNYVLSGGLDAGTAFMTILLFLALNNN-NIM-L
VIT_00s0508g000 VNYWCWGAMGIFFNIFVYRRFKG<span class="highlight">W</span><span class="highlight">W</span>ARHNYVLSAGLDAGVAFMAILSYFTLQDK-DINGM
                **: .*  : :. . . ::    ** * *****.*:***..**::* :::*: . :*  :

AT5G53520.1     D<span class="highlight">W</span><span class="highlight">W</span>G-NSGEGCPVAKCPTAKGVVVHGCPVF-
Bra029053.1-P   G<span class="highlight">W</span><span class="highlight">W</span>E-NSGEGFRAAKCPTARGVIVHGCPVF-
Bra029064.1-P   G<span class="highlight">W</span><span class="highlight">W</span>G-NSGEGCPVAKCPTAKGVIVHGCPVF-
Bra022658.1-P   D<span class="highlight">W</span><span class="highlight">W</span>G-NTGEGCPVAKCPTAKGVVVHGCPVF-
KRH64375        N<span class="highlight">W</span><span class="highlight">W</span>G-NNPEGCPLATCPTEKGIVVKGCPVH-
KRH53552        N<span class="highlight">W</span><span class="highlight">W</span>G-NNPEGCPLANCPTEKGIVVKGCPVH-
VIT_00s0508g000 Q<span class="highlight">W</span><span class="highlight">W</span>GLGIDDHCPLAHCPTAPAIKVEGCPVFN
                 **  .  :    * ***  .: *.****. 


<a href='WW_WW-index.html' target='_blank'>index</a>

        </pre>
    </body>
</html>
            